<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
<title>00092</title>
<script type="text/javascript">
function showhide(id)
{
    var e = document.getElementById(id);
    e.style.display = (e.style.display == 'block') ? 'none' : 'block';
}
</script>
<style>
rect {
transition: .6s fill;
fill: #D3D3D3;
opacity: 0;
}
rect:hover {
    fill: #D3D3D3;
    opacity: 0.2;
}

.outline {
    clear: both;
}

.svg-container {
    width: 100%;
    max-width: 5234px;
    margin-left: auto;
    margin-right: auto;
    display: block;
}

.svg-content {
    width: 100%;
}

.container {
    width: 100%;
}
</style>

</head>

<body>

<div class="container"><div class="spacer">&nbsp;</div><div class="svg-container"><svg version="1.1"  preserveAspectRatio="xMinYMin meet" class="svg-content" viewBox="0 0 5234 17332" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<image width="5234" height="17332" xlink:href="00092-assets/00092.png"></image>
<a xlink:href="00092" xlink:title="00092"><rect x="2772" y="162" width="28" height="28"/></a>
<a xlink:href="00192" xlink:title="00192"><rect x="2772" y="324" width="28" height="28"/></a>
<a xlink:href="00292" xlink:title="00292"><rect x="2772" y="412" width="28" height="28"/></a>
<a xlink:href="00392" xlink:title="00392"><rect x="2772" y="556" width="28" height="28"/></a>
<a xlink:href="00492" xlink:title="00492"><rect x="2772" y="718" width="28" height="28"/></a>
<a xlink:href="00592" xlink:title="00592"><rect x="2772" y="828" width="28" height="28"/></a>
<a xlink:href="00692" xlink:title="00692"><rect x="2772" y="920" width="28" height="28"/></a>
<a xlink:href="00792" xlink:title="00792"><rect x="2772" y="1006" width="28" height="28"/></a>
<a xlink:href="00892" xlink:title="00892"><rect x="2772" y="1086" width="28" height="28"/></a>
<a xlink:href="00992" xlink:title="00992"><rect x="2772" y="1166" width="28" height="28"/></a>
<a xlink:href="01092" xlink:title="01092"><rect x="2772" y="1252" width="28" height="28"/></a>
<a xlink:href="01192" xlink:title="01192"><rect x="2772" y="2472" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1073/pnas.1601895113" xlink:title="http://dx.doi.org/10.1073/pnas.1601895113"><rect x="1830" y="3268" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1126/science.aad2993" xlink:title="http://dx.doi.org/10.1126/science.aad2993"><rect x="1832" y="3448" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.cell.2016.11.003" xlink:title="http://dx.doi.org/10.1016/j.cell.2016.11.003"><rect x="1648" y="3584" width="214" height="33"/></a>
<a xlink:href="01292" xlink:title="01292"><rect x="2772" y="3738" width="28" height="28"/></a>
<a xlink:href="01392" xlink:title="01392"><rect x="2772" y="3870" width="28" height="28"/></a>
<a xlink:href="01492" xlink:title="01492"><rect x="2772" y="3950" width="28" height="28"/></a>
<a xlink:href="01592" xlink:title="01592"><rect x="2772" y="4030" width="28" height="28"/></a>
<a xlink:href="01692" xlink:title="01692"><rect x="2772" y="4726" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.alcohol.2016.10.010" xlink:title="http://dx.doi.org/10.1016/j.alcohol.2016.10.010"><rect x="798" y="4820" width="258" height="33"/></a>
<a xlink:href="01792" xlink:title="01792"><rect x="2772" y="5422" width="28" height="28"/></a>
<a xlink:href="01892" xlink:title="01892"><rect x="2772" y="5502" width="28" height="28"/></a>
<a xlink:href="01992" xlink:title="01992"><rect x="2772" y="5582" width="28" height="28"/></a>
<a xlink:href="02092" xlink:title="02092"><rect x="2772" y="7308" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.celrep.2016.08.066" xlink:title="http://dx.doi.org/10.1016/j.celrep.2016.08.066"><rect x="1858" y="6790" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/nature21683" xlink:title="http://dx.doi.org/10.1038/nature21683"><rect x="1342" y="6962" width="386" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1126/science.aam5339" xlink:title="http://dx.doi.org/10.1126/science.aam5339"><rect x="1306" y="7158" width="447" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1038/nature23262" xlink:title="http://dx.doi.org/10.1038/nature23262"><rect x="1298" y="6550" width="386" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.ccell.2014.09.010" xlink:title="http://dx.doi.org/10.1016/j.ccell.2014.09.010"><rect x="1792" y="6318" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/ncomms14665" xlink:title="http://dx.doi.org/10.1038/ncomms14665"><rect x="1330" y="5882" width="416" height="33"/></a>
<a xlink:href="http://advances.sciencemag.org/content/2/2/e1501340.full" xlink:title="http://advances.sciencemag.org/content/2/2/e1501340.full"><rect x="1768" y="7872" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.cell.2016.09.017" xlink:title="http://dx.doi.org/10.1016/j.cell.2016.09.017"><rect x="1844" y="8068" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1371/journal.pgen.1006391" xlink:title="http://dx.doi.org/10.1371/journal.pgen.1006391"><rect x="1840" y="8282" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/mp.2010.136" xlink:title="http://dx.doi.org/10.1038/mp.2010.136"><rect x="1282" y="8750" width="162" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1038/ncomms13171" xlink:title="http://dx.doi.org/10.1038/ncomms13171"><rect x="1838" y="8956" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1371/journal.pcbi.1005385" xlink:title="http://dx.doi.org/10.1371/journal.pcbi.1005385"><rect x="1278" y="8555" width="486" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1158/2159-8290.CD-16-1292" xlink:title="http://dx.doi.org/10.1158/2159-8290.CD-16-1292"><rect x="1282" y="7500" width="529" height="33"/></a>
<a xlink:href="02192" xlink:title="02192"><rect x="2772" y="9034" width="28" height="28"/></a>
<a xlink:href="02292" xlink:title="02292"><rect x="2772" y="9114" width="28" height="28"/></a>
<a xlink:href="02392" xlink:title="02392"><rect x="2772" y="9194" width="28" height="28"/></a>
<a xlink:href="02492" xlink:title="02492"><rect x="2772" y="9274" width="28" height="28"/></a>
<a xlink:href="02592" xlink:title="02592"><rect x="2772" y="12356" width="28" height="28"/></a>
<a xlink:href="http://link.springer.com/article/10.1007/s12304-015-9242-3/fulltext.html" xlink:title="http://link.springer.com/article/10.1007/s12304-015-9242-3/fulltext.html"><rect x="2130" y="10768" width="28" height="28"/></a>
<a xlink:href="https://doi.org/10.1007/s12304-015-9242-3" xlink:title="https://doi.org/10.1007/s12304-015-9242-3"><rect x="1612" y="10724" width="445" height="33"/></a>
<a xlink:href="http://rsbl.royalsocietypublishing.org/content/6/4/544.full.pdf " xlink:title="http://rsbl.royalsocietypublishing.org/content/6/4/544.full.pdf "><rect x="1450" y="11076" width="28" height="28"/></a>
<a xlink:href="http://bristol.academia.edu/SSahney/Papers" xlink:title="http://bristol.academia.edu/SSahney/Papers"><rect x="814" y="11070" width="28" height="28"/></a>
<a xlink:href="http://doi.org/10.1111/j.1461-0248.2009.01350.x" xlink:title="http://doi.org/10.1111/j.1461-0248.2009.01350.x"><rect x="2130" y="11290" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1101/gr.206706.116" xlink:title="http://dx.doi.org/10.1101/gr.206706.116"><rect x="1728" y="13476" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/nn.4388" xlink:title="http://dx.doi.org/10.1038/nn.4388"><rect x="1726" y="13656" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1093/molbev/msw127" xlink:title="http://dx.doi.org/10.1093/molbev/msw127"><rect x="1392" y="13904" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.anbehav.2016.08.009" xlink:title="http://dx.doi.org/10.1016/j.anbehav.2016.08.009"><rect x="2130" y="14186" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/srep34077" xlink:title="http://dx.doi.org/10.1038/srep34077"><rect x="2130" y="14436" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.gr.2016.08.003" xlink:title="http://dx.doi.org/10.1016/j.gr.2016.08.003"><rect x="1596" y="14606" width="197" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1257/aer.103.1.1" xlink:title="http://dx.doi.org/10.1257/aer.103.1.1"><rect x="2128" y="15102" width="28" height="28"/></a>
<a xlink:href="02692" xlink:title="02692"><rect x="2772" y="15438" width="28" height="28"/></a>
<a xlink:href="02792" xlink:title="02792"><rect x="2772" y="15518" width="28" height="28"/></a>
<a xlink:href="02892" xlink:title="02892"><rect x="2772" y="15598" width="28" height="28"/></a>
<a xlink:href="02992" xlink:title="02992"><rect x="2772" y="15678" width="28" height="28"/></a>
<a xlink:href="03092" xlink:title="03092"><rect x="2772" y="15758" width="28" height="28"/></a>
<a xlink:href="03192" xlink:title="03192"><rect x="2772" y="15838" width="28" height="28"/></a>
<a xlink:href="03292" xlink:title="03292"><rect x="2772" y="15918" width="28" height="28"/></a>
<a xlink:href="03392" xlink:title="03392"><rect x="2772" y="15998" width="28" height="28"/></a>
<a xlink:href="03492" xlink:title="03492"><rect x="2772" y="16078" width="28" height="28"/></a>
<a xlink:href="03592" xlink:title="03592"><rect x="2772" y="16158" width="28" height="28"/></a>
<a xlink:href="03692" xlink:title="03692"><rect x="2772" y="16238" width="28" height="28"/></a>
<a xlink:href="03792" xlink:title="03792"><rect x="2772" y="16318" width="28" height="28"/></a>
<a xlink:href="03892" xlink:title="03892"><rect x="2772" y="16398" width="28" height="28"/></a>
<a xlink:href="03992" xlink:title="03992"><rect x="2772" y="16478" width="28" height="28"/></a>
<a xlink:href="04092" xlink:title="04092"><rect x="2772" y="16558" width="28" height="28"/></a>
<a xlink:href="04192" xlink:title="04192"><rect x="2772" y="16638" width="28" height="28"/></a>
<a xlink:href="04292" xlink:title="04292"><rect x="2772" y="16718" width="28" height="28"/></a>
<a xlink:href="04392" xlink:title="04392"><rect x="2772" y="16798" width="28" height="28"/></a>
<a xlink:href="04492" xlink:title="04492"><rect x="2772" y="16878" width="28" height="28"/></a>
<a xlink:href="04592" xlink:title="04592"><rect x="2772" y="16958" width="28" height="28"/></a>
<a xlink:href="04692" xlink:title="04692"><rect x="2772" y="17038" width="28" height="28"/></a>
<a xlink:href="04792" xlink:title="04792"><rect x="2772" y="17118" width="28" height="28"/></a>
<a xlink:href="04892" xlink:title="04892"><rect x="2772" y="17198" width="28" height="28"/></a>
<a xlink:href="04992" xlink:title="04992"><rect x="2772" y="17278" width="28" height="28"/></a>
<a xlink:href="05092" xlink:title="05092"><rect x="3140" y="850" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.7554/eLife.19466" xlink:title="http://dx.doi.org/10.7554/eLife.19466"><rect x="3950" y="776" width="376" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1073/pnas.1611028113" xlink:title="http://dx.doi.org/10.1073/pnas.1611028113"><rect x="3940" y="1084" width="28" height="28"/></a>
<a xlink:href="05192" xlink:title="05192"><rect x="3140" y="1234" width="28" height="28"/></a>
<a xlink:href="http://darwin-online.org.uk/" xlink:title="http://darwin-online.org.uk/"><rect x="3386" y="1173" width="244" height="33"/></a>
<a xlink:href="http://wallace-online.org/" xlink:title="http://wallace-online.org/"><rect x="3386" y="1283" width="218" height="33"/></a>
<a xlink:href="05292" xlink:title="05292"><rect x="3140" y="1676" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.neuron.2017.01.033" xlink:title="http://dx.doi.org/10.1016/j.neuron.2017.01.033"><rect x="3736" y="1536" width="256" height="33"/></a>
<a xlink:href="https://www.sciencedaily.com/releases/2017/02/170223124227.htm" xlink:title="https://www.sciencedaily.com/releases/2017/02/170223124227.htm"><rect x="3726" y="1742" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.cell.2017.01.038" xlink:title="http://dx.doi.org/10.1016/j.cell.2017.01.038"><rect x="3736" y="1946" width="214" height="33"/></a>
<a xlink:href="05392" xlink:title="05392"><rect x="3140" y="2088" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.cell.2017.03.025" xlink:title="http://dx.doi.org/10.1016/j.cell.2017.03.025"><rect x="3386" y="2106" width="450" height="33"/></a>
<a xlink:href="05492" xlink:title="05492"><rect x="3140" y="2248" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1126/science.aah4204" xlink:title="http://dx.doi.org/10.1126/science.aah4204"><rect x="3386" y="2266" width="439" height="33"/></a>
<a xlink:href="05592" xlink:title="05592"><rect x="3140" y="2442" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/srep46337" xlink:title="http://dx.doi.org/10.1038/srep46337"><rect x="3386" y="2494" width="365" height="33"/></a>
<a xlink:href="05692" xlink:title="05692"><rect x="3140" y="2654" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.chom.2017.03.002" xlink:title="http://dx.doi.org/10.1016/j.chom.2017.03.002"><rect x="3386" y="2688" width="478" height="33"/></a>
<a xlink:href="05792" xlink:title="05792"><rect x="3140" y="2850" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1186/s40168-017-0260-z" xlink:title="http://dx.doi.org/10.1186/s40168-017-0260-z"><rect x="3386" y="2884" width="478" height="33"/></a>
<a xlink:href="05892" xlink:title="05892"><rect x="3140" y="3028" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/s41559-016-0038" xlink:title="http://dx.doi.org/10.1038/s41559-016-0038"><rect x="3386" y="3046" width="456" height="33"/></a>
<a xlink:href="05992" xlink:title="05992"><rect x="3140" y="3206" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/ncomms13229" xlink:title="http://dx.doi.org/10.1038/ncomms13229"><rect x="3980" y="3240" width="179" height="33"/></a>
<a xlink:href="06092" xlink:title="06092"><rect x="3140" y="3394" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/ncomms12461" xlink:title="http://dx.doi.org/10.1038/ncomms12461"><rect x="3726" y="3446" width="28" height="28"/></a>
<a xlink:href="06192" xlink:title="06192"><rect x="3140" y="3524" width="28" height="28"/></a>
<a xlink:href="06292" xlink:title="06292"><rect x="3140" y="3678" width="28" height="28"/></a>
<a xlink:href="https://mosaicscience.com/story/ancient-human-evolution-neanderthal-genetics" xlink:title="https://mosaicscience.com/story/ancient-human-evolution-neanderthal-genetics"><rect x="3386" y="3697" width="572" height="33"/></a>
<a xlink:href="https://mosaicscience.com/story/ancient-human-evolution-neanderthal-genetics" xlink:title="https://mosaicscience.com/story/ancient-human-evolution-neanderthal-genetics"><rect x="3386" y="3730" width="357" height="33"/></a>
<a xlink:href="06392" xlink:title="06392"><rect x="3140" y="3890" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1371/journal.pcbi.1005435" xlink:title="http://dx.doi.org/10.1371/journal.pcbi.1005435"><rect x="3386" y="3924" width="486" height="33"/></a>
<a xlink:href="06492" xlink:title="06492"><rect x="3140" y="4086" width="28" height="28"/></a>
<a xlink:href="https://www.nature.com/articles/ncomms15172" xlink:title="https://www.nature.com/articles/ncomms15172"><rect x="3386" y="4120" width="412" height="33"/></a>
<a xlink:href="06592" xlink:title="06592"><rect x="3140" y="4264" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/nature22075" xlink:title="http://dx.doi.org/10.1038/nature22075"><rect x="3386" y="4282" width="386" height="33"/></a>
<a xlink:href="06692" xlink:title="06692"><rect x="3140" y="4442" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1093/molbev/msx163" xlink:title="http://dx.doi.org/10.1093/molbev/msx163"><rect x="3386" y="4476" width="425" height="33"/></a>
<a xlink:href="06792" xlink:title="06792"><rect x="3140" y="4620" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.cell.2017.05.036" xlink:title="http://dx.doi.org/10.1016/j.cell.2017.05.036"><rect x="3386" y="4638" width="450" height="33"/></a>
<a xlink:href="06892" xlink:title="06892"><rect x="3140" y="4740" width="28" height="28"/></a>
<a xlink:href="06992" xlink:title="06992"><rect x="3140" y="4820" width="28" height="28"/></a>
<a xlink:href="07092" xlink:title="07092"><rect x="3140" y="4960" width="28" height="28"/></a>
<a xlink:href="07192" xlink:title="07192"><rect x="3140" y="6110" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.celrep.2015.03.049" xlink:title="http://dx.doi.org/10.1016/j.celrep.2015.03.049"><rect x="3682" y="5800" width="247" height="33"/></a>
<a xlink:href="http://www.pnas.org/cgi/doi/10.1073/pnas.1218525110" xlink:title="http://www.pnas.org/cgi/doi/10.1073/pnas.1218525110"><rect x="3682" y="5158" width="221" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1371/journal.pbio.2000633" xlink:title="http://dx.doi.org/10.1371/journal.pbio.2000633"><rect x="3682" y="5996" width="486" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1038/nature21725" xlink:title="http://dx.doi.org/10.1038/nature21725"><rect x="3682" y="6158" width="150" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.immuni.2017.02.016" xlink:title="http://dx.doi.org/10.1016/j.immuni.2017.02.016"><rect x="3682" y="6352" width="495" height="33"/></a>
<a xlink:href="https://www.sciencedaily.com/releases/2017/04/170403123508.htm" xlink:title="https://www.sciencedaily.com/releases/2017/04/170403123508.htm"><rect x="3672" y="6924" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.3389/fncel.2017.00120" xlink:title="http://dx.doi.org/10.3389/fncel.2017.00120"><rect x="3682" y="7096" width="446" height="33"/></a>
<a xlink:href="07292" xlink:title="07292"><rect x="3140" y="9298" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.cell.2015.11.004" xlink:title="http://dx.doi.org/10.1016/j.cell.2015.11.004"><rect x="3860" y="7834" width="28" height="28"/></a>
<a xlink:href="http://abcnews.go.com/Health/anxiety-head-gut/story?id=20229136" xlink:title="http://abcnews.go.com/Health/anxiety-head-gut/story?id=20229136"><rect x="4212" y="8138" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/srep31023" xlink:title="http://dx.doi.org/10.1038/srep31023"><rect x="3860" y="8284" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.3389/fmicb.2016.01230" xlink:title="http://dx.doi.org/10.3389/fmicb.2016.01230"><rect x="3860" y="8568" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1128/msphere.00199-16" xlink:title="http://dx.doi.org/10.1128/msphere.00199-16"><rect x="3860" y="8750" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1186/s12916-016-0698-z" xlink:title="http://dx.doi.org/10.1186/s12916-016-0698-z"><rect x="3860" y="8996" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/nm.4174" xlink:title="http://dx.doi.org/10.1038/nm.4174"><rect x="3860" y="9212" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.molmet.2016.10.003" xlink:title="http://dx.doi.org/10.1016/j.molmet.2016.10.003"><rect x="3860" y="9496" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.tins.2016.09.002" xlink:title="http://dx.doi.org/10.1016/j.tins.2016.09.002"><rect x="3860" y="7462" width="28" height="28"/></a>
<a xlink:href="https://www.sciencedaily.com/releases/2016/11/161114124958.htm" xlink:title="https://www.sciencedaily.com/releases/2016/11/161114124958.htm"><rect x="3870" y="9737" width="238" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1038/NCOMMS13329" xlink:title="http://dx.doi.org/10.1038/NCOMMS13329"><rect x="3870" y="9898" width="200" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.molcel.2016.10.25" xlink:title="http://dx.doi.org/10.1016/j.molcel.2016.10.25"><rect x="3870" y="10092" width="236" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.cell.2016.11.018" xlink:title="http://dx.doi.org/10.1016/j.cell.2016.11.018"><rect x="3870" y="10288" width="214" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.3390/e15041416" xlink:title="http://dx.doi.org/10.3390/e15041416"><rect x="3860" y="10560" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1371/journal.pone.0169303" xlink:title="http://dx.doi.org/10.1371/journal.pone.0169303"><rect x="3860" y="10810" width="28" height="28"/></a>
<a xlink:href="https://doi.org/10.3389/fmicb.2017.00004" xlink:title="https://doi.org/10.3389/fmicb.2017.00004"><rect x="3870" y="10980" width="219" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1038/srep41802" xlink:title="http://dx.doi.org/10.1038/srep41802"><rect x="3870" y="11176" width="128" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.chom.2017.06.006" xlink:title="http://dx.doi.org/10.1016/j.chom.2017.06.006"><rect x="3870" y="11372" width="478" height="33"/></a>
<a xlink:href="07392" xlink:title="07392"><rect x="3140" y="11828" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1371/journal.pgen.1006297" xlink:title="http://dx.doi.org/10.1371/journal.pgen.1006297"><rect x="3612" y="11738" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1073/pnas.1613843113" xlink:title="http://dx.doi.org/10.1073/pnas.1613843113"><rect x="3612" y="11986" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.celrep.2016.12.010" xlink:title="http://dx.doi.org/10.1016/j.celrep.2016.12.010"><rect x="3622" y="12156" width="247" height="33"/></a>
<a xlink:href="07492" xlink:title="07492"><rect x="3140" y="12686" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.7554/eLife.18678" xlink:title="http://dx.doi.org/10.7554/eLife.18678"><rect x="3672" y="12754" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.7554/eLife.18678" xlink:title="http://dx.doi.org/10.7554/eLife.18678"><rect x="3682" y="12710" width="140" height="33"/></a>
<a xlink:href="07592" xlink:title="07592"><rect x="3140" y="13232" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/s41598-017-04197-4" xlink:title="http://dx.doi.org/10.1038/s41598-017-04197-4"><rect x="3638" y="13250" width="497" height="33"/></a>
<a xlink:href="07692" xlink:title="07692"><rect x="3140" y="13500" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1073/pnas.1608765113" xlink:title="http://dx.doi.org/10.1073/pnas.1608765113"><rect x="3864" y="13648" width="28" height="28"/></a>
<a xlink:href="http://biology.usf.edu/ib/data/flyers/SCHWARTZ-FISH-LABELING-1-2015.pdf" xlink:title="http://biology.usf.edu/ib/data/flyers/SCHWARTZ-FISH-LABELING-1-2015.pdf"><rect x="4322" y="13352" width="28" height="28"/></a>
<a xlink:href="http://io9.com/confirmation-that-photosynthesizing-sea-slugs-steal-gen-1683702602" xlink:title="http://io9.com/confirmation-that-photosynthesizing-sea-slugs-steal-gen-1683702602"><rect x="4322" y="13432" width="28" height="28"/></a>
<a xlink:href="07792" xlink:title="07792"><rect x="3140" y="14030" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.gde.2015.03.005" xlink:title="http://dx.doi.org/10.1016/j.gde.2015.03.005"><rect x="4202" y="14046" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1073/pnas.1612290113" xlink:title="http://dx.doi.org/10.1073/pnas.1612290113"><rect x="3624" y="14310" width="221" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1088/1478-3975/8/1/015001" xlink:title="http://dx.doi.org/10.1088/1478-3975/8/1/015001"><rect x="3614" y="13864" width="28" height="28"/></a>
<a xlink:href="07892" xlink:title="07892"><rect x="3140" y="14738" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.it.2016.08.002" xlink:title="http://dx.doi.org/10.1016/j.it.2016.08.002"><rect x="3718" y="14514" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.molcel.2016.11.012" xlink:title="http://dx.doi.org/10.1016/j.molcel.2016.11.012"><rect x="4176" y="14684" width="251" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1038/nature14432" xlink:title="http://dx.doi.org/10.1038/nature14432"><rect x="4584" y="14846" width="386" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1172/JCI89927" xlink:title="http://dx.doi.org/10.1172/JCI89927"><rect x="4596" y="15040" width="352" height="33"/></a>
<a xlink:href="07992" xlink:title="07992"><rect x="3140" y="15144" width="28" height="28"/></a>
<a xlink:href="08092" xlink:title="08092"><rect x="3140" y="15224" width="28" height="28"/></a>
<a xlink:href="08192" xlink:title="08192"><rect x="3140" y="15304" width="28" height="28"/></a>
<a xlink:href="08292" xlink:title="08292"><rect x="3140" y="15384" width="28" height="28"/></a>
<a xlink:href="08392" xlink:title="08392"><rect x="3140" y="15464" width="28" height="28"/></a>
<a xlink:href="08492" xlink:title="08492"><rect x="3140" y="15544" width="28" height="28"/></a>
<a xlink:href="08592" xlink:title="08592"><rect x="3140" y="15624" width="28" height="28"/></a>
<a xlink:href="08692" xlink:title="08692"><rect x="3140" y="15704" width="28" height="28"/></a>
<a xlink:href="08792" xlink:title="08792"><rect x="3140" y="15784" width="28" height="28"/></a>
<a xlink:href="08892" xlink:title="08892"><rect x="3140" y="15864" width="28" height="28"/></a>
<a xlink:href="08992" xlink:title="08992"><rect x="3140" y="15944" width="28" height="28"/></a>
<a xlink:href="09092" xlink:title="09092"><rect x="3140" y="16024" width="28" height="28"/></a>
<a xlink:href="09192" xlink:title="09192"><rect x="3140" y="16104" width="28" height="28"/></a>
<a xlink:href="09292" xlink:title="09292"><rect x="3140" y="16184" width="28" height="28"/></a>
<a xlink:href="09392" xlink:title="09392"><rect x="3140" y="16264" width="28" height="28"/></a>
<a xlink:href="09492" xlink:title="09492"><rect x="3140" y="16344" width="28" height="28"/></a>
<a xlink:href="09592" xlink:title="09592"><rect x="3140" y="16424" width="28" height="28"/></a>
<a xlink:href="09692" xlink:title="09692"><rect x="3140" y="16504" width="28" height="28"/></a>
<a xlink:href="09792" xlink:title="09792"><rect x="3140" y="16584" width="28" height="28"/></a>
<a xlink:href="09892" xlink:title="09892"><rect x="3140" y="16664" width="28" height="28"/></a>
<a xlink:href="09992" xlink:title="09992"><rect x="3140" y="16744" width="28" height="28"/></a>
</svg>
</div>
</div>

<br/>
<a href="javascript:showhide('outlineDiv')">Outline show/hide</a>
<div id="outlineDiv" style="display:none;">
<h1>00092</h1>
<p>eve</p>
<h2>05092</h2>
<ul>
<li>diversity, or why evolution happens
<ul>
<li>
<p><em>origin and evolution of transporter substrate specificity within the npf family</em><br />
morten egevang jørgensen, deyang xu, christoph crocoll, david ramírez, mohammed saddik motawia, carl erik olsen, hussam hassan nour-eldin, barbara ann halkier 2017<br />
<a href="http://dx.doi.org/10.7554/eLife.19466">dx.doi.org/10.7554/eLife.19466</a></p>
<ul>
<li>
<p>Despite vast diversity in metabolites and the matching substrate specificity of their transporters, little is known about how evolution of transporter substrate specificities is linked to emergence of substrates via evolution of biosynthetic pathways. Transporter specificity towards the recently evolved glucosinolates characteristic of <em>Brassicales</em> is shown to evolve prior to emergence of glucosinolate biosynthesis. Furthermore, we show that glucosinolate transporters belonging to the ubiquitous NRT1/PTR FAMILY (NPF) likely evolved from transporters of the ancestral cyanogenic glucosides found across more than 2500 species outside of the <em>Brassicales</em>. Biochemical characterization of orthologs along the phylogenetic lineage from cassava to <em>A. thaliana,</em> suggests that alterations in the electrogenicity of the transporters accompanied changes in substrate specificity. Linking the evolutionary path of transporter substrate specificities to that of the biosynthetic pathways, exemplify how transporter substrate specificities originate and evolve as new biosynthesis pathways emerge.</p>
<p>All living cells are surrounded by membranes that protect them from the external environment. The membrane contains proteins called transporters, which move nutrients and other molecules (known as substrates) across the membrane. A variety of transporters have evolved to move the hundreds of thousands of different substrates found in nature.<br />
Plant cells make many different compounds to protect themselves from pests and diseases. A group of transporters known as the NPF family move some of these compounds across the cells outer membrane. The types of substrates they transport vary in different plants. In cassava, for example, NPF transporters move compounds called cyanogenic glucosides, which are poisonous to humans and other animals. On the other hand, NPF transporters in another plant called <em>Arabidopsis thaliana</em> can move bitter-tasting compounds called glucosinolates. The process that makes glucosinolates in plants evolved from the process that makes cyanogenic glucosides.<br />
Can transporters evolve the ability to move a new substrate before or after that substrate first appears? To answer this question, Jørgensen et al. studied the NPF family in <em>A. thaliana</em>, cassava and another plant called papaya that makes both cyanogenic glucosides and glucosinolates. The experiments suggest that NPF transporters able to move both cyanogenic glucosides and glucosinolates evolved before plants evolved the ability to make glucosinolates. Later in evolution, these multi-specific transporters specialized to only move glucosinolates. Jørgensen et al. also show that early glucosinolate transporters could move a broad variety of glucosinolates but later evolved to only transport particular types.<br />
These findings show how transporters and the processes that make compounds in cells may evolve together. A future challenge will be to understand the molecular changes in a transporter that make it specific for a certain substrate. This may help researchers to develop new ways of controlling the amount of toxic compounds in crops we eat by manipulating how the compounds are transported.</p>
</li>
</ul>
</li>
<li>
<p><em>environmental change explains cichlid adaptive radiation at lake malawi over the past 1.2 million years</em><br />
sarah j. ivory, margaret w. blome, john w. king, michael m. mcglue, julia e. cole, andrew s. cohen 2016<br />
http://dx.doi.org/10.1073/pnas.1611028113<br />
Link: <a href="http://dx.doi.org/10.1073/pnas.1611028113">dx.doi.org/10.1073/pnas.1611028113</a></p>
<ul>
<li>
<p>Tropical African lakes are well-known to house exceptionally biodiverse assemblages of fish and other aquatic fauna, which are thought to be at risk in the future. Although the modern assemblages are well-studied, direct evidence of the origin of this incredible wealth of species and the mechanisms that drive speciation are virtually unknown. We use a long sedimentary record from Lake Malawi to show that over the last 1.2 My both large-scale climatic and tectonic changes resulted in wet–dry transitions that led to extraordinary habitat variability and rapid diversification events. This work allows us to understand the environmental context of aquatic evolution in the most biodiverse tropical lake.<br />
Abstract</p>
<p>Long paleoecological records are critical for understanding evolutionary responses to environmental forcing and unparalleled tools for elucidating the mechanisms that lead to the development of regions of high biodiversity. We use a 1.2-My record from Lake Malawi, a textbook example of biological diversification, to document how climate and tectonics have driven ecosystem and evolutionary dynamics. Before ∼800 ka, Lake Malawi was much shallower than today, with higher frequency but much lower amplitude water-level and oxygenation changes. Since ∼800 ka, the lake has experienced much larger environmental fluctuations, best explained by a punctuated, tectonically driven rise in its outlet location and level. Following the reorganization of the basin, a change in the pacing of hydroclimate variability associated with the Mid-Pleistocene Transition resulted in hydrologic change dominated by precession rather than the high-latitude teleconnections recorded elsewhere. During this time, extended, deep lake phases have abruptly alternated with times of extreme aridity and ecosystem variability. Repeated crossings of hydroclimatic thresholds within the lake system were critical for establishing the rhythm of diversification, hybridization, and extinction that dominate the modern system. The chronology of these changes closely matches both the timing and pattern of phylogenetic history inferred independently for the lake’s extraordinary array of cichlid fish species, suggesting a direct link between environmental and evolutionary dynamics.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>05192</h2>
<ul>
<li>
<p>darwin online<br />
<a href="http://darwin-online.org.uk/">darwin-online.org.uk/</a></p>
<ul>
<li>make chapter–by–chapter comparison of all editions of <em>on the origin of species</em> using ithoughts</li>
</ul>
</li>
<li>
<p>wallace online<br />
<a href="http://wallace-online.org/">wallace-online.org/</a></p>
</li>
</ul>
<h2>05292</h2>
<ul>
<li>gene expression
<ul>
<li>
<p>x inactivation</p>
</li>
<li>
<p><em>diverse non-genetic, allele-specific expression effects shape genetic architecture at the cellular level in the mammalian brain</em><br />
wei-chao huang et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.neuron.2017.01.033">j.neuron.2017.01.033</a></p>
<ul>
<li>
<p>•In vivo genome-wide screen uncovers diverse non-genetic allelic effects<br />
•Non-genetic allelic effects are prevalent in the neonatal mouse brain<br />
•Allelic effects cause mosaics of mutant and wild-type cells for heterozygous mutations<br />
•Allelic effects exist in the primate brain and impact genes linked to mental illness</p>
<p>Interactions between genetic and epigenetic effects shape brain function, behavior, and the risk for mental illness. Random X inactivation and genomic imprinting are epigenetic allelic effects that are well known to influence genetic architecture and disease risk. Less is known about the nature, prevalence, and conservation of other potential epigenetic allelic effects in vivo in the mouse and primate brain. Here we devise genomics, in situ hybridization, and mouse genetics strategies to uncover diverse allelic effects in the brain that are not caused by imprinting or genetic variation. We found allelic effects that are developmental stage and cell type specific, that are prevalent in the neonatal brain, and that cause mosaics of monoallelic brain cells that differentially express wild-type and mutant alleles for heterozygous mutations. Finally, we show that diverse non-genetic allelic effects that impact mental illness risk genes exist in the macaque and human brain. Our findings have potential implications for mammalian brain genetics.</p>
</li>
</ul>
</li>
<li>
<p>“silencing one gene copy may be a way in which cells fine tune their genetic program at specific times during the lifecycle of the animal, or in discrete places”<br />
Link: <a href="https://www.sciencedaily.com/releases/2017/02/170223124227.htm">sciencedaily.com/releases/2017/02/170223124227.htm</a></p>
</li>
<li>
<p><em>impacts of neanderthal-introgressed sequences on the landscape of human gene expression</em><br />
rajiv c. mccoy, jon wakefield, joshua m. akey 2017<br />
<a href="http://dx.doi.org/10.1016/j.cell.2017.01.038">j.cell.2017.01.038</a></p>
<ul>
<li>Regulatory variation influencing gene expression is a key contributor to phenotypic diversity, both within and between species. Unfortunately, RNA degrades too rapidly to be recovered from fossil remains, limiting functional genomic insights about our extinct hominin relatives. Many Neanderthal sequences survive in modern humans due to ancient hybridization, providing an opportunity to assess their contributions to transcriptional variation and to test hypotheses about regulatory evolution. We developed a flexible Bayesian statistical approach to quantify allele-specific expression (ASE) in complex RNA-seq datasets. We identified widespread expression differences between Neanderthal and modern human alleles, indicating pervasive cis-regulatory impacts of introgression. Brain regions and testes exhibited significant downregulation of Neanderthal alleles relative to other tissues, consistent with natural selection influencing the tissue-specific regulatory landscape. Our study demonstrates that Neanderthal-inherited sequences are not silent remnants of ancient interbreeding but have measurable impacts on gene expression that contribute to variation in modern human phenotypes.</li>
<li>identified people who had one copy of gene from modern humans and other copy from neanderthals</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>05392</h2>
<ul>
<li>
<p><em>trade-off between transcriptome plasticity and genome evolution in cephalopods</em><br />
liscovitch-brauer et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.cell.2017.03.025">dx.doi.org/10.1016/j.cell.2017.03.025</a></p>
<ul>
<li>
<p>•Unlike other taxa, cephalopods diversify their proteomes extensively by RNA editing<br />
•Extensive recoding is specific to the behaviorally complex coleiods<br />
•Unlike mammals, cephalopod recoding is evolutionarily conserved and often adaptive<br />
•Transcriptome diversification comes at the expense of slowed-down genome evolution</p>
<p>RNA editing, a post-transcriptional process, allows the diversification of proteomes beyond the genomic blueprint; however it is infrequently used among animals for this purpose. Recent reports suggesting increased levels of RNA editing in squids thus raise the question of the nature and effects of these events. We here show that RNA editing is particularly common in behaviorally sophisticated coleoid cephalopods, with tens of thousands of evolutionarily conserved sites. Editing is enriched in the nervous system, affecting molecules pertinent for excitability and neuronal morphology. The genomic sequence flanking editing sites is highly conserved, suggesting that the process confers a selective advantage. Due to the large number of sites, the surrounding conservation greatly reduces the number of mutations and genomic polymorphisms in protein-coding regions. This trade-off between genome evolution and transcriptome plasticity highlights the importance of RNA recoding as a strategy for diversifying proteins, particularly those associated with neural function.</p>
</li>
</ul>
</li>
</ul>
<h2>05492</h2>
<ul>
<li>
<p><em>coupling between distant biofilms and emergence of nutrient time-sharing</em><br />
liu j et al. 2017<br />
<a href="http://dx.doi.org/10.1126/science.aah4204">dx.doi.org/10.1126/science.aah4204</a></p>
<ul>
<li>Bacteria within communities can interact to organize their behavior. It remains unclear whether such interactions extend beyond a single community to coordinate the behavior of distant populations. We discovered that two <em>Bacillus subtilis</em> biofilm communities undergoing metabolic oscillations become coupled through electrical signaling and synchronize their growth dynamics. Coupling increases competition by also synchronizing demand for limited nutrients. As predicted by mathematical modeling, we confirm that biofilms resolve this conflict by switching from in-phase to anti-phase oscillations. This results in time-sharing behavior where each community takes turns consuming nutrients. Time-sharing enables biofilms to counterintuitively increase growth under reduced nutrient supply. Distant biofilms can thus coordinate their behavior to resolve nutrient competition through time-sharing, a strategy used in engineered systems to allocate limited resources.</li>
</ul>
</li>
</ul>
<h2>05592</h2>
<ul>
<li>
<p><em>indolepropionic acid and novel lipid metabolites are associated with a lower risk of type 2 diabetes in the finnish diabetes prevention study</em><br />
vanessa d. de mello et al. 2017<br />
<a href="http://dx.doi.org/10.1038/srep46337">dx.doi.org/10.1038/srep46337</a></p>
<ul>
<li>Wide-scale profiling technologies including metabolomics broaden the possibility of novel discoveries related to the pathogenesis of type 2 diabetes (T2D). By applying non-targeted metabolomics approach, we investigated here whether serum metabolite profile predicts T2D in a well-characterized study population with impaired glucose tolerance by examining two groups of individuals who took part in the Finnish Diabetes Prevention Study (DPS); those who either early developed T2D (n = 96) or did not convert to T2D within the 15-year follow-up (n = 104). Several novel metabolites were associated with lower likelihood of developing T2D, including indole and lipid related metabolites. Higher indolepropionic acid was associated with reduced likelihood of T2D in the DPS. Interestingly, in those who remained free of T2D, indolepropionic acid and various lipid species were associated with better insulin secretion and sensitivity, respectively. Furthermore, these metabolites were negatively correlated with low-grade inflammation. We replicated the association between indolepropionic acid and T2D risk in one Finnish and one Swedish population. We suggest that indolepropionic acid, a gut microbiota-produced metabolite, is a potential biomarker for the development of T2D that may mediate its protective effect by preservation of β-cell function. Novel lipid metabolites associated with T2D may exert their effects partly through enhancing insulin sensitivity.</li>
</ul>
</li>
</ul>
<h2>05692</h2>
<ul>
<li>
<p><em>age-associated microbial dysbiosis promotes intestinal permeability, systemic inflammation, and macrophage dysfunction</em><br />
netusha thevaranjan et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.chom.2017.03.002">dx.doi.org/10.1016/j.chom.2017.03.002</a></p>
<ul>
<li>
<p>•Age-associated inflammation drives macrophage dysfunction and tissue damage<br />
•Mice under germ-free conditions are protected from age-associated inflammation<br />
•Co-housing germ-free mice with old, but not young, mice increases age-related inflammation<br />
•Age-related microbiota changes can be reversed by reducing TNF levels</p>
<p>Levels of inflammatory mediators in circulation are known to increase with age, but the underlying cause of this age-associated inflammation is debated. We find that, when maintained under germ-free conditions, mice do not display an age-related increase in circulating pro-inflammatory cytokine levels. A higher proportion of germ-free mice live to 600 days than their conventional counterparts, and macrophages derived from aged germ-free mice maintain anti-microbial activity. Co-housing germ-free mice with old, but not young, conventionally raised mice increases pro-inflammatory cytokines in the blood. In tumor necrosis factor (TNF)-deficient mice, which are protected from age-associated inflammation, age-related microbiota changes are not observed. Furthermore, age-associated microbiota changes can be reversed by reducing TNF using anti-TNF therapy. These data suggest that aging-associated microbiota promote inflammation and that reversing these age-related microbiota changes represents a potential strategy for reducing age-associated inflammation and the accompanying morbidity.</p>
</li>
</ul>
</li>
</ul>
<h2>05792</h2>
<ul>
<li>
<p><em>differences in gut microbial composition correlate with regional brain volumes in irritable bowel syndrome</em><br />
jennifer s. labus et al. 2017<br />
<a href="http://dx.doi.org/10.1186/s40168-017-0260-z">dx.doi.org/10.1186/s40168-017-0260-z</a></p>
<ul>
<li>The results confirm previous reports of gut microbiome-based IBS subgroups and identify for the first time brain structural alterations associated with these subgroups. They provide preliminary evidence for the involvement of specific microbes and their predicted metabolites in these correlations.</li>
</ul>
</li>
</ul>
<h2>05892</h2>
<ul>
<li>
<p><em>legacy effects of developmental stages determine the functional role of predators</em><br />
volker h. w. rudolf &amp; b. g. van allen 2017<br />
<a href="http://dx.doi.org/10.1038/s41559-016-0038">dx.doi.org/10.1038/s41559-016-0038</a></p>
<ul>
<li>Predators are instrumental in structuring natural communities and ecosystem processes. The strong effects of predators are often attributed to their high trophic position in the food web. However, most predators have to grow and move up the food chain before reaching their final trophic position, and during this developmental process their traits, interactions and abundances change. Here, we show that this process of ‘moving up’ the food chain during development strongly determines the ecological role of a predator. By experimentally manipulating the succession of developmental stages of a predatory salamander in a seasonal aquatic ecosystem, we found that the effects of this apex predator on the ecosystem typically declined with age and size. Furthermore, younger, smaller predator stages had long-lasting effects on community structure and ecosystem function that determined the effects of subsequent older, larger stages. Consequently, the legacy effects of early stages largely shaped the impact of the predator on the ecosystem, which could not simply be inferred from its final trophic position. Our results highlight that accounting for all life stages when managing natural populations is crucial to preserve the functioning of natural ecosystems, especially given that early life stages of species are often particularly vulnerable to natural and anthropogenic disturbances.</li>
</ul>
</li>
</ul>
<h2>05992</h2>
<ul>
<li>cascade: continuous and punctuated
<ul>
<li>
<p><em>self-motion evokes precise spike timing in the primate vestibular system</em><br />
mohsen jamali, maurice j. chacron, kathleen e. cullen 2016<br />
<a href="http://dx.doi.org/10.1038/ncomms13229">ncomms13229</a></p>
<ul>
<li>The accurate representation of self-motion requires the efficient processing of sensory input by the vestibular system. Conventional wisdom is that vestibular information is exclusively transmitted through changes in firing rate, yet under this assumption vestibular neurons display relatively poor detection and information transmission. Here, we carry out an analysis of the system’s coding capabilities by recording neuronal responses to repeated presentations of naturalistic stimuli. We find that afferents with greater intrinsic variability reliably discriminate between different stimulus waveforms through differential patterns of precise (∼6 ms) spike timing, while those with minimal intrinsic variability do not. A simple mathematical model provides an explanation for this result. Postsynaptic central neurons also demonstrate precise spike timing, suggesting that higher brain areas also represent self-motion using temporally precise firing. These findings demonstrate that two distinct sensory channels represent vestibular information: one using rate coding and the other that takes advantage of precise spike timing.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>06092</h2>
<ul>
<li>commensualism
<ul>
<li>
<p><em>fishing down nutrients on coral reefs</em><br />
jacob e. allgeier, abel valdivia, courtney cox, craig a. layman 2016<br />
http://dx.doi.org/10.1038/ncomms12461<br />
Link: <a href="http://dx.doi.org/10.1038/ncomms12461">dx.doi.org/10.1038/ncomms12461</a></p>
<ul>
<li>Fishing is widely considered a leading cause of biodiversity loss in marine environments, but the potential effect on ecosystem processes, such as nutrient fluxes, is less explored. Here, we test how fishing on Caribbean coral reefs influences biodiversity and ecosystem functions provided by the fish community, that is, fish-mediated nutrient capacity. Specifically, we modelled five processes of nutrient storage (in biomass) and supply (via excretion) of nutrients, as well as a measure of their multifunctionality, onto 143 species of coral reef fishes across 110 coral reef fish communities. These communities span a gradient from extreme fishing pressure to protected areas with little to no fishing. We find that in fished sites fish-mediated nutrient capacity is reduced almost 50%, despite no substantial changes in the number of species. Instead, changes in community size and trophic structure were the primary cause of shifts in ecosystem function. These findings suggest that a broader perspective that incorporates predictable impacts of fishing pressure on ecosystem function is imperative for effective coral reef conservation and management.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>06192</h2>
<h2>06292</h2>
<ul>
<li>
<p><em>what does it mean to be human?</em><br />
gaia vince discovers that analysing the genetics of ancient humans means changing ideas about our evolution 2017</p>
<p><a href="https://mosaicscience.com/story/ancient-human-evolution-neanderthal-genetics">mosaicscience.com/story/ancient-human-evolution-neanderthal-genetics</a></p>
<ul>
<li>
<p>The things that we thought we understood about Europeans are coming unstuck as we examine the genes of more ancient people. For example, it was generally accepted that pale skin evolved so we could get more vitamin D after moving north to where there was little sun and people had to cover up against the cold. But it turns out that it was the Yamnaya people from much further south, tall and brown-eyed, who brought pale skins to Europe. Northern Europeans before then were dark-skinned and got plenty of vitamin D from eating fish.<br />
It is the same with lactose tolerance. Around 90 per cent of Europeans have a genetic mutation that allows them to digest milk into adulthood, and scientists had assumed that this gene evolved in farmers in northern Europe, giving them an additional food supply to help survive the long winters. But Eske’s research using the genomes of hundreds of Bronze Age people, who lived after the advent of farming, has cast doubt on this theory too: “We found that the genetic trait was almost non-existent in the European population. This trait only became abundant in the northern European population within the last 2,000 years,” he says.<br />
It turns out that lactose tolerance genes were also introduced by the Yamnaya. “They had a slightly higher tolerance to milk than the European farmers and must have introduced it to the European gene pool. Maybe there was a disaster around 2,000 years ago that caused a population bottleneck and allowed the gene to take off. The Viking sagas talk about the sun becoming black – a major volcanic eruption – that could have caused a massive drop in population size, which could have been where some of that stock takes off with lactose.”</p>
</li>
<li>
<p>yamnaya<br />
<a href="https://mobile.nytimes.com/2015/06/16/science/dna-deciphers-roots-of-modern-europeans.html?_r=0&amp;referer=https://duckduckgo.com/">mobile.nytimes.com/2015/06/16/science/dna-deciphers-roots-of-modern-europeans.html</a></p>
</li>
<li>
<p><a href="http://www.nature.com/news/european-languages-linked-to-migration-from-the-east-1.16919">nature.com/news/european-languages-linked-to-migration-from-the-east-1.16919</a></p>
</li>
<li>
<p><em>ancient human genomes suggest three ancestral populations for present–day humans</em><br />
<a href="http://www.nature.com/articles/nature13673.epdf?referrer_access_token=dUGFQ4p5aSftizrGNtcLtdRgN0jAjWel9jnR3ZoTv0MaJb0jOMpBA1y27HQGlCUiHaqx24ysNx_tIj9u3K68hIb67qNeR62pjGLMM_oheXxudlZ3ujrsQdcQ3y1Hv43qAtla3QkjTusatPWqibFrGLtYM3oNKg5nWyQD1eagFah-7e4zquIYKbz-elLhtQJkPiggMbPOOQtzFYi1DswNxZwPkSWHeMCYkX--l9qLsMH8I6o9psIi6HKUXdio5DylzrTjc4FaiOtO4VzrDa3EPg%3D%3D&amp;tracking_referrer=www.nature.com">nature.com/articles/nature13673.epdf</a></p>
</li>
<li>
<p><em>neanderthal behaviour, diet, and disease inferred from ancient dna in dental calculus</em><br />
laura s. weyrich et al. 2017<br />
<a href="http://dx.doi.org/10.1038/nature21674">dx.doi.org/10.1038/nature21674</a></p>
<ul>
<li>Recent genomic data have revealed multiple interactions between Neanderthals and modern humans<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21674.html#ref1">27</a>, but there is currently little genetic evidence regarding Neanderthal behaviour, diet, or disease. Here we describe the shotgun-sequencing of ancient DNA from five specimens of Neanderthal calcified dental plaque (calculus) and the characterization of regional differences in Neanderthal ecology. At Spy cave, Belgium, Neanderthal diet was heavily meat based and included woolly rhinoceros and wild sheep (mouflon), characteristic of a steppe environment. In contrast, no meat was detected in the diet of Neanderthals from El Sidrón cave, Spain, and dietary components of mushrooms, pine nuts, and moss reflected forest gathering<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21674.html#ref2">28</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21674.html#ref3">29</a>. Differences in diet were also linked to an overall shift in the oral bacterial community (microbiota) and suggested that meat consumption contributed to substantial variation within Neanderthal microbiota. Evidence for self-medication was detected in an El Sidrón Neanderthal with a dental abscess<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21674.html#ref4">30</a> and a chronic gastrointestinal pathogen (<em>Enterocytozoon bieneusi</em>). Metagenomic data from this individual also contained a nearly complete genome of the archaeal commensal <em>Methanobrevibacter oralis</em> (10.2× depth of coverage)—the oldest draft microbial genome generated to date, at around 48,000 years old. DNA preserved within dental calculus represents a notable source of information about the behaviour and health of ancient hominin specimens, as well as a unique system that is useful for the study of long-term microbial evolution.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>06392</h2>
<ul>
<li>
<p><em>variable habitat conditions drive species covariation in the human microbiota</em><br />
charles k. fisher, thierry mora, aleksandra m. walczak 2017<br />
<a href="http://dx.doi.org/10.1371/journal.pcbi.1005435">dx.doi.org/10.1371/journal.pcbi.1005435</a></p>
<ul>
<li>
<p>Two species with similar resource requirements respond in a characteristic way to variations in their habitat—their abundances rise and fall in concert. We use this idea to learn how bacterial populations in the microbiota respond to habitat conditions that vary from person-to-person across the human population. Our mathematical framework shows that habitat fluctuations are sufficient for explaining intra-bodysite correlations in relative species abundances from the Human Microbiome Project. We explicitly show that the relative abundances of closely related species are positively correlated and can be predicted from taxonomic relationships. We identify a small set of functional pathways related to metabolism and maintenance of the cell wall that form the basis of a common resource sharing niche space of the human microbiota.</p>
<p>The human body is inhabited by a vast number of microorganisms comprising the human microbiota. The species composition of the microbiota varies considerably from person-to-person and the relative abundances of some species rise and fall in concert. We introduce a mathematical model where differences in habitat conditions cause most of the variability of the microbiota. A statistical analysis shows that variable habitat conditions are sufficient for explaining the patterns of variation observed across a healthy human population and, as a result, the correlation between the relative abundances of two species reflects how closely related they are rather than how they directly interact with each other.</p>
</li>
</ul>
</li>
</ul>
<h2>06492</h2>
<ul>
<li>
<p><em>antibiotic-producing symbionts dynamically transition between plant pathogenicity and insect-defensive mutualism</em><br />
laura v. flórez et al. 2017<br />
<a href="https://www.nature.com/articles/ncomms15172">nature.com/articles/ncomms15172</a></p>
<ul>
<li>Pathogenic and mutualistic bacteria associated with eukaryotic hosts often lack distinctive genomic features, suggesting regular transitions between these lifestyles. Here we present evidence supporting a dynamic transition from plant pathogenicity to insect-defensive mutualism in symbiotic <em>Burkholderia gladioli</em> bacteria. In a group of herbivorous beetles, these symbionts protect the vulnerable egg stage against detrimental microbes. The production of a blend of antibiotics by <em>B. gladioli</em>, including toxoflavin, caryoynencin and two new antimicrobial compounds, the macrolide lagriene and the isothiocyanate sinapigladioside, likely mediate this defensive role. In addition to vertical transmission, these insect symbionts can be exchanged via the host plant and retain the ability to initiate systemic plant infection at the expense of the plant’s fitness. Our findings provide a paradigm for the transition between pathogenic and mutualistic lifestyles and shed light on the evolution and chemical ecology of this defensive mutualism.</li>
</ul>
</li>
</ul>
<h2>06592</h2>
<ul>
<li>
<p><em>endothelial tlr4 and the microbiome drive cerebral cavernous malformations</em><br />
alan t. tang et al. 2017<br />
<a href="http://dx.doi.org/10.1038/nature22075">dx.doi.org/10.1038/nature22075</a></p>
<ul>
<li>Cerebral cavernous malformations (CCMs) are a cause of stroke and seizure for which no effective medical therapies yet exist. CCMs arise from the loss of an adaptor complex that negatively regulates MEKK3–KLF2/4 signalling in brain endothelial cells, but upstream activators of this disease pathway have yet to be identified. Here we identify endothelial Toll-like receptor 4 (TLR4) and the gut microbiome as critical stimulants of CCM formation. Activation of TLR4 by Gram-negative bacteria or lipopolysaccharide accelerates CCM formation, and genetic or pharmacologic blockade of TLR4 signalling prevents CCM formation in mice. Polymorphisms that increase expression of the <em>TLR4</em> gene or the gene encoding its co-receptor CD14 are associated with higher CCM lesion burden in humans. Germ-free mice are protected from CCM formation, and a single course of antibiotics permanently alters CCM susceptibility in mice. These studies identify unexpected roles for the microbiome and innate immune signalling in the pathogenesis of a cerebrovascular disease, as well as strategies for its treatment.</li>
</ul>
</li>
</ul>
<h2>06692</h2>
<ul>
<li>
<p><em>sexual dimorphism and retinal mosaic diversification following the evolution of a violet receptor in butterflies</em><br />
kyle j. mcculloch et al. 2017<br />
<a href="http://dx.doi.org/10.1093/molbev/msx163">dx.doi.org/10.1093/molbev/msx163</a></p>
<ul>
<li>Numerous animal lineages have expanded and diversified the opsin-based photoreceptors in their eyes underlying color vision behavior. However, the selective pressures giving rise to new photoreceptors and their spectral tuning remain mostly obscure. Previously, we identified a violet receptor (UV2) that is the result of a UV opsin gene duplication specific to <em>Heliconius</em> butterflies*.* At the same time the violet receptor evolved, <em>Heliconius</em> evolved UV-yellow coloration on their wings, due to the pigment 3-hydroxykynurenine (3-OHK) and the nanostructure architecture of the scale cells. In order to better understand the selective pressures giving rise to the violet receptor, we characterized opsin expression patterns using immunostaining (14 species) and RNA-Seq (18 species), and reconstructed evolutionary histories of visual traits in five major lineages within <em>Heliconius</em> and one species from the genus <em>Eueides.</em> Opsin expression patterns are hyperdiverse within <em>Heliconius</em>. We identified six unique retinal mosaics and three distinct forms of sexual dimorphism based on ommatidial types within the genus <em>Heliconius</em>. Additionally, phylogenetic analysis revealed independent losses of opsin expression, pseudogenization events, and relaxation of selection on <em>UVRh2</em> in one lineage. Despite this diversity, the newly evolved violet receptor is retained across most species and sexes surveyed. Discriminability modeling of behaviorally preferred 3-OHK yellow wing coloration suggests that the violet receptor may facilitate <em>Heliconius</em> color vision in the context of conspecific recognition. Our observations give insights into the selective pressures underlying the origins of new visual receptors.</li>
</ul>
</li>
</ul>
<h2>06792</h2>
<ul>
<li>
<p><em>microbial genetic composition tunes host longevity</em><br />
bing han et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.cell.2017.05.036">dx.doi.org/10.1016/j.cell.2017.05.036</a></p>
<ul>
<li>
<pre><code>    •Systematic analysis of longevity-promoting microbial genetic variations  
      •Colanic acid as a pro-longevity natural compound effective in different species  
      •Bacterial metabolites regulate host mitochondrial dynamics and UPRmit  
</code></pre>
<p>Homeostasis of the gut microbiota critically influences host health and aging. Developing genetically engineered probiotics holds great promise as a new therapeutic paradigm to promote healthy aging. Here, through screening 3,983 <em>Escherichia coli</em> mutants, we discovered that 29 bacterial genes, when deleted, increase longevity in the host <em>Caenorhabditis elegans</em>. A dozen of these bacterial mutants also protect the host from age-related progression of tumor growth and amyloid-beta accumulation. Mechanistically, we discovered that five bacterial mutants promote longevity through increased secretion of the polysaccharide colanic acid (CA), which regulates mitochondrial dynamics and unfolded protein response (UPRmt) in the host. Purified CA polymers are sufficient to promote longevity via ATFS-1, the host UPRmt-responsive transcription factor. Furthermore, the mitochondrial changes and longevity effects induced by CA are conserved across different species. Together, our results identified molecular targets for developing pro-longevity microbes and a bacterial metabolite acting on host mitochondria to promote longevity.</p>
</li>
<li>
<p>&quot;These findings are also interesting and have implications from the biological point of view in the way we understand host-microbe communication,&quot; Wang said. &quot;Mitochondria seem to have evolved from bacteria that millions of years ago entered primitive cells. Our finding suggests that products from bacteria today can still chime in the communication between mitochondria in our cells. We think that this type of communication is very important and here we have provided the first evidence of this. Fully understanding microbe-mitochondria communication can help us understand at a deeper level the interactions between microbes and their hosts.&quot;</p>
</li>
</ul>
</li>
</ul>
<h2>06892</h2>
<h2>06992</h2>
<h2>07092</h2>
<ul>
<li>transmit across host organisms
<ul>
<li>culture</li>
<li>niche construction</li>
</ul>
</li>
</ul>
<h2>07192</h2>
<ul>
<li>microbiome
<ul>
<li>
<p><em>animals in a bacterial world, a new imperative for the life sciences</em><br />
mcfall–ngai et al 2013<br />
<a href="http://www.pnas.org/cgi/doi/10.1073/pnas.1218525110">pnas.1218525110</a></p>
<ul>
<li>In the last two decades, the widespread application of genetic and genomic approaches has revealed a bacterial world astonishing in its ubiquity and diversity. This review examines how a growing knowledge of the vast range of animal–bacterial interactions, whether in shared ecosystems or intimate symbioses, is fundamentally altering our understanding of animal biology. Specifically, we highlight recent techno-logical and intellectual advances that have changed our thinking about five questions: how have bacteria facilitated the origin and evolution of animals; how do animals and bacteria affect each other's genomes; how does normal animal development depend on bacterial partners; how is homeostasis maintained between animals and their symbionts; and how can ecological approaches deepen our understanding of the multiple levels of animal–bacterial interaction. As answers to these fundamental questions emerge, all biologists will be challenged to broaden their appreciation of these interactions and to include investigations of the relationships between and among bacteria and their animal partners as we seek a better understanding of the natural world.</li>
<li>the New Synthesis of the 1930s and beyond must be reconsidered in terms of three areas in which it has proven weakest: symbiosis, de-velopment, and microbiology (115). One of these areas, microbiology, presents particular challenges both to the species concept, as formulated by Ernst Mayr in 1942, and to the concept that vertical transmission of genetic information is the only motor of selectable evolutionary change.</li>
</ul>
</li>
<li>
<p><em>i, superorganism: learning to love your inner ecosystem</em><br />
jon turney 2015<br />
9781848318229</p>
<ul>
<li>diverse ecosystems but similar collective metabolism</li>
<li>has full story of helicobacter pylori</li>
</ul>
</li>
<li>
<p><em>i contain multitudes: the microbes within us and a grander view of life</em><br />
ed jong 2016<br />
9780062368621</p>
</li>
<li>
<p><em>brain maker: the power of gut microbes to heal and protect your brain — for life</em><br />
david perlmutter 2015</p>
<ul>
<li>more controversial</li>
</ul>
</li>
<li>
<p><em>the gut microbiota of rural papua new guineans: composition, diversity patterns, and ecological processes</em><br />
inés martínez et al. 2015</p>
<p><a href="http://dx.doi.org/10.1016/j.celrep.2015.03.049">j.celrep.2015.03.049</a></p>
<ul>
<li>
<p>•The fecal microbiota in PNG is more diverse but less individualized than in the US<br />
•Most bacterial species are shared among PNG and the US, but abundance profiles differ<br />
•Impact of lifestyle on ecological assembly processes might explain these patterns<br />
•Westernization may decrease bacterial dispersal rates, altering microbiota structure</p>
<p>Although recent research revealed an impact of westernization on diversity and composition of the human gut microbiota, the exact consequences on metacommunity characteristics are insufficiently understood, and the underlying ecological mechanisms have not been elucidated. Here, we have compared the fecal microbiota of adults from two non-industrialized regions in Papua New Guinea (PNG) with that of United States (US) residents. Papua New Guineans harbor communities with greater bacterial diversity, lower inter-individual variation, vastly different abundance profiles, and bacterial lineages undetectable in US residents. A quantification of the ecological processes that govern community assembly identified bacterial dispersal as the dominant process that shapes the microbiome in PNG but not in the US. These findings suggest that the microbiome alterations detected in industrialized societies might arise from modern lifestyle factors limiting bacterial dispersal, which has implications for human health and the development of strategies aimed to redress the impact of westernization.</p>
</li>
</ul>
</li>
<li>
<p><em>stochastic assembly produces heterogeneous communities in the caenorhabditis elegans intestine</em><br />
nicole m. vega, jeff gore 2017<br />
<a href="http://dx.doi.org/10.1371/journal.pbio.2000633">dx.doi.org/10.1371/journal.pbio.2000633</a></p>
<ul>
<li>
<p>Host-associated bacterial communities vary extensively between individuals, but it can be very difficult to determine the sources of this heterogeneity. Here, we demonstrate that stochastic bacterial community assembly in the <em>Caenorhabditis elegans</em> intestine is sufficient to produce strong interworm heterogeneity in community composition. When worms are fed with two neutrally competing, fluorescently labeled bacterial strains, we observe stochastically driven bimodality in community composition, in which approximately half of the worms are dominated by each bacterial strain. A simple model incorporating stochastic colonization suggests that heterogeneity between worms is driven by the low rate at which bacteria successfully establish new intestinal colonies. We can increase this rate experimentally by feeding worms at high bacterial density; in these conditions, the bimodality disappears. These results demonstrate that demographic noise is a potentially important driver of diversity in bacterial community formation and suggest a role for <em>C</em>. <em>elegans</em> as a model system for ecology of host-associated communities.<br />
Author summary<br />
Host-associated bacterial communities—also known as microbiomes—vary extensively between individuals, even among clones exposed to the same environment. The sources of this variation are not entirely understood and can be very difficult to determine. In this manuscript, we demonstrate experimentally how randomness in bacterial colonization can result in large differences in the composition of host-associated bacterial communities, using the nematode worm <em>Caenorhabditis elegans</em> as a tractable host model. We find that the amount of variation between individual communities is a function of two rates relevant to how bacteria colonize the host intestine: the colonization rate and the birth rate. We can manipulate the degree of variation between communities by altering the colonization rate, using the amount of bacteria presented to the worms to control the rate at which migrants enter the intestine. When worms are fed with two neutrally competing, fluorescently labeled bacterial strains at low colonization rates, we are able to produce noise-induced bistability in this system, in which each community is dominated by bacteria of only one color. These results demonstrate the potential importance of noise and randomness as a driver of variation between communities and highlight the utility of the simple model organism <em>C</em>. <em>elegans</em> for studying questions relevant to host-associated microbial communities.</p>
</li>
<li>
<p>system that is stable without diffusion becomes unstable in the presence of diffusion. Turing was motivated to understand morphogenesis with this example of instability</p>
</li>
</ul>
</li>
<li>
<p><em>complex pectin metabolism by gut bacteria reveals novel catalytic functions</em><br />
didier ndeh et al. 2017<br />
<a href="http://dx.doi.org/10.1038/nature21725">nature21725</a></p>
<ul>
<li>The metabolism of carbohydrate polymers drives microbial diversity in the human gut microbiota. It is unclear, however, whether bacterial consortia or single organisms are required to depolymerize highly complex glycans. Here we show that the gut bacterium <em>Bacteroides thetaiotaomicron</em> uses the most structurally complex glycan known: the plant pectic polysaccharide rhamnogalacturonan-II, cleaving all but 1 of its 21 distinct glycosidic linkages. The deconstruction of rhamnogalacturonan-II side chains and backbone are coordinated to overcome steric constraints, and the degradation involves previously undiscovered enzyme families and catalytic activities. The degradation system informs revision of the current structural model of rhamnogalacturonan-II and highlights how individual gut bacteria orchestrate manifold enzymes to metabolize the most challenging glycan in the human diet.</li>
</ul>
</li>
<li>
<p><em>exposure to bacterial cpg dna protects from airway allergic inflammation by expanding regulatory lung interstitial macrophages</em><br />
catherine sabatel et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.immuni.2017.02.016">dx.doi.org/10.1016/j.immuni.2017.02.016</a></p>
<ul>
<li>Living in a microbe-rich environment reduces the risk of developing asthma. Exposure of humans or mice to unmethylated CpG DNA (CpG) from bacteria reproduces these protective effects, suggesting a major contribution of CpG to microbe-induced asthma resistance. However, how CpG confers protection remains elusive. We found that exposure to CpG expanded regulatory lung interstitial macrophages (IMs) from monocytes infiltrating the lung or mobilized from the spleen. Trafficking of IM precursors to the lung was independent of CCR2, a chemokine receptor required for monocyte mobilization from the bone marrow. Using a mouse model of allergic airway inflammation, we found that adoptive transfer of IMs isolated from CpG-treated mice recapitulated the protective effects of CpG when administered before allergen sensitization or challenge. IM-mediated protection was dependent on IL-10, given that <em>Il10−/−</em> CpG-induced IMs lacked regulatory effects. Thus, the expansion of regulatory lung IMs upon exposure to CpG might underlie the reduced risk of asthma development associated with a microbe-rich environment.</li>
</ul>
</li>
<li>
<p><em>microbes can help explain the evolution of host altruism</em><br />
ohad lewin-epstein, ranit aharonov &amp; lilach hadany 2017<br />
10.1038/ncomms14040</p>
<ul>
<li>The evolution of altruistic behaviour, which is costly to the donor but beneficial for the recipient, is among the most intriguing questions in evolutionary biology. Several theories have been proposed to explain it, including kin selection, group selection and reciprocity. Here we propose that microbes that manipulate their hosts to act altruistically could be favoured by selection, and may play a role in the widespread occurrence of altruism. Using computational models, we find that microbe-induced altruism can explain the evolution of host altruistic behaviour under wider conditions than host-centred theories, including in a fully mixed host population, without repeating interactions or individual recognition. Our results suggest that factors such as antibiotics that kill microbes might negatively affect cooperation in a wide range of organisms.</li>
</ul>
</li>
<li>
<p>A new study finds that a noninvasive electromagnetic brain stimulation technique helps obese people lose weight, partly by changing the composition of their intestinal bacteria -- the so-called gut microbiota. Results of the technique, called deep transcranial magnetic stimulation (dTMS), will be presented Sunday at ENDO 2017, the Endocrine Society's 99th annual meeting in Orlando, Fla.<br />
Link: <a href="https://www.sciencedaily.com/releases/2017/04/170403123508.htm">sciencedaily.com/releases/2017/04/170403123508.htm</a></p>
</li>
<li>
<p><em>the gut microbiota and autism spectrum disorders</em><br />
qinrui li, ying han, angel belle c. dy, randi j. hagerman 2017<br />
<a href="http://dx.doi.org/10.3389/fncel.2017.00120">dx.doi.org/10.3389/fncel.2017.00120</a></p>
<ul>
<li>Gastrointestinal (GI) symptoms are a common comorbidity in patients with autism spectrum disorder (ASD), but the underlying mechanisms are unknown. Many studies have shown alterations in the composition of the fecal flora and metabolic products of the gut microbiome in patients with ASD. The gut microbiota influences brain development and behaviors through the neuroendocrine, neuroimmune and autonomic nervous systems. In addition, an abnormal gut microbiota is associated with several diseases, such as inflammatory bowel disease (IBD), ASD and mood disorders. Here, we review the bidirectional interactions between the central nervous system and the gastrointestinal tract (brain-gut axis) and the role of the gut microbiota in the central nervous system (CNS) and ASD. Microbiome-mediated therapies might be a safe and effective treatment for ASD.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>07292</h2>
<ul>
<li>
<p>bacteriome<br />
influencing organism directly</p>
<ul>
<li>
<p><em>missing microbes: how the overuse of antibiotics is fuelling our modern plagues</em><br />
martin blaser 2014<br />
978-1250069276</p>
</li>
<li>
<p><em>psychobiotics and the manipulation of bacteria-brain signals</em><br />
sarkar et al. 2016<br />
http://dx.doi.org/10.1016/j.tins.2016.09.002<br />
Link: <a href="http://dx.doi.org/10.1016/j.tins.2016.09.002">dx.doi.org/10.1016/j.tins.2016.09.002</a></p>
<ul>
<li>Psychobiotics were previously defined as live bacteria (probiotics) which, when ingested, confer mental health benefits through interactions with commensal gut bacteria. We expand this definition to encompass prebiotics, which enhance the growth of beneficial gut bacteria. We review probiotic and prebiotic effects on emotional, cognitive, systemic, and neural variables relevant to health and disease. We discuss gut–brain signalling mechanisms enabling psychobiotic effects, such as metabolite production. Overall, knowledge of how the microbiome responds to exogenous influence remains limited. We tabulate several important research questions and issues, exploration of which will generate both mechanistic insights and facilitate future psychobiotic development. We suggest the definition of psychobiotics be expanded beyond probiotics and prebiotics to include other means of influencing the microbiome.</li>
</ul>
</li>
<li>
<p><em>gut microbiota orchestrates energy homeostasis during cold</em><br />
chevalier et al 2015<br />
http://dx.doi.org/10.1016/j.cell.2015.11.004<br />
Link: <a href="http://dx.doi.org/10.1016/j.cell.2015.11.004">dx.doi.org/10.1016/j.cell.2015.11.004</a></p>
<ul>
<li>Cold exposure markedly shifts the composition of the gut microbiota. This ‘‘cold microbiota’’ mediates remodeling of the fat and intestinal tissues, helping the host to withstand periods of high energy demand.</li>
<li>Cold exposure leads to marked changes in the gut microbiota composition</li>
<li>Cold microbiota transplantation increases insulin sensitivity and WAT browning</li>
<li>Cold exposure or cold transplantation increase the gut size and absorptive capacity</li>
<li>Reconstitution of cold-suppressed A. muciniphila reverts the increased caloric uptake</li>
</ul>
</li>
<li>
<p>mental resilience</p>
<ul>
<li><em>anxiety in your head could come from your gut</em><br />
susan donaldson James 2013<br />
Link: <a href="http://abcnews.go.com/Health/anxiety-head-gut/story?id=20229136">abcnews.go.com/Health/anxiety-head-gut/story</a></li>
</ul>
</li>
<li>
<p><em>the airway microbiome at birth</em><br />
charitharth vivek lal et al 2016<br />
http://dx.doi.org/10.1038/srep31023<br />
Link: <a href="http://dx.doi.org/10.1038/srep31023">dx.doi.org/10.1038/srep31023</a></p>
<ul>
<li>Alterations of pulmonary microbiome have been recognized in multiple respiratory disorders. It is critically important to ascertain if an airway microbiome exists at birth and if so, whether it is associated with subsequent lung disease. We found an established diverse and similar airway microbiome at birth in both preterm and term infants, which was more diverse and different from that of older preterm infants with established chronic lung disease (bronchopulmonary dysplasia). Consistent temporal dysbiotic changes in the airway microbiome were seen from birth to the development of bronchopulmonary dysplasia in extremely preterm infants. Genus Lactobacillus was decreased at birth in infants with chorioamnionitis and in preterm infants who subsequently went on to develop lung disease. Our results, taken together with previous literature indicating a placental and amniotic fluid microbiome, suggest fetal acquisition of an airway microbiome. We speculate that the early airway microbiome may prime the developing pulmonary immune system, and dysbiosis in its development may set the stage for subsequent lung disease.</li>
</ul>
</li>
<li>
<p><em>staphylococcus aureus shifts toward commensalism in response to corynebacterium species</em><br />
matthew m. ramsey, marcelo o. freire, rebecca a. gabrilska, kendra p. rumbaugh and katherine p. lemon 2016<br />
http://dx.doi.org/10.3389/fmicb.2016.01230<br />
Link: <a href="http://dx.doi.org/10.3389/fmicb.2016.01230">dx.doi.org/10.3389/fmicb.2016.01230</a></p>
<ul>
<li>Staphylococcus aureus–human interactions result in a continuum of outcomes from commensalism to pathogenesis. S. aureus is a clinically important pathogen that asymptomatically colonizes ~25% of humans as a member of the nostril and skin microbiota, where it resides with other bacteria including commensal Corynebacterium species. Commensal Corynebacterium spp. are also positively correlated with S. aureus in chronic polymicrobial diabetic foot infections, distinct from acute monomicrobial S. aureus infections. Recent work by our lab and others indicates that microbe–microbe interactions between S. aureus and human skin/nasal commensals, including Corynebacterium species, affect S. aureus behavior and fitness. Thus, we hypothesized that S. aureus interactions with Corynebacterium spp. diminish S. aureus virulence. We tested this by assaying for changes in S. aureus gene expression during in vitro mono- versus coculture with Corynebacterium striatum, a common skin and nasal commensal. We observed a broad shift in S. aureus gene transcription during in vitro growth with C. striatum, including increased transcription of genes known to exhibit increased expression during human nasal colonization and decreased transcription of virulence genes. S. aureus uses several regulatory pathways to transition between commensal and pathogenic states. One of these, the quorum signal accessory gene regulator (agr) system, was strongly inhibited in response to Corynebacterium spp. Phenotypically, S. aureus exposed to C. striatum exhibited increased adhesion to epithelial cells, reflecting a commensal state, and decreased hemolysin activity, reflecting an attenuation of virulence. Consistent with this, S. aureus displayed diminished fitness in experimental in vivo coinfection with C. striatum when compared to monoinfection. These data support a model in which S. aureus shifts from virulence toward a commensal state when exposed to commensal Corynebacterium species.</li>
</ul>
</li>
<li>
<p><em>extreme dysbiosis of the microbiome in critical illness</em><br />
daniel mcdonald et al 2016<br />
http://dx.doi.org/10.1128/msphere.00199-16<br />
Link: <a href="http://dx.doi.org/10.1128/msphere.00199-16">dx.doi.org/10.1128/msphere.00199-16</a></p>
<ul>
<li>Critical illness is hypothesized to associate with loss of “health- promoting” commensal microbes and overgrowth of pathogenic bacteria (dysbiosis). This dysbiosis is believed to increase susceptibility to nosocomial infections, sepsis, and organ failure. A trial with prospective monitoring of the intensive care unit (ICU) patient microbiome using culture-independent techniques to confirm and character- ize this dysbiosis is thus urgently needed. Characterizing ICU patient microbiome changes may provide first steps toward the development of diagnostic and thera- peutic interventions using microbiome signatures. To characterize the ICU patient microbiome, we collected fecal, oral, and skin samples from 115 mixed ICU patients across four centers in the United States and Canada. Samples were collected at two time points: within 48 h of ICU admission, and at ICU discharge or on ICU day 10. Sample collection and processing were performed according to Earth Microbiome Project protocols. We applied SourceTracker to assess the source composition of ICU patient samples by using Qiita, including samples from the American Gut Project (AGP), mammalian corpse decomposition samples, childhood (Global Gut study), and house surfaces. Our results demonstrate that critical illness leads to significant and rapid dysbiosis. Many taxons significantly depleted from ICU patients versus AGP healthy controls are key “health-promoting” organisms, and overgrowth of known pathogens was frequent. Source compositions of ICU patient samples are largely un- characteristic of the expected community type. Between time points and within a patient, the source composition changed dramatically. Our initial results show great promise for microbiome signatures as diagnostic markers and guides to therapeutic interventions in the ICU to repopulate the normal, “health-promoting” microbiome and thereby improve patient outcomes.</li>
</ul>
</li>
<li>
<p><em>long-term effects on luminal and mucosal microbiota and commonly acquired taxa in faecal microbiota transplantation for recurrent clostridium difficile infection</em><br />
jonna jalanka et al. 2016<br />
http://dx.doi.org/10.1186/s12916-016-0698-z<br />
Link: <a href="http://dx.doi.org/10.1186/s12916-016-0698-z">dx.doi.org/10.1186/s12916-016-0698-z</a></p>
<ul>
<li>
<p>Background<br />
Faecal microbiota transplantation (FMT) is an effective treatment for recurrent Clostridium difficile infection (rCDI). It restores the disrupted intestinal microbiota and subsequently suppresses C. difficile. The long-term stability of the intestinal microbiota and the recovery of mucosal microbiota, both of which have not been previously studied, are assessed herein. Further, the specific bacteria behind the treatment efficacy are also investigated.</p>
<p>Methods<br />
We performed a high-throughput microbiota profiling using a phylogenetic microarray analysis of 131 faecal and mucosal samples from 14 rCDI patients pre- and post-FMT during a 1-year follow-up and 23 samples from the three universal donors over the same period.</p>
<p>Results<br />
The FMT treatment was successful in all patients. FMT reverted the patients’ bacterial community to become dominated by Clostridium clusters IV and XIVa, the major anaerobic bacterial groups of the healthy gut. In the mucosa, the amount of facultative anaerobes decreased, whereas Bacteroidetes increased. Post-FMT, the patients’ microbiota profiles were more similar to their own donors than what is generally observed for unrelated subjects and this striking similarity was retained throughout the 1-year follow-up. Furthermore, the universal donor approach allowed us to identify bacteria commonly established in all CDI patients and revealed a commonly acquired core microbiota consisting of 24 bacterial taxa.</p>
<p>Conclusions<br />
FMT induces profound microbiota changes, therefore explaining the high clinical efficacy for rCDI. The identification of commonly acquired bacteria could lead to effective bacteriotherapeutic formulations. FMT can affect microbiota in the long-term and offers a means to modify it relatively permanently for the treatment of microbiota-associated diseases.</p>
</li>
</ul>
</li>
<li>
<p><em>dietary zinc alters the microbiota and decreases resistance to clostridium difficile infection</em><br />
joseph p zackular et al. 2016<br />
http://dx.doi.org/10.1038/nm.4174<br />
Link: <a href="http://dx.doi.org/10.1038/nm.4174">dx.doi.org/10.1038/nm.4174</a></p>
<ul>
<li>Clostridium difficile is the most commonly reported nosocomial pathogen in the United States and is an urgent public health concern worldwide1. Over the past decade, incidence, severity and costs associated with C. difficile infection (CDI) have increased dramatically2. CDI is most commonly initiated by antibiotic-mediated disruption of the gut microbiota; however, non-antibiotic-associated CDI cases are well documented and on the rise3, 4. This suggests that unexplored environmental, nutrient and host factors probably influence CDI. Here we show that excess dietary zinc (Zn) substantially alters the gut microbiota and, in turn, reduces the minimum amount of antibiotics needed to confer susceptibility to CDI. In mice colonized with C. difficile, excess dietary Zn severely exacerbated C. difficile–associated disease by increasing toxin activity and altering the host immune response. In addition, we show that the Zn-binding S100 protein calprotectin has antimicrobial effects against C. difficile and is an essential component of the innate immune response to CDI. Taken together, these data suggest that nutrient Zn levels have a key role in determining susceptibility to CDI and severity of disease, and that calprotectin-mediated metal limitation is an important factor in the host immune response to C. difficile.</li>
</ul>
</li>
<li>
<p><em>gut microbiota and glucometabolic alterations in response to recurrent partial sleep deprivation in normal-weight young individuals</em><br />
christian benedict, heike vogel, wenke jonas, anni woting, michael blaut, annette schürmann, jonathan cedernaes 2016<br />
http://dx.doi.org/10.1016/j.molmet.2016.10.003<br />
Link: <a href="http://dx.doi.org/10.1016/j.molmet.2016.10.003">dx.doi.org/10.1016/j.molmet.2016.10.003</a></p>
<ul>
<li>very basic first study to reveal changes in gut microbiota due to changes in sleep regime
<ul>
<li>Objective: Changes to the microbial community in the human gut have been proposed to promote metabolic disturbances that also occur after short periods of sleep loss (including insulin resistance). However, whether sleep loss affects the gut microbiota remains unknown.<br />
Methods: In a randomized within-subject crossover study utilizing a standardized in-lab protocol (with fixed meal times and exercise schedules), we studied nine normal-weight men at two occasions: after two nights of partial sleep deprivation (PSD; sleep opportunity 02:45-07:00h), and after two nights of normal sleep (NS; sleep opportunity 22:30-07:00h). Fecal samples were collected within 24 hours before, and after two in-lab nights, of either NS or PSD. In addition, participants underwent an oral glucose tolerance test following each sleep intervention.<br />
Results: Microbiota composition analysis (V4 16S rRNA gene sequencing) revealed that after two days of PSD vs. after two days of NS, individuals exhibited an increased Firmicutes:Bacteroidetes ratio and of the families Coriobacteriaceae and Erysipelotrichaceae, with decreases in Tenericutes (all P&lt;0.05) – previously all associated with metabolic perturbations in animal or human models. However, no PSD vs. NS effect on beta diversity or on fecal short-chain fatty acid concentrations was found. Fasting and postprandial insulin sensitivity decreased after PSD (all P&lt;0.05).<br />
Discussion: Our findings demonstrate that short-term sleep loss induces subtle effects on human microbiota. To what extent changes to the microbial community contribute to metabolic consequences of sleep loss warrants further investigations in larger and more prolonged sleep studies, to also assess how sleep loss impacts the microbiota in individuals who already are metabolically compromised.</li>
</ul>
</li>
</ul>
</li>
<li>
<p>prenatal exposure to a mother’s stress contributes to anxiety and cognitive problems that persist into adulthood, a phenomenon that could be explained by lasting – and potentially damaging – changes in the microbiome, according to new research in mice</p>
<p><a href="https://www.sciencedaily.com/releases/2016/11/161114124958.htm">161114124958.htm</a></p>
</li>
<li>
<p><em>akkermansia muciniphila mediates negative effects of ifnγ on glucose metabolism</em><br />
renee l. greer 2016<br />
<a href="http://dx.doi.org/10.1038/NCOMMS13329">NCOMMS13329</a></p>
<ul>
<li>Cross-talk between the gut microbiota and the host immune system regulates host metabolism, and its dysregulation can cause metabolic disease. Here, we show that the gut microbe <em>Akkermansia muciniphila</em> can mediate negative effects of IFNγ on glucose tolerance. In IFNγ-deficient mice, <em>A. muciniphila</em> is significantly increased and restoration of IFNγ levels reduces <em>A. muciniphila</em> abundance. We further show that IFNγ-knockout mice whose microbiota does not contain <em>A. muciniphila</em> do not show improvement in glucose tolerance and adding back <em>A. muciniphila</em> promoted enhanced glucose tolerance. We go on to identify Irgm1 as an IFNγ-regulated gene in the mouse ileum that controls gut <em>A. muciniphila</em> levels. <em>A. muciniphila</em> is also linked to IFNγ-regulated gene expression in the intestine and glucose parameters in humans, suggesting that this trialogue between IFNγ, <em>A. muciniphila</em> and glucose tolerance might be an evolutionally conserved mechanism regulating metabolic health in mice and humans.</li>
<li>our study revealed a new homeostatic regulatory process in mammalian organisms, where a member of different kingdom, <em>A. muciniphila,</em> constitutes an integral part of the interaction between the supposedly functionally distinct and distant systems of immunity and glucose metabolism. Furthermore, our results and other published work in mouse models and human subjects suggest that this transkingdom interaction may be common in mammals<a href="http://www.nature.com/articles/ncomms13329#ref25">25</a>,<a href="http://www.nature.com/articles/ncomms13329#ref26">26</a>,<a href="http://www.nature.com/articles/ncomms13329#ref27">27</a>. Over the years, biologists have drawn boundaries between systems and kingdoms. Our results highlight the fact that these boundaries must be crossed to fully understand the complexity of living organisms.</li>
</ul>
</li>
<li>
<p><em>diet-microbiota interactions mediate global epigenetic programming in multiple host tissues</em><br />
kimberly a. krautkramer et al. 2016<br />
<a href="http://dx.doi.org/10.1016/j.molcel.2016.10.25">j.molcel.2016.10.25</a></p>
<ul>
<li>
<p>Gut microbiota alter host histone acetylation and methylation in multiple tissues</p>
<p>Western diet suppresses microbiota-driven SCFA production and chromatin effects</p>
<p>SCFAs recapitulate microbiota-driven chromatin and transcriptional effects</p>
<p>Histone-modifying enzymes regulate transcription and are sensitive to availability of endogenous small-molecule metabolites, allowing chromatin to respond to changes in environment. The gut microbiota produces a myriad of metabolites that affect host physiology and susceptibility to disease; however, the underlying molecular events remain largely unknown. Here we demonstrate that microbial colonization regulates global histone acetylation and methylation in multiple host tissues in a diet-dependent manner: consumption of a “Western-type” diet prevents many of the microbiota-dependent chromatin changes that occur in a polysaccharide-rich diet. Finally, we demonstrate that supplementation of germ-free mice with short-chain fatty acids, major products of gut bacterial fermentation, is sufficient to recapitulate chromatin modification states and transcriptional responses associated with colonization. These findings have profound implications for understanding the complex functional interactions between diet, gut microbiota, and host health.</p>
</li>
</ul>
</li>
<li>
<p><em>gut microbiota regulate motor deficits and neuroinflammation in a model of parkinson’s disease</em><br />
timothy r. sampson et al. 2016<br />
<a href="http://dx.doi.org/10.1016/j.cell.2016.11.018">j.cell.2016.11.018</a></p>
<ul>
<li>
<p>Gut microbes promote α-synuclein-mediated motor deficits and brain pathology</p>
<p>Depletion of gut bacteria reduces microglia activation</p>
<p>SCFAs modulate microglia and enhance PD pathophysiology</p>
<p>Human gut microbiota from PD patients induce enhanced motor dysfunction in mice</p>
<p>The intestinal microbiota influence neurodevelopment, modulate behavior, and contribute to neurological disorders. However, a functional link between gut bacteria and neurodegenerative diseases remains unexplored. Synucleinopathies are characterized by aggregation of the protein α-synuclein (αSyn), often resulting in motor dysfunction as exemplified by Parkinson’s disease (PD). Using mice that overexpress αSyn, we report herein that gut microbiota are required for motor deficits, microglia activation, and αSyn pathology. Antibiotic treatment ameliorates, while microbial re-colonization promotes, pathophysiology in adult animals, suggesting that postnatal signaling between the gut and the brain modulates disease. Indeed, oral administration of specific microbial metabolites to germ-free mice promotes neuroinflammation and motor symptoms. Remarkably, colonization of αSyn-overexpressing mice with microbiota from PD-affected patients enhances physical impairments compared to microbiota transplants from healthy human donors. These findings reveal that gut bacteria regulate movement disorders in mice and suggest that alterations in the human microbiome represent a risk factor for PD.</p>
</li>
</ul>
</li>
<li>
<p><em>glyphosate’s suppression of cytochrome p450 enzymes and amino acid biosynthesis by the gut microbiome: pathways to modern diseases</em><br />
anthony samsel and stephanie seneff 2013<br />
http://dx.doi.org/10.3390/e15041416<br />
Link: <a href="http://dx.doi.org/10.3390/e15041416">dx.doi.org/10.3390/e15041416</a></p>
<ul>
<li>Glyphosate, the active ingredient in Roundup®, is the most popular herbicide used worldwide. The industry asserts it is minimally toxic to humans, but here we argue otherwise. Residues are found in the main foods of the Western diet, comprised primarily of sugar, corn, soy and wheat. Glyphosate's inhibition of cytochrome P450 (CYP) enzymes is an overlooked component of its toxicity to mammals. CYP enzymes play crucial roles in biology, one of which is to detoxify xenobiotics. Thus, glyphosate enhances the damaging effects of other food borne chemical residues and environmental toxins. Negative impact on the body is insidious and manifests slowly over time as inflammation damages cellular systems throughout the body. Here, we show how interference with CYP enzymes acts synergistically with disruption of the biosynthesis of aromatic amino acids by gut bacteria, as well as impairment in serum sulfate transport. Consequences are most of the diseases and conditions associated with a Western diet, which include gastrointestinal disorders, obesity, diabetes, heart disease, depression, autism, infertility, cancer and Alzheimer’s disease. We explain the documented effects of glyphosate and its ability to induce disease, and we show that glyphosate is the “textbook example” of exogenous semiotic entropy: the disruption of homeostasis by environmental toxins.</li>
</ul>
</li>
<li>
<p><em>intestinal serotonin transporter inhibition by toll-like receptor 2 activation. a feedback modulation</em><br />
latorre e, layunta e, grasa l, castro m, pardo j, gomollón f, et al. 2016<br />
http://dx.doi.org/10.1371/journal.pone.0169303<br />
Link: <a href="http://dx.doi.org/10.1371/journal.pone.0169303">dx.doi.org/10.1371/journal.pone.0169303</a></p>
<ul>
<li>TLR2 is a microbiota recognition receptor that has been described to contribute to intestinal homeostasis and to ameliorate inflammatory intestinal injury. In this context, serotonin (5-HT) has shown to be an essential intestinal physiological neuromodulator that is also involved in intestinal inflammatory diseases. Since the interaction between TLR2 activation and the intestinal serotoninergic system remains non-investigated, our main aim was to analyze the effect of TLR2 on intestinal serotonin transporter (SERT) activity and expression and the intracellular pathways involved. Caco-2/TC7 cells were used to analyze SERT and TLR2 molecular expression and SERT activity by measuring 5-HT uptake. The results showed that apical TLR2 activation inhibits SERT activity in Caco-2/TC7 cells mainly by reducing SERT protein level either in the plasma membrane, after short-term TLR2 activation or in both the plasma membrane and cell lysate, after long-term activation. cAMP/PKA pathway appears to mediate short-term inhibitory effect of TLR2 on SERT; however, p38 MAPK pathway has been shown to be involved in both short- and long-term TLR2 effect. Reciprocally, 5-HT long-term treatment yielded TLR2 down regulation in Caco-2/TC7 cells. Finally, results from <em>in vivo</em> showed an augmented intestinal SERT expression in mice <em>Tlr2-/-</em>, thus confirming our inhibitory effect of TLR2 on intestinal SERT <em>in vitro</em>. The present work infers that TLR2 may act in intestinal pathophysiology, not only by its inherent innate immune role, but also by regulating the intestinal serotoninergic system.</li>
</ul>
</li>
<li>
<p><em>the active component of aspirin, salicylic acid, promotes staphylococcus aureus biofilm formation in a pia-dependent manner</em><br />
cristian dotto et al. 2017<br />
<a href="https://doi.org/10.3389/fmicb.2017.00004">fmicb.2017.00004</a></p>
<ul>
<li>Aspirin has provided clear benefits to human health. But salicylic acid (SAL) -the main aspirin biometabolite- exerts several effects on eukaryote and prokaryote cells. SAL can affect, for instance, the expression of <em>Staphylococcus aureus</em> virulence factors. SAL can also form complexes with iron cations and it has been shown that different iron chelating molecules diminished the formation of <em>S. aureus</em> biofilm. The aim of this study was to elucidate whether the iron content limitation caused by SAL can modify the <em>S. aureus</em> metabolism and/or metabolic regulators thus changing the expression of the main polysaccharides involved in biofilm formation. The exposure of biofilm to 2 mM SAL induced a 27% reduction in the intracellular free Fe2+ concentration compared with the controls. In addition, SAL depleted 23% of the available free Fe2+ cation in culture media. These moderate iron-limited conditions promoted an intensification of biofilms formed by strain Newman and by <em>S. aureus</em> clinical isolates related to the USA300 and USA100 clones. The slight decrease in iron bioavailability generated by SAL was enough to induce the increase of PIA expression in biofilms formed by methicillin-resistant as well as methicillin-sensitive <em>S. aureus</em> strains. <em>S. aureus</em> did not produce capsular polysaccharide (CP) when it was forming biofilms under any of the experimental conditions tested. Furthermore, SAL diminished aconitase activity and stimulated the lactic fermentation pathway in bacteria forming biofilms. The polysaccharide composition of <em>S. aureus</em> biofilms was examined and FTIR spectroscopic analysis revealed a clear impact of SAL in a <em>codY</em>-dependent manner. Moreover, SAL negatively affected <em>codY</em> transcription in mature biofilms thus relieving the CodY repression of the <em>ica</em> operon. Treatment of mice with SAL induced a significant increase of <em>S aureus</em> colonization. It is suggested that the elevated PIA expression induced by SAL might be responsible for the high nasal colonization observed in mice. SAL-induced biofilms may contribute to <em>S. aureus</em> infection persistence in vegetarian individuals as well as in patients that frequently consume aspirin.</li>
</ul>
</li>
<li>
<p><em>reduction of abeta amyloid pathology in appps1 transgenic mice in the absence of gut microbiota</em><br />
t. harach et al. 2017<br />
<a href="http://dx.doi.org/10.1038/srep41802">srep41802</a></p>
<ul>
<li>Alzheimer’s disease is the most common form of dementia in the western world, however there is no cure available for this devastating neurodegenerative disorder. Despite clinical and experimental evidence implicating the intestinal microbiota in a number of brain disorders, its impact on Alzheimer’s disease is not known. To this end we sequenced bacterial 16S rRNA from fecal samples of Aβ precursor protein (APP) transgenic mouse model and found a remarkable shift in the gut microbiota as compared to non-transgenic wild-type mice. Subsequently we generated germ-free APP transgenic mice and found a drastic reduction of cerebral Aβ amyloid pathology when compared to control mice with intestinal microbiota. Importantly, colonization of germ-free APP transgenic mice with microbiota from conventionally-raised APP transgenic mice increased cerebral Aβ pathology, while colonization with microbiota from wild-type mice was less effective in increasing cerebral Aβ levels. Our results indicate a microbial involvement in the development of Abeta amyloid pathology, and suggest that microbiota may contribute to the development of neurodegenerative diseases.</li>
</ul>
</li>
<li>
<p><em>cutaneous leishmaniasis induces a transmissible dysbiotic skin microbiota that promotes skin inflammation</em><br />
ciara gimblet et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.chom.2017.06.006">dx.doi.org/10.1016/j.chom.2017.06.006</a></p>
<ul>
<li>
<p>Leishmania infection alters the skin microbiota of both humans and mice<br />
Dysbiosis is characterized by a dominance of <em>Staphylococcus</em> and/or <em>Streptococcus</em><br />
Naive mice acquire dysbiosis when co-housed with leishmania-infected mice<br />
Acquiring a dysbiotic microbiota prior to infection exacerbates skin inflammation</p>
<p>Skin microbiota can impact allergic and autoimmune responses, wound healing, and anti-microbial defense. We investigated the role of skin microbiota in cutaneous leishmaniasis and found that human patients infected with <em>Leishmania braziliensis</em> develop dysbiotic skin microbiota, characterized by increases in the abundance of <em>Staphylococcus</em> and/or <em>Streptococcus</em>. Mice infected with <em>L. major</em> exhibit similar changes depending upon disease severity. Importantly, this dysbiosis is not limited to the lesion site, but is transmissible to normal skin distant from the infection site and to skin from co-housed naive mice. This observation allowed us to test whether a pre-existing dysbiotic skin microbiota influences disease, and we found that challenging dysbiotic naive mice with <em>L. major</em> or testing for contact hypersensitivity results in exacerbated skin inflammatory responses. These findings demonstrate that a dysbiotic skin microbiota is not only a consequence of tissue stress, but also enhances inflammation, which has implications for many inflammatory cutaneous diseases.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>07392</h2>
<ul>
<li>virome
<ul>
<li>
<p>influencing bacteriome as well as entire organism directly</p>
</li>
<li>
<p><em>drosophila adaptation to viral infection through defensive symbiont evolution</em><br />
vitor g. faria, nelson e. martins, sara magalhães, tânia f. paulo, viola nolte, christian schlötterer, élio sucena, luis teixeira 2016<br />
http://dx.doi.org/10.1371/journal.pgen.1006297<br />
Link: <a href="http://dx.doi.org/10.1371/journal.pgen.1006297">dx.doi.org/10.1371/journal.pgen.1006297</a></p>
<ul>
<li>
<p>Microbial symbionts can modulate host interactions with biotic and abiotic factors. Such interactions may affect the evolutionary trajectories of both host and symbiont. Wolbachia protects Drosophila melanogaster against several viral infections and the strength of the protection varies between variants of this endosymbiont. Since Wolbachia is maternally transmitted, its fitness depends on the fitness of its host. Therefore, Wolbachia populations may be under selection when Drosophila is subjected to viral infection. Here we show that in D. melanogaster populations selected for increased survival upon infection with Drosophila C virus there is a strong selection coefficient for specific Wolbachia variants, leading to their fixation. Flies carrying these selected Wolbachia variants have higher survival and fertility upon viral infection when compared to flies with the other variants. These findings demonstrate how the interaction of a host with pathogens shapes the genetic composition of symbiont populations. Furthermore, host adaptation can result from the evolution of its symbionts, with host and symbiont functioning as a single evolutionary unit.</p>
<p>Author Summary</p>
<p>Animals live in close association with microbial partners that can shape many aspects of their lives. For instance, several insects carry bacteria that defend them against parasites and infectious diseases. The intracellular bacterium Wolbachia protects the fruit fly Drosophila melanogaster against viral infection. Natural populations of Drosophila carry different variants of Wolbachia, which differ from one another in the strength of this protection. Here we show that a population of Drosophila infected with viruses during several generations adapts to this challenge through turnover in Wolbachia composition. The Wolbachia variants that give higher protection to viruses, by increasing fly survival and fecundity upon infection, are strongly selected. This work demonstrates that the interaction of an animal with a pathogen can shape its associated microbial populations. We show that adaptation to pathogens can be achieved not only through selection of resistance on the host proper but also through the evolutionary shaping of its microbial community.</p>
</li>
</ul>
</li>
<li>
<p><em>predators catalyze an increase in chloroviruses by foraging on the symbiotic hosts of zoochlorellae</em><br />
john p. delong, zeina al-ameeli, garry duncan, james l. van etten, david d. dunigan 2016<br />
http://dx.doi.org/10.1073/pnas.1613843113<br />
Link: <a href="http://dx.doi.org/10.1073/pnas.1613843113">dx.doi.org/10.1073/pnas.1613843113</a></p>
<ul>
<li>
<p>Reproduction and growth of viruses depend on successful encounters with appropriate hosts. However, some hosts are difficult to encounter. In particular, chloroviruses cannot reach their target zoochlorellae hosts, because zoochlorellae are endosymbionts, living inside the cell of a protist that protects the zoochlorellae from the chlorovirus. The protist host is subject to predation, and we show that copepods foraging on zoochlorellae-bearing protists can disrupt the mutualism and pass endosymbiontic zoochlorellae through their guts, exposing them to chloroviruses. In this way, predators can catalyze the virus population growth by breaking down physical barriers between viruses and their endosymbiont hosts.</p>
<p>Virus population growth depends on contacts between viruses and their hosts. It is often unclear how sufficient contacts are made between viruses and their specific hosts to generate spikes in viral abundance. Here, we show that copepods, acting as predators, can bring aquatic viruses and their algal hosts into contact. Specifically, predation of the protist <em>Paramecium bursaria</em> by copepods resulted in a &gt;100-fold increase in the number of chloroviruses in 1 d. Copepod predation can be seen as an ecological “catalyst” by increasing contacts between chloroviruses and their hosts, zoochlorellae (endosymbiotic algae that live within paramecia), thereby facilitating viral population growth. When feeding, copepods passed <em>P. bursaria</em> through their digestive tract only partially digested, releasing endosymbiotic algae that still supported viral reproduction and resulting in a virus population spike. A simple predator–prey model parameterized for copepods consuming protists generates cycle periods for viruses consistent with those observed in natural ponds. Food webs are replete with similar symbiotic organisms, and we suspect the predator catalyst mechanism is capable of generating blooms for other endosymbiont-targeting viruses.</p>
</li>
</ul>
</li>
<li>
<p><em>trim28 controls a gene regulatory network based on endogenous retroviruses in human neural progenitor cells</em><br />
per ludvik brattås et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.celrep.2016.12.010">j.celrep.2016.12.010</a></p>
<ul>
<li>
<p>•Stage- and region-specific expression of ERVs during human brain development<br />
•TRIM28 binds to ERVs and induces hetereochromatin in human neural progenitor cells<br />
•Knockdown of TRIM28 in hNPCs results in the upregulation of ERV expression<br />
•Protein-coding genes located near upregulated ERVs are upregulated</p>
<p>Endogenous retroviruses (ERVs), which make up 8% of the human genome, have been proposed to participate in the control of gene regulatory networks. In this study, we find a region- and developmental stage-specific expression pattern of ERVs in the developing human brain, which is linked to a transcriptional network based on ERVs. We demonstrate that almost 10,000, primarily primate-specific, ERVs act as docking platforms for the co-repressor protein TRIM28 in human neural progenitor cells, which results in the establishment of local heterochromatin. Thereby, TRIM28 represses ERVs and consequently regulates the expression of neighboring genes. These results uncover a gene regulatory network based on ERVs that participates in control of gene expression of protein-coding transcripts important for brain development.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>07492</h2>
<ul>
<li>mycobiome
<ul>
<li>
<p><em>harbouring public good mutants within a pathogen population can increase both fitness and virulence</em><br />
richard j lindsay et al 2016<br />
<a href="http://dx.doi.org/10.7554/eLife.18678">eLife.18678</a></p>
<p>Link: <a href="http://dx.doi.org/10.7554/eLife.18678">dx.doi.org/10.7554/eLife.18678</a></p>
<ul>
<li>“paradoxic” increase of virulence in diverse population versus “pure” population</li>
<li>Existing theory, empirical, clinical and field research all predict that reducing the virulence of individuals within a pathogen population will reduce the overall virulence, rendering disease less severe. Here, we show that this seemingly successful disease management strategy can fail with devastating consequences for infected hosts. We deploy cooperation theory and a novel synthetic system involving the rice blast fungus <em>Magnaporthe oryzae</em>. In vivo infections of rice demonstrate that <em>M. oryzae</em> virulence is enhanced, quite paradoxically, when a public good mutant is present in a population of high-virulence pathogens. We reason that during infection, the fungus engages in multiple cooperative acts to exploit host resources. We establish a multi-trait cooperation model which suggests that the observed failure of the virulence reduction strategy is caused by the interference between different social traits. Multi-trait cooperative interactions are widespread, so we caution against the indiscriminant application of anti-virulence therapy as a disease-management strategy.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>07592</h2>
<ul>
<li>archaea
<ul>
<li>
<p><em>human age and skin physiology shape diversity and abundance of archaea on skin</em><br />
christine moissl-eichinger et al. 2017<br />
<a href="http://dx.doi.org/10.1038/s41598-017-04197-4">dx.doi.org/10.1038/s41598-017-04197-4</a></p>
<ul>
<li>The human skin microbiome acts as an important barrier protecting our body from pathogens and other environmental influences. Recent investigations have provided evidence that Archaea are a constant but highly variable component of the human skin microbiome, yet factors that determine their abundance changes are unknown. Here, we tested the hypothesis that the abundance of archaea on human skin is influenced by human age and skin physiology by quantitative PCR of 51 different skin samples taken from human subjects of various age. Our results reveal that archaea are more abundant in human subjects either older than 60 years or younger than 12 years as compared to middle-aged human subjects. These results, together with results obtained from spectroscopy analysis, allowed us gain first insights into a potential link of lower sebum levels and lipid content and thus reduced skin moisture with an increase in archaeal signatures. Amplicon sequencing of selected samples revealed the prevalence of specific eury- and mainly thaumarchaeal taxa, represented by a core archaeome of the human skin.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>07692</h2>
<ul>
<li>
<p>genome</p>
<ul>
<li>sea slugs taking genes and chloroplasts from bacteria
<ul>
<li>
<p>usf paper<br />
Link: <a href="http://biology.usf.edu/ib/data/flyers/SCHWARTZ-FISH-LABELING-1-2015.pdf">biology.usf.edu/ib/data/flyers/SCHWARTZ-FISH-LABELING-1-2015.pdf</a></p>
<ul>
<li>
<p>201412</p>
<p>Reference: Biol. Bull. 227: 300–312. (December 2014)</p>
<p>FISH Labeling Reveals a Horizontally Transferred Algal (Vaucheria litorea) Nuclear Gene on a Sea Slug (Elysia chlorotica) Chromosome</p>
<p>JULIE A. SCHWARTZ1, NICHOLAS E. CURTIS2, AND SIDNEY K. PIERCE1,3*<br />
1Department of Integrative Biology, University of South Florida, Tampa, Florida 33620; 2Department of Biology and Chemistry, Ave Maria University, Ave Maria, Florida 34142; and 3Department of Biology, University of Maryland, College Park, Maryland 20742</p>
</li>
</ul>
</li>
<li>
<p>press article<br />
Link: <a href="http://io9.com/confirmation-that-photosynthesizing-sea-slugs-steal-gen-1683702602">io9.com/confirmation-that-photosynthesizing-sea-slugs-steal-gen-1683702602</a></p>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p>horizontal gene transfer<br />
not only in simple organisms</p>
<ul>
<li>
<p><em>horizontal gene transfer is more frequent with increased heterotrophy and contributes to parasite adaptation</em><br />
zhenzhen yang et al. 2016<br />
http://dx.doi.org/10.1073/pnas.1608765113<br />
Link: <a href="http://dx.doi.org/10.1073/pnas.1608765113">dx.doi.org/10.1073/pnas.1608765113</a></p>
<ul>
<li>
<p>Horizontal gene transfer (HGT) is the nonsexual transfer and genomic integration of genetic materials between organisms. In eukaryotes, HGT appears rare, but parasitic plants may be exceptions, as haustorial feeding connections between parasites and their hosts provide intimate cellular contacts that could facilitate DNA transfer between unrelated species. Through analysis of genome-scale data, we identified &gt;50 expressed and likely functional HGT events in one family of parasitic plants. HGT reflected parasite preferences for different host plants and was much more frequent in plants with increasing parasitic dependency. HGT was strongly biased toward expression and protein types likely to contribute to haustorial function, suggesting that functional HGT of host genes may play an important role in adaptive evolution of parasites.<br />
Abstract</p>
<p>Horizontal gene transfer (HGT) is the transfer of genetic material across species boundaries and has been a driving force in prokaryotic evolution. HGT involving eukaryotes appears to be much less frequent, and the functional implications of HGT in eukaryotes are poorly understood. We test the hypothesis that parasitic plants, because of their intimate feeding contacts with host plant tissues, are especially prone to horizontal gene acquisition. We sought evidence of HGTs in transcriptomes of three parasitic members of Orobanchaceae, a plant family containing species spanning the full spectrum of parasitic capabilities, plus the free-living Lindenbergia. Following initial phylogenetic detection and an extensive validation procedure, 52 high-confidence horizontal transfer events were detected, often from lineages of known host plants and with an increasing number of HGT events in species with the greatest parasitic dependence. Analyses of intron sequences in putative donor and recipient lineages provide evidence for integration of genomic fragments far more often than retro-processed RNA sequences. Purifying selection predominates in functionally transferred sequences, with a small fraction of adaptively evolving sites. HGT-acquired genes are preferentially expressed in the haustorium—the organ of parasitic plants—and are strongly biased in predicted gene functions, suggesting that expression products of horizontally acquired genes are contributing to the unique adaptive feeding structure of parasitic plants.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>07792</h2>
<ul>
<li>cancer
<ul>
<li>
<p><em>cancer tumors as metazoa 1.0: tapping genes of ancient ancestors</em><br />
p c w davies and c h lineweaver 2011<br />
http://dx.doi.org/10.1088/1478-3975/8/1/015001<br />
Link: <a href="http://dx.doi.org/10.1088/1478-3975/8/1/015001">dx.doi.org/10.1088/1478-3975/8/1/015001</a></p>
<ul>
<li>The genes of cellular cooperation that evolved with multicellularity about a billion years ago are the same genes that malfunction to cause cancer. We hypothesize that cancer is an atavistic condition that occurs when genetic or epigenetic malfunction unlocks an ancient 'toolkit' of pre-existing adaptations, re-establishing the dominance of an earlier layer of genes that controlled loose-knit colonies of only partially differentiated cells, similar to tumors. The existence of such a toolkit implies that the progress of the neoplasm in the host organism differs distinctively from normal Darwinian evolution. Comparative genomics and the phylogeny of basal metazoans, opisthokonta and basal multicellular eukaryotes should help identify the relevant genes and yield the order in which they evolved. This order will be a rough guide to the reverse order in which cancer develops, as mutations disrupt the genes of cellular cooperation. Our proposal is consistent with current understanding of cancer and explains the paradoxical rapidity with which cancer acquires a suite of mutually-supportive complex abilities. Finally we make several predictions and suggest ways to test this model.</li>
</ul>
</li>
<li>
<p>canine transmissible venereal tumour</p>
<ul>
<li><em>the cancer which survived: insights from the genome of an 11 000 year-old cancer</em><br />
andrea strakova and elizabeth p murchison 2015<br />
http://dx.doi.org/10.1016/j.gde.2015.03.005<br />
Link: <a href="http://dx.doi.org/10.1016/j.gde.2015.03.005">dx.doi.org/10.1016/j.gde.2015.03.005</a></li>
</ul>
</li>
<li>
<p>tasmanian devils</p>
</li>
<li>
<p><em>cancer cells enter dormancy after cannibalizing mesenchymal stem/stromal cells (mscs)</em><br />
thomas j. bartosh, mujib ullah, suzanne zeitouni, joshua beaver, darwin j. prockop 2016<br />
<a href="http://dx.doi.org/10.1073/pnas.1612290113">pnas.1612290113</a></p>
<ul>
<li>Patients with breast cancer often develop malignant regrowth of residual drug-resistant dormant tumor cells years after primary treatment, a process defined as cancer relapse. Deciphering the causal basis of tumor dormancy therefore has obvious therapeutic significance. Because cancer cell behavior is strongly influenced by stromal cells, particularly the mesenchymal stem/stromal cells (MSCs) that are actively recruited into tumor-associated stroma, we assessed the impact of MSCs on breast cancer cell (BCC) dormancy. Using 3D cocultures to mimic the cellular interactions of an emerging tumor niche, we observed that MSCs sequentially surrounded the BCCs, promoted formation of cancer spheroids, and then were internalized/degraded through a process resembling the well-documented yet ill-defined clinical phenomenon of cancer cell cannibalism. This suspected feeding behavior was less appreciable in the presence of a rho kinase inhibitor and in 2D monolayer cocultures. Notably, cannibalism of MSCs enhanced survival of BCCs deprived of nutrients but suppressed their tumorigenicity, together suggesting the cancer cells entered dormancy. Transcriptome profiles revealed that the resulting BCCs acquired a unique molecular signature enriched in prosurvival factors and tumor suppressors, as well as inflammatory mediators that demarcate the secretome of senescent cells, also referred to as the senescence-associated secretory phenotype. Overall, our results provide intriguing evidence that cancer cells under duress enter dormancy after cannibalizing MSCs. Importantly, our practical 3D coculture model could provide a valuable tool to understand the antitumor activity of MSCs and cell cannibalism further, and therefore open new therapeutic avenues for the prevention of cancer recurrence.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>07892</h2>
<ul>
<li>immune system
<ul>
<li>
<p><em>shaping variation in the human immune system</em><br />
liston, linterman, and carr 2016<br />
http://dx.doi.org/10.1016/j.it.2016.08.002<br />
Link: <a href="http://dx.doi.org/10.1016/j.it.2016.08.002">dx.doi.org/10.1016/j.it.2016.08.002</a></p>
<ul>
<li>
<p>Immune responses demonstrate a high level of intra-species variation, compensating for the specialization capacity of pathogens. The recent advent of in-depth immune phenotyping projects in large-scale cohorts has allowed a first look into the factors that shape the inter-individual diversity of the human immune system. Genetic approaches have identified genetic diversity as drivers of 20–40% of the variation between the immune systems of individuals. The remaining 60–80% is shaped by intrinsic factors, with age being the predominant factor, as well as by environmental influences, where cohabitation and chronic viral infections were identified as key mediators. We review and integrate the recent in-depth large-scale studies on human immune diversity and its potential impact on health.</p>
<p>Trends<br />
Diversity within the human immune system is stable over the duration of months to years with an elastic response to immunological challenge, allowing the study of immune drivers.</p>
<p>Genetic variation accounts for 20–40% of immune variation, with enrichment of gene variants associated with autoimmunity, inflammatory disease, and susceptibility to infections among the identified genetic drivers.</p>
<p>Among the identified intrinsic drivers of immune variation, age is the most potent, driving a shift from a precursor-biased immune status to an inflammation-biased immune status.</p>
<p>A strong environmental effect on immune variation is observed, as revealed by cohabitation studies, with the strongest individual driver identified to date being chronic viral infection.</p>
</li>
</ul>
</li>
<li>
<p>bacterial immune system</p>
<ul>
<li>
<p><em>quorum sensing controls adaptive immunity through the regulation of multiple crispr-cas systems</em><br />
adrian g. patterson et al. 2016<br />
<a href="http://dx.doi.org/10.1016/j.molcel.2016.11.012">j.molcel.2016.11.012</a></p>
<ul>
<li>
<p>Quorum sensing regulates the type I-E, I-F, and III-A CRISPR-Cas systems in <em>Serratia</em></p>
<p>SmaR represses <em>cas</em> gene and CRISPR expression in the absence of AHL signals</p>
<p>Both interference and adaptation are modulated by quorum sensing</p>
<p>Bacteria coordinate their defenses based on cell density and the risk of infection</p>
<p>Bacteria commonly exist in high cell density populations, making them prone to viral predation and horizontal gene transfer (HGT) through transformation and conjugation. To combat these invaders, bacteria possess an arsenal of defenses, such as CRISPR-Cas adaptive immunity. Many bacterial populations coordinate their behavior as cell density increases, using quorum sensing (QS) signaling. In this study, we demonstrate that QS regulation results in increased expression of the type I-E, I-F, and III-A CRISPR-Cas systems in <em>Serratia</em> cells in high-density populations. Strains unable to communicate via QS were less effective at defending against invaders targeted by any of the three CRISPR-Cas systems. Additionally, the acquisition of immunity by the type I-E and I-F systems was impaired in the absence of QS signaling. We propose that bacteria can use chemical communication to modulate the balance between community-level defense requirements in high cell density populations and host fitness costs of basal CRISPR-Cas activity.</p>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p>immune system where we didn’t expect it</p>
<ul>
<li>brain
<ul>
<li>
<p><em>structural and functional features of central nervous system lymphatic vessels</em><br />
antoine louveau et al. 2015<br />
<a href="http://dx.doi.org/10.1038/nature14432">dx.doi.org/10.1038/nature14432</a></p>
<ul>
<li>One of the characteristics of the central nervous system is the lack of a classical lymphatic drainage system. Although it is now accepted that the central nervous system undergoes constant immune surveillance that takes place within the meningeal compartment<a href="http://www.nature.com/nature/journal/v523/n7560/full/nature14432.html#ref1">79</a>, <a href="http://www.nature.com/nature/journal/v523/n7560/full/nature14432.html#ref2">80</a>, <a href="http://www.nature.com/nature/journal/v523/n7560/full/nature14432.html#ref3">81</a>, the mechanisms governing the entrance and exit of immune cells from the central nervous system remain poorly understood<a href="http://www.nature.com/nature/journal/v523/n7560/full/nature14432.html#ref4">82</a>, <a href="http://www.nature.com/nature/journal/v523/n7560/full/nature14432.html#ref5">83</a>, <a href="http://www.nature.com/nature/journal/v523/n7560/full/nature14432.html#ref6">84</a>. In searching for T-cell gateways into and out of the meninges, we discovered functional lymphatic vessels lining the dural sinuses. These structures express all of the molecular hallmarks of lymphatic endothelial cells, are able to carry both fluid and immune cells from the cerebrospinal fluid, and are connected to the deep cervical lymph nodes. The unique location of these vessels may have impeded their discovery to date, thereby contributing to the long-held concept of the absence of lymphatic vasculature in the central nervous system. The discovery of the central nervous system lymphatic system may call for a reassessment of basic assumptions in neuroimmunology and sheds new light on the aetiology of neuroinflammatory and neurodegenerative diseases associated with immune system dysfunction.</li>
</ul>
</li>
</ul>
</li>
<li>testes
<ul>
<li>
<p><em>egress of sperm autoantigen from seminiferous tubules maintains systemic tolerance</em><br />
kenneth s.k. tung et al. 2017<br />
<a href="http://dx.doi.org/10.1172/JCI89927">dx.doi.org/10.1172/JCI89927</a></p>
<ul>
<li>Autoimmune responses to meiotic germ cell antigens (MGCA) that are expressed on sperm and testis occur in human infertility and after vasectomy. Many MGCA are also expressed as cancer/testis antigens (CTA) in human cancers, but the tolerance status of MGCA has not been investigated. MGCA are considered to be uniformly immunogenic and nontolerogenic, and the prevailing view posits that MGCA are sequestered behind the Sertoli cell barrier in seminiferous tubules. Here, we have shown that only some murine MGCA are sequestered. Nonsequestered MCGA (NS-MGCA) egressed from normal tubules, as evidenced by their ability to interact with systemically injected antibodies and form localized immune complexes outside the Sertoli cell barrier. NS-MGCA derived from cell fragments that were discarded by spermatids during spermiation. They egressed as cargo in residual bodies and maintained Treg-dependent physiological tolerance. In contrast, sequestered MGCA (S-MGCA) were undetectable in residual bodies and were nontolerogenic. Unlike postvasectomy autoantibodies, which have been shown to mainly target S-MGCA, autoantibodies produced by normal mice with transient Treg depletion that developed autoimmune orchitis exclusively targeted NS-MGCA. We conclude that spermiation, a physiological checkpoint in spermatogenesis, determines the egress and tolerogenicity of MGCA. Our findings will affect target antigen selection in testis and sperm autoimmunity and the immune responses to CTA in male cancer patients.</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>07992</h2>
<h2>08092</h2>
<h2>08192</h2>
<h2>08292</h2>
<h2>08392</h2>
<h2>08492</h2>
<h2>08592</h2>
<h2>08692</h2>
<h2>08792</h2>
<h2>08892</h2>
<h2>08992</h2>
<h2>09092</h2>
<h2>09192</h2>
<h2>09292</h2>
<h2>09392</h2>
<h2>09492</h2>
<h2>09592</h2>
<h2>09692</h2>
<h2>09792</h2>
<h2>09892</h2>
<h2>09992</h2>
<h2>04992</h2>
<h2>04892</h2>
<h2>04792</h2>
<h2>04692</h2>
<h2>04592</h2>
<h2>04492</h2>
<h2>04392</h2>
<h2>04292</h2>
<h2>04192</h2>
<h2>04092</h2>
<h2>03992</h2>
<h2>03892</h2>
<h2>03792</h2>
<h2>03692</h2>
<h2>03592</h2>
<h2>03492</h2>
<h2>03392</h2>
<h2>03292</h2>
<h2>03192</h2>
<h2>03092</h2>
<h2>02992</h2>
<h2>02892</h2>
<h2>02792</h2>
<h2>02692</h2>
<h2>02592</h2>
<ul>
<li>controversial theories
<ul>
<li>
<p>evolution through loss is different from evolution through gain. currently evolution through gain.</p>
</li>
<li>
<p>genetic basis for race</p>
<ul>
<li>
<p>nicholas wade 2014 a troublesome inheritance: genes, race, and human history</p>
<ul>
<li>holds that east asian and jewish success comes from evolutionary pressure selecting for higher intelligence, e.g. emergence of Pharisees leading to conversion away from judaism of those who refused to become literate, leaving a self-selected founder population</li>
<li>holds that reason arose from religious debate, leading to reason and conscience and thence a legal system and thence science</li>
<li>holds that genetic differences between races leads to difference in industry, rather than difference in privilege</li>
</ul>
</li>
<li>
<p><em>the 10,000 year explosion; how civilization accelerated human evolution</em><br />
gregory cochran, henry harpending 2009<br />
978-0-465-00221-4</p>
<ul>
<li>contraversial, especially regarding IQ in Ashkenazim, saying that medieval European prejudice and restrictions to societal livelihood along with in–group marriage created evolutionary pressure for higher intelligence among Ashkenazim but not other Jewish populations</li>
</ul>
</li>
</ul>
</li>
<li>
<p>cultural basis for healthy habits</p>
<ul>
<li>john durant 2013 the paleo manifesto: ancient wisdom for modern health
<ul>
<li>holds that better survival rate of jewish populations during plagues due to healthy practices encoded in cultural practices, e.g. hand washing valued compared to surrounding populations of christians who devalued cleanliness</li>
</ul>
</li>
</ul>
</li>
<li>
<p>geographic determinism</p>
<ul>
<li>jared diamond guns germs and steel
<ul>
<li>holds that &quot;history followed different courses for different peoples because of differences among peoples’ environments, not because of biological differences among peoples themselves.&quot;</li>
</ul>
</li>
</ul>
</li>
<li>
<p>better institutions</p>
<ul>
<li>niall ferguson
<ul>
<li>holds that open society led to innovation, led to rise of west
<ul>
<li>in which case west now in decline</li>
<li>where does open society come from?</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>dna dispose, but subjects decide. learning and the extended synthesis</em><br />
markus lindholm 2015<br />
<a href="https://doi.org/10.1007/s12304-015-9242-3">doi.org/10.1007/s12304-015-9242-3</a></p>
<p>Link: <a href="http://link.springer.com/article/10.1007/s12304-015-9242-3/fulltext.html">link.springer.com/article/10.1007/s12304-015-9242-3/fulltext.html</a></p>
<ul>
<li>Adaptation by means of natural selection depends on the ability of populations to maintain variation in heritable traits. According to the Modern Synthesis this variation is sustained by mutations and genetic drift. Epigenetics, evodevo, niche construction and cultural factors have more recently been shown to contribute to heritable variation, however, leading an increasing number of biologists to call for an extended view of speciation and evolution. An additional common feature across the animal kingdom is learning, defined as the ability to change behavior according to novel experiences or skills. Learning constitutes an additional source for phenotypic variation, and change in behavior may induce long lasting shifts in fitness, and hence favor evolutionary novelties. Based on published studies, I demonstrate how learning about food, mate choice and habitats has contributed substantially to speciation in the canonical story of Darwin’s finches on the Galapagos Islands. Learning cannot be reduced to genetics, because it demands decisions, which requires a subject. Evolutionary novelties may hence emerge both from shifts in allelic frequencies and from shifts in learned, subject driven behavior. The existence of two principally different sources of variation also prevents the Modern Synthesis from self-referring explanations.</li>
</ul>
</li>
<li>
<p>living space, not competition, is evolutionary driver?</p>
<ul>
<li>
<p>this is particularly important because &quot;evolutionary competition&quot; &quot;nature red in tooth and claw&quot; is a prime assumption of our society.</p>
</li>
<li>
<p>links between global taxonomic diversity, ecological diversity and the expansion of vertebrates on land<br />
sahney et al 2010<br />
Link: <a href="http://rsbl.royalsocietypublishing.org/content/6/4/544.full.pdf">rsbl.royalsocietypublishing.org/content/6/4/544.full.pdf </a></p>
<ul>
<li>http://rsbl.royalsocietypublishing.org/content/6/4/544.full.html#ref–list–1</li>
<li>sahney<br />
Link: <a href="http://bristol.academia.edu/SSahney/Papers">bristol.academia.edu/SSahney/Papers</a></li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>evolutionary rescue can prevent extinction following environmental change</em><br />
graham bell and andrew gonzalez 2009<br />
http://doi.org/10.1111/j.1461-0248.2009.01350.x<br />
Link: <a href="http://doi.org/10.1111/j.1461-0248.2009.01350.x">doi.org/10.1111/j.1461-0248.2009.01350.x</a></p>
<ul>
<li>
<p>basically, presence of a few adapted individuals in a large population allows numbers to recover in laboratory i.e. stable conditions. changing conditions not investigated here, only single step change.</p>
</li>
<li>
<p>pdf<br />
Link: <a href="http://gonzalezlab.weebly.com/uploads/5/7/4/1/5741805/bell_and_gonzalez_09__el.pdf">gonzalezlab.weebly.com/uploads/5/7/4/1/5741805/bell_and_gonzalez_09__el.pdf</a></p>
</li>
<li>
<p>The ubiquity of global change and its impacts on biodiversity poses a clear and urgent challenge for evolutionary biologists. In many cases, environmental change is so widespread and rapid that individuals can neither accommodate to them physiologically nor migrate to a more favourable site. Extinction will ensue unless the population adapts fast enough to counter the rate of decline. According to theory, whether populations can be rescued by evolution depends upon several crucial variables: population size, the supply of genetic variation, and the degree of maladaptation to the new environment. Using techniques in experimental evolution we tested the conditions for evolutionary rescue (ER). Hundreds of yeast populations were exposed to normally lethal concentrations of salt in conditions, where the frequency of rescue mutations was estimated and population size was manipulated. In a striking match with theory, we show that ER is possible, and that the recovery of the population may occur within 25 generations. We observed a clear threshold in population size for ER whereby the ancestral population size must be sufficiently large to counter stochastic extinction and contain resistant individuals. These results demonstrate that rapid evolution is an important component of the response of small populations to environmental change.</p>
</li>
</ul>
</li>
<li>
<p>interface theory of perception http://www.cogsci.uci.edu/~ddhoff/interface.pdf</p>
<ul>
<li>fits into our metaphor for evolution although we disagree with the tone and many of the arguments used.</li>
<li>he argues we have interfaces to the world and wonders why we do.</li>
<li>I say because this helps us to prepare for reality by enabling us to dream and think and tell stories!</li>
<li>beliefs are stories, stories are spells, spells are maps to the territory of reality. by understanding our maps we better navigate the territory.</li>
<li>absolutes do exist but only a few are useful. even the rule to not obey rules all the time should be broken sometimes so who is to say there might not be a few rules which might always apply? (likelihood low but not zero).</li>
<li>the beliefs in the world now are not static, they perpetuate. they are not only spells but become daemons. this is possession. the human world is possessed by some very evil daemons which we must face and defeat.</li>
<li>if the start of spells is the fall from paradise then the transcendence of spells is the return.</li>
<li>satisficing, what is good enough.</li>
<li>the author of the interface theory forgets that the theory of evolution is also an interface, according to his theory.</li>
<li>again his example of evolutionary game theory lacks the essential component of change which would render the theory useful. there may indeed be times when veracity outcompetes simple, even if those times are few and far between they would be significant for the survival of the less common variant. again another assumption is that there is only one outcome. organisms are not one–outcome values. it may be coincidence that different outcomes due to other factors enable survival when one factor changes.</li>
<li>so again the problem is reliance on stability as a criterion in deciding evolutionary fitness. he criticises others for conflating veracity with fitness but falls into a similar trap by conflating stability with fitness.</li>
</ul>
</li>
<li>
<p>is our next evolutionary step to learn to control these massive ponderous structures that we have built, of industry, of law, of everything? we are still operating as a society on the individual level and have poor control over the direction of our society.</p>
</li>
<li>
<p>“junk” dna is not junk</p>
<ul>
<li>
<p><em>genomic variation within alpha satellite dna influences centromere location on human chromosomes with metastable epialleles</em><br />
megan e aldrup-macdonald, molly e kuo, lori l sullivan, kimberline chew, beth a sullivan 2016;<br />
http://dx.doi.org/10.1101/gr.206706.116<br />
Link: <a href="http://dx.doi.org/10.1101/gr.206706.116">dx.doi.org/10.1101/gr.206706.116</a></p>
<ul>
<li>Alpha satellite is a tandemly organized type of repetitive DNA that comprises 5% of the genome and is found at all human centromeres. A defined number of 171-bp monomers are organized into chromosome-specific higher-order repeats (HORs) that are reiterated thousands of times. At least half of all human chromosomes have two or more distinct HOR alpha satellite arrays within their centromere regions. We previously showed that the two alpha satellite arrays of Homo sapiens Chromosome 17 (HSA17), D17Z1 and D17Z1-B, behave as centromeric epialleles, that is, the centromere, defined by chromatin containing the centromeric histone variant CENPA and recruitment of other centromere proteins, can form at either D17Z1 or D17Z1-B. Some individuals in the human population are functional heterozygotes in that D17Z1 is the active centromere on one homolog and D17Z1-B is active on the other. In this study, we aimed to understand the molecular basis for how centromere location is determined on HSA17. Specifically, we focused on D17Z1 genomic variation as a driver of epiallele formation. We found that D17Z1 arrays that are predominantly composed of HOR size and sequence variants were functionally less competent. They either recruited decreased amounts of the centromere-specific histone variant CENPA and the HSA17 was mitotically unstable, or alternatively, the centromere was assembled at D17Z1-B and the HSA17 was stable. Our study demonstrates that genomic variation within highly repetitive, noncoding DNA of human centromere regions has a pronounced impact on genome stability and basic chromosomal function.</li>
</ul>
</li>
<li>
<p><em>l1-associated genomic regions are deleted in somatic cells of the healthy human brain</em><br />
jennifer erwin et al. 2016<br />
http://dx.doi.org/10.1038/nn.4388<br />
Link: <a href="http://dx.doi.org/10.1038/nn.4388">dx.doi.org/10.1038/nn.4388</a></p>
<ul>
<li>The healthy human brain is a mosaic of varied genomes. Long interspersed element-1 (LINE-1 or L1) retrotransposition is known to create mosaicism by inserting L1 sequences into new locations of somatic cell genomes. Using a machine learning-based, single-cell sequencing approach, we discovered that somatic L1-associated variants (SLAVs) are composed of two classes: L1 retrotransposition insertions and retrotransposition-independent L1-associated variants. We demonstrate that a subset of SLAVs comprises somatic deletions generated by L1 endonuclease cutting activity. Retrotransposition-independent rearrangements in inherited L1s resulted in the deletion of proximal genomic regions. These rearrangements were resolved by microhomology-mediated repair, which suggests that L1-associated genomic regions are hotspots for somatic copy number variants in the brain and therefore a heritable genetic contributor to somatic mosaicism. We demonstrate that SLAVs are present in crucial neural genes, such as DLG2 (also called PSD93), and affect 44–63% of cells of the cells in the healthy brain.</li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>balancing evolution</em><br />
occurrence of harmful gene variants could be the price we pay for the genetic diversity that is otherwise highly beneficial to our survival</p>
<ul>
<li>
<p><em>excess of deleterious mutations around hla genes reveals evolutionary cost of balancing selection</em><br />
tobias l. lenz, victor spirin, daniel m. jordan, shamil r. sunyaev 2016<br />
http://dx.doi.org/10.1093/molbev/msw127<br />
Link: <a href="http://dx.doi.org/10.1093/molbev/msw127">dx.doi.org/10.1093/molbev/msw127</a></p>
<ul>
<li>Deleterious mutations are expected to evolve under negative selection and are usually purged from the population. However, deleterious alleles segregate in the human population and some disease-associated variants are maintained at considerable frequencies. Here, we test the hypothesis that balancing selection may counteract purifying selection in neighboring regions and thus maintain deleterious variants at higher frequency than expected from their detrimental fitness effect. We first show in realistic simulations that balancing selection reduces the density of polymorphic sites surrounding a locus under balancing selection, but at the same time markedly increases the population frequency of the remaining variants, including even substantially deleterious alleles. To test the predictions of our simulations empirically, we then use whole-exome sequencing data from 6,500 human individuals and focus on the most established example for balancing selection in the human genome, the major histocompatibility complex (MHC). Our analysis shows an elevated frequency of putatively deleterious coding variants in nonhuman leukocyte antigen (non-HLA) genes localized in the MHC region. The mean frequency of these variants declined with physical distance from the classical HLA genes, indicating dependency on genetic linkage. These results reveal an indirect cost of the genetic diversity maintained by balancing selection, which has hitherto been perceived as mostly advantageous, and have implications both for the evolution of recombination and also for the epidemiology of various MHC-associated diseases.</li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>social context, but not individual personality, alters immigrant viability in a spider with mixed social structure</em><br />
spencer j. ingley, jonathan n. pruitt, inon scharf, jessica purcell 2016<br />
http://dx.doi.org/10.1016/j.anbehav.2016.08.009<br />
Link: <a href="http://dx.doi.org/10.1016/j.anbehav.2016.08.009">dx.doi.org/10.1016/j.anbehav.2016.08.009</a></p>
<ul>
<li>Local adaptation in social animals can occur in individual- or group-level traits.<br />
•<br />
Misaligned traits at either level can reduce migration between environments.<br />
•<br />
We tested how social context and individual personality alter immigrant viability.<br />
•<br />
Social context, not personality, affected survival in foreign environments.<br />
•<br />
Spiders transplanted with their natal groups suffered the highest mortality rates.<br />
Immigrant viability is a major determinant of the realized rate of gene flow across populations. For social organisms, the social context in which immigrants disperse across contrasting environments may have important implications for their viability post dispersal. Here, we use social spiders whose individual personalities as well as group personality compositions vary across sites to test whether the strength of selection against immigrants (i.e. mortality rates) differs depending on whether spiders are transplanted (1) as individuals and remain alone, (2) join pre-existing colonies at their new non-native environment, or (3) move with their native group. We also tested for an effect of individual personality on survival. We found that social context, and not individual personality, affects individual survival in foreign environments with contrasting resource levels. Individuals that were transplanted with their native groups suffered higher mortality rates compared to individuals transplanted as singletons, regardless of whether or not they were assimilated into native colonies. Moving as individuals could thus provide an avenue for ongoing gene flow among populations from different resource environments. We found no depressed performance of control colonies that were transplanted across sites with resource levels similar to each colony's site of origin. These results are at odds with the intuition that dispersing as a group should generally enhance the viability of immigrants, at least in social species. We propose that these results could be explained by a mismatch in the ideal group compositions (personality compositions) favoured in different environments, despite a lack of selection on individual personality traits. These results provide a first glimpse into the relative roles of individual personality and social context in mediating gene flow among populations from divergent environments.</li>
</ul>
</li>
<li>
<p><em>demographic and reproductive plasticity across the depth distribution of a coral reef fish</em><br />
esther d. goldstein, evan k. d’alessandro, su sponaugle 2016<br />
http://dx.doi.org/10.1038/srep34077<br />
Link: <a href="http://dx.doi.org/10.1038/srep34077">dx.doi.org/10.1038/srep34077</a></p>
<ul>
<li>As humans expand into natural environments, populations of wild organisms may become relegated to marginal habitats at the boundaries of their distributions. In the ocean, mesophotic coral ecosystems (30–150 m) at the depth limit of photosynthetic reefs are hypothesized to act as refuges that are buffered from anthropogenic and natural disturbances, yet the viability and persistence of subpopulations in these peripheral habitats remains poorly understood. To assess the potential for mesophotic reefs to support robust coral reef fish populations, we compared population density and structure, growth, size, and reproductive output of the bicolor damselfish (Stegastes partitus) from shallow (&lt;10 m), deep shelf (20–30 m), and mesophotic reefs (60–70 m) across the Florida Platform. Population densities decreased and size and age distributions shifted toward older and larger individuals in deeper habitats. Otolith-derived ages indicated that S. partitus found on mesophotic reefs reach larger asymptotic sizes and have longer lifespans than fish in shallower habitats. Based on measurements of oocyte area and batch fecundity, mesophotic fish also have higher reproductive investment. These demographic patterns indicate that mesophotic fish populations composed of large, fecund individuals produce high condition larvae and rely on longevity of individuals for population persistence and viability.</li>
</ul>
</li>
<li>
<p><em>archean coastal-plain paleosols and life on land</em><br />
gregory j. retallack, david h. krinsley, robert fischer, joshua j. razink, kurt a. langworthy 2016<br />
<a href="http://dx.doi.org/10.1016/j.gr.2016.08.003">j.gr.2016.08.003</a></p>
<ul>
<li>
<p>There are coastal-plain paleosols in 3.0 Ga Farrel Quartzite, Western Australia.</p>
<p>Paleosols have organic surface (A) and sulfate-rich subsurface (By) horizon.</p>
<p>Comparable profiles are known from deserts of Chile, Antarctica, and Mars.</p>
<p>Microfossils in paleosols include actinobacteria, sulfur bacteria, methanogens.</p>
<p>Coastal-plain paleosols in the 3.0 Ga Farrel Quartzite of Western Australia have organic surface (A horizon) and sulfate-rich subsurface (By) horizons, like soils of the Atacama Desert of Chile, Dry Valleys of Antarctica, and 3.7 Ga paleosols of Mars. Farrel Quartzite paleosols include previously described microfossils, permineralized by silica in a way comparable with the Devonian Rhynie Chert, a well known permineralized Histosol. Five microfossil morphotypes in the Farrel Quartzite include a variety of spheroidal cells (<em>Archaeosphaeroides</em>) as well as distinctive large spindles (new genus provisionally assigned to cf. <em>Eopoikilofusa</em>). Previously published cell-specific carbon isotopic analyses of the Farrel Quartzite microfossils, and unusually abundant sulfate considering a likely anoxic atmosphere, allow interpretation of these morphotypes as a terrestrial community of actinobacteria, purple sulfur bacteria, and methanogenic Archaea.</p>
</li>
</ul>
</li>
<li>
<p><em>the 'out of africa' hypothesis, human genetic diversity, and comparative economic development</em><br />
quamrul ashraf, oded galor 2013<br />
http://dx.doi.org/10.1257/aer.103.1.1<br />
Link: <a href="http://dx.doi.org/10.1257/aer.103.1.1">dx.doi.org/10.1257/aer.103.1.1</a></p>
<ul>
<li>This research advances and empirically establishes the hypothesis that, in the course of the prehistoric exodus of Homo sapiens out of Africa, variation in migratory distance to various settlements across the globe affected genetic diversity and has had a persistent hump-shaped effect on comparative economic development, reflecting the trade-off between the beneficial and the detrimental effects of diversity on productivity. While the low diversity of Native American populations and the high diversity of African populations have been detrimental for the development of these regions, the intermediate levels of diversity associated with European and Asian populations have been conducive for development.</li>
<li>“greater diversity are also characterized, on average, by a lower prevalence of interpersonal trust and a higher intensity of scientific knowledge creation.”</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>02492</h2>
<h2>02392</h2>
<h2>02292</h2>
<h2>02192</h2>
<h2>02092</h2>
<ul>
<li>transmit to offspring
<ul>
<li>
<p>chromosome</p>
<ul>
<li>
<p>mutations and sexual recombination transmitted via DNA, distribution changes and also elimination via genetic drift and migration</p>
</li>
<li>
<p><em>defined chromosome structure in the genome-reduced bacterium mycoplasma pneumoniae</em><br />
marie trussart et al. 2017<br />
<a href="http://dx.doi.org/10.1038/ncomms14665">dx.doi.org/10.1038/ncomms14665</a></p>
<ul>
<li>DNA-binding proteins are central regulators of chromosome organization; however, in genome-reduced bacteria their diversity is largely diminished. Whether the chromosomes of such bacteria adopt defined three-dimensional structures remains unexplored. Here we combine Hi-C and super-resolution microscopy to determine the structure of the <em>Mycoplasma pneumoniae</em> chromosome at a 10 kb resolution. We find a defined structure, with a global symmetry between two arms that connect opposite poles, one bearing the chromosomal Ori and the other the midpoint. Analysis of local structures at a 3 kb resolution indicates that the chromosome is organized into domains ranging from 15 to 33 kb. We provide evidence that genes within the same domain tend to be co-regulated, suggesting that chromosome organization influences transcriptional regulation, and that supercoiling regulates local organization. This study extends the current understanding of bacterial genome organization and demonstrates that a defined chromosomal structure is a universal feature of living systems.</li>
<li>a higher 10 kb resolution Hi-C map of <em>C. crescentus</em><a href="http://www.nature.com/articles/ncomms14665#ref18">18</a> revealed that its genome is divided into 23 chromosome interacting domains (CIDs) or highly self-interacting regions, similar to the topologically associating domains (TADs) found in eukaryotes</li>
</ul>
</li>
</ul>
</li>
<li>
<p>evodevo<br />
evolutionary development biology</p>
<ul>
<li>alters developmental processes to create new and novel structures from the old gene networks (such as bone structures of the jaw deviating to the ossicles of the middle ear) or will conserve (molecular economy) a similar program in a host of organisms such as eye development genes in molluscs, insects, and vertebrates</li>
</ul>
</li>
<li>
<p>neochromosome</p>
<ul>
<li>cancerous mutation e.g. gared et al 2014 the architecture and evolution of cancer neochromosomes<br />
Link: <a href="http://dx.doi.org/10.1016/j.ccell.2014.09.010">dx.doi.org/10.1016/j.ccell.2014.09.010</a></li>
</ul>
</li>
<li>
<p>epigenome</p>
<ul>
<li>
<p>methylation during lifetime transmitted via germline</p>
</li>
<li>
<p><em>maternal h3k27me3 controls dna methylation-independent imprinting</em><br />
azusa inoue, lan jiang, falong lu, tsukasa suzuki, yi zhang 2017<br />
<a href="http://dx.doi.org/10.1038/nature23262">dx.doi.org/10.1038/nature23262</a></p>
<ul>
<li>Mammalian sperm and oocytes have different epigenetic landscapes and are organized in different fashions. After fertilization, the initially distinct parental epigenomes become largely equalized with the exception of certain loci, including imprinting control regions. How parental chromatin becomes equalized and how imprinting control regions escape from this reprogramming is largely unknown. Here we profile parental allele-specific DNase I hypersensitive sites in mouse zygotes and morula embryos, and investigate the epigenetic mechanisms underlying these allelic sites. Integrated analyses of DNA methylome and tri-methylation at lysine 27 of histone H3 (H3K27me3) chromatin immunoprecipitation followed by sequencing identify 76 genes with paternal allele-specific DNase I hypersensitive sites that are devoid of DNA methylation but harbour maternal allele-specific H3K27me3. Interestingly, these genes are paternally expressed in preimplantation embryos, and ectopic removal of H3K27me3 induces maternal allele expression. H3K27me3-dependent imprinting is largely lost in the embryonic cell lineage, but at least five genes maintain their imprinted expression in the extra-embryonic cell lineage. The five genes include all paternally expressed autosomal imprinted genes previously demonstrated to be independent of oocyte DNA methylation. Thus, our study identifies maternal H3K27me3 as a DNA methylation-independent imprinting mechanism.</li>
</ul>
</li>
<li>
<p><em>parent-of-origin dna methylation dynamics during mouse development</em><br />
yonatan stelzer, hao wu, yuelin song, chikdu s. shivalila, styliani markoulaki, rudolf jaenisch 2016<br />
http://dx.doi.org/10.1016/j.celrep.2016.08.066<br />
Link: <a href="http://dx.doi.org/10.1016/j.celrep.2016.08.066">dx.doi.org/10.1016/j.celrep.2016.08.066</a></p>
<ul>
<li>
<p>•In vivo tracing of parent-specific DNA methylation dynamics at single-cell resolution<br />
•Cell-type-specific methylation signatures at the Dlk-Dio3 IG-DMR during development<br />
•Dynamic parent- and cell-type-specific DNA methylation changes in the adult brain</p>
<p>Parent-specific differentially methylated regions (DMRs) are established during gametogenesis and regulate parent-specific expression of imprinted genes. Monoallelic expression of imprinted genes is essential for development, suggesting that imprints are faithfully maintained in embryos and adults. To test this hypothesis, we targeted a reporter for genomic methylation to the imprinted Dlk1-Dio3 intergenic DMR (IG-DMR) to assess the methylation of both parental alleles at single-cell resolution. Biallelic gain or loss of IG-DMR methylation occurred in a small fraction of mouse embryonic stem cells, significantly affecting developmental potency. Mice carrying the reporter in either parental allele showed striking parent-specific changes in IG-DMR methylation, causing substantial and consistent tissue- and cell-type-dependent signatures in embryos and postnatal animals. Furthermore, dynamics in DNA methylation persisted during adult neurogenesis, resulting in inter-individual diversity. This substantial cell-cell DNA methylation heterogeneity implies that dynamic DNA methylation variations in the adult may be of functional importance.</p>
</li>
</ul>
</li>
<li>
<p><em>krab zinc-finger proteins contribute to the evolution of gene regulatory networks</em><br />
michaël imbeault, pierre-yves helleboid, didier trono 2017<br />
<a href="http://dx.doi.org/10.1038/nature21683">dx.doi.org/10.1038/nature21683</a></p>
<ul>
<li>The human genome encodes some 350 Krüppel-associated box (KRAB) domain-containing zinc-finger proteins (KZFPs), the products of a rapidly evolving gene family that has been traced back to early tetrapods<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref1">105</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref2">106</a>. The function of most KZFPs is unknown, but a few have been demonstrated to repress transposable elements in embryonic stem (ES) cells by recruiting the transcriptional regulator TRIM28 and associated mediators of histone H3 Lys9 trimethylation (H3K9me3)-dependent heterochromatin formation and DNA methylation<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref3">107</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref4">108</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref5">109</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref6">110</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref7">111</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref8">112</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref9">113</a>. Depletion of TRIM28 in human or mouse ES cells triggers the upregulation of a broad range of transposable elements<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref4">108</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref10">114</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref11">115</a>, and recent data based on a few specific examples have pointed to an arms race between hosts and transposable elements as an important driver of KZFP gene selection<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref5">109</a>. Here, to obtain a global view of this phenomenon, we combined phylogenetic and genomic studies to investigate the evolutionary emergence of KZFP genes in vertebrates and to identify their targets in the human genome. First, we unexpectedly reassigned the root of the family to a common ancestor of coelacanths and tetrapods. Second, although we confirmed that the majority of KZFPs bind transposable elements and pinpoint cases of ongoing co-evolution, we found that most of their transposable element targets have lost all transposition potential. Third, by examining the interplay between human KZFPs and other transcriptional modulators, we obtained evidence that KZFPs exploit evolutionarily conserved fragments of transposable elements as regulatory platforms long after the arms race against these genetic invaders has ended. Together, our results demonstrate that KZFPs partner with transposable elements to build a largely species-restricted layer of epigenetic regulation.</li>
</ul>
</li>
<li>
<p><em>germ line-inherited h3k27me3 restricts enhancer function during maternal-to-zygotic transition</em><br />
zenk f et al. 2017<br />
<a href="http://dx.doi.org/10.1126/science.aam5339">dx.doi.org/10.1126/science.aam5339</a></p>
<ul>
<li>Gametes carry parental genetic material to the next generation. Stress-induced epigenetic changes in the germ line can be inherited and can have a profound impact on offspring development. However, the molecular mechanisms and consequences of transgenerational epigenetic inheritance are poorly understood. We found that <em>Drosophila</em> oocytes transmit the repressive histone mark H3K27me3 to their offspring. Maternal contribution of the histone methyltransferase Enhancer of zeste, the enzymatic component of Polycomb repressive complex 2, is required for active propagation of H3K27me3 during early embryogenesis. H3K27me3 in the early embryo prevents aberrant accumulation of the active histone mark H3K27ac at regulatory regions and precocious activation of lineage-specific genes at zygotic genome activation. Disruption of the germ line–inherited Polycomb epigenetic memory causes embryonic lethality that cannot be rescued by late zygotic reestablishment of H3K27me3. Thus, maternally inherited H3K27me3, propagated in the early embryo, regulates the activation of enhancers and lineage-specific genes during development.</li>
</ul>
</li>
</ul>
</li>
<li>
<p>…</p>
</li>
<li>
<p>epitranscriptome</p>
<ul>
<li>
<p><em>the epitranscriptome of noncoding rnas in cancer</em><br />
manel esteller, pier paolo pandolfi 2017<br />
<a href="http://dx.doi.org/10.1158/2159-8290.CD-16-1292">dx.doi.org/10.1158/2159-8290.CD-16-1292</a></p>
<ul>
<li>The activity of RNA is controlled by different types of post-transcriptional modifications, such as the addition of methyl groups and other chemical and structural changes, that have been recently described in human cells by high-throughput sequencing. Herein, we will discuss how the so-called epitranscriptome is disrupted in cancer and what the contribution of its writers, readers, and erasers to the process of cellular transformation is, particularly focusing on the epigenetic modifications of ncRNAs.</li>
</ul>
</li>
</ul>
</li>
<li>
<p>epigenetic memory</p>
<ul>
<li>
<p><em>reconsidering plant memory: intersections between stress recovery, rna turnover, and epigenetics</em><br />
crisp et al. 2016<br />
DOI: 10.1126/sciadv.1501340<br />
Link: <a href="http://advances.sciencemag.org/content/2/2/e1501340.full">advances.sciencemag.org/content/2/2/e1501340.full</a></p>
<ul>
<li>plant memory and forgetting</li>
<li>some plants also seem to have short-term memory not using DNA or RNA</li>
<li>“unlike animals, in <em>Arabidopsis</em>, DNA methylation can be inherited through meiotic cell divisions”</li>
</ul>
</li>
</ul>
</li>
<li>
<p>protein/prion</p>
<ul>
<li>
<p><em>intrinsically disordered proteins drive emergence and inheritance of biological traits</em><br />
sohini chakrabortee et al. 2016<br />
http://dx.doi.org/10.1016/j.cell.2016.09.017<br />
Link: <a href="http://dx.doi.org/10.1016/j.cell.2016.09.017">dx.doi.org/10.1016/j.cell.2016.09.017</a></p>
<ul>
<li>•Transient overexpression of many yeast proteins creates heritable biological traits<br />
•The traits have prion-like inheritance patterns and can involve gains of function<br />
•The traits are transmissible by protein only but do not arise from amyloid fibers<br />
•The traits arise from disordered protein sequences conserved from yeast to man<br />
Summary<br />
Prions are a paradigm-shifting mechanism of inheritance in which phenotypes are encoded by self-templating protein conformations rather than nucleic acids. Here, we examine the breadth of protein-based inheritance across the yeast proteome by assessing the ability of nearly every open reading frame (ORF; ∼5,300 ORFs) to induce heritable traits. Transient overexpression of nearly 50 proteins created traits that remained heritable long after their expression returned to normal. These traits were beneficial, had prion-like patterns of inheritance, were common in wild yeasts, and could be transmitted to naive cells with protein alone. Most inducing proteins were not known prions and did not form amyloid. Instead, they are highly enriched in nucleic acid binding proteins with large intrinsically disordered domains that have been widely conserved across evolution. Thus, our data establish a common type of protein-based inheritance through which intrinsically disordered proteins can drive the emergence of new traits and adaptive opportunities.</li>
</ul>
</li>
</ul>
</li>
<li>
<p>mitochondria</p>
<ul>
<li>
<p><em>genetic evidence for elevated pathogenicity of mitochondrial dna heteroplasmy in autism spectrum disorder</em><br />
wang y, picard m, gu z 2016<br />
http://dx.doi.org/10.1371/journal.pgen.1006391<br />
Link: <a href="http://dx.doi.org/10.1371/journal.pgen.1006391">dx.doi.org/10.1371/journal.pgen.1006391</a></p>
<ul>
<li>Mitochondria contain their own genome, the mitochondrial DNA (mtDNA), and are abundant in the brain where they produce energy and intracellular signals required for normal brain function and cognition. Mitochondrial dysfunction has been proposed as a cause of Autism Spectrum Disorder (ASD), but the genetic basis has not been established. By analyzing mtDNA sequences from 903 ASD children along with their unaffected siblings and mothers, we found a unique pattern of heteroplasmic mtDNA mutations (the coexistence of both mutant and wide-type mtDNA molecules in a cell) associated with increased risk of ASD. A large fraction of these mtDNA heteroplasmies are pathogenic and can be either inherited maternally or acquired during development. Our findings unveil an important issue about pathogenic mtDNA heteroplasmy transmission between mothers and children, which could underlie many neurodevelopmental disorders in children that are shown to involve mitochondrial dysfunction. Evaluating mtDNA heteroplasmies in high-risk families could help diagnosis and treatment of these diseases.</li>
</ul>
</li>
<li>
<p><em>classification and adaptive behavior prediction of children with autism spectrum disorder based upon multivariate data analysis of markers of oxidative stress and dna methylation</em><br />
daniel p. howsmon, uwe kruger, stepan melnyk, s. jill james, juergen hahn 2017<br />
<a href="http://dx.doi.org/10.1371/journal.pcbi.1005385">dx.doi.org/10.1371/journal.pcbi.1005385</a></p>
<ul>
<li>The number of diagnosed cases of Autism Spectrum Disorders (ASD) has increased dramatically over the last four decades; however, there is still considerable debate regarding the underlying pathophysiology of ASD. This lack of biological knowledge restricts diagnoses to be made based on behavioral observations and psychometric tools. However, physiological measurements should support these behavioral diagnoses in the future in order to enable earlier and more accurate diagnoses. Stepping towards this goal of incorporating biochemical data into ASD diagnosis, this paper analyzes measurements of metabolite concentrations of the folate-dependent one-carbon metabolism and transulfuration pathways taken from blood samples of 83 participants with ASD and 76 age-matched neurotypical peers. Fisher Discriminant Analysis enables multivariate classification of the participants as on the spectrum or neurotypical which results in 96.1% of all neurotypical participants being correctly identified as such while still correctly identifying 97.6% of the ASD cohort. Furthermore, kernel partial least squares is used to predict adaptive behavior, as measured by the Vineland Adaptive Behavior Composite score, where measurement of five metabolites of the pathways was sufficient to predict the Vineland score with an <em>R</em>2 of 0.45 after cross-validation. This level of accuracy for classification as well as severity prediction far exceeds any other approach in this field and is a strong indicator that the metabolites under consideration are strongly correlated with an ASD diagnosis but also that the statistical analysis used here offers tremendous potential for extracting important information from complex biochemical data sets.<br />
Author summary<br />
Autism spectrum disorder (ASD) encompasses a family of neurological disorders characterized by limited social interaction and restricted repetitive behaviors. The number of children diagnosed with ASD has grown exponentially over the last four decades and is now estimated to affect ∼1.5% of children. Although ASD is currently diagnosed and treated based solely on psychometric tools, a biochemical view applicable to at least a subset of ASD cases is emerging. Abnormalities in folate-dependent one carbon metabolism and transsulfuration pathways can summarize a large number of observations of genetic and environmental effects that increase ASD predisposition. However, these complex, highly interconnected pathways require more advanced statistical models than the typical univariate models presented in the literature. Therefore, we developed multivariate statistical models that classify participants based on their neurological status and predict adaptive behavior in ASD. We emphasize that these models are cross-validated, helping to ensure that the results will generalize to new samples. The models developed herein have much stronger predictability than any existing approaches from the scientific literature.</li>
</ul>
</li>
<li>
<p><em>mitochondrial dysfunction in autism spectrum disorders: a systematic review and meta-analysis</em><br />
d a rossignol, r e frye 2011<br />
<a href="http://dx.doi.org/10.1038/mp.2010.136">mp.2010.136</a></p>
<ul>
<li>A comprehensive literature search was performed to collate evidence of mitochondrial dysfunction in autism spectrum disorders (ASDs) with two primary objectives. First, features of mitochondrial dysfunction in the general population of children with ASD were identified. Second, characteristics of mitochondrial dysfunction in children with ASD and concomitant mitochondrial disease (MD) were compared with published literature of two general populations: ASD children without MD, and non-ASD children with MD. The prevalence of MD in the general population of ASD was 5.0% (95% confidence interval 3.2, 6.9%), much higher than found in the general population (~0.01%). The prevalence of abnormal biomarker values of mitochondrial dysfunction was high in ASD, much higher than the prevalence of MD. Variances and mean values of many mitochondrial biomarkers (lactate, pyruvate, carnitine and ubiquinone) were significantly different between ASD and controls. Some markers correlated with ASD severity. Neuroimaging, <em>in vitro</em> and post-mortem brain studies were consistent with an elevated prevalence of mitochondrial dysfunction in ASD. Taken together, these findings suggest children with ASD have a spectrum of mitochondrial dysfunction of differing severity. Eighteen publications representing a total of 112 children with ASD and MD (ASD/MD) were identified. The prevalence of developmental regression (52%), seizures (41%), motor delay (51%), gastrointestinal abnormalities (74%), female gender (39%), and elevated lactate (78%) and pyruvate (45%) was significantly higher in ASD/MD compared with the general ASD population. The prevalence of many of these abnormalities was similar to the general population of children with MD, suggesting that ASD/MD represents a distinct subgroup of children with MD. Most ASD/MD cases (79%) were not associated with genetic abnormalities, raising the possibility of secondary mitochondrial dysfunction. Treatment studies for ASD/MD were limited, although improvements were noted in some studies with carnitine, co-enzyme Q10 and B-vitamins. Many studies suffered from limitations, including small sample sizes, referral or publication biases, and variability in protocols for selecting children for MD workup, collecting mitochondrial biomarkers and defining MD. Overall, this evidence supports the notion that mitochondrial dysfunction is associated with ASD. Additional studies are needed to further define the role of mitochondrial dysfunction in ASD.</li>
</ul>
</li>
<li>
<p><em>mitochondrial function controls intestinal epithelial stemness and proliferation</em><br />
emanuel berger et al. 2016<br />
http://dx.doi.org/10.1038/ncomms13171<br />
Link: <a href="http://dx.doi.org/10.1038/ncomms13171">dx.doi.org/10.1038/ncomms13171</a></p>
<ul>
<li>Control of intestinal epithelial stemness is crucial for tissue homeostasis. Disturbances in epithelial function are implicated in inflammatory and neoplastic diseases of the gastrointestinal tract. Here we report that mitochondrial function plays a critical role in maintaining intestinal stemness and homeostasis. Using intestinal epithelial cell (IEC)-specific mouse models, we show that loss of HSP60, a mitochondrial chaperone, activates the mitochondrial unfolded protein response (MT-UPR) and results in mitochondrial dysfunction. HSP60-deficient crypts display loss of stemness and cell proliferation, accompanied by epithelial release of WNT10A and RSPO1. Sporadic failure of Cre-mediated <em>Hsp60</em> deletion gives rise to hyperproliferative crypt foci originating from OLFM4+ stem cells. These effects are independent of the MT-UPR-associated transcription factor CHOP. In conclusion, compensatory hyperproliferation of HSP60+ escaper stem cells suggests paracrine release of WNT-related factors from HSP60-deficient, functionally impaired IEC to be pivotal in the control of the proliferative capacity of the stem cell niche.</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>01992</h2>
<h2>01892</h2>
<h2>01792</h2>
<h2>01692</h2>
<ul>
<li>genetics and epigenetics
<ul>
<li>epigenetic change may be constant, with the net effect from sustained trend in any direction
<ul>
<li>
<p>perhaps conscious change in particular direction can lead to epigenetic change</p>
<ul>
<li>could the same be true of insight and innovation, that by providing a suitable environment you set the trend towards certain methylations being more common</li>
</ul>
</li>
<li>
<p>they continue to look for pathways in which, for example, serotonin directly influences behavior via chemical changes in the brain. but what if the real mechanism is actually indirect, that epigenetic changes are constantly happening and what serotonin does is tip the balance towards a certain population of epigenetic variance, which in turn causes the behavior we observe</p>
</li>
<li>
<p>when I get enough sleep, ideas and connections and creativity just seem to flow. could this be through epigenetic change too?</p>
</li>
<li>
<p><em>adolescent binge-pattern alcohol exposure alters genome-wide dna methylation patterns in the hypothalamus of alcohol-naïve male offspring</em><br />
annadorothea animes 2016<br />
<a href="http://dx.doi.org/10.1016/j.alcohol.2016.10.010">j.alcohol.2016.10.010</a></p>
<ul>
<li>Teenage binge drinking is a major health concern in the United States, with 21% of teenagers reporting binge-pattern drinking behavior in the last 30 days. Recently, our lab showed that alcohol-naïve offspring of rats exposed to alcohol during adolescence exhibited altered gene expression profiles in the hypo- thalamus, a brain region involved in stress regulation. We employed Enhanced Reduced Representation Bisulfite Sequencing as an unbiased approach to test the hypothesis that parental exposure to binge- pattern alcohol during adolescence alters DNA methylation profiles in their alcohol-naïve offspring. Wistar rats were administered a repeated binge-ethanol exposure paradigm during early (postnatal day (PND) 37e44) and late (PND 67e74) adolescent development. Animals were mated 24 h after the last ethanol dose and subsequent offspring were produced. Analysis of male PND7 offspring revealed that offspring of alcohol-exposed parents exhibited differential DNA methylation patterns in the hypothala- mus. The differentially methylated cytosines (DMCs) were distinct between offspring depending on which parent was exposed to ethanol. Moreover, novel DMCs were observed when both parents were exposed to ethanol and many DMCs from single parent ethanol exposure were not recapitulated with dual parent exposure. We also measured mRNA expression of several differentially methylated genes and some, but not all, showed correlative changes in expression. Importantly, methylation was not a direct predictor of expression levels, underscoring the complexity of transcriptional regulation. Overall, we demonstrate that adolescent binge ethanol exposure causes altered genome-wide DNA methylation patterns in the hypothalamus of alcohol-naïve offspring.</li>
</ul>
</li>
</ul>
</li>
<li>genetic sensitivity environment
<ul>
<li>(...) jed diamond in irritable male syndrome isbn 140507745x chap8 irritable male body quotes a study by remi cadoret, Iowa college medicine, comment by dean hamer, national cancer institute. saying that good genes or bad genes in good family, or good genes in bad family both fine. but bad genes in bad family not fine at all. &quot;shows that what is being inherited is not bad behaviour or aggression but rather a genetic sensitivity to the environment.&quot; he goes on to a conventional conclusion but what is important for us is that this sentence on its own, interpreted out of context, actually makes sense  for our way of thinking about genes and development.
<ul>
<li>It is a genetic sensitivity to the environment that nature “selects” for, not a particular trait or set of traits. What will help organisms survive is not an unchanging adherence to a given pattern but an adaptability to the conditions in which they find themselves.</li>
<li>check for studies of cumulative genetic drift as opposed to single mutations – this may be missing in literature.</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>01592</h2>
<h2>01492</h2>
<h2>01392</h2>
<h2>01292</h2>
<ul>
<li>invariant with time
<ul>
<li>scattershot
<ul>
<li>many outcomes, large range of abilities and differences</li>
<li>a few will be very well suited to the particular niche they happen to grow into</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>01192</h2>
<ul>
<li>varies with time
<ul>
<li>
<p>niche</p>
<ul>
<li>a circumstance in the world to which a physical characteristic of an organism is advantageous</li>
<li>niches change so those that fit them will be different over time</li>
<li>at any particular time the crucial limiting niche may be different (the one circumstance that all organisms of a certain type in a certain area must endure to survive)</li>
<li>study cross-fostering blue tits and great tits finds learned behaviour from foster parents establishes the niche the offspring inhabit to a higher degree than expected http://rspb.royalsocietypublishing.org/content/274/1606/19.full</li>
</ul>
</li>
<li>
<p>survival</p>
<ul>
<li>not selection</li>
<li>just those who happen to fit the current state of the current niche more likely to survive</li>
</ul>
</li>
<li>
<p>sexual reproduction</p>
<ul>
<li>illusion of choice</li>
<li>actually what may be happening is that the internal state of the selecting partner becomes effectively a niche</li>
</ul>
</li>
<li>
<p>fitness</p>
<ul>
<li>I don't believe there is a single maximum for evolutionary fitness in a single location, because the conditions, however local, change over time. just as we were talking about another paper, where my viewpoint is that in some circumstances, viridical vision may be fit, whereas in others it may be unfit.</li>
<li>We are still assuming that natural selection operates on the level of individual organisms (even though chance and expression of genes are major factors) so how is it that there can be some creatures that are unfit for the current situation?</li>
<li>The answer that it is not a single-run game, where all the unfit are eliminated in one round. Simplifications of evolutionary theory assume that the unfit just die out, when in fact what is fit is not set, and so what is fit in one situation is unfit in another and it is evolutionarily effective to have a range of possible options.</li>
<li>So the distribution over time of so-called fitness is not going to a simple one, even if momentarily a sample indicates a peak distribution around some criterion.</li>
</ul>
</li>
<li>
<p>circadian rhythms</p>
<ul>
<li>
<p><em>cell autonomous regulation of herpes and influenza virus infection by the circadian clock</em><br />
rachel s. edgara et al. 2016<br />
__http://dx.doi.org/10.1073/pnas.1601895113<br />
Link: <a href="http://dx.doi.org/10.1073/pnas.1601895113">dx.doi.org/10.1073/pnas.1601895113</a></p>
<ul>
<li>Viruses are intracellular pathogens that hijack host cell machinery and resources to replicate. Rather than being constant, host physiology is rhythmic, undergoing circadian (∼24 h) oscillations in many virus-relevant pathways, but whether daily rhythms impact on viral replication is unknown. We find that the time of day of host infection regulates virus progression in live mice and individual cells. Furthermore, we demonstrate that herpes and influenza A virus infections are enhanced when host circadian rhythms are abolished by disrupting the key clock gene transcription factor Bmal1. Intracellular trafficking, biosynthetic processes, protein synthesis, and chromatin assembly all contribute to circadian regulation of virus infection. Moreover, herpesviruses differentially target components of the molecular circadian clockwork. Our work demonstrates that viruses exploit the clockwork for their own gain and that the clock represents a novel target for modulating viral replication that extends beyond any single family of these ubiquitous pathogens.</li>
</ul>
</li>
<li>
<p><em>local modulation of human brain responses by circadian rhythmicity and sleep debt</em><br />
vincenzo muto et al 2016<br />
http://dx.doi.org/10.1126/science.aad2993<br />
Link: <a href="http://dx.doi.org/10.1126/science.aad2993">dx.doi.org/10.1126/science.aad2993</a></p>
<ul>
<li>Human performance is modulated by circadian rhythmicity and homeostatic sleep pressure. Whether and how this interaction is represented at the regional brain level has not been established. We quantified changes in brain responses to a sustained-attention task during 13 functional magnetic resonance imaging sessions scheduled across the circadian cycle, during 42 hours of wakefulness and after recovery sleep, in 33 healthy participants. Cortical responses showed significant circadian rhythmicity, the phase of which varied across brain regions. Cortical responses also significantly decreased with accrued sleep debt. Subcortical areas exhibited primarily a circadian modulation that closely followed the melatonin profile. These findings expand our understanding of the mechanisms involved in maintaining cognition during the day and its deterioration during sleep deprivation and circadian misalignment.</li>
<li>our results demonstrate a regional modulation of brain circadian rhythmicity. Several subcortical responses show a strong circadian modulation but no significant influence of sleep debt. By contrast, in most cortical areas, sleep pressure exerts a widespread negative influence on regional responses. This differential regulation of brain responses might explain the supposedly “compensatory” responses repeatedly reported<br />
in thalamic areas during sleep loss.</li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>microbiota diurnal rhythmicity programs host transcriptome oscillations</em><br />
christoph a. thaiss et al. 2016<br />
<a href="http://dx.doi.org/10.1016/j.cell.2016.11.003">j.cell.2016.11.003</a></p>
<ul>
<li>
<p>Intestinal microbiota biogeography and metabolome undergo diurnal oscillations</p>
<p>Circadian oscillations of serum metabolites are regulated by the microbiota</p>
<p>Microbiota rhythms program the circadian epigenetic and transcriptional landscape</p>
<p>The microbiota regulates the circadian liver transcriptome and detoxification pattern</p>
<p>The intestinal microbiota undergoes diurnal compositional and functional oscillations that affect metabolic homeostasis, but the mechanisms by which the rhythmic microbiota influences host circadian activity remain elusive. Using integrated multi-omics and imaging approaches, we demonstrate that the gut microbiota features oscillating biogeographical localization and metabolome patterns that determine the rhythmic exposure of the intestinal epithelium to different bacterial species and their metabolites over the course of a day. This diurnal microbial behavior drives, in turn, the global programming of the host circadian transcriptional, epigenetic, and metabolite oscillations. Surprisingly, disruption of homeostatic microbiome rhythmicity not only abrogates normal chromatin and transcriptional oscillations of the host, but also incites genome-wide de novo oscillations in both intestine and liver, thereby impacting diurnal fluctuations of host physiology and disease susceptibility. As such, the rhythmic biogeography and metabolome of the intestinal microbiota regulates the temporal organization and functional outcome of host transcriptional and epigenetic programs.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>01092</h2>
<ul>
<li>
<p><em>top of the world</em><br />
karen carpenter</p>
<ul>
<li>
<p>eng</p>
<ul>
<li>
<p>such a feeling’s coming over me<br />
there is wonder in most every thing i see<br />
not a cloud in the sky, got the sun in my eyes<br />
and i won’t be surprised if it’s a dream</p>
<p>everything i want the world to be<br />
is now coming true especially for me<br />
and the reason is clear, it’s because you are here<br />
you’re the nearest thing to heaven that i’ve seen</p>
<p>☆i’m on the top of the world looking down on creation<br />
　and the only explanation i can find<br />
　is the love that i’ve found ever since you’ve been around<br />
　your love’s put me at the top of the world</p>
<p>something in the wind has learned my name<br />
and it’s telling me that things are not the same<br />
in the leaves on the trees and the touch of the breeze<br />
there’s a pleasing sense of happiness for me</p>
<p>there is only one wish on my mind<br />
when this day is through i hope that i will find<br />
that tomorrow will be just the same for you and me<br />
all i need will be mine if you are here</p>
<p>☆repeat x2</p>
</li>
</ul>
</li>
<li>
<p>goose house<br />
Link: <a href="https://m.youtube.com/watch?v=3trYIk73p0k&amp;feature=youtube_gdata_player">m.youtube.com/watch</a></p>
</li>
</ul>
</li>
</ul>
<h2>00992</h2>
<h2>00892</h2>
<h2>00792</h2>
<h2>00692</h2>
<ul>
<li>villains force our evolution — if we do not overcome them, we go extinct.</li>
</ul>
<h2>00592</h2>
<ul>
<li>evolutionary biology is like the “antiques roadshow” — high on speculation, short on fact</li>
</ul>
<h2>00492</h2>
<ul>
<li>no longer appropriate to say “genetic inheritance”. more “inherited transmission” or better just “transmission”</li>
</ul>
<h2>00392</h2>
<ul>
<li>firstly it is an illusion that polymath is rarer than specialist. both are common. polymath like swarm behaviour, self–correcting. moves in manner not easily predictable using methods for individual movement.</li>
</ul>
<h2>00292</h2>
<ul>
<li>evolution is example of a method of preparation and adaptation to possible and current conditions</li>
</ul>
<h2>00192</h2>
<ul>
<li>evolution</li>
</ul>
<h2>00092</h2>
<ul>
<li>uranium U (οὐρανός ouranos, uranus)</li>
<li>eve brea
<ul>
<li>controlling a stinger</li>
<li>stinger</li>
</ul>
</li>
<li>IB</li>
</ul>

</div>

</body>
</html>
