<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
<title>12b6ddcbb4ef1e022f47434b81bdc8ca</title>
<script type="text/javascript">
function showhide(id)
{
    var e = document.getElementById(id);
    e.style.display = (e.style.display == 'block') ? 'none' : 'block';
}
</script>
<style>
rect {
transition: .6s fill;
fill: #D3D3D3;
opacity: 0;
}
rect:hover {
    fill: #D3D3D3;
    opacity: 0.2;
}

.outline {
    clear: both;
}

.svg-container {
    width: 100%;
    max-width: 2886px;
    margin-left: auto;
    margin-right: auto;
    display: block;
}

.svg-content {
    width: 100%;
}

.container {
    width: 100%;
}
</style>

</head>

<body>

<div class="container"><div class="spacer">&nbsp;</div><div class="svg-container"><svg version="1.1"  preserveAspectRatio="xMinYMin meet" class="svg-content" viewBox="0 0 2886 15428" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<image width="2886" height="15428" xlink:href="00094-12b6ddcbb4ef1e022f47434b81bdc8ca-assets/00094-12b6ddcbb4ef1e022f47434b81bdc8ca.png"></image>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05094" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05094"><rect x="1276" y="166" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05194" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05194"><rect x="1276" y="366" width="28" height="28"/></a>
<a xlink:href="http://www.pnas.org/cgi/doi/10.1073/pnas.1218525110" xlink:title="http://www.pnas.org/cgi/doi/10.1073/pnas.1218525110"><rect x="1522" y="304" width="221" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1128/mSystems.00055-18" xlink:title="http://dx.doi.org/10.1128/mSystems.00055-18"><rect x="1522" y="464" width="556" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05294" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05294"><rect x="1276" y="666" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05394" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05394"><rect x="1276" y="870" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05494" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05494"><rect x="1276" y="998" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05594" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05594"><rect x="1276" y="1384" width="28" height="28"/></a>
<a xlink:href="https://aeon.co/ideas/i-holobiont-are-you-and-your-microbes-a-community-or-a-single-entity" xlink:title="https://aeon.co/ideas/i-holobiont-are-you-and-your-microbes-a-community-or-a-single-entity"><rect x="1522" y="1160" width="576" height="33"/></a>
<a xlink:href="https://aeon.co/ideas/i-holobiont-are-you-and-your-microbes-a-community-or-a-single-entity" xlink:title="https://aeon.co/ideas/i-holobiont-are-you-and-your-microbes-a-community-or-a-single-entity"><rect x="1522" y="1194" width="531" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1371/journal.pgen.1006297" xlink:title="http://dx.doi.org/10.1371/journal.pgen.1006297"><rect x="1512" y="1668" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05694" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05694"><rect x="1276" y="1868" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/ncomms14040" xlink:title="http://dx.doi.org/10.1038/ncomms14040"><rect x="1522" y="1966" width="416" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05794" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05794"><rect x="1276" y="2128" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1186/s40168-018-0432-5" xlink:title="http://dx.doi.org/10.1186/s40168-018-0432-5"><rect x="1522" y="2162" width="552" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05894" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05894"><rect x="1276" y="2324" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1371/journal.pbio.2000633" xlink:title="http://dx.doi.org/10.1371/journal.pbio.2000633"><rect x="1522" y="2358" width="486" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05994" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/05994"><rect x="1276" y="2502" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/nature21725" xlink:title="http://dx.doi.org/10.1038/nature21725"><rect x="1522" y="2520" width="150" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06094" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06094"><rect x="1276" y="2840" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1073/pnas.1707009114" xlink:title="http://dx.doi.org/10.1073/pnas.1707009114"><rect x="1522" y="2714" width="457" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.cell.2016.11.003" xlink:title="http://dx.doi.org/10.1016/j.cell.2016.11.003"><rect x="1522" y="2876" width="214" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1098/rspb.2017.2274" xlink:title="http://dx.doi.org/10.1098/rspb.2017.2274"><rect x="1522" y="3036" width="499" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06194" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06194"><rect x="1276" y="3244" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06294" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06294"><rect x="1276" y="3346" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06394" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06394"><rect x="1276" y="3528" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1086/698021" xlink:title="http://dx.doi.org/10.1086/698021"><rect x="1926" y="3608" width="400" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06494" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06494"><rect x="1276" y="4107" width="28" height="28"/></a>
<a xlink:href="http://abcnews.go.com/Health/anxiety-head-gut/story?id=20229136" xlink:title="http://abcnews.go.com/Health/anxiety-head-gut/story?id=20229136"><rect x="1864" y="3743" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.molmet.2016.10.003" xlink:title="http://dx.doi.org/10.1016/j.molmet.2016.10.003"><rect x="1512" y="4025" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1186/s40168-017-0321-3" xlink:title="http://dx.doi.org/10.1186/s40168-017-0321-3"><rect x="1522" y="4164" width="481" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1038/s41586-018-0119-x" xlink:title="http://dx.doi.org/10.1038/s41586-018-0119-x"><rect x="1522" y="4324" width="549" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.cell.2018.07.048" xlink:title="http://dx.doi.org/10.1016/j.cell.2018.07.048"><rect x="1522" y="4484" width="521" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06594" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06594"><rect x="1276" y="4887" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.4049/jimmunol.1701755" xlink:title="http://dx.doi.org/10.4049/jimmunol.1701755"><rect x="1522" y="4762" width="526" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.3389/fncel.2017.00120" xlink:title="http://dx.doi.org/10.3389/fncel.2017.00120"><rect x="1522" y="4924" width="446" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1038/nature25973" xlink:title="http://dx.doi.org/10.1038/nature25973"><rect x="1522" y="5084" width="457" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06694" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06694"><rect x="1276" y="5501" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/s41586-018-0617-x" xlink:title="http://dx.doi.org/10.1038/s41586-018-0617-x"><rect x="1522" y="5244" width="549" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1038/s41586-018-0620-2" xlink:title="http://dx.doi.org/10.1038/s41586-018-0620-2"><rect x="1522" y="5404" width="552" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1128/mbio.01751-18" xlink:title="http://dx.doi.org/10.1128/mbio.01751-18"><rect x="1522" y="5598" width="491" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.chom.2019.01.011" xlink:title="http://dx.doi.org/10.1016/j.chom.2019.01.011"><rect x="1522" y="5794" width="549" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06794" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06794"><rect x="1276" y="5897" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06894" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06894"><rect x="1276" y="5977" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06994" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/06994"><rect x="1276" y="6195" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1080/19490976.2017.1353849" xlink:title="http://dx.doi.org/10.1080/19490976.2017.1353849"><rect x="1522" y="6150" width="547" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1038/s41598-018-20953-6" xlink:title="http://dx.doi.org/10.1038/s41598-018-20953-6"><rect x="1522" y="6312" width="568" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07094" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07094"><rect x="1276" y="6471" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.celrep.2016.12.010" xlink:title="http://dx.doi.org/10.1016/j.celrep.2016.12.010"><rect x="1522" y="6506" width="247" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07194" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07194"><rect x="1276" y="6863" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.chom.2017.11.004" xlink:title="http://dx.doi.org/10.1016/j.chom.2017.11.004"><rect x="1522" y="6702" width="549" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.chom.2017.11.003" xlink:title="http://dx.doi.org/10.1016/j.chom.2017.11.003"><rect x="1522" y="6898" width="549" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1503/cmaj.170809" xlink:title="http://dx.doi.org/10.1503/cmaj.170809"><rect x="1522" y="7094" width="463" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07294" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07294"><rect x="1276" y="7255" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/s41598-017-16606-9" xlink:title="http://dx.doi.org/10.1038/s41598-017-16606-9"><rect x="1522" y="7290" width="568" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07394" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07394"><rect x="1276" y="7393" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07494" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07494"><rect x="1276" y="7791" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.cell.2018.10.029" xlink:title="http://dx.doi.org/10.1016/j.cell.2018.10.029"><rect x="1522" y="7616" width="521" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1126/science.aan4834" xlink:title="http://dx.doi.org/10.1126/science.aan4834"><rect x="1522" y="7776" width="439" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1103/PhysRevLett.121.098101" xlink:title="http://dx.doi.org/10.1103/PhysRevLett.121.098101"><rect x="1522" y="7970" width="564" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1103/PhysRevLett.121.098101" xlink:title="http://dx.doi.org/10.1103/PhysRevLett.121.098101"><rect x="1522" y="8004" width="147" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07594" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07594"><rect x="1276" y="8244" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1128/mSphere.00327-17" xlink:title="http://dx.doi.org/10.1128/mSphere.00327-17"><rect x="1522" y="8164" width="469" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1128/mSphere.00193-18" xlink:title="http://dx.doi.org/10.1128/mSphere.00193-18"><rect x="1522" y="8358" width="540" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07694" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07694"><rect x="1276" y="8716" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.celrep.2015.03.049" xlink:title="http://dx.doi.org/10.1016/j.celrep.2015.03.049"><rect x="1522" y="8554" width="247" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1038/s41598-018-29032-2" xlink:title="http://dx.doi.org/10.1038/s41598-018-29032-2"><rect x="1522" y="8750" width="568" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.3389/fmicb.2018.02323" xlink:title="http://dx.doi.org/10.3389/fmicb.2018.02323"><rect x="1522" y="8946" width="526" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07794" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07794"><rect x="1276" y="9668" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/s41598-017-17487-8" xlink:title="http://dx.doi.org/10.1038/s41598-017-17487-8"><rect x="1522" y="9176" width="568" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1093/jn/nxy004" xlink:title="http://dx.doi.org/10.1093/jn/nxy004"><rect x="1522" y="9404" width="421" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1186/s40168-018-0559-4" xlink:title="http://dx.doi.org/10.1186/s40168-018-0559-4"><rect x="1522" y="9564" width="552" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.cels.2018.08.003" xlink:title="http://dx.doi.org/10.1016/j.cels.2018.08.003"><rect x="1522" y="9724" width="530" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.cell.2018.10.020" xlink:title="http://dx.doi.org/10.1016/j.cell.2018.10.020"><rect x="1522" y="9884" width="521" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1038/s41467-018-07275-x" xlink:title="http://dx.doi.org/10.1038/s41467-018-07275-x"><rect x="1522" y="10044" width="565" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.cell.2018.09.055" xlink:title="http://dx.doi.org/10.1016/j.cell.2018.09.055"><rect x="1522" y="10264" width="521" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07894" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07894"><rect x="1276" y="10450" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07994" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/07994"><rect x="1276" y="10530" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08094" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08094"><rect x="1276" y="10610" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08194" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08194"><rect x="1276" y="10690" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08294" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08294"><rect x="1276" y="10770" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08394" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08394"><rect x="1276" y="10850" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08494" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08494"><rect x="1276" y="10930" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08594" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08594"><rect x="1276" y="11010" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08694" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08694"><rect x="1276" y="11090" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08794" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08794"><rect x="1276" y="11170" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08894" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08894"><rect x="1276" y="11250" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08994" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/08994"><rect x="1276" y="11330" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09094" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09094"><rect x="1276" y="11648" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1111/joim.12805" xlink:title="http://dx.doi.org/10.1111/joim.12805"><rect x="1522" y="11620" width="439" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1172/jci.insight.95997" xlink:title="http://dx.doi.org/10.1172/jci.insight.95997"><rect x="1522" y="11780" width="498" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09194" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09194"><rect x="1276" y="12108" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.tins.2016.09.002" xlink:title="http://dx.doi.org/10.1016/j.tins.2016.09.002"><rect x="1512" y="11984" width="28" height="28"/></a>
<a xlink:href="https://www.sciencedaily.com/releases/2017/04/170403123508.htm" xlink:title="https://www.sciencedaily.com/releases/2017/04/170403123508.htm"><rect x="1512" y="12334" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09294" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09294"><rect x="1276" y="12568" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1038/s41586-018-0092-4" xlink:title="http://dx.doi.org/10.1038/s41586-018-0092-4"><rect x="1522" y="12472" width="552" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.cell.2018.07.016" xlink:title="http://dx.doi.org/10.1016/j.cell.2018.07.016"><rect x="1522" y="12700" width="521" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09394" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09394"><rect x="1276" y="12970" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1186/s12916-016-0698-z" xlink:title="http://dx.doi.org/10.1186/s12916-016-0698-z"><rect x="1512" y="13054" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09494" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09494"><rect x="1276" y="13208" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1001/jama.2017.17077" xlink:title="http://dx.doi.org/10.1001/jama.2017.17077"><rect x="1522" y="13260" width="517" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09594" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09594"><rect x="1276" y="13420" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.watres.2018.03.032" xlink:title="http://dx.doi.org/10.1016/j.watres.2018.03.032"><rect x="1522" y="13454" width="559" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09694" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09694"><rect x="1276" y="13780" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1073/pnas.1613843113" xlink:title="http://dx.doi.org/10.1073/pnas.1613843113"><rect x="1522" y="13734" width="528" height="33"/></a>
<a xlink:href="http://dx.doi.org/10.1038/s41396-017-0042-4" xlink:title="http://dx.doi.org/10.1038/s41396-017-0042-4"><rect x="1522" y="13896" width="552" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09794" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09794"><rect x="1276" y="14038" width="28" height="28"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.chom.2018.07.006" xlink:title="http://dx.doi.org/10.1016/j.chom.2018.07.006"><rect x="1522" y="14056" width="549" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09894" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09894"><rect x="1276" y="14726" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09994" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/09994"><rect x="1276" y="15374" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04994" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04994"><rect x="830" y="9808" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04894" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04894"><rect x="830" y="9704" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04794" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04794"><rect x="830" y="9624" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04694" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04694"><rect x="830" y="9504" width="28" height="28"/></a>
<a xlink:href="https://doi.org/10.1016/j.cels.2018.06.005" xlink:title="https://doi.org/10.1016/j.cels.2018.06.005"><rect x="100" y="9522" width="508" height="33"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04594" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04594"><rect x="830" y="9384" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04494" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04494"><rect x="830" y="9304" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04394" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04394"><rect x="830" y="9224" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04294" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04294"><rect x="830" y="9144" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04194" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04194"><rect x="830" y="9064" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04094" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/04094"><rect x="830" y="8984" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03994" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03994"><rect x="830" y="8904" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03894" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03894"><rect x="830" y="8824" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03794" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03794"><rect x="830" y="8744" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03694" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03694"><rect x="830" y="8664" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03594" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03594"><rect x="830" y="8584" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03494" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03494"><rect x="830" y="8504" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03394" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03394"><rect x="830" y="8424" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03294" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03294"><rect x="830" y="8344" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03194" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03194"><rect x="830" y="8264" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03094" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/03094"><rect x="830" y="8184" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02994" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02994"><rect x="830" y="8104" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02894" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02894"><rect x="830" y="8024" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02794" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02794"><rect x="830" y="7944" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02694" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02694"><rect x="830" y="7864" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02594" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02594"><rect x="830" y="7784" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02494" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02494"><rect x="830" y="7704" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02394" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02394"><rect x="830" y="7624" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02294" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02294"><rect x="830" y="7544" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02194" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02194"><rect x="830" y="7464" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02094" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/02094"><rect x="830" y="7384" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01994" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01994"><rect x="830" y="7304" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01894" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01894"><rect x="830" y="7224" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01794" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01794"><rect x="830" y="7144" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01694" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01694"><rect x="830" y="7064" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01594" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01594"><rect x="830" y="6984" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01494" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01494"><rect x="830" y="6904" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01394" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01394"><rect x="830" y="6824" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01294" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01294"><rect x="830" y="6744" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01194" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01194"><rect x="830" y="6664" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01094" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/01094"><rect x="830" y="6584" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00994" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00994"><rect x="830" y="6504" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00894" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00894"><rect x="830" y="6424" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00794" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00794"><rect x="830" y="6344" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00694" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00694"><rect x="830" y="6264" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00594" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00594"><rect x="830" y="6184" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00494" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00494"><rect x="830" y="6104" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00394" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00394"><rect x="830" y="6024" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00294" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00294"><rect x="830" y="5944" width="28" height="28"/></a>
<a xlink:href="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00194" xlink:title="file:///var/mobile/Library/Mobile%20Documents/iCloud~com~toketaware~ios~ithoughts/Documents/00194"><rect x="830" y="5864" width="28" height="28"/></a>
<a xlink:href="00094-12b6ddcbb4ef1e022f47434b81bdc8ca.html" xlink:title="00094-12b6ddcbb4ef1e022f47434b81bdc8ca.html"><rect x="830" y="5704" width="28" height="28"/></a>
</svg>
</div>
</div>

<br/>
<a href="javascript:showhide('outlineDiv')">Outline show/hide</a>
<div id="outlineDiv" style="display:none;">
<h1>00094</h1>
<p>le petit prince</p>
<h2>microbiome</h2>
<h2>05094</h2>
<h2>05194</h2>
<ul>
<li>
<p><em>animals in a bacterial world, a new imperative for the life sciences</em><br />
mcfall–ngai et al 2013<br />
<a href="http://www.pnas.org/cgi/doi/10.1073/pnas.1218525110">pnas.1218525110</a></p>
<ul>
<li>In the last two decades, the widespread application of genetic and genomic approaches has revealed a bacterial world astonishing in its ubiquity and diversity. This review examines how a growing knowledge of the vast range of animal–bacterial interactions, whether in shared ecosystems or intimate symbioses, is fundamentally altering our understanding of animal biology. Specifically, we highlight recent techno-logical and intellectual advances that have changed our thinking about five questions: how have bacteria facilitated the origin and evolution of animals; how do animals and bacteria affect each other's genomes; how does normal animal development depend on bacterial partners; how is homeostasis maintained between animals and their symbionts; and how can ecological approaches deepen our understanding of the multiple levels of animal–bacterial interaction. As answers to these fundamental questions emerge, all biologists will be challenged to broaden their appreciation of these interactions and to include investigations of the relationships between and among bacteria and their animal partners as we seek a better understanding of the natural world.</li>
<li>the New Synthesis of the 1930s and beyond must be reconsidered in terms of three areas in which it has proven weakest: symbiosis, de-velopment, and microbiology (115). One of these areas, microbiology, presents particular challenges both to the species concept, as formulated by Ernst Mayr in 1942, and to the concept that vertical transmission of genetic information is the only motor of selectable evolutionary change.</li>
</ul>
</li>
<li>
<p><em>phylogenetically novel uncultured microbial cells dominate earth microbiomes</em><br />
karen g. lloyd et al. 2018<br />
<a href="http://dx.doi.org/10.1128/mSystems.00055-18">http://dx.doi.org/10.1128/mSystems.00055-18</a></p>
<ul>
<li>To describe a microbe’s physiology, including its metabolism, environmental roles, and growth characteristics, it must be grown in a laboratory culture. Unfortunately, many phylogenetically novel groups have never been cultured, so their physiologies have only been inferred from genomics and environmental characteristics. Although the diversity, or number of different taxonomic groups, of uncultured clades has been studied well, their global abundances, or numbers of cells in any given environment, have not been assessed. We quantified the degree of similarity of 16S rRNA gene sequences from diverse environments in publicly available metagenome and metatranscriptome databases, which we show have far less of the culture bias present in primer-amplified 16S rRNA gene surveys, to those of their nearest cultured relatives. Whether normalized to scaffold read depths or not, the highest abundances of metagenomic 16S rRNA gene sequences belong to phylogenetically novel uncultured groups in seawater, freshwater, terrestrial subsurface, soil, hypersaline environments, marine sediment, hot springs, hydrothermal vents, nonhuman hosts, snow, and bioreactors (22% to 87% uncultured genera to classes and 0% to 64% uncultured phyla). The exceptions were human and human-associated environments, which were dominated by cultured genera (45% to 97%). We estimate that uncultured genera and phyla could comprise 7.3 × 1029 (81%) and 2.2 × 1029 (25%) of microbial cells, respectively. Uncultured phyla were overrepresented in metatranscriptomes relative to metagenomes (46% to 84% of sequences in a given environment), suggesting that they are viable. Therefore, uncultured microbes, often from deeply phylogenetically divergent groups, dominate nonhuman environments on Earth, and their undiscovered physiologies may matter for Earth systems.</li>
</ul>
</li>
</ul>
<h2>05294</h2>
<ul>
<li>
<p><em>i, superorganism: learning to love your inner ecosystem</em><br />
jon turney 2015 9781848318229</p>
<ul>
<li>diverse ecosystems but similar collective metabolism</li>
<li>has full story of helicobacter pylori</li>
</ul>
</li>
<li>
<p><em>gut: the inside story of the body’s most under–rated organ</em><br />
giulia enders 2015 9781771641500</p>
</li>
</ul>
<h2>05394</h2>
<ul>
<li><em>i contain multitudes: the microbes within us and a grander view of life</em><br />
ed jong 2016 9780062368621</li>
</ul>
<h2>05494</h2>
<ul>
<li><em>missing microbes: how the overuse of antibiotics is fuelling our modern plagues</em><br />
martin blaser 2014 978-1250069276</li>
</ul>
<h2>05594</h2>
<ul>
<li>
<p><em>i, holobiont. are you and your microbes a community or a single entity?</em><br />
derek skillings 2018<br />
<a href="https://aeon.co/ideas/i-holobiont-are-you-and-your-microbes-a-community-or-a-single-entity">https://aeon.co/ideas/i-holobiont-are-you-and-your-microbes-a-community-or-a-single-entity</a></p>
<p>“natural selection doesn’t just act on the genome of individual organisms: it acts on the hologenome of holobionts, which are seen as single units that can evolve at the level of the holobiont.”</p>
</li>
<li>
<p><em>drosophila adaptation to viral infection through defensive symbiont evolution</em><br />
vitor g. faria, nelson e. martins, sara magalhães, tânia f. paulo, viola nolte, christian schlötterer, élio sucena, luis teixeira 2016<br />
http://dx.doi.org/10.1371/journal.pgen.1006297<br />
Link: <a href="http://dx.doi.org/10.1371/journal.pgen.1006297">dx.doi.org/10.1371/journal.pgen.1006297</a></p>
<ul>
<li>
<p>Microbial symbionts can modulate host interactions with biotic and abiotic factors. Such interactions may affect the evolutionary trajectories of both host and symbiont. Wolbachia protects Drosophila melanogaster against several viral infections and the strength of the protection varies between variants of this endosymbiont. Since Wolbachia is maternally transmitted, its fitness depends on the fitness of its host. Therefore, Wolbachia populations may be under selection when Drosophila is subjected to viral infection. Here we show that in D. melanogaster populations selected for increased survival upon infection with Drosophila C virus there is a strong selection coefficient for specific Wolbachia variants, leading to their fixation. Flies carrying these selected Wolbachia variants have higher survival and fertility upon viral infection when compared to flies with the other variants. These findings demonstrate how the interaction of a host with pathogens shapes the genetic composition of symbiont populations. Furthermore, host adaptation can result from the evolution of its symbionts, with host and symbiont functioning as a single evolutionary unit.</p>
<p>Author Summary</p>
<p>Animals live in close association with microbial partners that can shape many aspects of their lives. For instance, several insects carry bacteria that defend them against parasites and infectious diseases. The intracellular bacterium Wolbachia protects the fruit fly Drosophila melanogaster against viral infection. Natural populations of Drosophila carry different variants of Wolbachia, which differ from one another in the strength of this protection. Here we show that a population of Drosophila infected with viruses during several generations adapts to this challenge through turnover in Wolbachia composition. The Wolbachia variants that give higher protection to viruses, by increasing fly survival and fecundity upon infection, are strongly selected. This work demonstrates that the interaction of an animal with a pathogen can shape its associated microbial populations. We show that adaptation to pathogens can be achieved not only through selection of resistance on the host proper but also through the evolutionary shaping of its microbial community.</p>
</li>
</ul>
</li>
</ul>
<h2>05694</h2>
<ul>
<li>
<p><em>brain maker: the power of gut microbes to heal and protect your brain — for life</em><br />
david perlmutter 2015</p>
<ul>
<li>more controversial</li>
</ul>
</li>
<li>
<p><em>microbes can help explain the evolution of host altruism</em><br />
ohad lewin-epstein, ranit aharonov &amp; lilach hadany 2017<br />
<a href="http://dx.doi.org/10.1038/ncomms14040">dx.doi.org/10.1038/ncomms14040</a></p>
<ul>
<li>The evolution of altruistic behaviour, which is costly to the donor but beneficial for the recipient, is among the most intriguing questions in evolutionary biology. Several theories have been proposed to explain it, including kin selection, group selection and reciprocity. Here we propose that microbes that manipulate their hosts to act altruistically could be favoured by selection, and may play a role in the widespread occurrence of altruism. Using computational models, we find that microbe-induced altruism can explain the evolution of host altruistic behaviour under wider conditions than host-centred theories, including in a fully mixed host population, without repeating interactions or individual recognition. Our results suggest that factors such as antibiotics that kill microbes might negatively affect cooperation in a wide range of organisms.</li>
</ul>
</li>
</ul>
<h2>05794</h2>
<ul>
<li>
<p><em>trees, fungi and bacteria: tripartite metatranscriptomics of a root microbiome responding to soil contamination</em><br />
e. gonzalez et al. 2018<br />
<a href="http://dx.doi.org/10.1186/s40168-018-0432-5">http://dx.doi.org/10.1186/s40168-018-0432-5</a></p>
<ul>
<li>From gene expression alone, the major responses of <em>Salix purpurea</em> cv. ‘Fish Creek’ to soil contamination do not seem to be direct degradation, immobilisation or exclusion of contaminants but rather widespread, extraordinarily complex, alterations to interactions with microbiota. The extremely diverse fungal community, revealed using a bioinformatics approach unconstrained by the requirement of <em>a priori</em> nucleotide sequence, responds dynamically to contamination both with changes to gene expression but also with substantial shifts in the community makeup. Much like the host plant, fungal gene expression was dominated by alterations in pathways associated to microbiome interactions as opposed to direct hydrocarbon degradation. Surprisingly, gene expression representing increased petroleum hydrocarbon metabolism equipment came from an <em>Enterobacteriaceae</em> species whose total expression was uniformly more abundant in roots of contaminated trees. It seems possible that, given the successful contamination tolerance, willows may depend on symbiosis to tolerate stressful environmental conditions instead of relying solely on their own metabolism.<br />
While polyadenylated bacterial sequences are often not quantified across experimental systems investigated with eukaryote expression in mind, the bacterial functionality observed here is thought-provoking, revealing a plausible set of expressed genes indicating tripartite mutualism. A finding which is useful in providing a convincing reminder that observing the entirety of generated data, even that which is commonly discarded, is always of potential value for exposing the unexpected. Most importantly, with regard to the attempted all-inclusive strategy of observing all expressed sequences within a biological sample, regardless of origin, is how extensively interpretation of expression could be confounded if plant, fungal or bacterial expression were investigated as responsive to treatment (here being contamination) in isolation, and thus how crucial it is to attempt observation of the entire microbiome and wider metaorganism. This metatranscriptomic approach, which marries an understanding of the uncertainty of microbiome biology with the strengths and limitations of current bioinformatics, should be broadly transferable through utility for biological samples containing expressed sequences from well-characterised, poorly characterised or entirely unknown organisms at a diversity of low or high complexity.</li>
</ul>
</li>
</ul>
<h2>05894</h2>
<ul>
<li>
<p><em>stochastic assembly produces heterogeneous communities in the caenorhabditis elegans intestine</em><br />
nicole m. vega, jeff gore 2017<br />
<a href="http://dx.doi.org/10.1371/journal.pbio.2000633">dx.doi.org/10.1371/journal.pbio.2000633</a></p>
<ul>
<li>
<p>Host-associated bacterial communities vary extensively between individuals, but it can be very difficult to determine the sources of this heterogeneity. Here, we demonstrate that stochastic bacterial community assembly in the <em>Caenorhabditis elegans</em> intestine is sufficient to produce strong interworm heterogeneity in community composition. When worms are fed with two neutrally competing, fluorescently labeled bacterial strains, we observe stochastically driven bimodality in community composition, in which approximately half of the worms are dominated by each bacterial strain. A simple model incorporating stochastic colonization suggests that heterogeneity between worms is driven by the low rate at which bacteria successfully establish new intestinal colonies. We can increase this rate experimentally by feeding worms at high bacterial density; in these conditions, the bimodality disappears. These results demonstrate that demographic noise is a potentially important driver of diversity in bacterial community formation and suggest a role for <em>C</em>. <em>elegans</em> as a model system for ecology of host-associated communities.<br />
Author summary<br />
Host-associated bacterial communities—also known as microbiomes—vary extensively between individuals, even among clones exposed to the same environment. The sources of this variation are not entirely understood and can be very difficult to determine. In this manuscript, we demonstrate experimentally how randomness in bacterial colonization can result in large differences in the composition of host-associated bacterial communities, using the nematode worm <em>Caenorhabditis elegans</em> as a tractable host model. We find that the amount of variation between individual communities is a function of two rates relevant to how bacteria colonize the host intestine: the colonization rate and the birth rate. We can manipulate the degree of variation between communities by altering the colonization rate, using the amount of bacteria presented to the worms to control the rate at which migrants enter the intestine. When worms are fed with two neutrally competing, fluorescently labeled bacterial strains at low colonization rates, we are able to produce noise-induced bistability in this system, in which each community is dominated by bacteria of only one color. These results demonstrate the potential importance of noise and randomness as a driver of variation between communities and highlight the utility of the simple model organism <em>C</em>. <em>elegans</em> for studying questions relevant to host-associated microbial communities.</p>
</li>
<li>
<p>system that is stable without diffusion becomes unstable in the presence of diffusion. Turing was motivated to understand morphogenesis with this example of instability</p>
</li>
</ul>
</li>
</ul>
<h2>05994</h2>
<ul>
<li>
<p><em>complex pectin metabolism by gut bacteria reveals novel catalytic functions</em><br />
didier ndeh et al. 2017<br />
<a href="http://dx.doi.org/10.1038/nature21725">nature21725</a></p>
<ul>
<li>
<p>single microorganisms in the human gut have the ability to disassemble the most complex of carbohydrates in our diet.<br />
It is the first time such a discovery has been made and it is hoped that this may be used to one day identify new pre- and pro-biotic products to enhance people’s health.<br />
Led by Professor Harry Gilbert, from the Institute for Cell and Molecular Biosciences at Newcastle University, UK, the study is published today (Wednesday) in the leading academic journal, <em>Nature</em>.<br />
Bacteria in the large bowel — the human gut — has a major impact on health and physiology as they help to disintegrate substances in food that we cannot digest, such as starches and fibre.<br />
The main source of nutrients available to the gut bacteria are carbohydrates from the human diet, which the body is unable to metabolise.<br />
The most complex of these carbohydrates is the plant polysaccharide, ‘rhamnogalacturonan II (RG-II)’, which can also be found at elevated levels in red wine.<br />
Previously it was thought that only groups of bacteria would be able to metabolise and breakdown RG-II, reflecting its complex structure. However, this research shows that single organisms present in the gut also have the ability to do this.<br />
Professor Gilbert said: “Our research reports how a highly complex biological process in the body is achieved.<br />
“This is an exciting step forward in the understanding of how human gut bacteria work and has implications for future research.”<br />
The team of international scientists found that RG-II is metabolised through the action of a type of bacterial enzyme, known as glycoside hydrolases, which target the complex carbohydrates sugars in the large bowel.<br />
The bacteria that can metabolise RG-II contain several genes that encode proteins that previously had no known action until now. The group have shown that seven of these genes produce glycoside hydrolases — which split the glycosidic linkage that joins sugars together in polysaccharides — and contribute to the breakdown of RG-II.<br />
Each of these seven glycoside hydrolases are founding members of a novel enzyme family. Three of the glycoside hydrolases that contribute to RG-II degradation break glycosidic linkages that have not previously been shown to be susceptible to biological attack, and these enzymes display novel catalytic functions.<br />
Professor Gilbert said: “This study has potential applications as understanding how this highly complex carbohydrate, which is an integral component of our diet, is utilised offers opportunities for developing new pre- and pro-biotic strategies to improve human health.</p>
</li>
<li>
<p><em>abstract</em> The metabolism of carbohydrate polymers drives microbial diversity in the human gut microbiota. It is unclear, however, whether bacterial consortia or single organisms are required to depolymerize highly complex glycans. Here we show that the gut bacterium <em>Bacteroides thetaiotaomicron</em> uses the most structurally complex glycan known: the plant pectic polysaccharide rhamnogalacturonan-II, cleaving all but 1 of its 21 distinct glycosidic linkages. The deconstruction of rhamnogalacturonan-II side chains and backbone are coordinated to overcome steric constraints, and the degradation involves previously undiscovered enzyme families and catalytic activities. The degradation system informs revision of the current structural model of rhamnogalacturonan-II and highlights how individual gut bacteria orchestrate manifold enzymes to metabolize the most challenging glycan in the human diet.</p>
</li>
</ul>
</li>
</ul>
<h2>06094</h2>
<ul>
<li>
<p><em>numerous uncharacterized and highly divergent microbes which colonize humans are revealed by circulating cell-free dna</em><br />
mark kowarsky et al. 2017<br />
<a href="http://dx.doi.org/10.1073/pnas.1707009114">dx.doi.org/10.1073/pnas.1707009114</a></p>
<ul>
<li>
<p>Through massive shotgun sequencing of circulating cell-free DNA from the blood of more than 1,000 independent samples, we identified hundreds of new bacteria and viruses which represent previously unidentified members of the human microbiome. Previous studies targeted specific niches such as feces, skin, or the oral cavity, whereas our approach of using blood effectively enables sampling of the entire body and reveals the colonization of niches which have been previously inaccessible. We were thus able to discover that the human body contains a vast and unexpected diversity of microbes, many of which have highly divergent relationships to the known tree of life.</p>
<p>Blood circulates throughout the human body and contains molecules drawn from virtually every tissue, including the microbes and viruses which colonize the body. Through massive shotgun sequencing of circulating cell-free DNA from the blood, we identified hundreds of new bacteria and viruses which represent previously unidentified members of the human microbiome. Analyzing cumulative sequence data from 1,351 blood samples collected from 188 patients enabled us to assemble 7,190 contiguous regions (contigs) larger than 1 kbp, of which 3,761 are novel with little or no sequence homology in any existing databases. The vast majority of these novel contigs possess coding sequences, and we have validated their existence both by finding their presence in independent experiments and by performing direct PCR amplification. When their nearest neighbors are located in the tree of life, many of the organisms represent entirely novel taxa, showing that microbial diversity within the human body is substantially broader than previously appreciated.</p>
</li>
</ul>
</li>
<li>
<p><em>microbiota diurnal rhythmicity programs host transcriptome oscillations</em><br />
christoph a. thaiss et al. 2016<br />
<a href="http://dx.doi.org/10.1016/j.cell.2016.11.003">j.cell.2016.11.003</a></p>
<ul>
<li>
<p>Intestinal microbiota biogeography and metabolome undergo diurnal oscillations</p>
<p>Circadian oscillations of serum metabolites are regulated by the microbiota</p>
<p>Microbiota rhythms program the circadian epigenetic and transcriptional landscape</p>
<p>The microbiota regulates the circadian liver transcriptome and detoxification pattern</p>
<p>The intestinal microbiota undergoes diurnal compositional and functional oscillations that affect metabolic homeostasis, but the mechanisms by which the rhythmic microbiota influences host circadian activity remain elusive. Using integrated multi-omics and imaging approaches, we demonstrate that the gut microbiota features oscillating biogeographical localization and metabolome patterns that determine the rhythmic exposure of the intestinal epithelium to different bacterial species and their metabolites over the course of a day. This diurnal microbial behavior drives, in turn, the global programming of the host circadian transcriptional, epigenetic, and metabolite oscillations. Surprisingly, disruption of homeostatic microbiome rhythmicity not only abrogates normal chromatin and transcriptional oscillations of the host, but also incites genome-wide de novo oscillations in both intestine and liver, thereby impacting diurnal fluctuations of host physiology and disease susceptibility. As such, the rhythmic biogeography and metabolome of the intestinal microbiota regulates the temporal organization and functional outcome of host transcriptional and epigenetic programs.</p>
</li>
</ul>
</li>
<li>
<p><em>hierarchical social networks shape gut microbial composition in wild verreaux’s sifaka</em><br />
amanda c. perofsky et al. 2017<br />
<a href="http://dx.doi.org/10.1098/rspb.2017.2274">http://dx.doi.org/10.1098/rspb.2017.2274</a></p>
<ul>
<li>In wild primates, social behaviour influences exposure to environmentally acquired and directly transmitted microorganisms. Prior studies indicate that gut microbiota reflect pairwise social interactions among chimpanzee and baboon hosts. Here, we demonstrate that higher-order social network structure—beyond just pairwise interactions—drives gut bacterial composition in wild lemurs, which live in smaller and more cohesive groups than previously studied anthropoid species. Using 16S rRNA gene sequencing and social network analysis of grooming contacts, we estimate the relative impacts of hierarchical (i.e. multilevel) social structure, individual demographic traits, diet, scent-marking, and habitat overlap on bacteria acquisition in a wild population of Verreaux’s sifaka (<em>Propithecus verreauxi</em>) consisting of seven social groups. We show that social group membership is clearly reflected in the microbiomes of individual sifaka, and that social groups with denser grooming networks have more homogeneous gut microbial compositions. Within social groups, adults, more gregarious individuals, and individuals that scent-mark frequently harbour the greatest microbial diversity. Thus, the community structure of wild lemurs governs symbiotic relationships by constraining transmission between hosts and partitioning environmental exposure to microorganisms. This social cultivation of mutualistic gut flora may be an evolutionary benefit of tight-knit group living.</li>
</ul>
</li>
</ul>
<h2>homeostasis</h2>
<h2>06194</h2>
<ul>
<li>communication between microbiome and host
<ul>
<li>
<p><em>microbiota derived short chain fatty acids promote histone crotonylation in the colon through histone deacetylases</em><br />
rachel fellows et al. 2017<br />
<a href="http://dx.doi.org/10.1038/s41467-017-02651-5">http://dx.doi.org/10.1038/s41467-017-02651-5</a></p>
<ul>
<li>The recently discovered histone post-translational modification crotonylation connects cellular metabolism to gene regulation. Its regulation and tissue-specific functions are poorly understood. We characterize histone crotonylation in intestinal epithelia and find that histone H3 crotonylation at lysine 18 is a surprisingly abundant modification in the small intestine crypt and colon, and is linked to gene regulation. We show that this modification is highly dynamic and regulated during the cell cycle. We identify class I histone deacetylases, HDAC1, HDAC2, and HDAC3, as major executors of histone decrotonylation. We show that known HDAC inhibitors, including the gut microbiota-derived butyrate, affect histone decrotonylation. Consistent with this, we find that depletion of the gut microbiota leads to a global change in histone crotonylation in the colon. Our results suggest that histone crotonylation connects chromatin to the gut microbiota, at least in part, via short-chain fatty acids and HDACs.</li>
</ul>
</li>
<li>
<p>…</p>
</li>
<li>
<p><em>gut microbiota orchestrates energy homeostasis during cold</em><br />
chevalier et al 2015<br />
http://dx.doi.org/10.1016/j.cell.2015.11.004<br />
Link: <a href="http://dx.doi.org/10.1016/j.cell.2015.11.004">dx.doi.org/10.1016/j.cell.2015.11.004</a></p>
<ul>
<li>Cold exposure markedly shifts the composition of the gut microbiota. This ‘‘cold microbiota’’ mediates remodeling of the fat and intestinal tissues, helping the host to withstand periods of high energy demand.</li>
<li>Cold exposure leads to marked changes in the gut microbiota composition</li>
<li>Cold microbiota transplantation increases insulin sensitivity and WAT browning</li>
<li>Cold exposure or cold transplantation increase the gut size and absorptive capacity</li>
<li>Reconstitution of cold-suppressed A. muciniphila reverts the increased caloric uptake</li>
</ul>
</li>
<li>
<p><em>a gut microbial factor modulates locomotor behaviour in drosophila</em><br />
catherine e. schretter et al. 2018<br />
<a href="http://dx.doi.org/10.1038/s41586-018-0634-9">http://dx.doi.org/10.1038/s41586-018-0634-9</a></p>
<ul>
<li>
<p>“This study provides additional evidence for a connection between the gut and the brain, and in particular outlines how gut bacteria may influence behavior, including movement,” said Margaret Sutherland, Ph.D., program director at NINDS.<br />
Researchers led by Sarkis K. Mazmanian, Ph.D., professor of microbiology at the California Institute of Technology in Pasadena, and graduate student Catherine E. Schretter, observed that germ-free flies, which did not carry bacteria, were hyperactive. For instance, they walked faster, over greater distances, and took shorter rests than flies that had normal levels of microbes. Dr. Mazmanian and his team investigated ways in which gut bacteria may affect behavior in fruit flies.<br />
“Locomotion is important for a number of activities such as mating and searching for food. It turns out that gut bacteria may be critical for fundamental behaviors in animals,” said Dr. Mazmanian.<br />
Fruit flies carry between five and 20 different species of bacteria and Dr. Mazmanian’s team treated the germ-free animals with individual strains of those microbes. When the flies received Lactobacillus brevis, their movements slowed down to normal speed. L. brevis was one of only two species of bacteria that restored normal behavior in the germ-free flies.<br />
Dr. Mazmanian’s group also discovered that the molecule xylose isomerase (Xi), a protein that breaks down sugar and is found in L. brevis, may be critical to this process. Isolating the molecule and treating germ-free flies with it was sufficient to slow down the speedwalkers.<br />
Additional experiments showed that Xi may regulate movement by fine-tuning levels of certain carbohydrates, such as trehalose, which is the main sugar found in flies and is similar to mammalian glucose. Flies that were given Xi had lower levels of trehalose than did untreated germ-free flies. When Xi-treated flies, which showed normal behavior, were given trehalose alone, they resumed fast movements suggesting that the sugar was able to reverse the effects of Xi.<br />
Next, the researchers looked into the flies’ nervous system to see what cells were involved in bacteria-directed movement. When Dr. Mazmanian’s team turned on neurons that produce the chemical octopamine, that activation canceled out the effect of L. brevis on the germ-free flies. As a result, the flies, which had previously slowed down after receiving the bacterium or Xi, resumed their speedwalking behavior. Turning on octopamine-producing nerve cells in flies with normal levels of bacteria also caused them to move faster. However, activating neurons that produce other brain chemicals did not influence the flies’ movements.<br />
According to Dr. Mazmanian, Schretter and their colleagues, Xi may be monitoring the flies’ metabolic state, including levels of nutrients, and then signaling to octopamine neurons whether they should turn on or off, resulting in changes in behavior.<br />
Instead of octopamine, mammals produce a comparable chemical called noradrenaline, which has been shown to control movement.<br />
“Gut bacteria may play a similar role in mammalian locomotion, and even in movement disorders such as Parkinson’s disease,” said Dr. Mazmanian.</p>
</li>
<li>
<p><em>abstract</em> While research into the biology of animal behaviour has primarily focused on the central nervous system, cues from peripheral tissues and the environment have been implicated in brain development and function<a href="https://www.nature.com/articles/s41586-018-0634-9#ref-CR1">9</a>. There is emerging evidence that bidirectional communication between the gut and the brain affects behaviours including anxiety, cognition, nociception and social interaction1,2,3,4,5,6,7,8,<a href="https://www.nature.com/articles/s41586-018-0634-9#ref-CR9">9</a>. Coordinated locomotor behaviour is critical for the survival and propagation of animals, and is regulated by internal and external sensory inputs<a href="https://www.nature.com/articles/s41586-018-0634-9#ref-CR10">10</a>,<a href="https://www.nature.com/articles/s41586-018-0634-9#ref-CR11">11</a>. However, little is known about how the gut microbiome influences host locomotion, or the molecular and cellular mechanisms involved. Here we report that germ-free status or antibiotic treatment results in hyperactive locomotor behaviour in the fruit fly Drosophila melanogaster. Increased walking speed and daily activity in the absence of a gut microbiome are rescued by mono-colonization with specific bacteria, including the fly commensal Lactobacillus brevis. The bacterial enzyme xylose isomerase from L. brevis recapitulates the locomotor effects of microbial colonization by modulating sugar metabolism in flies. Notably, thermogenetic activation of octopaminergic neurons or exogenous administration of octopamine, the invertebrate counterpart of noradrenaline, abrogates the effects of xylose isomerase on Drosophila locomotion. These findings reveal a previously unappreciated role for the gut microbiome in modulating locomotion, and identify octopaminergic neurons as mediators of peripheral microbial cues that regulate motor behaviour in animals.</p>
</li>
</ul>
</li>
<li>
<p><em>the drosophila immune deficiency pathway modulates enteroendocrine function and host metabolism</em><br />
layla kamareddine et al. 2018<br />
<a href="http://dx.doi.org/10.1016/j.cmet.2018.05.026">http://dx.doi.org/10.1016/j.cmet.2018.05.026</a></p>
<ul>
<li>
<p>•The innate immune deficiency (IMD) pathway regulates tachykinin transcription<br />
•Microbial activation of enteroendocrine IMD signaling mediates metabolic homeostasis<br />
•Acetate restores IMD pathway activation and metabolic homeostasis in axenic flies<br />
•Acetate signaling through IMD depends on the membrane-associated receptor PGRP-LC</p>
<p>Enteroendocrine cells (EEs) are interspersed between enterocytes and stem cells in the Drosophila intestinal epithelium. Like enterocytes, EEs express components of the immune deficiency (IMD) innate immune pathway, which activates transcription of genes encoding antimicrobial peptides. The discovery of large lipid droplets in intestines of IMD pathway mutants prompted us to investigate the role of the IMD pathway in the host metabolic response to its intestinal microbiota. Here we provide evidence that the short-chain fatty acid acetate is a microbial metabolic signal that activates signaling through the enteroendocrine IMD pathway in a PGRP-LC-dependent manner. This, in turn, increases transcription of the gene encoding the endocrine peptide Tachykinin (Tk), which is essential for timely larval development and optimal lipid metabolism and insulin signaling. Our findings suggest innate immune pathways not only provide the first line of defense against infection but also afford the intestinal microbiota control over host development and metabolism.</p>
</li>
</ul>
</li>
<li>
<p><em>non-classical immunity controls microbiota impact on skin immunity and tissue repair</em><br />
jonathan l. linehan et al. 2018<br />
<a href="http://dx.doi.org/10.1016/j.cell.2017.12.033">http://dx.doi.org/10.1016/j.cell.2017.12.033</a></p>
<ul>
<li>
<p>•Non-classical MHC class I molecules promote homeostatic immunity to the microbiota<br />
•Commensal-specific T cells express immunoregulatory and tissue repair signatures<br />
•Commensal-specific T cells accelerate wound closure</p>
<p>Mammalian barrier surfaces are constitutively colonized by numerous microorganisms. We explored how the microbiota was sensed by the immune system and the defining properties of such responses. Here, we show that a skin commensal can induce T cell responses in a manner that is restricted to non-classical MHC class I molecules. These responses are uncoupled from inflammation and highly distinct from pathogen-induced cells. Commensal-specific T cells express a defined gene signature that is characterized by expression of effector genes together with immunoregulatory and tissue-repair signatures. As such, non-classical MHCI-restricted commensal-specific immune responses not only promoted protection to pathogens, but also accelerated skin wound closure. Thus, the microbiota can induce a highly physiological and pleiotropic form of adaptive immunity that couples antimicrobial function with tissue repair. Our work also reveals that non-classical MHC class I molecules, an evolutionarily ancient arm of the immune system, can promote homeostatic immunity to the microbiota.</p>
</li>
</ul>
</li>
<li>
<p><em>intestinal serotonin transporter inhibition by toll-like receptor 2 activation. a feedback modulation</em><br />
latorre e et al. 2016<br />
<a href="http://dx.doi.org/10.1371/journal.pone.0169303">http://dx.doi.org/10.1371/journal.pone.0169303</a></p>
<ul>
<li>
<p>a protein known as TLR2, a critical detector of the microbiota found in the intestine. They found that it regulates levels of serotonin — a neurotransmitter which carries messages to the brain, and is also found in the gut, where it regulates our bowel routines.<br />
The research, carried out in cell cultures and verified in mice, provides strong evidence that microbiota can interfere with human physiology by modulating the serotonin transporter activity. Serotonin transporter is a target for numerous diseases and it seems that microbiota living in our guts is able to interfere with this transporter, controlling our serotonin levels.<br />
The finding, published in <em>PLOS ONE</em>, comes as scientists across the world are working to understand the complicated interactions between the “invisible world” of the microbiota in our bodies and the impact they have on our health and even our moods. Recently, scientists in California found evidence that the bacteria in the gut play a role in causing Parkinson’s Disease.<br />
It may also help explain how the microbiota in our guts affect our physiology. Inflammatory bowel disease is thought to be triggered when TLR2 is not functioning properly, but so far, the mechanisms behind this have not been fully understood. This study aimed to further this understanding, and was supported the Foundation for the Study of Inflammatory Bowel Diseases in Aragón (ARAINF), in Spain.<br />
Dr Eva Latorre, a postdoctoral researcher at the University of Exeter Medical School, said the new finding helped to further understanding in a fast-growing research area. She said: “This paper has concluded that the protein TLR2 alters the availability of serotonin, which is important in a range of conditions from depression to inflammatory bowel disease.</p>
</li>
<li>
<p><em>abstract</em> TLR2 is a microbiota recognition receptor that has been described to contribute to intestinal homeostasis and to ameliorate inflammatory intestinal injury. In this context, serotonin (5-HT) has shown to be an essential intestinal physiological neuromodulator that is also involved in intestinal inflammatory diseases. Since the interaction between TLR2 activation and the intestinal serotoninergic system remains non-investigated, our main aim was to analyze the effect of TLR2 on intestinal serotonin transporter (SERT) activity and expression and the intracellular pathways involved. Caco-2/TC7 cells were used to analyze SERT and TLR2 molecular expression and SERT activity by measuring 5-HT uptake. The results showed that apical TLR2 activation inhibits SERT activity in Caco-2/TC7 cells mainly by reducing SERT protein level either in the plasma membrane, after short-term TLR2 activation or in both the plasma membrane and cell lysate, after long-term activation. cAMP/PKA pathway appears to mediate short-term inhibitory effect of TLR2 on SERT; however, p38 MAPK pathway has been shown to be involved in both short- and long-term TLR2 effect. Reciprocally, 5-HT long-term treatment yielded TLR2 down regulation in Caco-2/TC7 cells. Finally, results from <em>in vivo</em> showed an augmented intestinal SERT expression in mice <em>Tlr2-/-</em>, thus confirming our inhibitory effect of TLR2 on intestinal SERT <em>in vitro</em>. The present work infers that TLR2 may act in intestinal pathophysiology, not only by its inherent innate immune role, but also by regulating the intestinal serotoninergic system.</p>
</li>
</ul>
</li>
<li>
<p><em>diet-microbiota interactions mediate global epigenetic programming in multiple host tissues</em><br />
kimberly a. krautkramer et al. 2016<br />
<a href="http://dx.doi.org/10.1016/j.molcel.2016.10.25">j.molcel.2016.10.25</a></p>
<ul>
<li>
<p>A plant-based diet, according to Federico Rey, a UW-Madison professor of bacteriology and also a co-corresponding author of the new report, yields a richer microbiome: “A good diet translates to a beautifully complex microbiome,” Rey says.<br />
“And we see that the gut microbiome affects the host epigenome in a diet-dependent manner. A plant-based diet seems to favor host-microbe communication.”<br />
The new Wisconsin study shows that a small set of short-chain fatty acids produced as the gut bacteria consume, metabolize and ferment nutrients from plants are important chemical messengers, communicating with the cells of the host through the epigenome. “One of the findings here is that microbial metabolism or fermentation of plant fiber results in the production of short-chain fatty acids. These molecules, and potentially many others, are partially responsible for the communication” with the epigenome, says Denu.<br />
In the study, the gut microbiota of the animals that were fed a diet rich in sugar and fat have a diminished capacity to communicate with host cells. According to the Wisconsin team, that may be a hint that the template for a healthy human microbiome was set in the distant past, when food from plants made up a larger portion of diet and sugar and fat were less available than in contemporary diets with more meat and processed foods.<br />
“As we move away from plant-based diets, we may be losing some of that communication between microbes and host,” notes Rey. “With a Western-type diet, it seems like the communication between microbes and host gets lost.”<br />
Foods rich in fat and sugar, especially processed foods, are more easily digested by the host, but are not necessarily a good source of food for the flora inhabiting the gut. The result is a less diverse microbiome and less communication to the host, according to the researchers.<br />
A surprising finding in the study is that the chemical communication between the microbiome and host cells is far reaching. In addition to talking to cells in the colon, the microbiome also seems to be communicating with cells in the liver and in fatty tissue far removed from the gut. That, says Denu, is more evidence of the importance of the microbiome to the well-being of its host.<br />
The kicker experiment in the study, says Denu, was providing mice raised in a germ-free environment with three different short-chain fatty acids that the study showed to be important messengers to the epigenome. The supplement was enough to promote the kind of healthy interplay between microbiota and host cells seen in mice given a diet high in plant fiber.<br />
“It helps show that the collection of three short-chain fatty acids produced in the plant-based diet are likely major communicators,” adds Denu. “We see that it is not just the microbe. It’s microbial metabolism.”</p>
</li>
<li>
<p><em>abstract</em> Gut microbiota alter host histone acetylation and methylation in multiple tissues</p>
<p>Western diet suppresses microbiota-driven SCFA production and chromatin effects</p>
<p>SCFAs recapitulate microbiota-driven chromatin and transcriptional effects</p>
<p>Histone-modifying enzymes regulate transcription and are sensitive to availability of endogenous small-molecule metabolites, allowing chromatin to respond to changes in environment. The gut microbiota produces a myriad of metabolites that affect host physiology and susceptibility to disease; however, the underlying molecular events remain largely unknown. Here we demonstrate that microbial colonization regulates global histone acetylation and methylation in multiple host tissues in a diet-dependent manner: consumption of a “Western-type” diet prevents many of the microbiota-dependent chromatin changes that occur in a polysaccharide-rich diet. Finally, we demonstrate that supplementation of germ-free mice with short-chain fatty acids, major products of gut bacterial fermentation, is sufficient to recapitulate chromatin modification states and transcriptional responses associated with colonization. These findings have profound implications for understanding the complex functional interactions between diet, gut microbiota, and host health.</p>
</li>
</ul>
</li>
<li>
<p><em>akkermansia muciniphila mediates negative effects of ifnγ on glucose metabolism</em><br />
renee l. greer 2016<br />
<a href="http://dx.doi.org/10.1038/NCOMMS13329">NCOMMS13329</a></p>
<ul>
<li>Cross-talk between the gut microbiota and the host immune system regulates host metabolism, and its dysregulation can cause metabolic disease. Here, we show that the gut microbe <em>Akkermansia muciniphila</em> can mediate negative effects of IFNγ on glucose tolerance. In IFNγ-deficient mice, <em>A. muciniphila</em> is significantly increased and restoration of IFNγ levels reduces <em>A. muciniphila</em> abundance. We further show that IFNγ-knockout mice whose microbiota does not contain <em>A. muciniphila</em> do not show improvement in glucose tolerance and adding back <em>A. muciniphila</em> promoted enhanced glucose tolerance. We go on to identify Irgm1 as an IFNγ-regulated gene in the mouse ileum that controls gut <em>A. muciniphila</em> levels. <em>A. muciniphila</em> is also linked to IFNγ-regulated gene expression in the intestine and glucose parameters in humans, suggesting that this trialogue between IFNγ, <em>A. muciniphila</em> and glucose tolerance might be an evolutionally conserved mechanism regulating metabolic health in mice and humans.</li>
<li>our study revealed a new homeostatic regulatory process in mammalian organisms, where a member of different kingdom, <em>A. muciniphila,</em> constitutes an integral part of the interaction between the supposedly functionally distinct and distant systems of immunity and glucose metabolism. Furthermore, our results and other published work in mouse models and human subjects suggest that this transkingdom interaction may be common in mammals<a href="http://www.nature.com/articles/ncomms13329#ref25">25</a>,<a href="http://www.nature.com/articles/ncomms13329#ref26">26</a>,<a href="http://www.nature.com/articles/ncomms13329#ref27">27</a>. Over the years, biologists have drawn boundaries between systems and kingdoms. Our results highlight the fact that these boundaries must be crossed to fully understand the complexity of living organisms.</li>
</ul>
</li>
</ul>
</li>
<li>fitness and longevity
<ul>
<li>
<p><em>microbiome interactions shape host fitness</em><br />
alison l. gould et al. 2018<br />
<a href="http://dx.doi.org/10.1073/pnas.1809349115">http://dx.doi.org/10.1073/pnas.1809349115</a></p>
<ul>
<li>
<p>UC Santa Barbara physicists Eric Jones and Jean Carlson have developed a mathematical approach to analyze and model interactions between gut bacteria in fruit flies. This method could lead to a more sophisticated understanding of the complex interactions between human gut microbes.</p>
<p>Their finding appear in the Proceedings of the National Academy of Sciences.</p>
<p>&quot;Especially over the past 20 years or so, scientists have been finding that the microbiome interacts with the rest of your body, with your immune system, with your brain,&quot; said Jones, a graduate student researcher in Carlson's lab. &quot;Many diseases are associated with certain microbial compositions in the gut.&quot;</p>
<p>The human gut microbiome as yet is too diverse to fully analyze. Instead, the research team, led by Carnegie Institution for Science biologist Will Ludington, used the fruit fly as a model organism to tease apart how the presence of particular gut bacteria could lead to physical and behavioral effects in the host organism.</p>
<p>In their paper, &quot;Microbiome interactions shape host fitness,&quot; Carlson, Jones, Ludington and colleagues examine the interactions between five core species of bacteria found in the fly gut, and calculate how the presence or absence of individual species influences aspects of the fly's fitness, including lifespan, fertility and development. &quot;The classic way we think about bacterial species is in a black-and-white context as agents of disease -- either you have it or you don't,&quot; Ludington said. &quot;Our work shows that isn't the case for the microbiome. The effects of a particular species depend on the context of which other species are also present.&quot;</p>
<p>Building on previous research that found the presence versus the absence of bacteria affected the longevity of an organism (sterile hosts lived longer), the researchers' work on this project revealed that the situation is far more nuanced. For example, the presence of certain bacteria might increase the host's fecundity, while others might decrease longevity. &quot;As we examined the total of what we call a fly's fitness -- it's chances of surviving and creating offspring -- we found that there was a tradeoff between having a short lifespan with lots of offspring, versus having a long lifespan with few offspring,&quot; Ludington explained. &quot;This tradeoff was mediated by microbiome interactions.&quot;</p>
<p>To decipher these interactions, Ludington performed a combinatorial assay, rearing 32 batches of flies each inhabited by a unique combination of the five bacteria. For each bacterial combination, Ludington measured the fly's development, fecundity and longevity. The analysis of the interactions required Carlson and Jones to develop new mathematical approaches.</p>
<p>&quot;One model that often would be a starting point would be to consider the interactions between pairs of bacteria,&quot; said Carlson, whose research delves into the physics of complex systems. &quot;This research shows us that a strictly pairwise model does not capture all of the observed fly traits.&quot;</p>
<p>What the study shows, the researchers said, is that the interactions between the bacterial populations are as significant to the host's overall fitness as their presence -- the microbiome's influence cannot be solely attributed to the presence or absence of individual species. &quot;In a sense,&quot; said Jones, &quot;the microbiome's influence on the host is more than the sum of its parts.&quot;</p>
<p>The newly developed models could be extended to better understand the interactions of the thousands of different species of bacteria in the human microbiome, which could, in turn, shed light on the many connections to microbiome-affiliated diseases including mood disorders, neurological dysfunctions, autoimmune diseases and antibiotic-resistant superbugs.</p>
<p>&quot;In many cases infections are caused by bacteria that we all have in ourselves all the time, and are kept in check by native gut bacteria,&quot; Carlson said. It's not so much that the infection is some new, horrible bacteria, she explained, but that the populations of other bacteria have changed, resulting in unrestricted growth for the infectious bacteria.</p>
<p>&quot;It's really about understanding the population dynamics of these systems,&quot; she said.</p>
</li>
<li>
<p><em>abstract</em> Gut bacteria can affect key aspects of host fitness, such as development, fecundity, and lifespan, while the host, in turn, shapes the gut microbiome. However, it is unclear to what extent individual species versus community interactions within the microbiome are linked to host fitness. Here, we combinatorially dissect the natural microbiome of <em>Drosophila melanogaster</em> and reveal that interactions between bacteria shape host fitness through life history tradeoffs. Empirically, we made germ-free flies colonized with each possible combination of the five core species of fly gut bacteria. We measured the resulting bacterial community abundances and fly fitness traits, including development, reproduction, and lifespan. The fly gut promoted bacterial diversity, which, in turn, accelerated development, reproduction, and aging: Flies that reproduced more died sooner. From these measurements, we calculated the impact of bacterial interactions on fly fitness by adapting the mathematics of genetic epistasis to the microbiome. Development and fecundity converged with higher diversity, suggesting minimal dependence on interactions. However, host lifespan and microbiome abundances were highly dependent on interactions between bacterial species. Higher-order interactions (involving three, four, and five species) occurred in 13–44% of possible cases depending on the trait, with the same interactions affecting multiple traits, a reflection of the life history tradeoff. Overall, we found these interactions were frequently context-dependent and often had the same magnitude as individual species themselves, indicating that the interactions can be as important as the individual species in gut microbiomes.</p>
</li>
</ul>
</li>
<li>
<p><em>microbial genetic composition tunes host longevity</em><br />
bing han et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.cell.2017.05.036">dx.doi.org/10.1016/j.cell.2017.05.036</a></p>
<ul>
<li>
<pre><code>    •Systematic analysis of longevity-promoting microbial genetic variations  
      •Colanic acid as a pro-longevity natural compound effective in different species  
      •Bacterial metabolites regulate host mitochondrial dynamics and UPRmit  
</code></pre>
<p>Homeostasis of the gut microbiota critically influences host health and aging. Developing genetically engineered probiotics holds great promise as a new therapeutic paradigm to promote healthy aging. Here, through screening 3,983 <em>Escherichia coli</em> mutants, we discovered that 29 bacterial genes, when deleted, increase longevity in the host <em>Caenorhabditis elegans</em>. A dozen of these bacterial mutants also protect the host from age-related progression of tumor growth and amyloid-beta accumulation. Mechanistically, we discovered that five bacterial mutants promote longevity through increased secretion of the polysaccharide colanic acid (CA), which regulates mitochondrial dynamics and unfolded protein response (UPRmt) in the host. Purified CA polymers are sufficient to promote longevity via ATFS-1, the host UPRmt-responsive transcription factor. Furthermore, the mitochondrial changes and longevity effects induced by CA are conserved across different species. Together, our results identified molecular targets for developing pro-longevity microbes and a bacterial metabolite acting on host mitochondria to promote longevity.</p>
</li>
<li>
<p>&quot;These findings are also interesting and have implications from the biological point of view in the way we understand host-microbe communication,&quot; Wang said. &quot;Mitochondria seem to have evolved from bacteria that millions of years ago entered primitive cells. Our finding suggests that products from bacteria today can still chime in the communication between mitochondria in our cells. We think that this type of communication is very important and here we have provided the first evidence of this. Fully understanding microbe-mitochondria communication can help us understand at a deeper level the interactions between microbes and their hosts.&quot;</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>06294</h2>
<ul>
<li>promoted by diet
<ul>
<li>
<p>human microbiome produces protective propionate when fed fibre</p>
<ul>
<li>
<p><em>the short-chain fatty acid propionate protects from hypertensive cardiovascular damage</em><br />
hendrik bartolomaeus et al. 2018<br />
<a href="http://dx.doi.org/10.1161/circulationaha.118.036652">http://dx.doi.org/10.1161/circulationaha.118.036652</a></p>
<ul>
<li>
<p>&quot;You are what you eat,&quot; as the proverb goes. But to a large extent our well-being also depends on what bacterial guests in our digestive tract consume. That's because gut flora help the human body to utilize food and produce essential micronutrients, including vitamins.</p>
<p>Beneficial gut microbes can produce metabolites from dietary fiber, including a fatty acid called propionate. This substance protects against the harmful consequences of high blood pressure. A Berlin research team from the Experimental and Clinical Research Center (ECRC), a joint institution of the Max Delbrück Center for Molecular Medicine (MDC) and Charité -- Universitätsmedizin Berlin, shows why this is the case. Their study has been published in advance online in the journal Circulation.</p>
<p>The researchers fed propionate to mice with elevated blood pressure. Afterwards, the animals had less pronounced damage to the heart or abnormal enlargement of the organ, making them less susceptible to cardiac arrhythmia. Vascular damage, such as atherosclerosis, also decreased in mice. &quot;Propionate works against a range of impairments in cardiovascular function caused by high blood pressure,&quot; says MDC researcher and research group leader Professor Dominik N. Müller. &quot;This may be a promising treatment option, particularly for patients who have too little of this fatty acid.&quot;</p>
<p>Detour via the immune system</p>
<p>&quot;Our study made it clear that the substance takes a detour via the immune system and thus affects the heart and blood vessels,&quot; say Dr. Nicola Wilck and Hendrik Bartolomaeus from the ECRC, who have been working together on the project for nearly five years. In particular, T helper cells, which enhance inflammatory processes and contribute to high blood pressure, were calmed.</p>
<p>This has a direct effect on the functional ability of the heart, for example. The research team triggered heart arrhythmia in 70 percent of the untreated mice through targeted electrical stimuli. However, only one-fifth of the animals treated with the fatty acid were susceptible to an irregular heartbeat. Further investigations with ultrasound, tissue sections, and single-cell analyses showed that propionate also reduced blood pressure-related damage to the animals' cardiovascular system, significantly increasing their survival rate.</p>
<p>But when researchers deactivated a certain T cell subtype in the mice's bodies, known as regulatory T cells, the positive effects of propionate disappeared. The immune cells are therefore indispensable for the substance's beneficial effect. A research group under Johannes Stegbauer, an adjunct professor at Düsseldorf University Hospital, confirmed the team's findings in a second animal model.</p>
<p>Short-chain fatty acid as a therapeutic option</p>
<p>The results explain why a diet rich in fiber, which has been recommended by nutrition organizations for many years, helps prevent cardiovascular diseases. Whole-grain products and fruits, for example, contain cellulose and inulin fibers, from which gut bacteria produce the beneficial molecules like propionate, a short-chain fatty acid with a backbone of just three carbon atoms.</p>
<p>&quot;Previously, it had not been clear which fatty acid is behind the positive effects and how it works,&quot; says Wilck. The study opens up new avenues in the treatment of cardiovascular diseases. &quot;It might make sense to administer propionate or a chemical precursor directly as a drug&quot; -- for example, when the blood of those affected contains too little of the substance.</p>
<p>Propionate still has to prove itself in everyday clinical practice. The research team now hopes to validate their findings by examining the substance's effects on human subjects. It is already known that propionate is safe for human consumption and can also be produced economically: The substance has been used for centuries as a preservative, for example. It is already approved as a food additive. &quot;With these favorable conditions, hopefully propionate will soon make the leap from the lab to patients who need it,&quot; says Wilck.</p>
</li>
<li>
<p><em>abstract</em> <strong>Background:</strong> Arterial hypertension and its organ sequelae show characteristics of T cell mediated inflammatory diseases. Experimental anti-inflammatory therapies have been shown to ameliorate hypertensive end-organ damage. Recently, the CANTOS study targeting interleukin-1β demonstrated that anti-inflammatory therapy reduces cardiovascular risk. The gut microbiome plays pivotal role in immune homeostasis and cardiovascular health. Short-chain fatty acids (SCFA) are produced from dietary fiber by gut bacteria and affect host immune homeostasis. Here, we investigated effects of the SCFA propionate in two different mouse models of hypertensive cardiovascular damage.<br />
<strong>Methods:</strong> To investigate the effect of SCFA on hypertensive cardiac damage and atherosclerosis, wild-type NMRI (WT) or ApoE-/- deficient mice received propionate (200mM) or control in the drinking water. To induce hypertension, WT mice were infused with Angiotensin (Ang)II (1.44mg/kg/d s.c.) for 14 days. To accelerate the development of atherosclerosis, ApoE-/- mice were infused with AngII (0.72mg/kg/d s.c.) for 28 days. Cardiac damage and atherosclerosis were assessed using histology, echocardiography, <em>in vivo</em> electrophysiology, immunofluorescence, and flow cytometry. Blood pressure was measured by radiotelemetry. Regulatory T cell (Treg) depletion using PC61 antibody was used to examine the mode of action of propionate.<br />
<strong>Results:</strong> Propionate significantly attenuated cardiac hypertrophy, fibrosis, vascular dysfunction, and hypertension in both models. Susceptibility to cardiac ventricular arrhythmias was significantly reduced in propionate-treated AngII-infused WT mice. Aortic atherosclerotic lesion area was significantly decreased in propionate-treated ApoE-/-. Systemic inflammation was mitigated by propionate treatment, quantified as a reduction in splenic effector memory T cell frequencies and splenic T helper 17 cells in both models, and a decrease in local cardiac immune cell infiltration in WT mice. Cardioprotective effects of propionate were abrogated in Treg-depleted AngII-infused mice, suggesting the effect is Treg-dependent.<br />
<strong>Conclusions:</strong> Our data emphasize an immune-modulatory role of SCFAs and their importance for cardiovascular health. The data suggest that lifestyle modifications leading to augmented SCFA production could be a beneficial non-pharmacological preventive strategy for patients with hypertensive cardiovascular disease.</p>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>gut bacteria selectively promoted by dietary fibers alleviate type 2 diabetes</em><br />
liping zhao et al. 2018<br />
<a href="http://dx.doi.org/10.1126/science.aao5774">http://dx.doi.org/10.1126/science.aao5774</a></p>
<ul>
<li>The gut microbiota benefits humans via short-chain fatty acid (SCFA) production from carbohydrate fermentation, and deficiency in SCFA production is associated with type 2 diabetes mellitus (T2DM). We conducted a randomized clinical study of specifically designed isoenergetic diets, together with fecal shotgun metagenomics, to show that a select group of SCFA-producing strains was promoted by dietary fibers and that most other potential producers were either diminished or unchanged in patients with T2DM. When the fiber-promoted SCFA producers were present in greater diversity and abundance, participants had better improvement in hemoglobin A1c levels, partly via increased glucagon-like peptide-1 production. Promotion of these positive responders diminished producers of metabolically detrimental compounds such as indole and hydrogen sulfide. Targeted restoration of these SCFA producers may present a novel ecological approach for managing T2DM.</li>
</ul>
</li>
<li>
<p><em>the gut microbiota mediates the anti-seizure effects of the ketogenic diet</em><br />
christine a. olson et al. 2018<br />
<a href="http://dx.doi.org/10.1016/j.cell.2018.04.027">http://dx.doi.org/10.1016/j.cell.2018.04.027</a></p>
<ul>
<li>
<p>•Changes in the gut microbiota are required for the anti-seizure effects of the KD<br />
•Specific KD-associated bacteria mediate and confer the anti-seizure effects of the KD<br />
•KD microbiota regulate amino acid γ-glutamylation and hippocampal GABA/glutamate</p>
<p>The ketogenic diet (KD) is used to treat refractory epilepsy, but the mechanisms underlying its neuroprotective effects remain unclear. Here, we show that the gut microbiota is altered by the KD and required for protection against acute electrically induced seizures and spontaneous tonic-clonic seizures in two mouse models. Mice treated with antibiotics or reared germ free are resistant to KD-mediated seizure protection. Enrichment of, and gnotobiotic co-colonization with, KD-associated Akkermansia and Parabacteroides restores seizure protection. Moreover, transplantation of the KD gut microbiota and treatment with Akkermansia and Parabacteroides each confer seizure protection to mice fed a control diet. Alterations in colonic lumenal, serum, and hippocampal metabolomic profiles correlate with seizure protection, including reductions in systemic gamma-glutamylated amino acids and elevated hippocampal GABA/glutamate levels. Bacterial cross-feeding decreases gamma-glutamyltranspeptidase activity, and inhibiting gamma-glutamylation promotes seizure protection in vivo. Overall, this study reveals that the gut microbiota modulates host metabolism and seizure susceptibility in mice.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>06394</h2>
<ul>
<li>
<p>promoted by diversity</p>
<ul>
<li>
<p><em>staphylococcus aureus shifts toward commensalism in response to corynebacterium species</em><br />
matthew m. ramsey et al. 2016<br />
<a href="http://dx.doi.org/10.3389/fmicb.2016.01230">http://dx.doi.org/10.3389/fmicb.2016.01230</a></p>
<ul>
<li>Staphylococcus aureus–human interactions result in a continuum of outcomes from commensalism to pathogenesis. S. aureus is a clinically important pathogen that asymptomatically colonizes ~25% of humans as a member of the nostril and skin microbiota, where it resides with other bacteria including commensal Corynebacterium species. Commensal Corynebacterium spp. are also positively correlated with S. aureus in chronic polymicrobial diabetic foot infections, distinct from acute monomicrobial S. aureus infections. Recent work by our lab and others indicates that microbe–microbe interactions between S. aureus and human skin/nasal commensals, including Corynebacterium species, affect S. aureus behavior and fitness. Thus, we hypothesized that S. aureus interactions with Corynebacterium spp. diminish S. aureus virulence. We tested this by assaying for changes in S. aureus gene expression during in vitro mono- versus coculture with Corynebacterium striatum, a common skin and nasal commensal. We observed a broad shift in S. aureus gene transcription during in vitro growth with C. striatum, including increased transcription of genes known to exhibit increased expression during human nasal colonization and decreased transcription of virulence genes. S. aureus uses several regulatory pathways to transition between commensal and pathogenic states. One of these, the quorum signal accessory gene regulator (agr) system, was strongly inhibited in response to Corynebacterium spp. Phenotypically, S. aureus exposed to C. striatum exhibited increased adhesion to epithelial cells, reflecting a commensal state, and decreased hemolysin activity, reflecting an attenuation of virulence. Consistent with this, S. aureus displayed diminished fitness in experimental in vivo coinfection with C. striatum when compared to monoinfection. These data support a model in which S. aureus shifts from virulence toward a commensal state when exposed to commensal Corynebacterium species.</li>
</ul>
</li>
<li>
<p><em>harbouring public good mutants within a pathogen population can increase both fitness and virulence</em><br />
richard j lindsay et al 2016<br />
<a href="http://dx.doi.org/10.7554/eLife.18678">eLife.18678</a></p>
<p>Link: <a href="http://dx.doi.org/10.7554/eLife.18678">dx.doi.org/10.7554/eLife.18678</a></p>
<ul>
<li>“paradoxic” increase of virulence in diverse population versus “pure” population</li>
<li>Existing theory, empirical, clinical and field research all predict that reducing the virulence of individuals within a pathogen population will reduce the overall virulence, rendering disease less severe. Here, we show that this seemingly successful disease management strategy can fail with devastating consequences for infected hosts. We deploy cooperation theory and a novel synthetic system involving the rice blast fungus <em>Magnaporthe oryzae</em>. In vivo infections of rice demonstrate that <em>M. oryzae</em> virulence is enhanced, quite paradoxically, when a public good mutant is present in a population of high-virulence pathogens. We reason that during infection, the fungus engages in multiple cooperative acts to exploit host resources. We establish a multi-trait cooperation model which suggests that the observed failure of the virulence reduction strategy is caused by the interference between different social traits. Multi-trait cooperative interactions are widespread, so we caution against the indiscriminant application of anti-virulence therapy as a disease-management strategy.</li>
</ul>
</li>
<li>
<p><em>differential human gut microbiome assemblages during soil-transmitted helminth infections in indonesia and liberia</em><br />
bruce a. rosa et al. 2018<br />
<a href="http://dx.doi.org/10.1186/s40168-018-0416-5">http://dx.doi.org/10.1186/s40168-018-0416-5</a></p>
<ul>
<li><em>abstract</em> we identify specific members of the gut microbiome that discriminate between moderately/heavily STH-infected and non-infected states across very diverse geographical regions using two different statistical methods. We also identify microbiome-encoded biological functions associated with the STH infections, which are associated potentially with STH survival strategies, and changes in the host environment. These results provide a novel insight of the cross-kingdom interactions in the human gut ecosystem by unlocking the microbiome assemblages at taxonomic, genetic, and functional levels so that advances towards key mechanistic studies can be made.</li>
<li>Studying communities in Liberia and Indonesia, the researchers found that the gut microbiomes of people able to clear the infections without drugs were more alike and differed markedly from the microbiomes of those who could not clear the infections without treatment.<br />
“People who have sustained infections or who experience multiple infections have a different microbiome to start with compared with those who do not have as much trouble with infection,” said senior author Makedonka Mitreva, PhD, an associate professor of medicine. “Our work included samples from a placebo-controlled trial of drugs against these parasitic worms. It suggests that the microbiomes of people who retained infection are somehow compromised to begin with. Something about their microbiomes makes them more prone to getting infected and to maintaining a chronic infection.”<br />
According to the World Health Organization, about one quarter of the world’s population — over 1.5 billion people — is infected with parasitic worms called helminths. These worm infections are most common in tropical and subtropical areas with poor sanitation.<br />
The researchers identified 12 microbes associated with worm-infected individuals and one type of bacteria associated with uninfected individuals. In particular, gut bacteria associated with increased inflammation were linked to healthy uninfected individuals, perhaps because this type of inflammatory environment makes it harder for the worms to establish themselves in the gut.<br />
The researchers, including first author Bruce A. Rosa, PhD, a staff scientist, analyzed about 400 fecal samples from 250 people living in different villages in Liberia and Indonesia. Mitreva said one of the study’s strengths is that it established common gut microbiome patterns associated with high and low risk of worm infections in two distant geographic regions — West Africa and Southeast Asia, where such infections are prevalent.<br />
The infections spread easily when soil is contaminated with human feces that contain worm eggs. Children are particularly vulnerable because the infections can cause malnutrition, leading to impaired growth and development. Even when infections are treated with appropriate drugs, they often recur because the infective stages of the parasite are ever present in the soil. The main species that affect human health are large roundworm, whipworm and hookworm.<br />
“There are good drugs against most — but not all — of these infections. We are beginning to see more responses to treatment that are less than optimal, indicating the worms are starting to develop resistance to some of these treatments,” said Mitreva, also an assistant director of the university’s McDonnell Genome Institute. “The big problem is reinfection. Even if the therapy works and the infection is cleared, the exposure to contaminated soil is so pervasive that new infections are extremely common. Building on the information from this study, our ultimate long-term goal is to develop a way to alter the gut microbiome so that it protects people from reinfection.”<br />
Currently, antihelminthic drugs are used to deworm people who are infected, especially school-age children and women of reproductive age. Since the study found characteristics of the microbiome that are discriminative of infection, Mitreva said this information could be used to predict who is most likely to develop severe and chronic infections and direct more preventive efforts to those individuals.<br />
“Ideally, we would like to be in a position to suggest some local fermented foods that could alter the microbiome and result in a decreased rate of reinfection,” Mitreva said. “Rather than give more antihelminthic drugs, we want to help people fight off the infection by themselves. Right now, we are looking at the bacteria that we found were associated with protection against infection and studying their effects in mice with intestinal worm infections.”</li>
</ul>
</li>
<li>
<p><em>a commensal strain of staphylococcus epidermidis protects against skin neoplasia</em><br />
teruaki nakatsuji et al. 2018<br />
<a href="http://dx.doi.org/10.1126/sciadv.aao4502">http://dx.doi.org/10.1126/sciadv.aao4502</a></p>
<ul>
<li>We report the discovery that strains of <em>Staphylococcus epidermidis</em> produce 6-<em>N</em>-hydroxyaminopurine (6-HAP), a molecule that inhibits DNA polymerase activity. In culture, 6-HAP selectively inhibited proliferation of tumor lines but did not inhibit primary keratinocytes. Resistance to 6-HAP was associated with the expression of mitochondrial amidoxime reducing components, enzymes that were not observed in cells sensitive to this compound. Intravenous injection of 6-HAP in mice suppressed the growth of B16F10 melanoma without evidence of systemic toxicity. Colonization of mice with an <em>S. epidermidis</em> strain producing 6-HAP reduced the incidence of ultraviolet-induced tumors compared to mice colonized by a control strain that did not produce 6-HAP. <em>S. epidermidis</em> strains producing 6-HAP were found in the metagenome from multiple healthy human subjects, suggesting that the microbiome of some individuals may confer protection against skin cancer. These findings show a new role for skin commensal bacteria in host defense.</li>
</ul>
</li>
<li>
<p><em>human age and skin physiology shape diversity and abundance of archaea on skin</em><br />
christine moissl-eichinger et al. 2017<br />
<a href="http://dx.doi.org/10.1038/s41598-017-04197-4">dx.doi.org/10.1038/s41598-017-04197-4</a></p>
<ul>
<li>The human skin microbiome acts as an important barrier protecting our body from pathogens and other environmental influences. Recent investigations have provided evidence that Archaea are a constant but highly variable component of the human skin microbiome, yet factors that determine their abundance changes are unknown. Here, we tested the hypothesis that the abundance of archaea on human skin is influenced by human age and skin physiology by quantitative PCR of 51 different skin samples taken from human subjects of various age. Our results reveal that archaea are more abundant in human subjects either older than 60 years or younger than 12 years as compared to middle-aged human subjects. These results, together with results obtained from spectroscopy analysis, allowed us gain first insights into a potential link of lower sebum levels and lipid content and thus reduced skin moisture with an increase in archaeal signatures. Amplicon sequencing of selected samples revealed the prevalence of specific eury- and mainly thaumarchaeal taxa, represented by a core archaeome of the human skin.</li>
</ul>
</li>
</ul>
</li>
<li>
<p>promoted by exercise</p>
<ul>
<li>
<p><em>exercise alters gut microbiota composition and function in lean and obese humans</em><br />
jacob m. allen et al. 2017<br />
<a href="http://dx.doi.org/10.1249/mss.0000000000001495">dx.doi.org/10.1249/mss.0000000000001495</a></p>
<ul>
<li>Exercise is associated with altered gut microbial composition, but studies have not investigated whether the gut microbiota and associated metabolites are modulated by exercise training in humans. We explored the impact of six weeks of endurance exercise on the composition, functional capacity, and metabolic output of the gut microbiota in lean and obese adults with multiple-day dietary controls prior to outcome variable collection.<br />
METHODS<br />
Thirty-two lean (n=18 [9 female]) and obese (n=14 [11 female]), previously sedentary subjects participated in six weeks of supervised, endurance-based exercise training (3 days/wk) that progressed from 30 to 60 minutes/day and from moderate (60% of heart rate reserve [HRR]) to vigorous intensity (75% HRR). Subsequently, participants subsequently returned to a sedentary lifestyle activity for a six week washout period. Fecal samples were collected before and after six weeks of exercise, as well as after the sedentary washout period, with 3-day dietary controls in place prior to each collection.<br />
RESULTS<br />
β-diversity analysis revealed that exercise-induced alterations of the gut microbiota were dependent on obesity status. Exercise increased fecal concentrations of short chain fatty acids (SCFAs) in lean, but not obese, participants. Exercise-induced shifts in metabolic output of the microbiota paralleled changes in bacterial genes and taxa capable of SCFA production. Lastly, exercise-induced changes in the microbiota were largely reversed once exercise training ceased.<br />
CONCLUSION<br />
These findings suggest that exercise training induces compositional and functional changes in the human gut microbiota that are dependent on obesity status, independent of diet and contingent on the sustainment of exercise.</li>
</ul>
</li>
</ul>
</li>
<li>
<p>ecological restoration</p>
<ul>
<li>
<p><em>gut microbiota and human health: insights from ecological restoration</em><br />
matthew r. orr et al. 2018<br />
<a href="http://dx.doi.org/10.1086/698021">http://dx.doi.org/10.1086/698021</a></p>
<ul>
<li>Dysbiosis of the gut facilitates chronic diseases such as cancer, obesity, and autoimmune disorders. Successful treatment may require restoration of functions performed by symbiotic microbes. The field of ecological restoration has restored species diversity and ecosystem function of natural habitats since the latter part of the 20th century, but its possible applications to medicine are unexplored. Here we show that approaches developed to restore natural ecosystems may aid in the treatment of chronic disease by helping to characterize target conditions for healthy microbe communities, by identifying the importance of donor diversity in fecal microbiota transplants, and by elevating the importance of diet, patient involvement, and personalized medicine in gut health. Dialogue between the fields of restoration ecology and medicine may help to guide future directions in gut research and clinical care.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>06494</h2>
<ul>
<li>
<p>mental resilience</p>
<ul>
<li><em>anxiety in your head could come from your gut</em><br />
susan donaldson James 2013<br />
Link: <a href="http://abcnews.go.com/Health/anxiety-head-gut/story?id=20229136">abcnews.go.com/Health/anxiety-head-gut/story</a></li>
</ul>
</li>
<li>
<p><em>gut microbiota and glucometabolic alterations in response to recurrent partial sleep deprivation in normal-weight young individuals</em><br />
christian benedict, heike vogel, wenke jonas, anni woting, michael blaut, annette schürmann, jonathan cedernaes 2016<br />
http://dx.doi.org/10.1016/j.molmet.2016.10.003<br />
Link: <a href="http://dx.doi.org/10.1016/j.molmet.2016.10.003">dx.doi.org/10.1016/j.molmet.2016.10.003</a></p>
<ul>
<li>very basic first study to reveal changes in gut microbiota due to changes in sleep regime
<ul>
<li>Objective: Changes to the microbial community in the human gut have been proposed to promote metabolic disturbances that also occur after short periods of sleep loss (including insulin resistance). However, whether sleep loss affects the gut microbiota remains unknown.<br />
Methods: In a randomized within-subject crossover study utilizing a standardized in-lab protocol (with fixed meal times and exercise schedules), we studied nine normal-weight men at two occasions: after two nights of partial sleep deprivation (PSD; sleep opportunity 02:45-07:00h), and after two nights of normal sleep (NS; sleep opportunity 22:30-07:00h). Fecal samples were collected within 24 hours before, and after two in-lab nights, of either NS or PSD. In addition, participants underwent an oral glucose tolerance test following each sleep intervention.<br />
Results: Microbiota composition analysis (V4 16S rRNA gene sequencing) revealed that after two days of PSD vs. after two days of NS, individuals exhibited an increased Firmicutes:Bacteroidetes ratio and of the families Coriobacteriaceae and Erysipelotrichaceae, with decreases in Tenericutes (all P&lt;0.05) – previously all associated with metabolic perturbations in animal or human models. However, no PSD vs. NS effect on beta diversity or on fecal short-chain fatty acid concentrations was found. Fasting and postprandial insulin sensitivity decreased after PSD (all P&lt;0.05).<br />
Discussion: Our findings demonstrate that short-term sleep loss induces subtle effects on human microbiota. To what extent changes to the microbial community contribute to metabolic consequences of sleep loss warrants further investigations in larger and more prolonged sleep studies, to also assess how sleep loss impacts the microbiota in individuals who already are metabolically compromised.</li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>microbial regulation of microrna expression in the amygdala and prefrontal cortex</em><br />
alan e. hoban et al. 2017<br />
<a href="http://dx.doi.org/10.1186/s40168-017-0321-3">dx.doi.org/10.1186/s40168-017-0321-3</a></p>
<ul>
<li>One of the most exciting findings in the past decade in relation to psychiatric disorders has been the discovery that the gut microbiota may act as a key regulator of the brain and behaviour [1, 2, 3]. Proof-of-principal experiments using microbiota-deficient or germ-free (GF) rodents have been extremely useful in investigating the impact of the absence of the gut microbiota on brain development and behaviour <a href="http://dx.doi.org/10.1371/journal.pbio.2000633">4</a>, especially regarding the ability of the gut microbiota to influence normal anxiety and fear behaviours [5, 6, 7]. GF animals not only display changes in basal anxiety but also have deficits in sociability, cognition and increased depressive-like behaviours [8, 9, 10]. Some studies that use antibiotic exposure in order to deplete the gut microbiota have also shown similarly altered behavioural phenotypes [11, 12, 13]. Preclinical evidence also highlights that certain probiotics and prebiotics have anxiolytic-like activity [14, 15] which highlights the potential of microbiota-mediated therapeutic approaches for treating anxiety-related disorders.<br />
The most reproducible finding with regard to behavioural effects in rodent models of microbial disturbances is a marked influence on anxiety-related behaviours <a href="http://dx.doi.org/10.1371/journal.pbio.2000633">4</a>. The neural systems that govern such behaviours are complex, with an array of brain regions that interconnect in order to process emotional stimuli and allow for appropriate responses <a href="http://www.nature.com/articles/ncomms13329#ref26">16</a>. Specifically, the amygdala and PFC are key loci that control anxiety and the response to fearful stimuli <a href="http://www.nature.com/articles/ncomms13329#ref27">17</a>. Both regions display abnormalities in GF animals including hypermyelination in the PFC <a href="http://dx.doi.org/10.1016/j.cell.2017.05.036">18</a>, altered morphology of the amygdala <a href="http://dx.doi.org/10.7554/eLife.18678">19</a> and impaired amygdala-dependent fear memory recall <a href="http://dx.doi.org/10.1016/j.cell.2016.11.003">7</a>. A large body of evidence implicates dysfunction of the neural circuits connecting these two brain regions in the pathophysiology of fear- and anxiety-like disorders [20, 21].<br />
Over the past few years, attempts to understand the mechanisms underlying psychiatric disorders have focused attention on the role of miRNAs <a href="http://abcnews.go.com/Health/anxiety-head-gut/story?id=20229136">22</a>. miRNAs are an expanding class of endogenous small non-coding RNAs, which function as post-transcriptional regulators of gene expression, primarily through translational repression. It has been demonstrated that brain-specific miRNA candidates within the amygdala and PFC regulate anxiety- and fear-related behaviours in mice [23, 24, 25, 26].<br />
We have previously shown that the absence of microbial exposure throughout life results in altered transcriptional regulation in both the amygdala and PFC [18, 27]. We therefore hypothesized that alterations in miRNA expression may be associated with some of the noted molecular changes in these animals. To this end, we used a number of complementary approaches. Firstly, we conducted unbiased Illumina® NGS for miRNAs in the amygdala and PFC of conventionally raised (CON), GF and colonized GF mice (exGF) to investigate whether miRNA expression can be influenced by the gut microbiota. To delineate specific miRNAs that may be influenced by the gut microbiota during adulthood following normal assembly patterns in early life, we then examined miRNA expression in the central nervous system (CNS) of adult rodents following sustained microbiota knockdown with antibiotics. We hypothesized that there would be extensive reorganization of miRNA expression in these two key brain regions (amygdala and PFC) of microbiota-deficient rodents.</li>
</ul>
</li>
<li>
<p><em>microglial control of astrocytes in response to microbial metabolites</em><br />
veit rothhammer et al. 2018<br />
<a href="http://dx.doi.org/10.1038/s41586-018-0119-x">http://dx.doi.org/10.1038/s41586-018-0119-x</a></p>
<ul>
<li>Microglia and astrocytes modulate inflammation and neurodegeneration in the central nervous system (CNS). Microglia modulate pro-inflammatory and neurotoxic activities in astrocytes, but the mechanisms involved are not completely understood. Here we report that TGFα and VEGF-B produced by microglia regulate the pathogenic activities of astrocytes in the experimental autoimmune encephalomyelitis (EAE) mouse model of multiple sclerosis. Microglia-derived TGFα acts via the ErbB1 receptor in astrocytes to limit their pathogenic activities and EAE development. Conversely, microglial VEGF-B triggers FLT-1 signalling in astrocytes and worsens EAE. VEGF-B and TGFα also participate in the microglial control of human astrocytes. Furthermore, expression of TGFα and VEGF-B in CD14+ cells correlates with the multiple sclerosis lesion stage. Finally, metabolites of dietary tryptophan produced by the commensal flora control microglial activation and TGFα and VEGF-B production, modulating the transcriptional program of astrocytes and CNS inflammation through a mechanism mediated by the aryl hydrocarbon receptor. In summary, we identified positive and negative regulators that mediate the microglial control of astrocytes. Moreover, these findings define a pathway through which microbial metabolites limit pathogenic activities of microglia and astrocytes, and suppress CNS inflammation. This pathway may guide new therapies for multiple sclerosis and other neurological disorders.</li>
</ul>
</li>
<li>
<p><em>self-maintaining gut macrophages are essential for intestinal homeostasis</em><br />
sebastiaan de schepper et al. 2018<br />
<a href="http://dx.doi.org/10.1016/j.cell.2018.07.048">http://dx.doi.org/10.1016/j.cell.2018.07.048</a></p>
<ul>
<li>
<p>Self-maintaining macrophages (gMacs) colonize distinct niches of the gut<br />
gMacs have a unique transcriptional profile depending on the niche in which they reside<br />
gMacs support enteric neurons and submucosal vasculature<br />
Loss of gMacs results in vascular leakage, reduced intestinal secretion, and transit</p>
<p>Macrophages are highly heterogeneous tissue-resident immune cells that perform a variety of tissue-supportive functions. The current paradigm dictates that intestinal macrophages are continuously replaced by incoming monocytes that acquire a pro-inflammatory or tissue-protective signature. Here, we identify a self-maintaining population of macrophages that arise from both embryonic precursors and adult bone marrow-derived monocytes and persists throughout adulthood. Gene expression and imaging studies of self-maintaining macrophages revealed distinct transcriptional profiles that reflect their unique localization (i.e., closely positioned to blood vessels, submucosal and myenteric plexus, Paneth cells, and Peyer’s patches). Depletion of self-maintaining macrophages resulted in morphological abnormalities in the submucosal vasculature and loss of enteric neurons, leading to vascular leakage, impaired secretion, and reduced intestinal motility. These results provide critical insights in intestinal macrophage heterogeneity and demonstrate the strategic role of self-maintaining macrophages in gut homeostasis and intestinal physiology.</p>
</li>
</ul>
</li>
</ul>
<h2>development</h2>
<h2>06594</h2>
<ul>
<li>
<p><em>cutting edge: critical roles for microbiota-mediated regulation of the immune system in a prenatal immune activation model of autism</em><br />
catherine r. lammert et al. 2018<br />
<a href="http://dx.doi.org/10.4049/jimmunol.1701755">http://dx.doi.org/10.4049/jimmunol.1701755</a></p>
<ul>
<li>Recent studies suggest that autism is often associated with dysregulated immune responses and altered microbiota composition. This has led to growing speculation about potential roles for hyperactive immune responses and the microbiome in autism. Yet how microbiome–immune cross-talk contributes to neurodevelopmental disorders currently remains poorly understood. In this study, we report critical roles for prenatal microbiota composition in the development of behavioral abnormalities in a murine maternal immune activation (MIA) model of autism that is driven by the viral mimetic polyinosinic-polycytidylic acid. We show that preconception microbiota transplantation can transfer susceptibility to MIA-associated neurodevelopmental disease and that this is associated with modulation of the maternal immune response. Furthermore, we find that ablation of IL-17a signaling provides protection against the development of neurodevelopmental abnormalities in MIA offspring. Our findings suggest that microbiota landscape can influence MIA-induced neurodevelopmental disease pathogenesis and that this occurs as a result of microflora-associated calibration of gestational IL-17a responses.</li>
</ul>
</li>
<li>
<p><em>the gut microbiota and autism spectrum disorders</em><br />
qinrui li et al. 2017<br />
<a href="http://dx.doi.org/10.3389/fncel.2017.00120">dx.doi.org/10.3389/fncel.2017.00120</a></p>
<ul>
<li>Gastrointestinal (GI) symptoms are a common comorbidity in patients with autism spectrum disorder (ASD), but the underlying mechanisms are unknown. Many studies have shown alterations in the composition of the fecal flora and metabolic products of the gut microbiome in patients with ASD. The gut microbiota influences brain development and behaviors through the neuroendocrine, neuroimmune and autonomic nervous systems. In addition, an abnormal gut microbiota is associated with several diseases, such as inflammatory bowel disease (IBD), ASD and mood disorders. Here, we review the bidirectional interactions between the central nervous system and the gastrointestinal tract (brain-gut axis) and the role of the gut microbiota in the central nervous system (CNS) and ASD. Microbiome-mediated therapies might be a safe and effective treatment for ASD.</li>
</ul>
</li>
<li>
<p><em>environment dominates over host genetics in shaping human gut microbiota</em><br />
daphna rothschild et al. 2018<br />
<a href="http://dx.doi.org/10.1038/nature25973">http://dx.doi.org/10.1038/nature25973</a></p>
<ul>
<li>Human gut microbiome composition is shaped by multiple factors but the relative contribution of host genetics remains elusive. Here we examine genotype and microbiome data from 1,046 healthy individuals with several distinct ancestral origins who share a relatively common environment, and demonstrate that the gut microbiome is not significantly associated with genetic ancestry, and that host genetics have a minor role in determining microbiome composition. We show that, by contrast, there are significant similarities in the compositions of the microbiomes of genetically unrelated individuals who share a household, and that over 20% of the inter-person microbiome variability is associated with factors related to diet, drugs and anthropometric measurements. We further demonstrate that microbiome data significantly improve the prediction accuracy for many human traits, such as glucose and obesity measures, compared to models that use only host genetic and environmental data. These results suggest that microbiome alterations aimed at improving clinical outcomes may be carried out across diverse genetic backgrounds.</li>
</ul>
</li>
</ul>
<h2>06694</h2>
<ul>
<li>
<p><em>temporal development of the gut microbiome in early childhood from the teddy study</em><br />
christopher j. stewart et al. 2018<br />
<a href="http://dx.doi.org/10.1038/s41586-018-0617-x">http://dx.doi.org/10.1038/s41586-018-0617-x</a></p>
<ul>
<li>The development of the microbiome from infancy to childhood is dependent on a range of factors, with microbial–immune crosstalk during this time thought to be involved in the pathobiology of later life diseases1,2,3,4,5,6,7,8,9 such as persistent islet autoimmunity and type 1 diabetes10,11,12. However, to our knowledge, no studies have performed extensive characterization of the microbiome in early life in a large, multi-centre population. Here we analyse longitudinal stool samples from 903 children between 3 and 46 months of age by 16S rRNA gene sequencing (n = 12,005) and metagenomic sequencing (n = 10,867), as part of the The Environmental Determinants of Diabetes in the Young (TEDDY) study. We show that the developing gut microbiome undergoes three distinct phases of microbiome progression: a developmental phase (months 3–14), a transitional phase (months 15–30), and a stable phase (months 31–46). Receipt of breast milk, either exclusive or partial, was the most significant factor associated with the microbiome structure. Breastfeeding was associated with higher levels of Bifidobacterium species (B. breve and B. bifidum), and the cessation of breast milk resulted in faster maturation of the gut microbiome, as marked by the phylum Firmicutes. Birth mode was also significantly associated with the microbiome during the developmental phase, driven by higher levels of Bacteroides species (particularly B. fragilis) in infants delivered vaginally. Bacteroides was also associated with increased gut diversity and faster maturation, regardless of the birth mode. Environmental factors including geographical location and household exposures (such as siblings and furry pets) also represented important covariates. A nested case–control analysis revealed subtle associations between microbial taxonomy and the development of islet autoimmunity or type 1 diabetes. These data determine the structural and functional assembly of the microbiome in early life and provide a foundation for targeted mechanistic investigation into the consequences of microbial–immune crosstalk for long-term health.</li>
</ul>
</li>
<li>
<p><em>the human gut microbiome in early-onset type 1 diabetes from the teddy study</em><br />
tommi vatanen et al. 2018<br />
<a href="http://dx.doi.org/10.1038/s41586-018-0620-2">http://dx.doi.org/10.1038/s41586-018-0620-2</a></p>
<ul>
<li>Type 1 diabetes (T1D) is an autoimmune disease that targets pancreatic islet beta cells and incorporates genetic and environmental factors<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR1">26</a>, including complex genetic elements<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR2">27</a>, patient exposures<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR3">28</a> and the gut microbiome<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR4">29</a>. Viral infections<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR5">30</a> and broader gut dysbioses<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR6">31</a> have been identified as potential causes or contributing factors; however, human studies have not yet identified microbial compositional or functional triggers that are predictive of islet autoimmunity or T1D. Here we analyse 10,913 metagenomes in stool samples from 783 mostly white, non-Hispanic children. The samples were collected monthly from three months of age until the clinical end point (islet autoimmunity or T1D) in the The Environmental Determinants of Diabetes in the Young (TEDDY) study, to characterize the natural history of the early gut microbiome in connection to islet autoimmunity, T1D diagnosis, and other common early life events such as antibiotic treatments and probiotics. The microbiomes of control children contained more genes that were related to fermentation and the biosynthesis of short-chain fatty acids, but these were not consistently associated with particular taxa across geographically diverse clinical centres, suggesting that microbial factors associated with T1D are taxonomically diffuse but functionally more coherent. When we investigated the broader establishment and development of the infant microbiome, both taxonomic and functional profiles were dynamic and highly individualized, and dominated in the first year of life by one of three largely exclusive Bifidobacterium species (B. bifidum, B. breve or B. longum) or by the phylum Proteobacteria. In particular, the strain-specific carriage of genes for the utilization of human milk oligosaccharide within a subset of B. longum was present specifically in breast-fed infants. These analyses of TEDDY gut metagenomes provide, to our knowledge, the largest and most detailed longitudinal functional profile of the developing gut microbiome in relation to islet autoimmunity, T1D and other early childhood events. Together with existing evidence from human cohorts<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR7">32</a>,<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR8">33</a> and a T1D mouse model<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR9">34</a>, these data support the protective effects of short-chain fatty acids in early-onset human T1D.</li>
</ul>
</li>
<li>
<p><em>gut microbiota in the first 2 years of life and the association with body mass index at age 12 in a norwegian birth cohort</em><br />
maggie a. stanislawski et al. 2018<br />
<a href="http://dx.doi.org/10.1128/mbio.01751-18">http://dx.doi.org/10.1128/mbio.01751-18</a></p>
<ul>
<li>Childhood obesity is a growing problem worldwide. Recent research suggests that the gut microbiota may play an important and potentially causal role in the development of obesity and may be one mechanism that explains the transgenerational transmission of obesity risk. Here we examine the early-life gut microbiota at days 4, 10, 30, 120, 365, and 730 and the association with body mass index (BMI) z-scores at age 12 in a Norwegian prospective cohort (<em>n</em> = 165), and evaluate how these BMI-associated taxa relate to maternal overweight/obesity (Ow/Ob) and excessive gestational weight gain (GWG). We performed 16S rRNA gene sequencing on the gut microbiota samples. Taxonomic phylogeny at days 10 and 730 was significantly associated with childhood BMI, and the gut microbiota taxa at two years of age explained over 50% of the variation in childhood BMI in this cohort. The subset of the early-life taxa within the gut microbiota that best predicted later childhood BMI showed substantial overlap with the maternal taxa most strongly associated with maternal Ow/Ob and excessive GWG. Our results show an association between the infant gut microbiota and later BMI, and they offer preliminary evidence that the infant gut microbiota, particularly at 2 years of age, may have potential to help identify children at risk for obesity.<br />
<strong>IMPORTANCE</strong> Understanding the role of the early-life gut microbiota in obesity is important because there may be opportunities for preventive strategies. We examined the relationships between infant gut microbiota at six times during the first two years of life and BMI at age 12 in a birth cohort of 165 children and their mothers. We found that the gut microbiota from early life to two years shows an increasingly strong association with childhood BMI. This study provides preliminary evidence that the gut microbiome at 2 years of age may offer useful information to help to identify youth who are at risk for obesity, which could facilitate more-targeted early prevention efforts.</li>
</ul>
</li>
<li>
<p><em>composition and variation of the human milk microbiota are influenced by maternal and early-life factors</em><br />
shirin moossavi et al. 2019<br />
<a href="http://dx.doi.org/10.1016/j.chom.2019.01.011">http://dx.doi.org/10.1016/j.chom.2019.01.011</a></p>
<ul>
<li>
<p>Although previously considered sterile, breastmilk is now known to contain a low abundance of bacteria. While the complexities of how maternal microbiota influence the infant microbiota are still unknown, this complex community of bacteria in breastmilk may help to establish the infant gut microbiota. Disruptions in this process could alter the infant microbiota, causing predisposition to chronic diseases such as allergies, asthma, and obesity. Although recent studies on human milk microbiota suggest that it might be affected by various factors, these findings have not been reproduced in large-scale studies, and the determinants of milk microbiota are still mostly unknown.</p>
<p>To address this gap in knowledge, Azad and her collaborators carried out bacterial gene sequencing on milk samples from 393 healthy mothers three to four months after giving birth. They used this information to examine how the milk microbiota composition is affected by maternal factors, early life events, breastfeeding practices, and other milk components. The researchers found a high degree of variability in the milk microbiota across mothers. Among the many factors analyzed, the mode of breastfeeding -- whether mothers provided milk with or without a pump -- was the only consistent factor directly associated with the milk microbiota composition.</p>
<p>Specifically, indirect breastfeeding (defined as at least one serving of pumped milk in the preceding two weeks) was associated with a higher abundance of potential opportunistic pathogens, such as Stenotrophomonas and Pseudomonadaceae.&quot;Increased exposure to potential pathogens in breastmilk could pose a risk of respiratory infection in the infant, potentially explaining why infants fed pumped milk are at increased risk for pediatric asthma compared to those fed exclusively at the breast,&quot; says first author Shirin Moossavi of the University of Manitoba. To determine if this is the case, there will need to be additional research into how changes in the milk microbiota affect colonization of the infant gut microbiome, which influences health.</p>
<p>By contrast, direct breastfeeding without a pump was associated with microbes typically found in the mouth, as well as higher overall bacterial richness and diversity. Taken together, the findings suggest that direct breastfeeding facilitates the acquisition of oral microbiota from infants, whereas indirect breastfeeding leads to enrichment with environmental (pump-associated) bacteria.</p>
<p>&quot;Our study contributes new evidence to the ongoing debate regarding the origins of milk microbiota,&quot; Azad says. &quot;Contrary or in addition to the hypothesis that milk bacteria come from the mother's gut, our results suggest that the infant's oral bacteria are important in shaping the milk microbiota. Mechanistic studies are needed to confirm this, but if true, it could provide exciting new opportunities for understanding and modifying the milk microbiota.&quot;</p>
<p>Intriguingly, the researchers also observed differences in the milk microbiota depending on the infant sex. These findings further support the idea that the milk microbiota is partially derived from the infant oral cavity, which may differ in male versus female infants.</p>
<p>According to the authors, this topic is ripe for further investigation. Collectively, the many factors they evaluated explained less than a third of the total variation observed in milk microbiota composition, indicating that other unmeasured factors contribute to the large inter-individual variation in milk microbiota profiles.</p>
<p>In future studies, the researchers will further explore the composition and function of the milk microbiota. In addition to bacteria, they will profile fungi in the milk samples. They also plan to investigate how the milk microbiota influences both the gut microbiota of infants and infant development and health. Specifically, their projects will examine the association of milk microbiota with infant growth, asthma, and allergies. &quot;This work could have important implications for microbiota-based strategies for early-life prevention of chronic conditions,&quot; Azad says.</p>
</li>
<li>
<p><em>abstract</em> •Milk microbiota variability is affected by maternal factors and other milk components<br />
•Some factors have phylum-specific effects<br />
•Some variations in milk microbiota are sex-specific<br />
•Feeding method (at the breast versus pumped) was strongly associated with milk microbiota</p>
<p>Breastmilk contains a complex community of bacteria that may help seed the infant gut microbiota. The composition and determinants of milk microbiota are poorly understood. Among 393 mother-infant dyads from the CHILD cohort, we found that milk microbiota at 3–4 months postpartum was dominated by inversely correlated Proteobacteria and Firmicutes, and exhibited discrete compositional patterns. Milk microbiota composition and diversity were associated with maternal factors (BMI, parity, and mode of delivery), breastfeeding practices, and other milk components in a sex-specific manner. Causal modeling identified mode of breastfeeding as a key determinant of milk microbiota composition. Specifically, providing pumped breastmilk was consistently associated with multiple microbiota parameters including enrichment of potential pathogens and depletion of bifidobacteria. Further, these data support the retrograde inoculation hypothesis, whereby the infant oral cavity impacts the milk microbiota. Collectively, these results identify features and determinants of human milk microbiota composition, with potential implications for infant health and development.</p>
</li>
</ul>
</li>
</ul>
<h2>06794</h2>
<ul>
<li>airway
<ul>
<li>
<p><em>the airway microbiome at birth</em><br />
charitharth vivek lal et al 2016<br />
http://dx.doi.org/10.1038/srep31023<br />
Link: <a href="http://dx.doi.org/10.1038/srep31023">dx.doi.org/10.1038/srep31023</a></p>
<ul>
<li>Alterations of pulmonary microbiome have been recognized in multiple respiratory disorders. It is critically important to ascertain if an airway microbiome exists at birth and if so, whether it is associated with subsequent lung disease. We found an established diverse and similar airway microbiome at birth in both preterm and term infants, which was more diverse and different from that of older preterm infants with established chronic lung disease (bronchopulmonary dysplasia). Consistent temporal dysbiotic changes in the airway microbiome were seen from birth to the development of bronchopulmonary dysplasia in extremely preterm infants. Genus Lactobacillus was decreased at birth in infants with chorioamnionitis and in preterm infants who subsequently went on to develop lung disease. Our results, taken together with previous literature indicating a placental and amniotic fluid microbiome, suggest fetal acquisition of an airway microbiome. We speculate that the early airway microbiome may prime the developing pulmonary immune system, and dysbiosis in its development may set the stage for subsequent lung disease.</li>
</ul>
</li>
<li>
<p><em>exposure to bacterial cpg dna protects from airway allergic inflammation by expanding regulatory lung interstitial macrophages</em><br />
catherine sabatel et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.immuni.2017.02.016">dx.doi.org/10.1016/j.immuni.2017.02.016</a></p>
<ul>
<li>Living in a microbe-rich environment reduces the risk of developing asthma. Exposure of humans or mice to unmethylated CpG DNA (CpG) from bacteria reproduces these protective effects, suggesting a major contribution of CpG to microbe-induced asthma resistance. However, how CpG confers protection remains elusive. We found that exposure to CpG expanded regulatory lung interstitial macrophages (IMs) from monocytes infiltrating the lung or mobilized from the spleen. Trafficking of IM precursors to the lung was independent of CCR2, a chemokine receptor required for monocyte mobilization from the bone marrow. Using a mouse model of allergic airway inflammation, we found that adoptive transfer of IMs isolated from CpG-treated mice recapitulated the protective effects of CpG when administered before allergen sensitization or challenge. IM-mediated protection was dependent on IL-10, given that <em>Il10−/−</em> CpG-induced IMs lacked regulatory effects. Thus, the expansion of regulatory lung IMs upon exposure to CpG might underlie the reduced risk of asthma development associated with a microbe-rich environment.</li>
</ul>
</li>
<li>
<p><em>sex-specific impact of asthma during pregnancy on infant gut microbiota</em><br />
petya t. koleva et al. 2017<br />
<a href="http://dx.doi.org/10.1183/13993003.00280-2017">http://dx.doi.org/10.1183/13993003.00280-2017</a></p>
<ul>
<li>Asthma during pregnancy is associated with retardation of fetal growth in a sex-specific manner. Lactobacilli microbes influence infant growth. This study aimed to determine whether lactobacilli and other microbes are reduced in the gut of infants born to an asthmatic mother, and whether this differs by the sex of the infant.<br />
Mother-infant pairs (N=1021) from the Canadian Healthy Infant Longitudinal Development full-term cohort were studied. The abundance of infant faecal microbiota at 3–4 months, profiled by gene sequencing, was compared between both women with and without asthma treatment during pregnancy. Infant sex, maternal ethnicity, pre-pregnancy overweight and atopy status, birth mode, breastfeeding status and intrapartum antibiotic treatment were tested as covariates.<br />
Independent of birth mode and other covariates, male, Caucasian infants born to women with prenatal asthma harboured fewer lactobacilli in the gut at 3–4 months of age. If asthmatic mothers had pre-pregnancy overweight, the abundance of <em>Lactobacillus</em> in males was further reduced in the infant gut, whereas the microbiota of female infants was enriched with <em>Bacteroidaceae</em>. Similar differences in infant gut microbial composition according to maternal prenatal asthma status were also more evident among women with food or environmental allergies.<br />
Gut lactobacilli were less abundant in male infants, but <em>Bacteroidaceae</em> were more abundant in female infants at 3–4 months of age, following maternal asthma during pregnancy.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>06894</h2>
<h2>06994</h2>
<ul>
<li>
<p><em>serum cortisol mediates the relationship between fecal ruminococcus and brain n-acetylaspartate in the young pig</em><br />
austin t. mudd et al. 2017<br />
<a href="http://dx.doi.org/10.1080/19490976.2017.1353849">dx.doi.org/10.1080/19490976.2017.1353849</a></p>
<ul>
<li>A dynamic relationship between the gut microbiota and brain is pivotal in neonatal development. Dysbiosis of the microbiome may result in altered neurodevelopment; however, it is unclear which specific members of microbiota are most influential and what factors might mediate the relationship between the gut and the brain. Twenty-four vaginally-derived male piglets were subjected to magnetic resonance spectroscopy at 30 d of age. Ascending colon contents, feces, and blood were collected and analyzed for volatile fatty acids, microbiota relative abundance by 16s rRNA, and serum metabolites, respectively. A mediation analysis was performed to assess the mediatory effect of serum biomarkers on the relationship between microbiota and neurometabolites. Results indicated fecal <em>Ruminococcus</em> and <em>Butyricimonas</em> predicted brain N-acetylaspartate (NAA). Analysis of serum biomarkers indicated <em>Ruminococcus</em> independently predicted serum serotonin and cortisol. A 3-step mediation indicated: i) <em>Ruminococcus</em> negatively predicted NAA, ii) <em>Ruminococcus</em> negatively predicted cortisol, and iii) a significant indirect effect (i.e., the effect of fecal <em>Ruminococcus</em> through cortisol on NAA) was observed and the direct effect became insignificant. Thus, serum cortisol fully mediated the relationship between fecal <em>Ruminococcus</em> and brain NAA. Using magnetic resonance spectroscopy, this study used a statistical mediation analysis and provides a novel perspective into the potential underlying mechanisms through which the microbiota may shape brain development. This is the first study to link <em>Ruminococcus</em>, cortisol, and NAA <em>in vivo</em>, and these findings are substantiated by previous literature indicating these factors may be influential in the etiology of neurodevelopmental disorders.</li>
</ul>
</li>
<li>
<p><em>pyrazines from bacteria and ants: convergent chemistry within an ecological niche</em><br />
eduardo a. silva-junior et al. 2018<br />
<a href="http://dx.doi.org/10.1038/s41598-018-20953-6">http://dx.doi.org/10.1038/s41598-018-20953-6</a></p>
<ul>
<li>Ants use pheromones to coordinate their communal activity. Volatile pyrazines, for instance, mediate food resource gathering and alarm behaviors in different ant species. Here we report that leaf-cutter ant-associated bacteria produce a family of pyrazines that includes members previously identified as ant trail and alarm pheromones. We found that L-threonine induces the bacterial production of the trail pheromone pyrazines, which are common for the host leaf-cutter ants. Isotope feeding experiments revealed that L-threonine along with sodium acetate were the biosynthetic precursors of these natural products and a biosynthetic pathway was proposed.</li>
</ul>
</li>
</ul>
<h2>07094</h2>
<ul>
<li>
<p><em>trim28 controls a gene regulatory network based on endogenous retroviruses in human neural progenitor cells</em><br />
per ludvik brattås et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.celrep.2016.12.010">j.celrep.2016.12.010</a></p>
<ul>
<li>
<p>•Stage- and region-specific expression of ERVs during human brain development<br />
•TRIM28 binds to ERVs and induces hetereochromatin in human neural progenitor cells<br />
•Knockdown of TRIM28 in hNPCs results in the upregulation of ERV expression<br />
•Protein-coding genes located near upregulated ERVs are upregulated</p>
<p>Endogenous retroviruses (ERVs), which make up 8% of the human genome, have been proposed to participate in the control of gene regulatory networks. In this study, we find a region- and developmental stage-specific expression pattern of ERVs in the developing human brain, which is linked to a transcriptional network based on ERVs. We demonstrate that almost 10,000, primarily primate-specific, ERVs act as docking platforms for the co-repressor protein TRIM28 in human neural progenitor cells, which results in the establishment of local heterochromatin. Thereby, TRIM28 represses ERVs and consequently regulates the expression of neighboring genes. These results uncover a gene regulatory network based on ERVs that participates in control of gene expression of protein-coding transcripts important for brain development.</p>
</li>
</ul>
</li>
</ul>
<h2>07194</h2>
<ul>
<li>
<p><em>bifidobacteria or fiber protects against diet-induced microbiota-mediated colonic mucus deterioration</em><br />
bjoern o. schroeder et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.chom.2017.11.004">http://dx.doi.org/10.1016/j.chom.2017.11.004</a></p>
<ul>
<li>
<p>•Western style diet impairs penetrability and growth of the colonic mucus layer in mice<br />
Mucus defects are accompanied by distinct alterations in microbiota composition<br />
•Microbiota transplant from chow-fed mice prevents mucus aberrations in mice fed a WSD<br />
•Mucus aberrations can be prevented by <em>Bifidobacterium longum</em> and inulin treatment</p>
<p>Diet strongly affects gut microbiota composition, and gut bacteria can influence the colonic mucus layer, a physical barrier that separates trillions of gut bacteria from the host. However, the interplay between a Western style diet (WSD), gut microbiota composition, and the intestinal mucus layer is less clear. Here we show that mice fed a WSD have an altered colonic microbiota composition that causes increased penetrability and a reduced growth rate of the inner mucus layer. Both barrier defects can be prevented by transplanting microbiota from chow-fed mice. In addition, we found that administration of <em>Bifidobacterium longum</em> was sufficient to restore mucus growth, whereas administration of the fiber inulin prevented increased mucus penetrability in WSD-fed mice. We hypothesize that the presence of distinct bacteria is crucial for proper mucus function. If confirmed in humans, these findings may help to better understand diseases with an affected mucus layer, such as ulcerative colitis.</p>
</li>
</ul>
</li>
<li>
<p><em>mediated nourishment of gut microbiota protects against diet-induced obesity by restoring il-22-mediated colonic health</em><br />
jun zou et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.chom.2017.11.003">http://dx.doi.org/10.1016/j.chom.2017.11.003</a></p>
<ul>
<li>
<p>•The fermentable fiber inulin prevented high-fat diet (HFD)-induced metabolic syndrome<br />
•HFD enriched with inulin increased gut epithelial proliferation, prevented colon atrophy<br />
•Inulin restored HFD-induced microbiota depletion and microbiota-mucosa separation<br />
•Inulin effects are microbiota and IL-22, but not short-chain fatty acid, dependent</p>
<p>Dietary supplementation with fermentable fiber suppresses adiposity and the associated parameters of metabolic syndrome. Microbiota-generated fiber-derived short-chain fatty acids (SCFAs) and free fatty acid receptors including GPR43 are thought to mediate these effects. We find that while fermentable (inulin), but not insoluble (cellulose), fiber markedly protected mice against high-fat diet (HFD)-induced metabolic syndrome, the effect was not significantly impaired by either inhibiting SCFA production or genetic ablation of GPR43. Rather, HFD decimates gut microbiota, resulting in loss of enterocyte proliferation, leading to microbiota encroachment, low-grade inflammation (LGI), and metabolic syndrome. Enriching HFD with inulin restored microbiota loads, interleukin-22 (IL-22) production, enterocyte proliferation, and antimicrobial gene expression in a microbiota-dependent manner, as assessed by antibiotic and germ-free approaches. Inulin-induced IL-22 expression, which required innate lymphoid cells, prevented microbiota encroachment and protected against LGI and metabolic syndrome. Thus, fermentable fiber protects against metabolic syndrome by nourishing microbiota to restore IL-22-mediated enterocyte function.</p>
</li>
</ul>
</li>
<li>
<p><em>postnatal exposure to household disinfectants, infant gut microbiota and subsequent risk of overweight in children</em><br />
mon h. tun et al. 2018<br />
<a href="http://dx.doi.org/10.1503/cmaj.170809">http://dx.doi.org/10.1503/cmaj.170809</a></p>
<ul>
<li>Antibacterial cleaning products have the capacity to change the environmental microbiome and alter risk for child overweight. Our study provides novel information regarding the impact of these products on infant gut microbial composition and outcomes of overweight in the same population. We found <em>Lachnospiraceae</em> to be enriched in infant gut microbiota with frequent postnatal use of domestic disinfectants but not eco-friendly products; genus <em>Clostridium</em> and <em>Haemophilus</em> were reduced concurrently. Evidence of statistical mediation with <em>Lachnospiraceae</em> abundance showed a role for this disinfectant-related change to gut microbiota in causing overweight. We did not observe mediation for infant fecal <em>Enterobacteriaceae</em>, suggesting an alternate pathway for the association between postnatal eco-friendly product use and reduced child overweight. Further study is required on the mechanisms through which household cleaning products alter gut microbial composition and the subsequent role this might have on metabolic disease</li>
</ul>
</li>
</ul>
<h2>07294</h2>
<ul>
<li>
<p><em>intrapartum antibiotics for gbs prophylaxis alter colonization patterns in the early infant gut microbiome of low risk infants</em><br />
jennifer c. stearns et al. 2017<br />
<a href="http://dx.doi.org/10.1038/s41598-017-16606-9">http://dx.doi.org/10.1038/s41598-017-16606-9</a></p>
<ul>
<li>Early life microbial colonization and succession is critically important to healthy development with impacts on metabolic and immunologic processes throughout life. A longitudinal prospective cohort was recruited from midwifery practices to include infants born at full term gestation to women with uncomplicated pregnancies. Here we compare bacterial community succession in infants born vaginally, with no exposure to antibiotics (n = 53), with infants who were exposed to intrapartum antibiotic prophylaxis (IAP) for Group B <em>Streptococcus</em> (GBS; n = 14), and infants born by C-section (n = 7). Molecular profiles  of the 16 S rRNA genes indicate that there is a delay in the expansion of <em>Bifidobacterium</em>, which was the dominate infant gut colonizer, over the first 12 weeks and a persistence of <em>Escherichia</em> when IAP for GBS exposure is present during vaginal labour. Longer duration of IAP exposure increased the magnitude of the effect on <em>Bifidobacterium</em> populations, suggesting a longer delay in microbial community maturation. As with prior studies, we found altered gut colonisation following C-section that included a notable lack of Bacteroidetes. This study found that exposure of infants to IAP for GBS during vaginal birth affected aspects of gut microbial ecology that, although dramatic at early time points, disappeared by 12 weeks of age in most infants.</li>
</ul>
</li>
</ul>
<h2>07394</h2>
<h2>comparative studies</h2>
<h2>07494</h2>
<ul>
<li>
<p><em>us immigration westernizes the human gut microbiome</em><br />
pajau vangay et al. 2018<br />
<a href="http://dx.doi.org/10.1016/j.cell.2018.10.029">http://dx.doi.org/10.1016/j.cell.2018.10.029</a></p>
<ul>
<li>
<p>•US immigration is associated with loss of gut microbiome diversity<br />
•US immigrants lose bacterial enzymes associated with plant fiber degradation<br />
•Bacteroides strains displace Prevotella strains according to time spent in the USA<br />
•Loss of diversity increases with obesity and is compounded across generations</p>
<p>Many US immigrant populations develop metabolic diseases post immigration, but the causes are not well understood. Although the microbiome plays a role in metabolic disease, there have been no studies measuring the effects of US immigration on the gut microbiome. We collected stool, dietary recalls, and anthropometrics from 514 Hmong and Karen individuals living in Thailand and the United States, including first- and second-generation immigrants and 19 Karen individuals sampled before and after immigration, as well as from 36 US-born European American individuals. Using 16S and deep shotgun metagenomic DNA sequencing, we found that migration from a non-Western country to the United States is associated with immediate loss of gut microbiome diversity and function in which US-associated strains and functions displace native strains and functions. These effects increase with duration of US residence and are compounded by obesity and across generations.</p>
</li>
</ul>
</li>
<li>
<p><em>seasonal cycling in the gut microbiome of the hadza hunter-gatherers of tanzania</em><br />
samuel a. smits et al. 2017<br />
<a href="http://dx.doi.org/10.1126/science.aan4834">dx.doi.org/10.1126/science.aan4834</a></p>
<ul>
<li>Although humans have cospeciated with their gut-resident microbes, it is difficult to infer features of our ancestral microbiome. Here, we examine the microbiome profile of 350 stool samples collected longitudinally for more than a year from the Hadza hunter-gatherers of Tanzania. The data reveal annual cyclic reconfiguration of the microbiome, in which some taxa become undetectable only to reappear in a subsequent season. Comparison of the Hadza data set with data collected from 18 populations in 16 countries with varying lifestyles reveals that gut community membership corresponds to modernization: Notably, the taxa within the Hadza that are the most seasonally volatile similarly differentiate industrialized and traditional populations. These data indicate that some dynamic lineages of microbes have decreased in prevalence and abundance in modernized populations.</li>
</ul>
</li>
<li>
<p><em>frequency- and amplitude-dependent microbial population dynamics during cycles of feast and famine</em><br />
jason merritt, seppe kuehn 2018<br />
<a href="http://dx.doi.org/10.1103/PhysRevLett.121.098101">http://dx.doi.org/10.1103/PhysRevLett.121.098101</a></p>
<ul>
<li>In nature microbial populations are subject to fluctuating nutrient levels. Nutrient fluctuations are important for evolutionary and ecological dynamics in microbial communities since they impact growth rates, population sizes, and biofilm formation. Here we use automated continuous-culture devices and high-throughput imaging to show that when populations of <em>Escherichia coli</em> are subjected to cycles of nutrient excess (feasts) and scarcity (famine) their abundance dynamics during famines depend on the frequency and amplitude of feasts. We show that frequency and amplitude dependent dynamics in planktonic populations arise from nutrient and history dependent rates of aggregation and dispersal. A phenomenological model recapitulates our experimental observations. Our results show that the statistical properties of environmental fluctuations have substantial impacts on spatial structure in bacterial populations driving large changes in abundance dynamics.</li>
</ul>
</li>
</ul>
<h2>07594</h2>
<ul>
<li>
<p><em>the gut microbiota of healthy aged chinese is similar to that of the healthy young</em><br />
gaorui bian 2017<br />
<a href="http://dx.doi.org/10.1128/mSphere.00327-17">dx.doi.org/10.1128/mSphere.00327-17</a></p>
<ul>
<li>The microbiota of the aged is variously described as being more or less diverse than that of younger cohorts, but the comparison groups used and the definitions of the aged population differ between experiments. The differences are often described by null hypothesis statistical tests, which are notoriously irreproducible when dealing with large multivariate samples. We collected and examined the gut microbiota of a cross-sectional cohort of more than 1,000 very healthy Chinese individuals who spanned ages from 3 to over 100 years. The analysis of 16S rRNA gene sequencing results used a compositional data analysis paradigm coupled with measures of effect size, where ordination, differential abundance, and correlation can be explored and analyzed in a unified and reproducible framework. Our analysis showed several surprising results compared to other cohorts. First, the overall microbiota composition of the healthy aged group was similar to that of people decades younger. Second, the major differences between groups in the gut microbiota profiles were found before age 20. Third, the gut microbiota differed little between individuals from the ages of 30 to &gt;100. Fourth, the gut microbiota of males appeared to be more variable than that of females. Taken together, the present findings suggest that the microbiota of the healthy aged in this cross-sectional study differ little from that of the healthy young in the same population, although the minor variations that do exist depend upon the comparison cohort.</li>
</ul>
</li>
<li>
<p><em>changes in the gut microbiota of urban subjects during an immersion in the traditional diet and lifestyle of a rainforest village</em><br />
kelly v. ruggles et al. 2018<br />
<a href="http://dx.doi.org/10.1128/mSphere.00193-18">http://dx.doi.org/10.1128/mSphere.00193-18</a></p>
<ul>
<li>
<p>Because the children’s gut microbiota exhibited more plasticity, these results raise an interesting possibility that urban children who eat a more traditional, high-fiber, low-fat and low-processed diet early in life might cultivate a more diverse set of gut microbes. Conversely, adults may have a limited response due to their low microbiome plasticity.<br />
Dominguez-Bello was not terribly surprised that the adults’ gut and other microbiota changed so little: “If you take traditional people and bring them to New York, give them antibiotics and McDonald’s to eat everyday, it’s not surprising that they lose diversity,” she says. “But if, as an urban dweller, you’ve already lost that gut microbe diversity and you move to a high-diversity diet, maybe you cannot ‘bloom’ diversity because you simply don’t have those microbes present anymore.”</p>
</li>
<li>
<p>People living traditional lifestyles have higher gut microbiota diversity than urban subjects. We hypothesized that shifting lifestyles from an urban environment to a traditional rainforest village would lead to changes in the microbiota of visitors, which would become more similar to the microbiota of villagers. Here, we characterized at different time points the microbiota of 7 urban visitors (5 adults and 2 children) staying in a rainforest Amerindian village for 16 days and compared them with a reference collection of samples from age-matched local villagers. We performed a 16S rRNA gene survey of samples from multiple body sites (including fecal, oral, nasal, and skin samples) using Illumina MiSeq sequencing. The main factor segregating the microbiotas of each body site was the human group (i.e., visitors versus villagers), with the visitor microbiota tending to have lower alpha diversity; the lowered alpha diversity was statistically significant in the microbiota of skin and in the children’s fecal and oral microbiota. During the rainforest period, all visitors experienced microbiota changes within their personal cloud of variation. For all body sites, the microbiota conformations in the visitor children better matched the microbiota conformations in villagers of the same age than did those of the visitor adults, which showed a lower “microbiota age” than the microbiota of the villagers. The results suggest higher stability in the adult microbiota, with the less resilient children’s microbiota responding more to dietary changes.</p>
</li>
</ul>
</li>
</ul>
<h2>07694</h2>
<ul>
<li>
<p><em>the gut microbiota of rural papua new guineans: composition, diversity patterns, and ecological processes</em><br />
inés martínez et al. 2015</p>
<p><a href="http://dx.doi.org/10.1016/j.celrep.2015.03.049">j.celrep.2015.03.049</a></p>
<ul>
<li>
<p>•The fecal microbiota in PNG is more diverse but less individualized than in the US<br />
•Most bacterial species are shared among PNG and the US, but abundance profiles differ<br />
•Impact of lifestyle on ecological assembly processes might explain these patterns<br />
•Westernization may decrease bacterial dispersal rates, altering microbiota structure</p>
<p>Although recent research revealed an impact of westernization on diversity and composition of the human gut microbiota, the exact consequences on metacommunity characteristics are insufficiently understood, and the underlying ecological mechanisms have not been elucidated. Here, we have compared the fecal microbiota of adults from two non-industrialized regions in Papua New Guinea (PNG) with that of United States (US) residents. Papua New Guineans harbor communities with greater bacterial diversity, lower inter-individual variation, vastly different abundance profiles, and bacterial lineages undetectable in US residents. A quantification of the ecological processes that govern community assembly identified bacterial dispersal as the dominant process that shapes the microbiome in PNG but not in the US. These findings suggest that the microbiome alterations detected in industrialized societies might arise from modern lifestyle factors limiting bacterial dispersal, which has implications for human health and the development of strategies aimed to redress the impact of westernization.</p>
</li>
</ul>
</li>
<li>
<p><em>impact of edible cricket consumption on gut microbiota in healthy adults, a double-blind, randomized crossover trial</em><br />
valerie j. stull et al. 2018<br />
<a href="http://dx.doi.org/10.1038/s41598-018-29032-2">http://dx.doi.org/10.1038/s41598-018-29032-2</a></p>
<ul>
<li>Edible insects are often considered a nutritious, protein-rich, environmentally sustainable alternative to traditional livestock with growing popularity among North American consumers. While the nutrient composition of several insects is characterized, all potential health impacts have not been evaluated. In addition to high protein levels, crickets contain chitin and other fibers that may influence gut health. In this study, we evaluated the effects of consuming 25 grams/day whole cricket powder on gut microbiota composition, while assessing safety and tolerability. Twenty healthy adults participated in this six-week, double-blind, crossover dietary intervention. Participants were randomized into two study arms and consumed either cricket-containing or control breakfast foods for 14 days, followed by a washout period and assignment to the opposite treatment. Blood and stool samples were collected at baseline and after each treatment period to assess liver function and microbiota changes. Results demonstrate cricket consumption is tolerable and non-toxic at the studied dose. Cricket powder supported growth of the probiotic bacterium, Bifidobacterium animalis, which increased 5.7-fold. Cricket consumption was also associated with reduced plasma TNF-α. These data suggest that eating crickets may improve gut health and reduce systemic inflammation; however, more research is needed to understand these effects and underlying mechanisms.</li>
</ul>
</li>
<li>
<p><em>six-week endurance exercise alters gut metagenome that is not reflected in systemic metabolism in over-weight women</em><br />
eveliina munukka et al. 2018<br />
<a href="http://dx.doi.org/10.3389/fmicb.2018.02323">http://dx.doi.org/10.3389/fmicb.2018.02323</a></p>
<ul>
<li>Recent studies suggest that exercise alters the gut microbiome. We determined whether six-weeks endurance exercise, without changing diet, affected the gut metagenome and systemic metabolites of overweight women. Previously sedentary overweight women (<em>n</em> = 19) underwent a six-weeks endurance exercise intervention, but two were excluded due to antibiotic therapy. The gut microbiota composition and functions were analyzed by 16S rRNA gene amplicon sequencing and metagenomics. Body composition was analyzed with DXA X-ray densitometer and serum metabolomics with NMR metabolomics. Total energy and energy-yielding nutrient intakes were analyzed from food records using Micro-Nutrica software. Serum clinical variables were determined with KONELAB instrument. Soluble Vascular Adhesion Protein 1 (VAP-1) was measured with ELISA and its’ enzymatic activity as produced hydrogen peroxide. The exercise intervention was effective, as maximal power and maximum rate of oxygen consumption increased while android fat mass decreased. No changes in diet were observed. Metagenomic analysis revealed taxonomic shifts including an increase in <em>Akkermansia</em> and a decrease in <em>Proteobacteria</em>. These changes were independent of age, weight, fat % as well as energy and fiber intake. Training slightly increased Jaccard distance of genus level β-diversity. Training did not alter the enriched metagenomic pathways, which, according to Bray Curtis dissimilarity analysis, may have been due to that only half of the subjects’ microbiomes responded considerably to exercise. Nevertheless, tranining decreased the abundance of several genes including those related to fructose and amino acid metabolism. These metagenomic changes, however, were not translated into major systemic metabolic changes as only two metabolites, phospholipids and cholesterol in large VLDL particles, decreased after exercise. Training also decreased the amine oxidase activity of pro-inflammatory VAP-1, whereas no changes in CRP were detected. All clinical blood variables were within normal range, yet exercise slightly increased glucose and decreased LDL and HDL. In conclusion, exercise training modified the gut microbiome without greatly affecting systemic metabolites or body composition. Based on our data and existing literature, we propose that especially <em>Akkermansia</em> and <em>Proteobacteria</em> are exercise-responsive taxa. Our results warrant the need for further studies in larger cohorts to determine whether exercise types other than endurance exercise also modify the gut metagenome.</li>
</ul>
</li>
</ul>
<h2>07794</h2>
<ul>
<li>
<p><em>heat-killed lactobacillus casei confers broad protection against influenza a virus primary infection and develops heterosubtypic immunity against future secondary infection</em><br />
yu-jin jung et al. 2017<br />
<a href="http://dx.doi.org/10.1038/s41598-017-17487-8">http://dx.doi.org/10.1038/s41598-017-17487-8</a></p>
<ul>
<li>Lactic acid bacteria (LAB) are the common probiotics. Here, we investigated the antiviral protective effects of heat-killed LAB strain <em>Lactobacillus casei</em> DK128 (DK128) on influenza viruses. Intranasal treatment of mice with DK128 conferred protection against different subtypes of influenza viruses by lessening weight loss and lowering viral loads. Protection via heat-killed DK128 was correlated with an increase in alveolar macrophage cells in the lungs and airways, early induction of virus specific antibodies, reduced levels of pro-inflammatory cytokines and innate immune cells. Importantly, the mice that were protected against primary viral infection as a result of heat-killed DK128 pretreatment developed subsequent heterosubtypic immunity against secondary virus infection. For protection against influenza virus via heat-killed DK128 pretreatment, B cells and partially CD4 T cells but not CD8 T cells were required as inferred from studies using knockout mouse models. Our study provides insight into how hosts can be equipped with innate and adaptive immunity via heat-killed DK128 treatment to protect against influenza virus, supporting that heat-killed LAB may be developed as anti-virus probiotics.</li>
</ul>
</li>
<li>
<p><em>walnut consumption alters the gastrointestinal microbiota, microbially derived secondary bile acids, and health markers in healthy adults: a randomized controlled trial</em><br />
hannah holscher et al. 2018<br />
<a href="http://dx.doi.org/10.1093/jn/nxy004">http://dx.doi.org/10.1093/jn/nxy004</a></p>
<ul>
<li>Walnut consumption affected the composition and function of the human gastrointestinal microbiota, increasing the relative abundances of Firmicutes species in butyrate-producing Clostridium clusters XIVa and IV, including <em>Faecalibacterium</em> and <em>Roseburia</em>, and reducing microbially derived, proinflammatory secondary bile acids and LDL cholesterol. These results suggest that the gastrointestinal microbiota may contribute to the underlying mechanisms of the beneficial health effects of walnut consumption.</li>
</ul>
</li>
<li>
<p><em>daylight exposure modulates bacterial communities associated with household dust</em><br />
ashkaan k. fahimipour et al. 2018<br />
<a href="http://dx.doi.org/10.1186/s40168-018-0559-4">http://dx.doi.org/10.1186/s40168-018-0559-4</a></p>
<ul>
<li>Microbial communities associated with indoor dust abound in the built environment. The transmission of sunlight through windows is a key building design consideration, but the effects of light exposure on dust communities remain unclear. We report results of an experiment and computational models designed to assess the effects of light exposure and wavelengths on the structure of the dust microbiome. Specifically, we placed household dust in replicate model “rooms” with windows that transmitted visible, ultraviolet, or no light and measured taxonomic compositions, absolute abundances, and viabilities of the resulting bacterial communities.<br />
Results<br />
Light exposure per se led to lower abundances of viable bacteria and communities that were compositionally distinct from dark rooms, suggesting preferential inactivation of some microbes over others under daylighting conditions. Differences between communities experiencing visible and ultraviolet light wavelengths were relatively minor, manifesting primarily in abundances of dead human-derived taxa. Daylighting was associated with the loss of a few numerically dominant groups of related microorganisms and apparent increases in the abundances of some rare groups, suggesting that a small number of microorganisms may have exhibited modest population growth under lighting conditions. Although biological processes like population growth on dust could have generated these patterns, we also present an alternate statistical explanation using sampling models from ecology; simulations indicate that artefactual, apparent increases in the abundances of very rare taxa may be a null expectation following the selective inactivation of dominant microorganisms in a community.<br />
Conclusions<br />
Our experimental and simulation-based results indicate that dust contains living bacterial taxa that can be inactivated following changes in local abiotic conditions and suggest that the bactericidal potential of ordinary window-filtered sunlight may be similar to ultraviolet wavelengths across dosages that are relevant to real buildings.</li>
</ul>
</li>
<li>
<p><em>inferring metabolic mechanisms of interaction within a defined gut microbiota</em><br />
gregory l. medlock et al. 2018<br />
<a href="http://dx.doi.org/10.1016/j.cels.2018.08.003">http://dx.doi.org/10.1016/j.cels.2018.08.003</a></p>
<ul>
<li>
<p>•<em>In vitro</em> monoculture and co-culture of six members of a defined murine microbiota<br />
•Growth and metabolome profiling to identify ecological and metabolic interactions<br />
•Development and application of a constant yield expectation (ConYE) model<br />
•Testing of inferred cross-feeding that contributed to commensal interaction</p>
<p>The diversity and number of species present within microbial communities create the potential for a multitude of interspecies metabolic interactions. Here, we develop, apply, and experimentally test a framework for inferring metabolic mechanisms associated with interspecies interactions. We perform pairwise growth and metabolome profiling of co-cultures of strains from a model mouse microbiota. We then apply our framework to dissect emergent metabolic behaviors that occur in co-culture. Based on one of the inferences from this framework, we identify and interrogate an amino acid cross-feeding interaction and validate that the proposed interaction leads to a growth benefit <em>in vitro</em>. Our results reveal the type and extent of emergent metabolic behavior in microbial communities composed of gut microbes. We focus on growth-modulating interactions, but the framework can be applied to interspecies interactions that modulate any phenotype of interest within microbial communities.</p>
</li>
</ul>
</li>
<li>
<p><em>microbial interkingdom interactions in roots promote arabidopsis survival</em><br />
paloma durán et al. 2018<br />
<a href="http://dx.doi.org/10.1016/j.cell.2018.10.020">http://dx.doi.org/10.1016/j.cell.2018.10.020</a></p>
<ul>
<li>
<p>•Roots of healthy plants are colonized by multi-kingdom microbial consortia<br />
•Bacterial Root Commensals (BRCs) shape fungal and oomycetal community structure<br />
•BRCs protect plants against fungi and oomycetes<br />
•Biocontrol activity of BRCs is a redundant trait and essential for plant survival</p>
<p>Roots of healthy plants are inhabited by soil-derived bacteria, fungi, and oomycetes that have evolved independently in distinct kingdoms of life. How these microorganisms interact and to what extent those interactions affect plant health are poorly understood. We examined root-associated microbial communities from three Arabidopsis thaliana populations and detected mostly negative correlations between bacteria and filamentous microbial eukaryotes. We established microbial culture collections for reconstitution experiments using germ-free A. thaliana. In plants inoculated with mono- or multi-kingdom synthetic microbial consortia, we observed a profound impact of the bacterial root microbiota on fungal and oomycetal community structure and diversity. We demonstrate that the bacterial microbiota is essential for plant survival and protection against root-derived filamentous eukaryotes. Deconvolution of 2,862 binary bacterial-fungal interactions ex situ, combined with community perturbation experiments in planta, indicate that biocontrol activity of bacterial root commensals is a redundant trait that maintains microbial interkingdom balance for plant health.</p>
</li>
</ul>
</li>
<li>
<p><em>coral-associated bacteria demonstrate phylosymbiosis and cophylogeny</em><br />
f. joseph pollock et al. 2018<br />
<a href="http://dx.doi.org/10.1038/s41467-018-07275-x">http://dx.doi.org/10.1038/s41467-018-07275-x</a></p>
<ul>
<li>Scleractinian corals’ microbial symbionts influence host health, yet how coral microbiomes assembled over evolution is not well understood. We survey bacterial and archaeal communities in phylogenetically diverse Australian corals representing more than 425 million years of diversification. We show that coral microbiomes are anatomically compartmentalized in both modern microbial ecology and evolutionary assembly. Coral mucus, tissue, and skeleton microbiomes differ in microbial community composition, richness, and response to host vs. environmental drivers. We also find evidence of coral-microbe phylosymbiosis, in which coral microbiome composition and richness reflect coral phylogeny. Surprisingly, the coral skeleton represents the most biodiverse coral microbiome, and also shows the strongest evidence of phylosymbiosis. Interactions between bacterial and coral phylogeny significantly influence the abundance of four groups of bacteria–including Endozoicomonas-like bacteria, which divide into host-generalist and host-specific subclades. Together these results trace microbial symbiosis across anatomy during the evolution of a basal animal lineage.</li>
</ul>
</li>
<li>
<p>causal</p>
</li>
<li>
<p><em>microbially produced imidazole propionate impairs insulin signaling through mtorc1</em><br />
ara koh et al. 2018<br />
<a href="http://dx.doi.org/10.1016/j.cell.2018.09.055">http://dx.doi.org/10.1016/j.cell.2018.09.055</a></p>
<ul>
<li>
<p>•Imidazole propionate levels are increased in subjects with type 2 diabetes (T2D)<br />
•Imidazole propionate is produced from histidine by T2D-associated bacteria<br />
•Imidazole propionate impairs glucose tolerance and insulin signaling<br />
•Imidazole propionate inhibits IRS via activation of p38γ/p62/mTORC1</p>
<p>Interactions between the gut microbiota, diet, and the host potentially contribute to the development of metabolic diseases. Here, we identify imidazole propionate as a microbially produced histidine-derived metabolite that is present at higher concentrations in subjects with versus without type 2 diabetes. We show that imidazole propionate is produced from histidine in a gut simulator at higher concentrations when using fecal microbiota from subjects with versus without type 2 diabetes and that it impairs glucose tolerance when administered to mice. We further show that imidazole propionate impairs insulin signaling at the level of insulin receptor substrate through the activation of p38γ MAPK, which promotes p62 phosphorylation and, subsequently, activation of mechanistic target of rapamycin complex 1 (mTORC1). We also demonstrate increased activation of p62 and mTORC1 in liver from subjects with type 2 diabetes. Our findings indicate that the microbial metabolite imidazole propionate may contribute to the pathogenesis of type 2 diabetes.</p>
</li>
</ul>
</li>
</ul>
<h2>dysbiosis</h2>
<h2>07894</h2>
<ul>
<li>induced by antibiotics
<ul>
<li>
<p><em>a nationwide study in denmark of the association between treated infections and the subsequent risk of treated mental disorders in children and adolescents</em><br />
ole köhler-forsberg et al. 2018<br />
<a href="http://dx.doi.org/10.1001/jamapsychiatry.2018.3428">http://dx.doi.org/10.1001/jamapsychiatry.2018.3428</a></p>
<p>effect of antibiotics was oddly minimised in abstract, conclusions, and media attention of this article, but if you know microbiome studies the incidental revelations in the paper itself are smoking guns.</p>
<ul>
<li>Antibiotics were the anti-infective agents associated with the largest risk increase in mental disorders (Table 1), whereas antivirals and antimycotics did not increase the risk estimates.</li>
<li>the risks were increased for schizophrenia spectrum disorders, OCD, personality and behavior disorders, mental retardation, ASD, ADHD, ODD/CD, and tic disorders. The results for anti-infective agents were higher among individuals with prior hospitalizations due to infections and increased in a dose-response association.</li>
<li>To our knowledge, the present study is the first to indicate that any treated infection, including less severe infection, is associated with an increased risk of a wide range of childhood and adolescent mental disorders.</li>
</ul>
</li>
<li>
<p><em>antibiotic-induced changes to the host metabolic environment inhibit drug efficacy and alter immune function</em><br />
jason h. yang et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.chom.2017.10.020">http://dx.doi.org/10.1016/j.chom.2017.10.020</a></p>
<ul>
<li>
<p>Antibiotic treatment depletes central metabolism intermediates in the peritoneum<br />
Antibiotic treatment elicits microbiome-independent changes in host metabolites<br />
Metabolites altered by antibiotic treatment during infection inhibit drug efficacy<br />
Antibiotic treatment impairs phagocytic killing by inhibiting respiratory activity</p>
<p>Bactericidal antibiotics alter microbial metabolism as part of their lethality and can damage mitochondria in mammalian cells. In addition, antibiotic susceptibility is sensitive to extracellular metabolites, but it remains unknown whether metabolites present at an infection site can affect either treatment efficacy or immune function. Here, we quantify local metabolic changes in the host microenvironment following antibiotic treatment for a peritoneal <em>Escherichia coli</em> infection. Antibiotic treatment elicits microbiome-independent changes in local metabolites, but not those distal to the infection site, by acting directly on host cells. The metabolites induced during treatment, such as AMP, reduce antibiotic efficacy and enhance phagocytic killing. Moreover, antibiotic treatment impairs immune function by inhibiting respiratory activity in immune cells. Collectively, these results highlight the immunomodulatory potential of antibiotics and reveal the local metabolic microenvironment to be an important determinant of infection resolution.</p>
</li>
</ul>
</li>
<li>
<p><em>recovery of gut microbiota of healthy adults following antibiotic exposure</em><br />
albert palleja et al. 2018<br />
<a href="http://dx.doi.org/10.1038/s41564-018-0257-9">http://dx.doi.org/10.1038/s41564-018-0257-9</a></p>
<ul>
<li>“Interestingly, Clostridium spp. were particularly likely to colonize after the antibiotic intervention, perhaps having survived as spores. This is relevant as Clostridium difficile infections are associated with the use of broad-spectrum antibiotics in acute care settings, which reduce host resistance to colonization and expansion of this pathogen.”</li>
<li><em>abstract</em> To minimize the impact of antibiotics, gut microorganisms harbour and exchange antibiotics resistance genes, collectively called their resistome. Using shotgun sequencing-based metagenomics, we analysed the partial eradication and subsequent regrowth of the gut microbiota in 12 healthy men over a 6-month period following a 4-day intervention with a cocktail of 3 last-resort antibiotics: meropenem, gentamicin and vancomycin. Initial changes included blooms of enterobacteria and other pathobionts, such as Enterococcus faecalis and Fusobacterium nucleatum, and the depletion of Bifidobacterium species and butyrate producers. The gut microbiota of the subjects recovered to near-baseline composition within 1.5 months, although 9 common species, which were present in all subjects before the treatment, remained undetectable in most of the subjects after 180 days. Species that harbour β-lactam resistance genes were positively selected for during and after the intervention. Harbouring glycopeptide or aminoglycoside resistance genes increased the odds of de novo colonization, however, the former also decreased the odds of survival. Compositional changes under antibiotic intervention in vivo matched results from in vitro susceptibility tests. Despite a mild yet long-lasting imprint following antibiotics exposure, the gut microbiota of healthy young adults are resilient to a short-term broad-spectrum antibiotics intervention and their antibiotics resistance gene carriage modulates their recovery processes.</li>
</ul>
</li>
<li>
<p><em>ciprofloxacin impairs mitochondrial dna replication initiation through inhibition of topoisomerase 2</em><br />
anu hangas et al. 2018<br />
<a href="http://dx.doi.org/10.1093/nar/gky793">http://dx.doi.org/10.1093/nar/gky793</a></p>
<ul>
<li>
<p>Fluoroquinolones are some of the most frequently used broad-spectrum antibacterial antibiotics, and usually prescribed for respiratory, urinary or ear infections. While they are generally well tolerated, some patients receiving fluoroquinolones develop severe health problems, among them tendon rupture, permanent nerve damage or depression. The reasons for these side effects are yet unclear.<br />
A study carried out at the University of Eastern Finland and published in <em>Nucleic Acids Research</em> investigated the effect of ciprofloxacin on mitochondria, the important cell organelles in our body that produce the energy for cellular function. Mitochondria possess their own small circular genome, which requires topoisomerase enzymes for its maintenance. Topoisomerases regulate the topology of DNA and untangle for instance knots and overwound stretches of a genome by cutting and reconnecting the DNA sequence. While fluoroquinolones are designed to inhibit the bacterial topoisomerase gyrase, which leads to the death of the bacterium, they also inhibit the topoisomerase 2 of our own cells.<br />
“We noticed that topoisomerase 2 is especially important in the replication of the mitochondrial genome, as it regulates the winding of this small DNA molecule by removing positive twists,” Academy Research Fellow Steffi Goffart from the University of Eastern Finland says.<br />
Ciprofloxacin stopped this normal maintenance and transcription of mitochondrial DNA by changing mtDNA topology, causing impaired mitochondrial energy production and blocking cellular growth and differentiation. This dramatic impact on mitochondrial DNA is the likely cause for most negative side effects experienced by patients, and also a reason to use fluoroquinolone antibiotics with great caution.</p>
</li>
<li>
<p><em>abstract</em> Maintenance of topological homeostasis is vital for gene expression and genome replication in all organisms. Similar to other circular genomes, also mitochondrial DNA (mtDNA) is known to exist in various different topological forms, although their functional significance remains unknown. We report here that both known type II topoisomerases Top2α and Top2β are present in mammalian mitochondria, with especially Top2β regulating the supercoiling state of mtDNA. Loss of Top2β or its inhibition by ciprofloxacin results in accumulation of positively supercoiled mtDNA, followed by cessation of mitochondrial transcription and replication initiation, causing depletion of mtDNA copy number. These mitochondrial effects block both cell proliferation and differentiation, possibly explaining some of the side effects associated with fluoroquinolone antibiotics. Our results show for the first time the importance of topology for maintenance of mtDNA homeostasis and provide novel insight into the mitochondrial effects of fluoroquinolones.</p>
</li>
</ul>
</li>
<li>
<p><em>characterization of shifts of koala (phascolarctos cinereus) intestinal microbial communities associated with antibiotic treatment</em><br />
katherine e. dahlhausen et al. 2018<br />
<a href="https://doi.org/10.7717/peerj.4452">https://doi.org/10.7717/peerj.4452</a></p>
<ul>
<li>Koalas (<em>Phascolarctos cinereus</em>) are arboreal marsupials native to Australia that eat a specialized diet of almost exclusively eucalyptus leaves. Microbes in koala intestines are known to break down otherwise toxic compounds, such as tannins, in eucalyptus leaves. Infections by <em>Chlamydia</em>, obligate intracellular bacterial pathogens, are highly prevalent in koala populations. If animals with <em>Chlamydia</em> infections are received by wildlife hospitals, a range of antibiotics can be used to treat them. However, previous studies suggested that koalas can suffer adverse side effects during antibiotic treatment. This study aimed to use 16S rRNA gene sequences derived from koala feces to characterize the intestinal microbiome of koalas throughout antibiotic treatment and identify specific taxa associated with koala health after treatment. Although differences in the alpha diversity were observed in the intestinal flora between treated and untreated koalas and between koalas treated with different antibiotics, these differences were not statistically significant. The alpha diversity of microbial communities from koalas that lived through antibiotic treatment versus those who did not was significantly greater, however. Beta diversity analysis largely confirmed the latter observation, revealing that the overall communities were different between koalas on antibiotics that died versus those that survived or never received antibiotics. Using both machine learning and OTU (operational taxonomic unit) co-occurrence network analyses, we found that OTUs that are very closely related to <em>Lonepinella koalarum</em>, a known tannin degrader found by culture-based methods to be present in koala intestines, was correlated with a koala’s health status. This is the first study to characterize the time course of effects of antibiotics on koala intestinal microbiomes. Our results suggest it may be useful to pursue alternative treatments for <em>Chlamydia</em> infections without the use of antibiotics or the development of <em>Chlamydia</em>-specific antimicrobial compounds that do not broadly affect microbial communities.</li>
</ul>
</li>
<li>
<p><em>the kinases hipa and hipa7 phosphorylate different substrate pools in escherichia coli to promote multidrug tolerance</em><br />
maja semanjski et al. 2018<br />
<a href="http://dx.doi.org/10.1126/scisignal.aat5750">http://dx.doi.org/10.1126/scisignal.aat5750</a></p>
<ul>
<li>The bacterial serine-threonine protein kinase HipA promotes multidrug tolerance by phosphorylating the glutamate-tRNA ligase (GltX), leading to a halt in translation, inhibition of growth, and induction of a physiologically dormant state (persistence). The HipA variant HipA7 substantially increases persistence despite being less efficient at inhibiting cell growth. We postulated that this phenotypic difference was caused by differences in the substrates targeted by both kinases. We overproduced HipA and HipA7 in Escherichia coli and identified their endogenous substrates by SILAC-based quantitative phosphoproteomics. We confirmed that GltX was the main substrate of both kinase variants and likely the primary determinant of persistence. When HipA and HipA7 were moderately overproduced from plasmids, HipA7 targeted only GltX, but HipA phosphorylated several additional substrates involved in translation, transcription, and replication, such as ribosomal protein L11 (RplK) and the negative modulator of replication initiation, SeqA. HipA7 showed reduced kinase activity compared to HipA and targeted a substrate pool similar to that of HipA only when produced from a high–copy number plasmid. The kinase variants also differed in autophosphorylation, which was substantially reduced for HipA7. When produced endogenously from the chromosome, HipA showed no activity because of inhibition by the antitoxin HipB, whereas HipA7 phosphorylated GltX and phage shock protein PspA. Initial testing did not reveal a connection between HipA-induced phosphorylation of RplK and persistence or growth inhibition, suggesting that other HipA-specific substrates were likely responsible for growth inhibition. Our results contribute to the understanding of HipA7 action and present a resource for elucidating HipA-related persistence.</li>
</ul>
</li>
<li>
<p>antibiotic-associated hemorrhagic colitis (AAHC)<br />
one mechanism for antibiotics to cause damage, through imbalance of toxin-producing part of microbiome</p>
<ul>
<li>
<p><em>biosynthesis of the enterotoxic pyrrolobenzodiazepine natural product tilivalline</em><br />
elisabeth dornisch et al. 2017<br />
<a href="http://dx.doi.org/10.1002/anie.201707737">dx.doi.org/10.1002/anie.201707737</a></p>
<ul>
<li>The nonribosomal enterotoxin tilivalline was the first naturally occurring pyrrolobenzodiazepine to be linked to disease in the human intestine. Since the producing organism <em>Klebsiella oxytoca</em> is part of the intestinal microbiota and the pyrrolobenzodiazepine causes the pathogenesis of colitis it is important to understand the biosynthesis and regulation of tilivalline activity. Here we report the biosynthesis of tilivalline and show that this nonribosomal peptide assembly pathway initially generates tilimycin, a simple pyrrolobenzodiazepine with cytotoxic properties. Tilivalline results from the non-enzymatic spontaneous reaction of tilimycin with biogenetically generated indole. Through a chemical total synthesis of tilimycin we could corroborate the predictions made about the biosynthesis. Production of two cytotoxic pyrrolobenzodiazepines with distinct functionalities by human gut resident <em>Klebsiella oxytoca</em> has important implications for intestinal disease.</li>
<li>The enteric bacterium <em>Klebsiella oxytoca</em> presents a striking example. In some patients treated with penicillin antibiotics, rapid proliferation of this organism results in antibiotic-associated hemorrhagic colitis (AAHC).<a href="http://dx.doi.org/10.1016/j.cell.2016.11.003">7</a> We showed in previous work that the indol-3-yl-substituted pyrrolobenzodiazepine (PBD) tilivalline (1) produced by <em>K. oxytoca</em> was required for the pathogenesis of colitis in an animal model of AAHC.<a href="http://dx.doi.org/10.1016/j.cell.2015.11.004">8</a> Naturally occurring PBDs form a family of antitumor antibiotics produced by Gram-positive soil bacteria.<a href="https://www.nature.com/articles/s41586-018-0634-9#ref-CR1">9</a> Synthesis of a natural PBD by a Gram-negative resident of the gut is thus surprising. We identified the toxin biosynthetic gene cluster by genetic mutation and localized the region to a unique pathogenicity island on the genome of cytotoxic <em>K. oxytoca</em> strains.<a href="http://dx.doi.org/10.1016/j.cell.2015.11.004">8</a></li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>oral administration of antibiotics increased the potential mobility of bacterial resistance genes in the gut of the fish piaractus mesopotamicus</em><br />
johan s. sáenz et al. 2019<br />
<a href="http://dx.doi.org/10.1186/s40168-019-0632-7">http://dx.doi.org/10.1186/s40168-019-0632-7</a></p>
<ul>
<li>
<p>they studied Piaractus mesopotamicus, a South American species known as pacu that is often raised in aquaculture. The fish received the antibiotic florfenicol in their food for 34 days. During this time and after the application period, the researchers took samples from the digestive tract of the fish and looked for relevant genetic changes in the gut bacteria.</p>
<p>Resistance genes hop around the genome</p>
<p>&quot;As expected, administration of the antibiotic induced an increase in the genes responsible for resistance to that antibiotic,&quot; explains COMI doctoral student Johan Sebastian Sáenz Medina, lead author of the paper. &quot;One example are genes for pump proteins, which simply remove the active substance from the bacteria again. However, we were particularly surprised by the different mechanisms that we could detect by which antibiotic resistance genes are spread amongst gut bacteria of the fish,&quot; Sáenz Medina explains. &quot;This suggests that the bacteria also exchange resistance through viruses, known as phages, and transposons.&quot;</p>
<p>Further metagenomic studies confirmed that these mobile genetic elements induce a fast distribution of resistance genes among genomes of different organisms. So far it has been postulated that only plasmids (in essence, easily transferable mini-chromosomes) are mainly responsible for the exchange of resistance genes.</p>
<p>&quot;The finding that resistance is also extensively transferred between bacteria without the involvement of plasmids is really quite surprising,&quot; says Michael Schloter.</p>
</li>
<li>
<p><em>abstract</em> Aquaculture is on the rise worldwide, and the use of antibiotics is fostering higher production intensity. However, recent findings suggest that the use of antibiotics comes at the price of increased antibiotic resistance. Yet, the effect of the oral administration of antibiotics on the mobility of microbial resistance genes in the fish gut is not well understood. In the present study, Piaractus mesopotamicus was used as a model to evaluate the effect of the antimicrobial florfenicol on the diversity of the gut microbiome as well as antibiotic resistance genes (ARGs) and mobile genetic elements (MGEs) using a metagenomic approach.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>07994</h2>
<ul>
<li>induced by pesticides, toxins
<ul>
<li>
<p><em>bisphenol-a alters microbiota metabolites derived from aromatic amino acids and worsens disease activity during colitis</em><br />
jennifer aa deluca et al. 2018<br />
<a href="http://dx.doi.org/10.1177/1535370218782139">http://dx.doi.org/10.1177/1535370218782139</a></p>
<ul>
<li>Inflammatory bowel disease is a complex collection of disorders. Microbial dysbiosis as well as exposure to toxins including xenoestrogens are thought to be risk factors for inflammatory bowel disease development and relapse. Bisphenol-A has been shown to exert estrogenic activity in the colon and alter intestinal function, but the role that xenoestrogens, such as bisphenol-A , play in colonic inflammation has been previously described but with conflicting results. We investigated the ability of bisphenol-A to exacerbate colonic inflammation and alter microbiota metabolites derived from aromatic amino acids in an acute dextran sulfate sodium-induced colitis model. Female C57BL/6 mice were ovariectomized and exposed to bisphenol-A daily for 15 days. Disease activity measures include body weight, fecal consistency, and rectal bleeding. Colons were scored for inflammation, injury, and nodularity. Alterations in the levels of microbiota metabolites derived from aromatic amino acids known to reflect phenotypic changes in the gut microbiome were analyzed. Bisphenol-A exposure increased mortality and worsened disease activity as well as inflammation and nodularity scores in the middle colon region following dextran sulfate sodium exposure. Unique patterns of metabolites were associated with bisphenol-A consumption. Regardless of dextran sulfate sodium treatment, bisphenol-A reduced levels of tryptophan and several metabolites associated with decreased inflammation in the colon. This is the first study to show that bisphenol-A treatment alone can reduce microbiota metabolites derived from aromatic amino acids in the colon which may be associated with increased colonic inflammation and inflammatory bowel disease.</li>
</ul>
</li>
<li>
<p><em>glyphosate perturbs the gut microbiota of honey bees</em><br />
erick v. s. motta et al. 2018<br />
<a href="https://doi.org/10.1073/pnas.1803880115">https://doi.org/10.1073/pnas.1803880115</a></p>
<ul>
<li>Glyphosate, the primary herbicide used globally for weed control, targets the 5-enolpyruvylshikimate-3-phosphate synthase (EPSPS) enzyme in the shikimate pathway found in plants and some microorganisms. Thus, glyphosate may affect bacterial symbionts of animals living near agricultural sites, including pollinators such as bees. The honey bee gut microbiota is dominated by eight bacterial species that promote weight gain and reduce pathogen susceptibility. The gene encoding EPSPS is present in almost all sequenced genomes of bee gut bacteria, indicating that they are potentially susceptible to glyphosate. We demonstrated that the relative and absolute abundances of dominant gut microbiota species are decreased in bees exposed to glyphosate at concentrations documented in the environment. Glyphosate exposure of young workers increased mortality of bees subsequently exposed to the opportunistic pathogen <em>Serratia marcescens</em>. Members of the bee gut microbiota varied in susceptibility to glyphosate, largely corresponding to whether they possessed an EPSPS of class I (sensitive to glyphosate) or class II (insensitive to glyphosate). This basis for differences in sensitivity was confirmed using in vitro experiments in which the EPSPS gene from bee gut bacteria was cloned into <em>Escherichia coli</em>. All strains of the core bee gut species, <em>Snodgrassella alvi</em>, encode a sensitive class I EPSPS, and reduction in <em>S. alvi</em> levels was a consistent experimental result. However, some <em>S. alvi</em> strains appear to possess an alternative mechanism of glyphosate resistance. Thus, exposure of bees to glyphosate can perturb their beneficial gut microbiota, potentially affecting bee health and their effectiveness as pollinators.</li>
</ul>
</li>
<li>
<p><em>glyphosate’s suppression of cytochrome p450 enzymes and amino acid biosynthesis by the gut microbiome: pathways to modern diseases</em><br />
anthony samsel and stephanie seneff 2013<br />
<a href="http://dx.doi.org/10.3390/e15041416">http://dx.doi.org/10.3390/e15041416</a></p>
<ul>
<li>Glyphosate, the active ingredient in Roundup®, is the most popular herbicide used worldwide. The industry asserts it is minimally toxic to humans, but here we argue otherwise. Residues are found in the main foods of the Western diet, comprised primarily of sugar, corn, soy and wheat. Glyphosate's inhibition of cytochrome P450 (CYP) enzymes is an overlooked component of its toxicity to mammals. CYP enzymes play crucial roles in biology, one of which is to detoxify xenobiotics. Thus, glyphosate enhances the damaging effects of other food borne chemical residues and environmental toxins. Negative impact on the body is insidious and manifests slowly over time as inflammation damages cellular systems throughout the body. Here, we show how interference with CYP enzymes acts synergistically with disruption of the biosynthesis of aromatic amino acids by gut bacteria, as well as impairment in serum sulfate transport. Consequences are most of the diseases and conditions associated with a Western diet, which include gastrointestinal disorders, obesity, diabetes, heart disease, depression, autism, infertility, cancer and Alzheimer’s disease. We explain the documented effects of glyphosate and its ability to induce disease, and we show that glyphosate is the “textbook example” of exogenous semiotic entropy: the disruption of homeostasis by environmental toxins.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08094</h2>
<ul>
<li>induced by other medicines
<ul>
<li>
<p><em>the active component of aspirin, salicylic acid, promotes staphylococcus aureus biofilm formation in a pia-dependent manner</em><br />
cristian dotto et al. 2017<br />
<a href="https://doi.org/10.3389/fmicb.2017.00004">fmicb.2017.00004</a></p>
<ul>
<li>Aspirin has provided clear benefits to human health. But salicylic acid (SAL) -the main aspirin biometabolite- exerts several effects on eukaryote and prokaryote cells. SAL can affect, for instance, the expression of <em>Staphylococcus aureus</em> virulence factors. SAL can also form complexes with iron cations and it has been shown that different iron chelating molecules diminished the formation of <em>S. aureus</em> biofilm. The aim of this study was to elucidate whether the iron content limitation caused by SAL can modify the <em>S. aureus</em> metabolism and/or metabolic regulators thus changing the expression of the main polysaccharides involved in biofilm formation. The exposure of biofilm to 2 mM SAL induced a 27% reduction in the intracellular free Fe2+ concentration compared with the controls. In addition, SAL depleted 23% of the available free Fe2+ cation in culture media. These moderate iron-limited conditions promoted an intensification of biofilms formed by strain Newman and by <em>S. aureus</em> clinical isolates related to the USA300 and USA100 clones. The slight decrease in iron bioavailability generated by SAL was enough to induce the increase of PIA expression in biofilms formed by methicillin-resistant as well as methicillin-sensitive <em>S. aureus</em> strains. <em>S. aureus</em> did not produce capsular polysaccharide (CP) when it was forming biofilms under any of the experimental conditions tested. Furthermore, SAL diminished aconitase activity and stimulated the lactic fermentation pathway in bacteria forming biofilms. The polysaccharide composition of <em>S. aureus</em> biofilms was examined and FTIR spectroscopic analysis revealed a clear impact of SAL in a <em>codY</em>-dependent manner. Moreover, SAL negatively affected <em>codY</em> transcription in mature biofilms thus relieving the CodY repression of the <em>ica</em> operon. Treatment of mice with SAL induced a significant increase of <em>S aureus</em> colonization. It is suggested that the elevated PIA expression induced by SAL might be responsible for the high nasal colonization observed in mice. SAL-induced biofilms may contribute to <em>S. aureus</em> infection persistence in vegetarian individuals as well as in patients that frequently consume aspirin.</li>
</ul>
</li>
<li>
<p><em>nonsteroidal anti-inflammatory drugs alter the microbiota and exacerbate clostridium difficile colitis while dysregulating the inflammatory response</em><br />
damian maseda et al. 2019<br />
<a href="http://dx.doi.org/10.1128/mbio.02282-18">http://dx.doi.org/10.1128/mbio.02282-18</a></p>
<ul>
<li>
<p>The researchers followed two groups of antibiotic-treated mice for one week after infection with C. difficile. One group had been treated with an NSAID called indomethacin prior to infection, and the other hadn't. Only about 20 percent of the mice treated with the NSAID survived to the end of the observation period, compared to about 80 percent of the mice that hadn't been exposed to the NSAID.</p>
<p>Aronoff and his collaborators determined that even brief exposure to the NSAID prior to C. difficile inoculation increased the severity of infections and shortened survival. Further cellular and genetic analyses revealed that the NSAID exposure altered the gut microbiota and depleted the production of prostaglandins, hormone-like substances known to play an important role in gastrointestinal health. Those observations align with previous studies reporting that NSAIDs can cause or exacerbate an inflammatory disease called colitis, also by inhibiting the body's production of prostaglandins.</p>
<p>In the new study, the researchers conclude that NSAID-driven changes worsened C. difficile infections by impairing epithelial cells -- the main defense system in the intestine against infectious taxa -- and by disturbing the normal immune response. They studied at the impact of only one NSAID, indomethacin, but Aronoff says he thinks the findings might extend to other common NSAIDs, including ibuprofen and aspirin, since they all have roughly the same biological mechanism.</p>
<p>&quot;Ultimately, these new results might guide how we treat people with C. diff, particularly with pain management,&quot; says Aronoff. &quot;Right now, it's too early for our results to guide clinical care, but they should be a stimulus for future studies.&quot;</p>
</li>
<li>
<p><em>abstract</em> Clostridium difficile infection (CDI) is a major public health threat worldwide. The use of nonsteroidal anti-inflammatory drugs (NSAIDs) is associated with enhanced susceptibility to and severity of CDI; however, the mechanisms driving this phenomenon have not been elucidated. NSAIDs alter prostaglandin (PG) metabolism by inhibiting cyclooxygenase (COX) enzymes. Here, we found that treatment with the NSAID indomethacin prior to infection altered the microbiota and dramatically increased mortality and the intestinal pathology associated with CDI in mice. We demonstrated that in C. difficile-infected animals, indomethacin treatment led to PG deregulation, an altered proinflammatory transcriptional and protein profile, and perturbed epithelial cell junctions. These effects were paralleled by increased recruitment of intestinal neutrophils and CD4+ cells and also by a perturbation of the gut microbiota. Together, these data implicate NSAIDs in the disruption of protective COX-mediated PG production during CDI, resulting in altered epithelial integrity and associated immune responses.</p>
</li>
</ul>
</li>
<li>
<p><em>dietary zinc alters the microbiota and decreases resistance to clostridium difficile infection</em><br />
joseph p zackular et al. 2016<br />
<a href="http://dx.doi.org/10.1038/nm.4174">http://dx.doi.org/10.1038/nm.4174</a></p>
<ul>
<li>Clostridium difficile is the most commonly reported nosocomial pathogen in the United States and is an urgent public health concern worldwide1. Over the past decade, incidence, severity and costs associated with C. difficile infection (CDI) have increased dramatically2. CDI is most commonly initiated by antibiotic-mediated disruption of the gut microbiota; however, non-antibiotic-associated CDI cases are well documented and on the rise3, 4. This suggests that unexplored environmental, nutrient and host factors probably influence CDI. Here we show that excess dietary zinc (Zn) substantially alters the gut microbiota and, in turn, reduces the minimum amount of antibiotics needed to confer susceptibility to CDI. In mice colonized with C. difficile, excess dietary Zn severely exacerbated C. difficile–associated disease by increasing toxin activity and altering the host immune response. In addition, we show that the Zn-binding S100 protein calprotectin has antimicrobial effects against C. difficile and is an essential component of the innate immune response to CDI. Taken together, these data suggest that nutrient Zn levels have a key role in determining susceptibility to CDI and severity of disease, and that calprotectin-mediated metal limitation is an important factor in the host immune response to C. difficile.</li>
</ul>
</li>
<li>
<p><em>zno nanoparticles affect intestinal function in an in vitro model</em><br />
fabiola moreno-olivas et al. 2018<br />
<a href="http://dx.doi.org/10.1039/c7fo02038d">http://dx.doi.org/10.1039/c7fo02038d</a></p>
<ul>
<li>Zinc oxide nanoparticles (ZnO NP) may be present in food packaging, which would put consumers at risk of NP ingestion. There is little information on the amount of ZnO NP that are present in food packaging and the effects of ZnO exposure on intestinal function. To estimate physiologically relevant ZnO exposures, foods that are naturally low in zinc (Zn), but are commonly packaged with ZnO NP, such as tuna, corn, and asparagus, were analyzed with inductively coupled plasma mass spectrometry (ICP-MS). It was found that the Zn present in a serving of these foods is approximately one hundred times higher than the recommended dietary allowance. An <em>in vitro</em> model of the small intestine composed of Caco-2 and HT29-MTX cells was used to investigate the effects of ZnO NP exposure. Cells were exposed to physiologically realistic doses of pristine NP in culture medium and to NP subjected to an <em>in vitro</em> digestion to better reflect the transformation that the NP may undergo once they enter the human GI tract. Uptake and/or transport of iron (Fe), Zn, glucose, and fatty acids were assessed and intestinal alkaline phosphatase (IAP) levels were measured before and after NP exposure. The findings show that there is a 75% decrease in Fe transport and a 30% decrease in glucose transport following ZnO NP exposure. These decreases were consistent with gene expression changes for their transporters. There is also evidence that the ZnO NP affect the microvilli of the intestinal cells, therefore reducing the amount of surface area available to absorb nutrients. These results suggest that the ingestion of physiologically relevant doses of ZnO NP can alter intestinal function in an <em>in vitro</em> model of the human small intestine.</li>
</ul>
</li>
<li>
<p><em>measuring artificial sweeteners toxicity using a bioluminescent bacterial panel</em><br />
dorin harpaz et al. 2018<br />
<a href="http://dx.doi.org/10.3390/molecules23102454">http://dx.doi.org/10.3390/molecules23102454</a></p>
<ul>
<li><em>abstract</em> Artificial sweeteners have become increasingly controversial due to their questionable influence on consumers’ health. They are introduced in most foods and many consume this added ingredient without their knowledge. Currently, there is still no consensus regarding the health consequences of artificial sweeteners intake as they have not been fully investigated. Consumption of artificial sweeteners has been linked with adverse effects such as cancer, weight gain, metabolic disorders, type-2 diabetes and alteration of gut microbiota activity. Moreover, artificial sweeteners have been identified as emerging environmental pollutants, and can be found in receiving waters, i.e., surface waters, groundwater aquifers and drinking waters. In this study, the relative toxicity of six FDA-approved artificial sweeteners (aspartame, sucralose, saccharine, neotame, advantame and acesulfame potassium-k (ace-k)) and that of ten sport supplements containing these artificial sweeteners, were tested using genetically modified bioluminescent bacteria from <em>E. coli</em>. The bioluminescent bacteria, which luminesce when they detect toxicants, act as a sensing model representative of the complex microbial system. Both induced luminescent signals and bacterial growth were measured. Toxic effects were found when the bacteria were exposed to certain concentrations of the artificial sweeteners. In the bioluminescence activity assay, two toxicity response patterns were observed, namely, the induction and inhibition of the bioluminescent signal. An inhibition response pattern may be observed in the response of sucralose in all the tested strains: TV1061 (MLIC = 1 mg/mL), DPD2544 (MLIC = 50 mg/mL) and DPD2794 (MLIC = 100 mg/mL). It is also observed in neotame in the DPD2544 (MLIC = 2 mg/mL) strain. On the other hand, the induction response pattern may be observed in its response in saccharin in TV1061 (MLIndC = 5 mg/mL) and DPD2794 (MLIndC = 5 mg/mL) strains, aspartame in DPD2794 (MLIndC = 4 mg/mL) strain, and ace-k in DPD2794 (MLIndC = 10 mg/mL) strain. The results of this study may help in understanding the relative toxicity of artificial sweeteners on <em>E. coli</em>, a sensing model representative of the gut bacteria. Furthermore, the tested bioluminescent bacterial panel can potentially be used for detecting artificial sweeteners in the environment, using a specific mode-of-action pattern</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08194</h2>
<ul>
<li>stress, mechanical and psychological
<ul>
<li>
<p><em>stressor interaction networks suggest antibiotic resistance co-opted from stress responses to temperature</em><br />
mauricio cruz-loya et al. 2018<br />
<a href="http://dx.doi.org/10.1038/s41396-018-0241-7">http://dx.doi.org/10.1038/s41396-018-0241-7</a></p>
<ul>
<li>Environmental factors like temperature, pressure, and pH partly shaped the evolution of life. As life progressed, new stressors (e.g., poisons and antibiotics) arose as part of an arms race among organisms. Here we ask if cells co-opted existing mechanisms to respond to new stressors, or whether new responses evolved de novo. We use a network-clustering approach based purely on phenotypic growth measurements and interactions among the effects of stressors on population growth. We apply this method to two types of stressors—temperature and antibiotics—to discover the extent to which their cellular responses overlap in Escherichia coli. Our clustering reveals that responses to low and high temperatures are clearly separated, and each is grouped with responses to antibiotics that have similar effects to cold or heat, respectively. As further support, we use a library of transcriptional fluorescent reporters to confirm heat-shock and cold-shock genes are induced by antibiotics. We also show strains evolved at high temperatures are more sensitive to antibiotics that mimic the effects of cold. Taken together, our results strongly suggest that temperature stress responses have been co-opted to deal with antibiotic stress.</li>
</ul>
</li>
<li>
<p><em>self-induced mechanical stress can trigger biofilm formation in uropathogenic escherichia coli</em><br />
eric k. chu  et al. 2018<br />
<a href="https://doi.org/10.1038/s41467-018-06552-z">https://doi.org/10.1038/s41467-018-06552-z</a></p>
<ul>
<li>Bacterial biofilms represent an important medical problem; however, the mechanisms of the onset of biofilm formation are poorly understood. Here, using new controlled methods allowing high-throughput and reproducible biofilm growth, we show that biofilm formation is linked to self-imposed mechanical stress. In growing uropathogenic Escherichia coli colonies, we report that mechanical stress can initially emerge from the physical stress accompanying colony confinement within micro-cavities or hydrogel environments reminiscent of the cytosol of host cells. Biofilm formation can then be enhanced by a nutrient access-modulated feedback loop, in which biofilm matrix deposition can be particularly high in areas of increased mechanical and biological stress, with the deposited matrix further enhancing the stress levels. This feedback regulation can lead to adaptive and diverse biofilm formation guided by the environmental stresses. Our results suggest previously unappreciated mechanisms of the onset and progression of biofilm growth</li>
</ul>
</li>
<li>
<p>prenatal exposure to a mother’s stress contributes to anxiety and cognitive problems that persist into adulthood, a phenomenon that could be explained by lasting – and potentially damaging – changes in the microbiome, according to new research in mice</p>
<p><a href="https://www.sciencedaily.com/releases/2016/11/161114124958.htm">161114124958.htm</a></p>
</li>
</ul>
</li>
</ul>
<h2>08294</h2>
<ul>
<li>nitrogen related
<ul>
<li>
<p><em>nitrated meat products are associated with mania in humans and altered behavior and brain gene expression in rats</em><br />
seva g. khambadkone et al. 2018<br />
<a href="https://doi.org/10.1038/s41380-018-0105-6">https://doi.org/10.1038/s41380-018-0105-6</a></p>
<ul>
<li>Mania is a serious neuropsychiatric condition associated with significant morbidity and mortality. Previous studies have suggested that environmental exposures can contribute to mania pathogenesis. We measured dietary exposures in a cohort of individuals with mania and other psychiatric disorders as well as in control individuals without a psychiatric disorder. We found that a history of eating nitrated dry cured meat but not other meat or fish products was strongly and independently associated with current mania (adjusted odds ratio 3.49, 95% confidence interval (CI) 2.24–5.45, p &lt; 8.97 × 10−8). Lower odds of association were found between eating nitrated dry cured meat and other psychiatric disorders. We further found that the feeding of meat preparations with added nitrate to rats resulted in hyperactivity reminiscent of human mania, alterations in brain pathways that have been implicated in human bipolar disorder, and changes in intestinal microbiota. These findings may lead to new methods for preventing mania and for developing novel therapeutic interventions.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08394</h2>
<ul>
<li>parkinsons, alzheimers
<ul>
<li>
<p><em>the vermiform appendix impacts the risk of developing parkinson’s disease</em><br />
bryan a. killinger et al. 2018<br />
<a href="http://dx.doi.org/10.1126/scitranslmed.aar5280">http://dx.doi.org/10.1126/scitranslmed.aar5280</a></p>
<ul>
<li>
<p>“ “Our results point to the appendix as a site of origin for Parkinson’s and provide a path forward for devising new treatment strategies that leverage the gastrointestinal tract’s role in the development of the disease,” said Viviane Labrie, Ph.D., an assistant professor at Van Andel Research Institute (VARI) and senior author of the study. “Despite having a reputation as largely unnecessary, the appendix actually plays a major part in our immune systems, in regulating the makeup of our gut bacteria and now, as shown by our work, in Parkinson’s disease.”<br />
The reduced risk for Parkinson’s was only apparent when the appendix and the alpha-synuclein contained within it were removed early in life, years before the onset of Parkinson’s, suggesting that the appendix may be involved in disease initiation. Removal of the appendix after the disease process starts, however, had no effect on disease progression.<br />
In a general population, people who had an appendectomy were 19 percent less likely to develop Parkinson’s. This effect was magnified in people who live in rural areas, with appendectomies resulting in a 25 percent reduction in disease risk. Parkinson’s often is more prevalent in rural populations, a trend that has been associated with increased exposure to pesticides.<br />
The study also demonstrated that appendectomy can delay disease progression in people who go on to develop Parkinson’s, pushing back diagnosis by an average of 3.6 years. Because there are no definitive tests for Parkinson’s, people often are diagnosed after motor symptoms such as tremor or rigidity arise. By then, the disease typically is quite advanced, with significant damage to the area of the brain that regulates voluntary movement.<br />
Conversely, appendectomies had no apparent benefit in people whose disease was linked to genetic mutations passed down through their families, a group that comprises fewer than 10 percent of cases.<br />
“Our findings today add a new layer to our understanding of this incredibly complex disease,” said Bryan Killinger, Ph.D., the study’s first author and a postdoctoral fellow in Labrie’s laboratory. “We have shown that the appendix is a hub for the accumulation of clumped forms of alpha-synuclein proteins, which are implicated in Parkinson’s disease. This knowledge will be invaluable as we explore new prevention and treatment strategies.”<br />
Labrie and her team also found clumps of alpha-synuclein in the appendixes of healthy people of all ages as well as people with Parkinson’s, raising new questions about the mechanisms that give rise to the disease and propel its progression. Clumped alpha-synuclein is considered to be a key hallmark of Parkinson’s; previously, it was thought to only be present in people with the disease.<br />
“We were surprised that pathogenic forms of alpha-synuclein were so pervasive in the appendixes of people both with and without Parkinson’s. It appears that these aggregates — although toxic when in the brain — are quite normal when in the appendix. This clearly suggests their presence alone cannot be the cause of the disease,” Labrie said. “Parkinson’s is relatively rare — less than 1 percent of the population — so there has to be some other mechanism or confluence of events that allows the appendix to affect Parkinson’s risk. That’s what we plan to look at next; which factor or factors tip the scale in favor of Parkinson’s?”<br />
Data for the study were gleaned from an in-depth characterization and visualization of alpha-synuclein forms in the appendix, which bore a remarkable resemblance to those found in the Parkinson’s disease brain, as well as analyses of two large health-record databases. The first dataset was garnered from the Swedish National Patient Registry, a one-of-a-kind database that contains de-identified medical diagnoses and surgical histories for the Swedish population beginning in 1964, and Statistics Sweden, a Swedish governmental agency responsible for official national statistics. The team at VARI collaborated with researchers at Lund University, Sweden, to comb through records for 1,698,000 people followed up to 52 years, a total of nearly 92 million person-years. The second dataset was from the Parkinson’s Progression Marker Initiative (PPMI), which includes details about patient diagnosis, age of onset, demographics and genetic information. ”</p>
</li>
<li>
<p>“Misfolded α-synuclein is a pathological hallmark of Parkinson’s disease (PD). Killinger et al. now report that the human appendix contains an abundance of misfolded α-synuclein and that removal of the appendix decreased the risk of developing PD. The appendix of both PD cases and healthy individuals contained abnormally cleaved and aggregated forms of α-synuclein, analogous to those found in postmortem brain tissue from patients with PD. Furthermore, α-synuclein derived from the appendix seeded rapid aggregation of recombinant α-synuclein in vitro. In two large-scale epidemiological studies, the authors demonstrated that an appendectomy occurring decades prior reduced the risk of developing PD, suggesting that the appendix may be implicated in PD initiation.”</p>
</li>
<li>
<p><em>abstract</em> The pathogenesis of Parkinson’s disease (PD) involves the accumulation of aggregated α-synuclein, which has been suggested to begin in the gastrointestinal tract. Here, we determined the capacity of the appendix to modify PD risk and influence pathogenesis. In two independent epidemiological datasets, involving more than 1.6 million individuals and over 91 million person-years, we observed that removal of the appendix decades before PD onset was associated with a lower risk for PD, particularly for individuals living in rural areas, and delayed the age of PD onset. We also found that the healthy human appendix contained intraneuronal α-synuclein aggregates and an abundance of PD pathology–associated α-synuclein truncation products that are known to accumulate in Lewy bodies, the pathological hallmark of PD. Lysates of human appendix tissue induced the rapid cleavage and oligomerization of full-length recombinant α-synuclein. Together, we propose that the normal human appendix contains pathogenic forms of α-synuclein that affect the risk of developing PD.</p>
</li>
</ul>
</li>
<li>
<p><em>gut microbiota regulate motor deficits and neuroinflammation in a model of parkinson’s disease</em><br />
timothy r. sampson et al. 2016<br />
<a href="http://dx.doi.org/10.1016/j.cell.2016.11.018">j.cell.2016.11.018</a></p>
<ul>
<li>
<p>Gut microbes promote α-synuclein-mediated motor deficits and brain pathology</p>
<p>Depletion of gut bacteria reduces microglia activation</p>
<p>SCFAs modulate microglia and enhance PD pathophysiology</p>
<p>Human gut microbiota from PD patients induce enhanced motor dysfunction in mice</p>
<p>The intestinal microbiota influence neurodevelopment, modulate behavior, and contribute to neurological disorders. However, a functional link between gut bacteria and neurodegenerative diseases remains unexplored. Synucleinopathies are characterized by aggregation of the protein α-synuclein (αSyn), often resulting in motor dysfunction as exemplified by Parkinson’s disease (PD). Using mice that overexpress αSyn, we report herein that gut microbiota are required for motor deficits, microglia activation, and αSyn pathology. Antibiotic treatment ameliorates, while microbial re-colonization promotes, pathophysiology in adult animals, suggesting that postnatal signaling between the gut and the brain modulates disease. Indeed, oral administration of specific microbial metabolites to germ-free mice promotes neuroinflammation and motor symptoms. Remarkably, colonization of αSyn-overexpressing mice with microbiota from PD-affected patients enhances physical impairments compared to microbiota transplants from healthy human donors. These findings reveal that gut bacteria regulate movement disorders in mice and suggest that alterations in the human microbiome represent a risk factor for PD.</p>
</li>
</ul>
</li>
<li>
<p><em>the role of microbial amyloid in neurodegeneration</em><br />
robert p. friedland, matthew r. chapman 2017<br />
<a href="http://dx.doi.org/10.1371/journal.ppat.1006654">http://dx.doi.org/10.1371/journal.ppat.1006654</a></p>
<ul>
<li>It has become apparent that the intestinal microbiota orchestrates important aspects of our metabolism, immunity, and development. Recent work has demonstrated that the microbiota also influences brain function in healthy and diseased individuals. Of great interest are reports that intestinal bacteria play a role in the pathogenic cascade of both Parkinson and Alzheimer diseases. These neurodegenerative disorders both involve misfolding of endogenous proteins that spreads from one region of the body to another in a manner analogous to prions. The mechanisms of how the microbiota influences or is correlated with disease require elaboration. Microbial proteins or metabolites may influence neurodegeneration through the promotion of amyloid formation by human proteins or by enhancing inflammatory responses to endogenous neuronal amyloids. We review the current knowledge concerning bacterial amyloids and their potential to influence cerebral amyloid aggregation and neuroinflammation. We propose the term “mapranosis” to describe the process of microbiota-associated proteopathy and neuroinflammation. The study of amyloid proteins made by the microbiota and their influence on health and disease is in its infancy. This is a promising area for therapeutic intervention because there are many ways to alter our microbial partners and their products, including amyloid proteins.</li>
</ul>
</li>
<li>
<p><em>reduction of abeta amyloid pathology in appps1 transgenic mice in the absence of gut microbiota</em><br />
t. harach et al. 2017<br />
<a href="http://dx.doi.org/10.1038/srep41802">srep41802</a></p>
<ul>
<li>Alzheimer’s disease is the most common form of dementia in the western world, however there is no cure available for this devastating neurodegenerative disorder. Despite clinical and experimental evidence implicating the intestinal microbiota in a number of brain disorders, its impact on Alzheimer’s disease is not known. To this end we sequenced bacterial 16S rRNA from fecal samples of Aβ precursor protein (APP) transgenic mouse model and found a remarkable shift in the gut microbiota as compared to non-transgenic wild-type mice. Subsequently we generated germ-free APP transgenic mice and found a drastic reduction of cerebral Aβ amyloid pathology when compared to control mice with intestinal microbiota. Importantly, colonization of germ-free APP transgenic mice with microbiota from conventionally-raised APP transgenic mice increased cerebral Aβ pathology, while colonization with microbiota from wild-type mice was less effective in increasing cerebral Aβ levels. Our results indicate a microbial involvement in the development of Abeta amyloid pathology, and suggest that microbiota may contribute to the development of neurodegenerative diseases.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08494</h2>
<ul>
<li>cancer
<ul>
<li>
<p><em>breast tissue, oral and urinary microbiomes in breast cancer</em><br />
hannah wang et al. 2017<br />
<a href="http://dx.doi.org/10.18632/oncotarget.21490">dx.doi.org/10.18632/oncotarget.21490</a></p>
<ul>
<li>It has long been proposed that the gut microbiome contributes to breast carcinogenesis by modifying systemic estrogen levels. This is often cited as a possible mechanism linking breast cancer and high-fat, low-fiber diets as well as antibiotic exposure, associations previously identified in population-based studies. More recently, a distinct microbiome has been identified within breast milk and tissue, but few studies have characterized differences in the breast tissue microbiota of patients with and without cancer, and none have investigated distant body-site microbiomes outside of the gut. We hypothesize that cancerous breast tissue is associated with a microbiomic profile distinct from that of benign breast tissue, and that microbiomes of more distant sites, the oral cavity and urinary tract, will reflect dysbiosis as well. Fifty-seven women with invasive breast cancer undergoing mastectomy and 21 healthy women undergoing cosmetic breast surgery were enrolled. The bacterial 16S rRNA gene was amplified from urine, oral rinse and surgically collected breast tissue, sequenced, and processed through a QIIME-based bioinformatics pipeline. Cancer patient breast tissue microbiomes clustered significantly differently from non-cancer patients (<em>p</em>=0.03), largely driven by decreased relative abundance of <em>Methylobacterium</em> in cancer patients (median 0.10 vs. 0.24, <em>p</em>=0.03). There were no significant differences in oral rinse samples. Differences in urinary microbiomes were largely explained by menopausal status, with peri/postmenopausal women showing decreased levels of <em>Lactobacillus</em>. Independent of menopausal status, however, cancer patients had increased levels of gram-positive organisms including <em>Corynebacterium</em> (<em>p</em>&lt;0.01), <em>Staphylococcus</em> (<em>p</em>=0.02)<em>, Actinomyces</em> (<em>p</em>&lt;0.01), and Propionibacteriaceae (<em>p</em>&lt;0.01). Our observations suggest that the local breast microbiota differ in patients with and without breast cancer. Cancer patient urinary microbiomes were characterized by increased levels of gram-positive organisms in this study, but need to be further studied in larger cohorts.</li>
</ul>
</li>
<li>
<p><em>patients with familial adenomatous polyposis harbor colonic biofilms containing tumorigenic bacteria</em><br />
christine m. dejea et al. 2018<br />
<a href="http://dx.doi.org/10.1126/science.aah3648">http://dx.doi.org/10.1126/science.aah3648</a></p>
<ul>
<li>Individuals with sporadic colorectal cancer (CRC) frequently harbor abnormalities in the composition of the gut microbiome; however, the microbiota associated with precancerous lesions in hereditary CRC remains largely unknown. We studied colonic mucosa of patients with familial adenomatous polyposis (FAP), who develop benign precursor lesions (polyps) early in life. We identified patchy bacterial biofilms composed predominately of <em>Escherichia coli</em> and <em>Bacteroides fragilis</em>. Genes for colibactin (<em>clbB</em>) and <em>Bacteroides fragilis</em> toxin (<em>bft</em>), encoding secreted oncotoxins, were highly enriched in FAP patients’ colonic mucosa compared to healthy individuals. Tumor-prone mice cocolonized with <em>E. coli</em> (expressing colibactin), and enterotoxigenic <em>B. fragilis</em> showed increased interleukin-17 in the colon and DNA damage in colonic epithelium with faster tumor onset and greater mortality, compared to mice with either bacterial strain alone. These data suggest an unexpected link between early neoplasia of the colon and tumorigenic bacteria.</li>
</ul>
</li>
<li>
<p><em>bacteroides fragilis toxin coordinates a pro-carcinogenic inflammatory cascade via targeting of colonic epithelial cells</em><br />
liam chung et al. 2018<br />
<a href="http://dx.doi.org/10.1016/j.chom.2018.01.007">http://dx.doi.org/10.1016/j.chom.2018.01.007</a></p>
<ul>
<li>
<p>•<em>B. fragilis</em> toxin-induced tumorigenesis requires epithelial IL17 and Stat3 signaling<br />
•IL17 targets colonic epithelial cells (CECs) to promote ETBF-mediated carcinogenesis<br />
•IL17-activated NF-κb signaling in CECs triggers C-X-C chemokine expression<br />
•NF-κB-induced chemokines direct pro-tumoral myeloid infiltration to distal colon</p>
<p>Pro-carcinogenic bacteria have the potential to initiate and/or promote colon cancer, in part via immune mechanisms that are incompletely understood. Using <em>ApcMin</em> mice colonized with the human pathobiont enterotoxigenic <em>Bacteroides fragilis</em> (ETBF) as a model of microbe-induced colon tumorigenesis, we show that the <em>Bacteroides fragilis</em> toxin (BFT) triggers a pro-carcinogenic, multi-step inflammatory cascade requiring IL-17R, NF-κB, and Stat3 signaling in colonic epithelial cells (CECs). Although necessary, Stat3 activation in CECs is not sufficient to trigger ETBF colon tumorigenesis. Notably, IL-17-dependent NF-κB activation in CECs induces a proximal to distal mucosal gradient of C-X-C chemokines, including CXCL1, that mediates the recruitment of CXCR2-expressing polymorphonuclear immature myeloid cells with parallel onset of ETBF-mediated distal colon tumorigenesis. Thus, BFT induces a pro-carcinogenic signaling relay from the CEC to a mucosal Th17 response that results in selective NF-κB activation in distal colon CECs, which collectively triggers myeloid-cell-dependent distal colon tumorigenesis.</p>
</li>
</ul>
</li>
<li>
<p><em>a causal mechanism for childhood acute lymphoblastic leukaemia</em><br />
mel greaves 2018<br />
<a href="http://dx.doi.org/10.1038/s41568-018-0015-6">http://dx.doi.org/10.1038/s41568-018-0015-6</a></p>
<ul>
<li>In this Review, I present evidence supporting a multifactorial causation of childhood acute lymphoblastic leukaemia (ALL), a major subtype of paediatric cancer. ALL evolves in two discrete steps. First, in utero initiation by fusion gene formation or hyperdiploidy generates a covert, pre-leukaemic clone. Second, in a small fraction of these cases, the postnatal acquisition of secondary genetic changes (primarily V(D)J recombination-activating protein (RAG) and activation-induced cytidine deaminase (AID)-driven copy number alterations in the case of ETS translocation variant 6 (<em>ETV6</em>)–runt-related transcription factor 1 (<em>RUNX1</em>)+ ALL) drives conversion to overt leukaemia. Epidemiological and modelling studies endorse a dual role for common infections. Microbial exposures earlier in life are protective but, in their absence, later infections trigger the critical secondary mutations. Risk is further modified by inherited genetics, chance and, probably, diet. Childhood ALL can be viewed as a paradoxical consequence of progress in modern societies, where behavioural changes have restrained early microbial exposure. This engenders an evolutionary mismatch between historical adaptations of the immune system and contemporary lifestyles. Childhood ALL may be a preventable cancer.</li>
</ul>
</li>
<li>
<p><em>activated atf6 induces intestinal dysbiosis and innate immune response to promote colorectal tumorigenesis</em><br />
o.i. coleman et al. 2018<br />
<a href="http://dx.doi.org/10.1053/j.gastro.2018.07.028">http://dx.doi.org/10.1053/j.gastro.2018.07.028</a></p>
<ul>
<li>In patients with CRC, ATF6 was associated with reduced time of disease-free survival. In studies of nATF6IEC mice, we found sustained intestinal activation of ATF6 in the colon to promote dysbiosis and microbiota-dependent tumorigenesis.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08594</h2>
<ul>
<li>immune system, inflammation, allergy
<ul>
<li>
<p><em>nlrp1 restricts butyrate producing commensals to exacerbate inflammatory bowel disease</em><br />
hazel tye et al. 2018<br />
<a href="http://dx.doi.org/10.1038/s41467-018-06125-0">http://dx.doi.org/10.1038/s41467-018-06125-0</a></p>
<ul>
<li><em>abstract</em> Anti-microbial signaling pathways are normally triggered by innate immune receptors when detecting pathogenic microbes to provide protective immunity. Here we show that the inflammasome sensor Nlrp1 aggravates DSS-induced experimental mouse colitis by limiting beneficial, butyrate-producing Clostridiales in the gut. The colitis-protective effects of Nlrp1 deficiency are thus reversed by vancomycin treatment, but recapitulated with butyrate supplementation in wild-type mice. Moreover, an activating mutation in Nlrp1a increases IL-18 and IFNγ production, and decreases colonic butyrate to exacerbate colitis. We also show that, in patients with ulcerative colitis, increased NLRP1 in inflamed regions of the colon is associated with increased IFN-γ. In this context, NLRP1, IL-18 or IFN-γ expression negatively correlates with the abundance of Clostridiales in human rectal mucosal biopsies. Our data identify the NLRP1 inflammasome to be a key negative regulator of protective, butyrate-producing commensals, which therefore promotes inflammatory bowel disease.</li>
</ul>
</li>
<li>
<p><em>translocation of a gut pathobiont drives autoimmunity in mice and humans</em><br />
s. manfredo vieira et al. 2018<br />
<a href="http://dx.doi.org/10.1126/science.aar7201">http://dx.doi.org/10.1126/science.aar7201</a></p>
<ul>
<li>Despite multiple associations between the microbiota and immune diseases, their role in autoimmunity is poorly understood. We found that translocation of a gut pathobiont, <em>Enterococcus gallinarum</em>, to the liver and other systemic tissues triggers autoimmune responses in a genetic background predisposing to autoimmunity. Antibiotic treatment prevented mortality in this model, suppressed growth of <em>E. gallinarum</em> in tissues, and eliminated pathogenic autoantibodies and T cells. Hepatocyte–<em>E. gallinarum</em> cocultures induced autoimmune-promoting factors. Pathobiont translocation in monocolonized and autoimmune-prone mice induced autoantibodies and caused mortality, which could be prevented by an intramuscular vaccine targeting the pathobiont. <em>E. gallinarum</em>–specific DNA was recovered from liver biopsies of autoimmune patients, and cocultures with human hepatocytes replicated the murine findings; hence, similar processes apparently occur in susceptible humans. These discoveries show that a gut pathobiont can translocate and promote autoimmunity in genetically predisposed hosts.</li>
</ul>
</li>
<li>
<p><em>role of short chain fatty acids in controlling tregs and immunopathology during mucosal infection</em><br />
natarajan bhaskaran et al. 2018<br />
<a href="http://dx.doi.org/10.3389/fmicb.2018.01995">http://dx.doi.org/10.3389/fmicb.2018.01995</a></p>
<ul>
<li>Interactions between mucosal tissues and commensal microbes control appropriate host immune responses and inflammation, but very little is known about these interactions. Here we show that the depletion of resident bacteria using antibiotics (Abx) causes oral and gut immunopathology during oropharyngeal candidiasis (OPC) infection. Antibiotic treatment causes reduction in the frequency of Foxp3+ regulatory cells (Tregs) and IL-17A producers, with a concomitant increase in oral tissue pathology. While <em>C. albicans</em> (CA) is usually controlled in the oral cavity, antibiotic treatment led to CA dependent oral and gut inflammation. A combination of short chain fatty acids (SCFA) controlled the pathology in Abx treated mice, correlating to an increase in the frequency of Foxp3+, IL-17A+, and Foxp3+IL-17A+ double positive (Treg17) cells in tongue and oral draining lymph nodes. However, SCFA treatment did not fully reverse the gut inflammation suggesting that resident microbiota have SCFA independent homeostatic mechanisms in gut mucosa. We also found that SCFA potently induce Foxp3 and IL-17A expression in CD4+ T cells, depending on the cytokine milieu <em>in vitro</em>. Depletion of Tregs alone in FDTR mice recapitulated oral inflammation in CA infected mice, showing that Abx mediated reduction of Tregs was involved in infection induced pathology. SCFA did not control inflammation in Treg depleted mice in CA infected FDTR mice, showing that Foxp3+ T cell induction was required for the protective effect mediated by SCFA. Taken together, our data reveal that SCFA derived from resident bacteria play a critical role in controlling immunopathology by regulating T cell cytokines during mucosal infections. This study has broader implications on protective effects of resident microbiota in regulating pathological infections.</li>
</ul>
</li>
<li>
<p><em>age-associated microbial dysbiosis promotes intestinal permeability, systemic inflammation, and macrophage dysfunction</em><br />
netusha thevaranjan et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.chom.2017.03.002">dx.doi.org/10.1016/j.chom.2017.03.002</a></p>
<ul>
<li>
<p>•Age-associated inflammation drives macrophage dysfunction and tissue damage<br />
•Mice under germ-free conditions are protected from age-associated inflammation<br />
•Co-housing germ-free mice with old, but not young, mice increases age-related inflammation<br />
•Age-related microbiota changes can be reversed by reducing TNF levels</p>
<p>Levels of inflammatory mediators in circulation are known to increase with age, but the underlying cause of this age-associated inflammation is debated. We find that, when maintained under germ-free conditions, mice do not display an age-related increase in circulating pro-inflammatory cytokine levels. A higher proportion of germ-free mice live to 600 days than their conventional counterparts, and macrophages derived from aged germ-free mice maintain anti-microbial activity. Co-housing germ-free mice with old, but not young, conventionally raised mice increases pro-inflammatory cytokines in the blood. In tumor necrosis factor (TNF)-deficient mice, which are protected from age-associated inflammation, age-related microbiota changes are not observed. Furthermore, age-associated microbiota changes can be reversed by reducing TNF using anti-TNF therapy. These data suggest that aging-associated microbiota promote inflammation and that reversing these age-related microbiota changes represents a potential strategy for reducing age-associated inflammation and the accompanying morbidity.</p>
</li>
</ul>
</li>
<li>
<p><em>recurrent infection progressively disables host protection against intestinal inflammation</em><br />
won ho yang et al. 2017<br />
<a href="http://dx.doi.org/10.1126/science.aao5610">http://dx.doi.org/10.1126/science.aao5610</a></p>
<ul>
<li>We have discovered an environmental and pathogenic origin of chronic intestinal inflammation using a model of recurrent human food poisoning. We further show that this disease can be prevented by either IAP augmentation or Neu inhibition, both of which may represent prophylactic and therapeutic approaches to human colitis and the IBDs.</li>
</ul>
</li>
<li>
<p><em>identification of the ligand of pru p 3, a peach ltp</em><br />
nuria cubells-baeza et al. 2017<br />
<a href="http://dx.doi.org/10.1007/s11103-017-0590-z">http://dx.doi.org/10.1007/s11103-017-0590-z</a></p>
<ul>
<li>
<p>certain compounds carried by allergic proteins would be collaborating agents needed in the processes that trigger immune responses with the appearance of allergic symptoms.<br />
This study was carried out by researchers from Mount Sinai Hospital in New York, Swiss Allergy Centre in Zurich and Institute of Applied Molecular Medicine of Universidad San Pablo CEU in Madrid. The obtained results will help develop preventive methods and effective treatments for allergies.<br />
In spite of a great effort, fundamental questions on the molecular and immunological origin of allergies are still ignored. Many studies have sought solutions to an ancient enigma: why some proteins cause allergies despite being very similar to others that are harmless. Researchers achieved to identify the proteins which are common allergens are in pollen, mites, domestic animals, food…<br />
Unfortunately, the characteristics of these proteins linked to allergens have been not found yet. However, in recent years a hypothesis developed in collaboration with the allergens group of CBGP has gained importance: certain compounds carried by the allergen proteins, known as ligands, would act as necessary collaborator agents in the allergic sensitization phase.<br />
Pru p 3 was the protein selected for this study. This protein is responsible for the peach allergy, being very common in Mediterranean countries. The CBGP team has recently identified in a study the natural ligand of Pru p 3 as a compound formed by an alkaloid attached to a hydrocarbon tail. In the current study, the team has found direct evidence of the participation of the Pru p 3 ligand in the processes of the immune system recognition in the allergic sensitization phase.<br />
Results reveal that the ligand is recognized by a type of cellular receptor called CD1d in the cell surface where the antigens appear, that is, substances able to provoke a response of the immune system to produce antibodies. The CD1d expressions are responsible for presenting lipid antigens activating cells of the immune system called iNKT (invariant natural killer T-cells). Once activated, these iNKT cells produce an enormous amount of substances that cause the characteristic symptoms of allergic disorders.<br />
Since many allergens transport diverse compounds, the discovery of Pru p 3 lipid-ligand as an adjuvant to promote allergic sensitization through its recognition by CD1d expressions open new horizons. This new discovery could be a general essential feature of the mechanism underlying the phenomenon of allergenicity.</p>
</li>
<li>
<p><em>abstract</em> The allergen Pru p 3, a peach lipid transfer protein, has been well studied. However, its physiological function remains to be elucidated. Our results showed that Pru p 3 usually carries a lipid ligand that play an essential role in its function in plants. Using ESI-qToF, we observed that the ligand was a derivative of camptothecin binding to phytosphingosine, wich that is inserted into the hydrophobic tunnel of the protein. In addition, the described ligand displayed topoisomerase I activity inhibition and self-fluorescence, both recognized as camptothecin properties. During flower development, the highest expression of <em>Pru p 3</em> was detected in the styles of pollinated flowers, in contrast to its non-expression in unpollinated pistils, where expression decreased after anthesis. During ripening, the expression of Pru p 3 were observed mainly in peel but not in pulp. In this sense, Pru p 3 protein was also localized in trichomes covering the fruit epidermis.</p>
</li>
</ul>
</li>
<li>
<p><em>loss of paneth cell autophagy causes acute susceptibility to toxoplasma gondii-mediated inflammation</em><br />
elise burger et al. 2018<br />
<a href="http://dx.doi.org/10.1016/j.chom.2018.01.001">http://dx.doi.org/10.1016/j.chom.2018.01.001</a></p>
<ul>
<li>
<p>•Basal Paneth cell autophagy is driven by the microbiota and IFN-γ<br />
•Paneth cell autophagy protects against acute infection<br />
•Loss of Paneth cell autophagy results in impaired intestinal permeability<br />
•Loss of Paneth cell autophagy leads to TNF- and IFN-γ mediated intestinal pathology</p>
<p>The protozoan parasite <em>Toxoplasma gondii</em> triggers severe small intestinal immunopathology characterized by IFN-γ- and intestinal microbiota-mediated inflammation, Paneth cell loss, and bacterial dysbiosis. Paneth cells are a prominent secretory epithelial cell type that resides at the base of intestinal crypts and releases antimicrobial peptides. We demonstrate that the microbiota triggers basal Paneth cell-specific autophagy via induction of IFN-γ, a known trigger of autophagy, to maintain intestinal homeostasis. Deletion of the autophagy protein Atg5 specifically in Paneth cells results in exaggerated intestinal inflammation characterized by complete destruction of the intestinal crypts resembling that seen in pan-epithelial Atg5-deficient mice. Additionally, lack of functional autophagy in Paneth cells within intestinal organoids and <em>T. gondii</em>-infected mice causes increased sensitivity to the proinflammatory cytokine TNF along with increased intestinal permeability, leading to exaggerated microbiota- and IFN-γ-dependent intestinal immunopathology. Thus, Atg5 expression in Paneth cells is essential for tissue protection against cytokine-mediated immunopathology during acute gastrointestinal infection.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08694</h2>
<ul>
<li>obesity, hypertension
<ul>
<li>
<p><em>molecular phenomics and metagenomics of hepatic steatosis in non-diabetic obese women</em><br />
lesley hoyles et al. 2018<br />
<a href="http://dx.doi.org/10.1038/s41591-018-0061-3">http://dx.doi.org/10.1038/s41591-018-0061-3</a></p>
<ul>
<li>Hepatic steatosis is a multifactorial condition that is often observed in obese patients and is a prelude to non-alcoholic fatty liver disease. Here, we combine shotgun sequencing of fecal metagenomes with molecular phenomics (hepatic transcriptome and plasma and urine metabolomes) in two well-characterized cohorts of morbidly obese women recruited to the FLORINASH study. We reveal molecular networks linking the gut microbiome and the host phenome to hepatic steatosis. Patients with steatosis have low microbial gene richness and increased genetic potential for the processing of dietary lipids and endotoxin biosynthesis (notably from Proteobacteria), hepatic inflammation and dysregulation of aromatic and branched-chain amino acid metabolism. We demonstrated that fecal microbiota transplants and chronic treatment with phenylacetic acid, a microbial product of aromatic amino acid metabolism, successfully trigger steatosis and branched-chain amino acid metabolism. Molecular phenomic signatures were predictive (area under the curve = 87%) and consistent with the gut microbiome having an effect on the steatosis phenome (&gt;75% shared variation) and, therefore, actionable via microbiome-based therapies.</li>
</ul>
</li>
<li>
<p><em>glucocorticoids and gut bacteria: “the galf hypothesis” in the metagenomic era</em><br />
david j. morris, jason m. ridlon 2017<br />
<a href="http://dx.doi.org/10.1016/j.steroids.2017.06.002">dx.doi.org/10.1016/j.steroids.2017.06.002</a></p>
<ul>
<li>
<p>The gut microbiota are implicated causally in essential hypertension.<br />
A proposed mechanism of essential hypertension is production of endogenous of glycerrhetinic acid-like factors (GALFs)<br />
GALFs act as competitive inhibitors of renal 11β-hydroxysteroid dehydrogenase.<br />
Gut bacteria are capable of converting host glucocorticoids into GALFs.</p>
<p>A new concept is emerging in biomedical sciences: the gut microbiota is a virtual ‘organ’ with endocrine function. Here, we explore the literature pertaining to the role of gut microbial metabolism of endogenous adrenocorticosteroids as a contributing factor in the etiology of essential hypertension. A body of literature demonstrates that bacterial products of glucocorticoid metabolism are absorbed into the portal circulation, and pass through the kidney before excretion into urine. Apparent mineralocorticoid excess (AME) syndrome patients were found to have congenital mutations resulting in non-functional renal 11β-hydroxysteroid dehydrogenase-2 (11β-HSD2) and severe hypertension often lethal in childhood. 11β-HSD2 acts as a “guardian” enzyme protecting the mineralocorticoid receptor from excess cortisol, preventing sodium and water retention in the normotensive state. Licorice root, whose active ingredient, glycerrhetinic acid (GA), inhibits renal 11β-HSD2, and thereby causes hypertension in some individuals. Bacterially derived glucocorticoid metabolites may cause hypertension in some patients by a similar mechanism. Parallel observations in gut microbiology coupled with screening of endogenous steroids as inhibitors of 11β-HSD2 have implicated particular gut bacteria in essential hypertension through the production of glycerrhetinic acid-like factors (GALFs). A protective role of GALFs produced by gut bacteria in the etiology of colorectal cancer is also explored.</p>
</li>
</ul>
</li>
<li>
<p><em>characterization of metabolic responses to healthy diets and association with blood pressure: application to the optimal macronutrient intake trial for heart health (omniheart), a randomized controlled study</em><br />
ruey leng loo et al. 2018<br />
<a href="http://dx.doi.org/10.1093/ajcn/nqx072">http://dx.doi.org/10.1093/ajcn/nqx072</a></p>
<ul>
<li>we identified 2 gut microbial-host co-metabolites associated with BP: phenylacetylglutamine and 4-cresyl sulfate, deriving from phenylalanine and tyrosine, respectively, resulting from bacterial putrefaction of protein in the distal colon. The gut microbiota, in particular <em>Firmicutes</em> and <em>Bacteroidetes</em>, can adapt to dietary changes and induce changes in host metabolism (35): an increase of <em>Firmicutes</em> to <em>Bacteroidetes</em> ratio has been demonstrated in spontaneous hypertensive rats (36). Other researchers have manipulated gut microbiota balance via probiotic administration with consequent beneficial effects on BP levels (37). More recently, blood concentrations of phenylacetylglutamine were found to be strongly anti-correlated with BP, consistent with our results, and with carotid-femoral pulse-wave velocity, a measure of aortic stiffness (38). Although 4-cresyl sulfate has never been formally linked to BP, its dietary excretion has been shown to be highly correlated with that of phenylacetylglutamine (16).</li>
<li>The association between gut-microbial co-metabolites and BP is further evidenced in the direct association we found between BP and hippurate, originating from the conversion of benzoic acid by gut microflora via the shikimate pathway (39). In contrast to our results, hypertensive rats showed an anti-correlation between hippurate and BP (40) but interpolation from animal data to human must be performed with care due to the differences in gut microbiomes between species. An inverse association between excretion of hippurate and BP has been reported in humans but this association was not significant after adjusting for BMI, alcohol intake, and urinary excretion of sodium and potassium (41). A controlled feeding study by Wu et al. (35) showed that changes in the gut microbiome occurred within 24 h of initiating a change in diet and that BMI and weight loss can also influence the gut microbiome composition. However, in our dietary intervention study, all participants consumed a consistent healthy dietary pattern for 6 wk and maintained their body weight, with micronutrients being indexed to the energy level of their diets. Our data, therefore, suggest modulation of diets can affect gut microbiome activity and that this may lead to a direct effect on BP regulation.</li>
<li>We observed an inverse association of <em>N</em>-methyl-2-pyridone-5-carboxamide (tryptophan–NAD metabolite) and BP. Bartus et al. (42) showed that ingestion of 1*-<em>methylnicotinamide in hypertriglyceridemic rats resulted in an increase of 1</em>-<em>methylnicotinamide and its metabolites such as <em>N</em>-methyl-2-pyridone-5-carboxamide and found that ingestion of 1</em>-<em>methylnicotinamide in both the diabetic and hypertriglyceridemic rats can ameliorate the nitric oxide-dependent vasodilation, a surrogate marker for atherosclerosis. Others have found that 1</em>-*methylnicotinamide demonstrates anti-thrombotic activity (43). Our findings further support the beneficial impact of <em>N</em>-methyl-2-pyridone-5-carboxamide on CVD health. We suggest the tryptophan–NAD pathway may offer a new target for pharmacological treatment of hypertension.</li>
<li>We also confirmed the association of dietary markers with BP including: a direct association between BP and carnitine (a marker for protein ingestion); and an inverse association with proline-betaine (citrus fruit ingestion). Our results are consistent with previous studies linking hypertension with blood concentration of carnitine (44) and variations in BP following carnitine treatment in rats (45). Similarly our results support the previously postulated benefit of citrus fruit intake in reduction of BP (34). Specifically for the OmniProt diet, despite the increased excretion of carnitine, a marker which was linked to higher BP, overall beneficial reductions in CVD risk factors (both BP levels and lipid profiles) were elicited and these benefits persisted for those who were considered as typical (HDR) as well as variable (VDR) responders. The specific mechanisms for this remain unclear although it may be hypothesized that the altered large-bowel microbiome following protein-rich dietary intervention may play a significant role.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08794</h2>
<ul>
<li>brain interaction
<ul>
<li>
<p><em>endothelial tlr4 and the microbiome drive cerebral cavernous malformations</em><br />
alan t. tang et al. 2017<br />
<a href="http://dx.doi.org/10.1038/nature22075">dx.doi.org/10.1038/nature22075</a></p>
<ul>
<li>Cerebral cavernous malformations (CCMs) are a cause of stroke and seizure for which no effective medical therapies yet exist. CCMs arise from the loss of an adaptor complex that negatively regulates MEKK3–KLF2/4 signalling in brain endothelial cells, but upstream activators of this disease pathway have yet to be identified. Here we identify endothelial Toll-like receptor 4 (TLR4) and the gut microbiome as critical stimulants of CCM formation. Activation of TLR4 by Gram-negative bacteria or lipopolysaccharide accelerates CCM formation, and genetic or pharmacologic blockade of TLR4 signalling prevents CCM formation in mice. Polymorphisms that increase expression of the <em>TLR4</em> gene or the gene encoding its co-receptor CD14 are associated with higher CCM lesion burden in humans. Germ-free mice are protected from CCM formation, and a single course of antibiotics permanently alters CCM susceptibility in mice. These studies identify unexpected roles for the microbiome and innate immune signalling in the pathogenesis of a cerebrovascular disease, as well as strategies for its treatment.</li>
</ul>
</li>
<li>
<p><em>differences in gut microbial composition correlate with regional brain volumes in irritable bowel syndrome</em><br />
jennifer s. labus et al. 2017<br />
<a href="http://dx.doi.org/10.1186/s40168-017-0260-z">dx.doi.org/10.1186/s40168-017-0260-z</a></p>
<ul>
<li>The results confirm previous reports of gut microbiome-based IBS subgroups and identify for the first time brain structural alterations associated with these subgroups. They provide preliminary evidence for the involvement of specific microbes and their predicted metabolites in these correlations.</li>
</ul>
</li>
<li>
<p><em>bidirectional brain-gut interactions and chronic pathological changes after traumatic brain injury in mice</em><br />
elise l. ma et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.bbi.2017.06.018">http://dx.doi.org/10.1016/j.bbi.2017.06.018</a></p>
<ul>
<li>
<p>•Experimental TBI induces chronic structural and functional changes in the colon.<br />
•Barrier dysfunction after TBI is linked to decreased colonic claudin-1 expression.<br />
•TBI alters the number of activated enteric glial cells in the colon mucosa.<br />
•Enteric <em>C. rodentium</em> infection weeks after TBI exacerbates brain neuropathology.</p>
<p>Objectives<br />
<a href="http://www.sciencedirect.com/topics/neuroscience/traumatic-brain-injury">Traumatic brain injury</a> (TBI) has complex effects on the gastrointestinal tract that are associated with TBI-related morbidity and mortality. We examined changes in <a href="http://www.sciencedirect.com/topics/neuroscience/mucous-membrane">mucosal</a> barrier properties and enteric <a href="http://www.sciencedirect.com/topics/neuroscience/glial-cells">glial cell</a> response in the gut after experimental TBI in mice, as well as effects of the enteric pathogen <a href="http://www.sciencedirect.com/topics/immunology-and-microbiology/citrobacter-rodentium"><em>Citrobacter rodentium</em></a> (<em>Cr</em>) on both gut and brain after injury.<br />
Methods<br />
Moderate-level TBI was induced in C57BL/6 mice by controlled cortical impact (CCI). Mucosal barrier function was assessed by transepithelial resistance, fluorescent-labelled dextran flux, and quantification of <a href="http://www.sciencedirect.com/topics/neuroscience/tight-junction">tight junction</a> proteins. Enteric glial cell number and activation were measured by Sox10 expression and <a href="http://www.sciencedirect.com/topics/neuroscience/glial-fibrillary-acidic-protein">GFAP</a> reactivity, respectively. Separate groups of mice were challenged with <em>Cr</em> infection during the chronic phase of TBI, and <a href="http://www.sciencedirect.com/topics/immunology-and-microbiology/host-immune-response">host immune response</a>, barrier integrity, enteric glial cell reactivity, and progression of brain injury and inflammation were assessed.<br />
Results<br />
Chronic CCI induced changes in colon morphology, including increased mucosal depth and smooth muscle thickening. At day 28 post-CCI, increased paracellular permeability and decreased claudin-1 mRNA and protein expression were observed in the absence of inflammation in the colon. Colonic glial cell GFAP and Sox10 expression were significantly increased 28 days after brain injury. Clearance of <em>Cr</em> and <a href="http://www.sciencedirect.com/topics/neuroscience/downregulation-and-upregulation">upregulation</a> of Th1/Th17 <a href="http://www.sciencedirect.com/topics/neuroscience/cytokines">cytokines</a> in the colon were unaffected by CCI; however, colonic paracellular flux and enteric glial cell GFAP expression were significantly increased. Importantly, <em>Cr</em> infection in chronically-injured mice worsened the brain lesion injury and increased astrocyte- and microglial-mediated inflammation.<br />
Conclusion<br />
These experimental studies demonstrate chronic and bidirectional <a href="http://www.sciencedirect.com/topics/immunology-and-microbiology/gutbrain-axis">brain-gut</a> interactions after TBI, which may negatively impact late outcomes after brain injury.</p>
</li>
</ul>
</li>
<li>
<p><em>gaba-modulating bacteria of the human gut microbiota</em><br />
philip strandwitz et al. 2019<br />
<a href="http://dx.doi.org/10.1038/s41564-018-0307-3">http://dx.doi.org/10.1038/s41564-018-0307-3</a></p>
<ul>
<li>
<p>Gut microbiota, the entire collection of microorganisms found in that habitat, affect many important functions, including the immune response and the nervous system. Nevertheless, many microorganisims residing in the human gut remain uncultured, which the research team called &quot;an obstacle for understanding their biological roles&quot; in the Nature Microbiology article.</p>
<p>More such microorganisms probably remain uncultured because they require key growth factors that are provided by neighboring bacteria in their natural environments, but not under artificial laboratory conditions. During an extensive screening process, the team found that KLE1738 required the presence of Bacteroides fragilis, a common human gut bacterium, to grow.</p>
<p>Further biological testing and purification led to the isolation of GABA as the growth factor produced by Bacteroides fragilis. GABA was, in fact, the only nutrient tested during the experiments that supported the growth of KLE1738.</p>
<p>In the next research phase, the team explored the possible connection between Bacteroides and depression. Stool samples and functional magnetic resonance imaging measurements of brain activity were collected from 23 subjects suffering from clinically diagnosed depression.</p>
<p>The researchers found an inverse relationship between the relative abundance of fecal Bacteroidesand functional connectivity in a part of the brain associated with elevated activity during depression. This means that low abundance of Bacteroides was associated with high activity in that part of the brain, and vice versa.</p>
<p>&quot;A good first step is to repeat our findings in additional human cohorts, which we are actively exploring,&quot; said Strandwitz of further research. &quot;When it comes to depression, animal models are often difficult to translate, which is why we are so excited about human studies. &quot;</p>
<p>Recent work published in the journals Science and Cell have identified the presence of sensory neurons in the gut that are hard-wired to the brain. &quot;It would be great to explore whether microbial GABA can act as a signal via that pathway,&quot; said Anukriti Sharma, a co-author of the Nature Microbiology article and a postdoctoral scholar at Argonne</p>
</li>
<li>
<p><em>abstract</em> The gut microbiota affects many important host functions, including the immune response and the nervous system1. However, while substantial progress has been made in growing diverse microorganisms of the microbiota2, 23–65% of species residing in the human gut remain uncultured3,4, which is an obstacle for understanding their biological roles. A likely reason for this unculturability is the absence in artificial media of key growth factors that are provided by neighbouring bacteria in situ5,6. In the present study, we used co-culture to isolate KLE1738, which required the presence of Bacteroides fragilis to grow. Bioassay-driven purification of B. fragilis supernatant led to the isolation of the growth factor, which, surprisingly, is the major inhibitory neurotransmitter GABA (γ-aminobutyric acid). GABA was the only tested nutrient that supported the growth of KLE1738, and a genome analysis supported a GABA-dependent metabolism mechanism. Using growth of KLE1738 as an indicator, we isolated a variety of GABA-producing bacteria, and found that Bacteroides ssp. produced large quantities of GABA. Genome-based metabolic modelling of the human gut microbiota revealed multiple genera with the predicted capability to produce or consume GABA. A transcriptome analysis of human stool samples from healthy individuals showed that GABA-producing pathways are actively expressed by Bacteroides, Parabacteroides and Escherichia species. By coupling 16S ribosmal RNA sequencing with functional magentic resonance imaging in patients with major depressive disorder, a disease associated with an altered GABA-mediated response, we found that the relative abundance levels of faecal Bacteroides are negatively correlated with brain signatures associated with depression.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08894</h2>
<ul>
<li>transmissible disease
<ul>
<li>
<p><em>cutaneous leishmaniasis induces a transmissible dysbiotic skin microbiota that promotes skin inflammation</em><br />
ciara gimblet et al. 2017<br />
<a href="http://dx.doi.org/10.1016/j.chom.2017.06.006">dx.doi.org/10.1016/j.chom.2017.06.006</a></p>
<ul>
<li>
<p>Leishmania infection alters the skin microbiota of both humans and mice<br />
Dysbiosis is characterized by a dominance of <em>Staphylococcus</em> and/or <em>Streptococcus</em><br />
Naive mice acquire dysbiosis when co-housed with leishmania-infected mice<br />
Acquiring a dysbiotic microbiota prior to infection exacerbates skin inflammation</p>
<p>Skin microbiota can impact allergic and autoimmune responses, wound healing, and anti-microbial defense. We investigated the role of skin microbiota in cutaneous leishmaniasis and found that human patients infected with <em>Leishmania braziliensis</em> develop dysbiotic skin microbiota, characterized by increases in the abundance of <em>Staphylococcus</em> and/or <em>Streptococcus</em>. Mice infected with <em>L. major</em> exhibit similar changes depending upon disease severity. Importantly, this dysbiosis is not limited to the lesion site, but is transmissible to normal skin distant from the infection site and to skin from co-housed naive mice. This observation allowed us to test whether a pre-existing dysbiotic skin microbiota influences disease, and we found that challenging dysbiotic naive mice with <em>L. major</em> or testing for contact hypersensitivity results in exacerbated skin inflammatory responses. These findings demonstrate that a dysbiotic skin microbiota is not only a consequence of tissue stress, but also enhances inflammation, which has implications for many inflammatory cutaneous diseases.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08994</h2>
<ul>
<li>diagnosis
<ul>
<li>
<p><em>extreme dysbiosis of the microbiome in critical illness</em><br />
daniel mcdonald et al 2016<br />
http://dx.doi.org/10.1128/msphere.00199-16<br />
Link: <a href="http://dx.doi.org/10.1128/msphere.00199-16">dx.doi.org/10.1128/msphere.00199-16</a></p>
<ul>
<li>Critical illness is hypothesized to associate with loss of “health- promoting” commensal microbes and overgrowth of pathogenic bacteria (dysbiosis). This dysbiosis is believed to increase susceptibility to nosocomial infections, sepsis, and organ failure. A trial with prospective monitoring of the intensive care unit (ICU) patient microbiome using culture-independent techniques to confirm and character- ize this dysbiosis is thus urgently needed. Characterizing ICU patient microbiome changes may provide first steps toward the development of diagnostic and thera- peutic interventions using microbiome signatures. To characterize the ICU patient microbiome, we collected fecal, oral, and skin samples from 115 mixed ICU patients across four centers in the United States and Canada. Samples were collected at two time points: within 48 h of ICU admission, and at ICU discharge or on ICU day 10. Sample collection and processing were performed according to Earth Microbiome Project protocols. We applied SourceTracker to assess the source composition of ICU patient samples by using Qiita, including samples from the American Gut Project (AGP), mammalian corpse decomposition samples, childhood (Global Gut study), and house surfaces. Our results demonstrate that critical illness leads to significant and rapid dysbiosis. Many taxons significantly depleted from ICU patients versus AGP healthy controls are key “health-promoting” organisms, and overgrowth of known pathogens was frequent. Source compositions of ICU patient samples are largely un- characteristic of the expected community type. Between time points and within a patient, the source composition changed dramatically. Our initial results show great promise for microbiome signatures as diagnostic markers and guides to therapeutic interventions in the ICU to repopulate the normal, “health-promoting” microbiome and thereby improve patient outcomes.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>probiotics</h2>
<h2>09094</h2>
<ul>
<li>
<p><em>lactobacillus reuteri reduces bone loss in older women with low bone mineral density - a randomized, placebo-controlled, double-blind, clinical trial</em><br />
anna g. nilsson et al. 2018<br />
<a href="http://dx.doi.org/10.1111/joim.12805">http://dx.doi.org/10.1111/joim.12805</a></p>
<ul>
<li>The importance of the gut microbiome for bone metabolism in mice has recently been demonstrated, but no studies are available in humans. <em>Lactobacillus reuteri</em> ATCCPTA 6475 (<em>L. reuteri</em> 6475) has been reported to increase bone mineral density (BMD) in mice but its effect on the human skeleton is unknown. The objective of this trial was to investigate if <em>L. reuteri</em> 6475 affects bone loss in older women with low BMD. In this double‐blind, placebo‐controlled study, women from the population who were 75 to 80 years old and had low BMD were randomized to orally receive 1010 colony‐forming units of <em>L. reuteri</em> 6475 daily or placebo. The predefined primary end‐point was relative change after 12 months in tibia total volumetric BMD (vBMD).<br />
Ninety women were included and 70 completed the study. <em>L. reuteri</em> 6475 reduced loss of total vBMD compared to placebo both in the intention‐to‐treat (ITT) analysis (‐0.83% [95% confidence interval [CI], ‐1.47 to ‐0.19%] vs. ‐1.85% [95% CI, ‐2.64 to ‐1.07%]; mean difference 1.02% [95% CI, 0.02 to 2.03]) and per protocol analysis (‐0.93% [95% CI, ‐1.45 to ‐0.40] vs. ‐1.86% [95% CI, ‐2.35 to ‐1.36]; mean difference 0.93% [95% CI, 0.21 to 1.65]. In general, similar but smaller effects were observed in the secondary bone variable outcomes, but these differences did not reach statistical significance in the ITT population. Adverse events did not differ between groups. In conclusion, supplementation with <em>L. reuteri</em> 6475 should be further explored as a novel approach to prevent age‐associated bone loss and osteoporosis.</li>
</ul>
</li>
<li>
<p><em>targeting the gut microbiome to treat the osteoarthritis of obesity</em><br />
eric m. schott et al. 2018<br />
<a href="http://dx.doi.org/10.1172/jci.insight.95997">http://dx.doi.org/10.1172/jci.insight.95997</a></p>
<ul>
<li>Obesity is a risk factor for osteoarthritis (OA), the greatest cause of disability in the US. The impact of obesity on OA is driven by systemic inflammation, and increased systemic inflammation is now understood to be caused by gut microbiome dysbiosis. Oligofructose, a nondigestible prebiotic fiber, can restore a lean gut microbial community profile in the context of obesity, suggesting a potentially novel approach to treat the OA of obesity. Here, we report that — compared with the lean murine gut — obesity is associated with loss of beneficial <em>Bifidobacteria</em>, while key proinflammatory species gain in abundance. A downstream systemic inflammatory signature culminates with macrophage migration to the synovium and accelerated knee OA. Oligofructose supplementation restores the lean gut microbiome in obese mice, in part, by supporting key commensal microflora, particularly <em>Bifidobacterium pseudolongum</em>. This is associated with reduced inflammation in the colon, circulation, and knee and protection from OA. This observation of a gut microbiome–OA connection sets the stage for discovery of potentially new OA therapeutics involving strategic manipulation of specific microbial species inhabiting the intestinal space.</li>
</ul>
</li>
</ul>
<h2>09194</h2>
<ul>
<li>
<p><em>psychobiotics and the manipulation of bacteria-brain signals</em><br />
sarkar et al. 2016<br />
http://dx.doi.org/10.1016/j.tins.2016.09.002<br />
Link: <a href="http://dx.doi.org/10.1016/j.tins.2016.09.002">dx.doi.org/10.1016/j.tins.2016.09.002</a></p>
<ul>
<li>Psychobiotics were previously defined as live bacteria (probiotics) which, when ingested, confer mental health benefits through interactions with commensal gut bacteria. We expand this definition to encompass prebiotics, which enhance the growth of beneficial gut bacteria. We review probiotic and prebiotic effects on emotional, cognitive, systemic, and neural variables relevant to health and disease. We discuss gut–brain signalling mechanisms enabling psychobiotic effects, such as metabolite production. Overall, knowledge of how the microbiome responds to exogenous influence remains limited. We tabulate several important research questions and issues, exploration of which will generate both mechanistic insights and facilitate future psychobiotic development. We suggest the definition of psychobiotics be expanded beyond probiotics and prebiotics to include other means of influencing the microbiome.</li>
</ul>
</li>
<li>
<p>A new study finds that a noninvasive electromagnetic brain stimulation technique helps obese people lose weight, partly by changing the composition of their intestinal bacteria -- the so-called gut microbiota. Results of the technique, called deep transcranial magnetic stimulation (dTMS), will be presented Sunday at ENDO 2017, the Endocrine Society's 99th annual meeting in Orlando, Fla.<br />
Link: <a href="https://www.sciencedaily.com/releases/2017/04/170403123508.htm">sciencedaily.com/releases/2017/04/170403123508.htm</a></p>
</li>
</ul>
<h2>09294</h2>
<ul>
<li>
<p><em>an exclusive metabolic niche enables strain engraftment in the gut microbiota</em><br />
elizabeth stanley shepherd et al. 2018<br />
<a href="http://dx.doi.org/10.1038/s41586-018-0092-4">http://dx.doi.org/10.1038/s41586-018-0092-4</a></p>
<ul>
<li>The dense microbial ecosystem in the gut is intimately connected to numerous facets of human biology, and manipulation of the gut microbiota has broad implications for human health. In the absence of profound perturbation, the bacterial strains that reside within an individual are mostly stable over time<a href="https://www.nature.com/articles/s41586-018-0092-4#ref-CR1">70</a>. By contrast, the fate of exogenous commensal and probiotic strains applied to an established microbiota is variable, generally unpredictable and greatly influenced by the background microbiota<a href="https://www.nature.com/articles/s41586-018-0092-4#ref-CR2">71</a>,<a href="https://www.nature.com/articles/s41586-018-0092-4#ref-CR3">72</a>. Therefore, analysis of the factors that govern strain engraftment and abundance is of critical importance to the emerging field of microbiome reprogramming. Here we generate an exclusive metabolic niche in mice via administration of a marine polysaccharide, porphyran, and an exogenous <em>Bacteroides</em> strain harbouring a rare gene cluster for porphyran utilization. Privileged nutrient access enables reliable engraftment of the exogenous strain at predictable abundances in mice harbouring diverse communities of gut microbes. This targeted dietary support is sufficient to overcome priority exclusion by an isogenic strain<a href="https://www.nature.com/articles/s41586-018-0092-4#ref-CR4">73</a>, and enables strain replacement. We demonstrate transfer of the 60-kb porphyran utilization locus into a naive strain of <em>Bacteroides</em>, and show finely tuned control of strain abundance in the mouse gut across multiple orders of magnitude by varying porphyran dosage. Finally, we show that this system enables the introduction of a new strain into the colonic crypt ecosystem. These data highlight the influence of nutrient availability in shaping microbiota membership, expand the ability to perform a broad spectrum of investigations in the context of a complex microbiota, and have implications for cell-based therapeutic strategies in the gut.</li>
</ul>
</li>
<li>
<p><em>cooperative metabolic adaptations in the host can favor asymptomatic infection and select for attenuated virulence in an enteric pathogen</em><br />
karina k. sanchez et al. 2018<br />
<a href="http://dx.doi.org/10.1016/j.cell.2018.07.016">http://dx.doi.org/10.1016/j.cell.2018.07.016</a></p>
<ul>
<li>
<p>Lethal dose 50 can be used to identify cooperative defense mechanisms<br />
Dietary iron promotes cooperative defenses against a lethal enteric infection<br />
Insulin resistance reduces gut glucose absorption<br />
Cooperative defenses drive selection for attenuated pathogen strains</p>
<p>Pathogen virulence exists on a continuum. The strategies that drive symptomatic or asymptomatic infections remain largely unknown. We took advantage of the concept of lethal dose 50 (LD50) to ask which component of individual non-genetic variation between hosts defines whether they survive or succumb to infection. Using the enteric pathogen Citrobacter, we found no difference in pathogen burdens between healthy and symptomatic populations. Iron metabolism-related genes were induced in asymptomatic hosts compared to symptomatic or naive mice. Dietary iron conferred complete protection without influencing pathogen burdens, even at 1000× the lethal dose of Citrobacter. Dietary iron induced insulin resistance, increasing glucose levels in the intestine that were necessary and sufficient to suppress pathogen virulence. A short course of dietary iron drove the selection of attenuated Citrobacter strains that can transmit and asymptomatically colonize naive hosts, demonstrating that environmental factors and cooperative metabolic strategies can drive conversion of pathogens toward commensalism.</p>
</li>
</ul>
</li>
</ul>
<h2>transplantation</h2>
<h2>09394</h2>
<ul>
<li>
<p><em>long-term effects on luminal and mucosal microbiota and commonly acquired taxa in faecal microbiota transplantation for recurrent clostridium difficile infection</em><br />
jonna jalanka et al. 2016<br />
http://dx.doi.org/10.1186/s12916-016-0698-z<br />
Link: <a href="http://dx.doi.org/10.1186/s12916-016-0698-z">dx.doi.org/10.1186/s12916-016-0698-z</a></p>
<ul>
<li>
<p>Background<br />
Faecal microbiota transplantation (FMT) is an effective treatment for recurrent Clostridium difficile infection (rCDI). It restores the disrupted intestinal microbiota and subsequently suppresses C. difficile. The long-term stability of the intestinal microbiota and the recovery of mucosal microbiota, both of which have not been previously studied, are assessed herein. Further, the specific bacteria behind the treatment efficacy are also investigated.</p>
<p>Methods<br />
We performed a high-throughput microbiota profiling using a phylogenetic microarray analysis of 131 faecal and mucosal samples from 14 rCDI patients pre- and post-FMT during a 1-year follow-up and 23 samples from the three universal donors over the same period.</p>
<p>Results<br />
The FMT treatment was successful in all patients. FMT reverted the patients’ bacterial community to become dominated by Clostridium clusters IV and XIVa, the major anaerobic bacterial groups of the healthy gut. In the mucosa, the amount of facultative anaerobes decreased, whereas Bacteroidetes increased. Post-FMT, the patients’ microbiota profiles were more similar to their own donors than what is generally observed for unrelated subjects and this striking similarity was retained throughout the 1-year follow-up. Furthermore, the universal donor approach allowed us to identify bacteria commonly established in all CDI patients and revealed a commonly acquired core microbiota consisting of 24 bacterial taxa.</p>
<p>Conclusions<br />
FMT induces profound microbiota changes, therefore explaining the high clinical efficacy for rCDI. The identification of commonly acquired bacteria could lead to effective bacteriotherapeutic formulations. FMT can affect microbiota in the long-term and offers a means to modify it relatively permanently for the treatment of microbiota-associated diseases.</p>
</li>
</ul>
</li>
</ul>
<h2>09494</h2>
<ul>
<li>
<p><em>effect of oral capsule– vs colonoscopy-delivered fecal microbiota transplantation on recurrent clostridium difficile infection: a randomized clinical trial</em><br />
dina kao et al. 2017<br />
<a href="http://dx.doi.org/10.1001/jama.2017.17077">http://dx.doi.org/10.1001/jama.2017.17077</a></p>
<ul>
<li>Question  Does the clinical efficacy of fecal microbiota transplantation (FMT) in treating recurrent <em>Clostridium difficile</em> infection (RCDI) depend on the route of delivery?<br />
Findings  In this noninferiority randomized clinical trial that included 116 adults with RCDI, the proportion without recurrence over 12 weeks was 96.2% after a single treatment in a group treated with oral capsules and in a group treated via colonoscopy, meeting the noninferiority margin of 15%.<br />
Meaning  FMT by oral capsules may be an effective approach to treating RCDI.<br />
Abstract<br />
Importance  Fecal microbiota transplantation (FMT) is effective in preventing recurrent <em>Clostridium difficile</em> infection (RCDI). However, it is not known whether clinical efficacy differs by route of delivery.<br />
Objective  To determine whether FMT by oral capsule is noninferior to colonoscopy delivery in efficacy.<br />
Design, Setting, and Participants  Noninferiority, unblinded, randomized trial conducted in 3 academic centers in Alberta, Canada. A total of 116 adult patients with RCDI were enrolled between October 2014 and September 2016, with follow-up to December 2016. The noninferiority margin was 15%.<br />
Interventions  Participants were randomly assigned to FMT by capsule or by colonoscopy at a 1:1 ratio.<br />
Main Outcomes and Measures  The primary outcome was the proportion of patients without RCDI 12 weeks after FMT. Secondary outcomes included (1) serious and minor adverse events, (2) changes in quality of life by the 36-Item Short Form Survey on a scale of 0 (worst possible quality of life) to 100 (best quality of life), and (3) patient perception on a scale of 1 (not at all unpleasant) to 10 (extremely unpleasant) and satisfaction on a scale of 1 (best) to 10 (worst).<br />
Results  Among 116 patients randomized (mean [SD] age, 58 <a href="http://dx.doi.org/10.7554/eLife.18678">19</a> years; 79 women [68%]), 105 (91%) completed the trial, with 57 patients randomized to the capsule group and 59 to the colonoscopy group. In per-protocol analysis, prevention of RCDI after a single treatment was achieved in 96.2% in both the capsule group (51/53) and the colonoscopy group (50/52) (difference, 0%; 1-sided 95% CI, −6.1% to infinity; <em>P</em> &lt; .001), meeting the criterion for noninferiority. One patient in each group died of underlying cardiopulmonary illness unrelated to FMT. Rates of minor adverse events were 5.4% for the capsule group vs 12.5% for the colonoscopy group. There was no significant between-group difference in improvement in quality of life. A significantly greater proportion of participants receiving capsules rated their experience as “not at all unpleasant” (66% vs 44%; difference, 22% [95% CI, 3%-40%]; <em>P</em> = .01).<br />
Conclusions and Relevance  Among adults with RCDI, FMT via oral capsules was not inferior to delivery by colonoscopy for preventing recurrent infection over 12 weeks. Treatment with oral capsules may be an effective approach to treating RCDI.</li>
</ul>
</li>
</ul>
<h2>09594</h2>
<ul>
<li>
<p><em>monitoring biofilm function in new and matured full-scale slow sand filters using flow cytometric histogram image comparison (chic)</em><br />
sandy chan et al. 2018<br />
<a href="http://dx.doi.org/10.1016/j.watres.2018.03.032">http://dx.doi.org/10.1016/j.watres.2018.03.032</a></p>
<ul>
<li>
<p>•Removal of the schmutzdecke did not affect the function of established SSF.<br />
•Use of preconditioned sand accelerated deep sand biofilm maturation in new SSF.<br />
•FCM bacterial profiles correlated with the presence of microbial indicators.<br />
•FCM is useful for process control of SSFs.</p>
<p>While slow sand filters (SSFs) have produced drinking water for more than a hundred years, understanding of their associated microbial communities is limited. In this study, bacteria in influent and effluent water from full-scale SSFs were explored using flow cytometry (FCM) with cytometric histogram image comparison (CHIC) analysis; and routine microbial counts for heterotrophs, total coliforms and Escherichia coli. To assess if FCM can monitor biofilm function, SSFs differing in age and sand composition were compared. FCM profiles from two established filters were indistinguishable. To examine biofilm in the deep sand bed, SSFs were monitored during a scraping event, when the top layer of sand and the schmutzdecke are removed to restore flow through the filter. The performance of an established SSF was stable: total organic carbon (TOC), pH, numbers of heterotrophs, coliforms, E. coli, and FCM bacterial profile were unaffected by scraping. However, the performance of two newly-built SSFs containing new and mixed sand was compromised: breakthrough of both microbial indicators and TOC occurred following scraping. The compromised performance of the new SSFs was reflected in distinct effluent bacterial communities; and, the presence of microbial indicators correlated to influent bacterial communities. This demonstrated that FCM can monitor SSF performance. Removal of the top layer of sand did not alter the effluent water from the established SSF, but did affect that of the SSFs containing new sand. This suggests that the impact of the surface biofilm on effluent water is greater when the deep sand bed biofilm is not established.</p>
</li>
</ul>
</li>
</ul>
<h2>virome</h2>
<h2>09694</h2>
<ul>
<li>
<p><em>predators catalyze an increase in chloroviruses by foraging on the symbiotic hosts of zoochlorellae</em><br />
john p. delong et al. 2017 al. 2016<br />
<a href="http://dx.doi.org/10.1073/pnas.1613843113">http://dx.doi.org/10.1073/pnas.1613843113</a></p>
<ul>
<li>
<p>Reproduction and growth of viruses depend on successful encounters with appropriate hosts. However, some hosts are difficult to encounter. In particular, chloroviruses cannot reach their target zoochlorellae hosts, because zoochlorellae are endosymbionts, living inside the cell of a protist that protects the zoochlorellae from the chlorovirus. The protist host is subject to predation, and we show that copepods foraging on zoochlorellae-bearing protists can disrupt the mutualism and pass endosymbiontic zoochlorellae through their guts, exposing them to chloroviruses. In this way, predators can catalyze the virus population growth by breaking down physical barriers between viruses and their endosymbiont hosts.</p>
<p>Virus population growth depends on contacts between viruses and their hosts. It is often unclear how sufficient contacts are made between viruses and their specific hosts to generate spikes in viral abundance. Here, we show that copepods, acting as predators, can bring aquatic viruses and their algal hosts into contact. Specifically, predation of the protist <em>Paramecium bursaria</em> by copepods resulted in a &gt;100-fold increase in the number of chloroviruses in 1 d. Copepod predation can be seen as an ecological “catalyst” by increasing contacts between chloroviruses and their hosts, zoochlorellae (endosymbiotic algae that live within paramecia), thereby facilitating viral population growth. When feeding, copepods passed <em>P. bursaria</em> through their digestive tract only partially digested, releasing endosymbiotic algae that still supported viral reproduction and resulting in a virus population spike. A simple predator–prey model parameterized for copepods consuming protists generates cycle periods for viruses consistent with those observed in natural ponds. Food webs are replete with similar symbiotic organisms, and we suspect the predator catalyst mechanism is capable of generating blooms for other endosymbiont-targeting viruses.</p>
</li>
</ul>
</li>
<li>
<p><em>deposition rates of viruses and bacteria above the atmospheric boundary layer</em><br />
isabel reche et al. 2018<br />
<a href="http://dx.doi.org/10.1038/s41396-017-0042-4">http://dx.doi.org/10.1038/s41396-017-0042-4</a></p>
<ul>
<li>Aerosolization of soil-dust and organic aggregates in sea spray facilitates the long-range transport of bacteria, and likely viruses across the free atmosphere. Although long-distance transport occurs, there are many uncertainties associated with their deposition rates. Here, we demonstrate that even in pristine environments, above the atmospheric boundary layer, the downward flux of viruses ranged from 0.26 × 109 to &gt;7 × 109 m−2 per day. These deposition rates were 9–461 times greater than the rates for bacteria, which ranged from 0.3 × 107 to &gt;8 × 107 m−2 per day. The highest relative deposition rates for viruses were associated with atmospheric transport from marine rather than terrestrial sources. Deposition rates of bacteria were significantly higher during rain events and Saharan dust intrusions, whereas, rainfall did not significantly influence virus deposition. Virus deposition rates were positively correlated with organic aerosols &lt;0.7 μm, whereas, bacteria were primarily associated with organic aerosols &gt;0.7 μm, implying that viruses could have longer residence times in the atmosphere and, consequently, will be dispersed further. These results provide an explanation for enigmatic observations that viruses with very high genetic identity can be found in very distant and different environments.</li>
</ul>
</li>
</ul>
<h2>09794</h2>
<ul>
<li>
<p><em>vesicle-cloaked virus clusters are optimal units for inter-organismal viral transmission</em><br />
marianita santiana et al. 2018<br />
<a href="http://dx.doi.org/10.1016/j.chom.2018.07.006">http://dx.doi.org/10.1016/j.chom.2018.07.006</a></p>
<ul>
<li>
<p>Rotaviruses and noroviruses are shed in stool as viral clusters inside vesicles<br />
Vesicles containing virus clusters remain intact during fecal-oral transmission<br />
Vesicles achieve a high MOI and induce severe disease<br />
Vesicle-cloaked viral clusters are more virulent units than free viral particles</p>
<p>In enteric viral infections, such as those with rotavirus and norovirus, individual viral particles shed in stool are considered the optimal units of fecal-oral transmission. We reveal that rotaviruses and noroviruses are also shed in stool as viral clusters enclosed within vesicles that deliver a high inoculum to the receiving host. Cultured cells non-lytically release rotaviruses and noroviruses inside extracellular vesicles. In addition, stools of infected hosts contain norovirus and rotavirus within vesicles of exosomal or plasma membrane origin. These vesicles remain intact during fecal-oral transmission and thereby transport multiple viral particles collectively to the next host, enhancing both the MOI and disease severity. Vesicle-cloaked viruses are non-negligible populations in stool and have a disproportionately larger contribution to infectivity than free viruses. Our findings indicate that vesicle-cloaked viruses are highly virulent units of fecal-oral transmission and highlight a need for antivirals targeting vesicles and virus clustering.</p>
</li>
</ul>
</li>
</ul>
<h2>09894</h2>
<ul>
<li>not yet read
<ul>
<li>
<p><em>microbia: a journey into the unseen world around you</em><br />
eugenia bone 2018</p>
</li>
<li>
<p><em>the hidden half of nature: the microbial roots of life and health</em><br />
david montgomery, anne biklé 2015</p>
</li>
<li>
<p><em>the mind-gut connection: how the hidden conversation within our bodies impacts our mood, our choices, and our overall health</em><br />
emeran mayer 2016</p>
</li>
<li>
<p><em>follow your gut: the enormous impact of tiny microbes</em><br />
rob knight 2015 not yet read</p>
</li>
<li>
<p><em>the psychobiotic revolution: mood, food and the new science of the gut-brain connection</em><br />
toreadnext</p>
</li>
<li>
<p><em>deadly companions how microbes shaped our history</em><br />
dorothy crawford 2018 toreadnext</p>
</li>
<li>
<p><em>human microbes the power within health healing and beyond</em><br />
vasu appanna 2018 toreadnext</p>
</li>
<li>
<p><em>natural defense: enlisting bugs and germs to protect our food and health</em><br />
emily monosson 2017</p>
</li>
<li>
<p><em>your baby’s microbiome: the critical role of vaginal birth and breastfeeding for lifelong health</em><br />
toni harman,  alex wakeford 2018</p>
</li>
</ul>
</li>
</ul>
<h2>09994</h2>
<h2>04994</h2>
<ul>
<li><em>bugs as drugs: therapeutic microbes for the prevention and treatment of disease</em><br />
robert britton, patrice cani 2018 to read next</li>
</ul>
<h2>04894</h2>
<h2>04794</h2>
<h2>04694</h2>
<ul>
<li>
<p><em>signal percolation within a bacterial community</em><br />
joseph w. larkin et al. 2018<br />
<a href="https://doi.org/10.1016/j.cels.2018.06.005">https://doi.org/10.1016/j.cels.2018.06.005</a></p>
<ul>
<li>
<p>•Biofilm cells exhibit heterogeneity (firing/non-firing) in electrical signaling<br />
•Firing cells arrange themselves into a theoretically predicted percolated network<br />
•Percolation model accounts for signal transmission in wild-type and mutant biofilms<br />
•Cost-benefit balance can organize the community at the critical percolation point</p>
<p>Signal transmission among cells enables long-range coordination in biological systems. However, the scarcity of quantitative measurements hinders the development of theories that relate signal propagation to cellular heterogeneity and spatial organization. We address this problem in a bacterial community that employs electrochemical cell-to-cell communication. We developed a model based on percolation theory, which describes how signals propagate through a heterogeneous medium. Our model predicts that signal transmission becomes possible when the community is organized near a critical phase transition between a disconnected and a fully connected conduit of signaling cells. By measuring population-level signal transmission with single-cell resolution in wild-type and genetically modified communities, we confirm that the spatial distribution of signaling cells is organized at the predicted phase transition. Our findings suggest that at this critical point, the population-level benefit of signal transmission outweighs the single-cell level cost. The bacterial community thus appears to be organized according to a theoretically predicted spatial heterogeneity that promotes efficient signal transmission.</p>
</li>
</ul>
</li>
</ul>
<h2>04594</h2>
<h2>04494</h2>
<h2>04394</h2>
<h2>04294</h2>
<h2>04194</h2>
<h2>04094</h2>
<h2>03994</h2>
<h2>03894</h2>
<h2>03794</h2>
<h2>03694</h2>
<h2>03594</h2>
<h2>03494</h2>
<h2>03394</h2>
<h2>03294</h2>
<h2>03194</h2>
<h2>03094</h2>
<h2>02994</h2>
<h2>02894</h2>
<h2>02794</h2>
<h2>02694</h2>
<h2>02594</h2>
<h2>02494</h2>
<h2>02394</h2>
<h2>02294</h2>
<h2>02194</h2>
<h2>02094</h2>
<h2>01994</h2>
<h2>01894</h2>
<h2>01794</h2>
<h2>01694</h2>
<h2>01594</h2>
<h2>01494</h2>
<h2>01394</h2>
<h2>01294</h2>
<h2>01194</h2>
<h2>01094</h2>
<h2>00994</h2>
<h2>00894</h2>
<h2>00794</h2>
<h2>00694</h2>
<h2>00594</h2>
<h2>00494</h2>
<h2>00394</h2>
<h2>00294</h2>
<h2>00194</h2>
<h2>00094</h2>
<ul>
<li>plutonium Pu (pluto)</li>
<li>le petit prince
<ul>
<li>talking to a snake</li>
<li>snake</li>
</ul>
</li>
<li>ID</li>
</ul>

</div>

</body>
</html>
